(function(e){function o(o){for(var i,a,l=o[0],s=o[1],c=o[2],d=0,h=[];d<l.length;d++)a=l[d],Object.prototype.hasOwnProperty.call(r,a)&&r[a]&&h.push(r[a][0]),r[a]=0;for(i in s)Object.prototype.hasOwnProperty.call(s,i)&&(e[i]=s[i]);p&&p(o);while(h.length)h.shift()();return n.push.apply(n,c||[]),t()}function t(){for(var e,o=0;o<n.length;o++){for(var t=n[o],i=!0,l=1;l<t.length;l++){var s=t[l];0!==r[s]&&(i=!1)}i&&(n.splice(o--,1),e=a(a.s=t[0]))}return e}var i={},r={app:0},n=[];function a(o){if(i[o])return i[o].exports;var t=i[o]={i:o,l:!1,exports:{}};return e[o].call(t.exports,t,t.exports,a),t.l=!0,t.exports}a.m=e,a.c=i,a.d=function(e,o,t){a.o(e,o)||Object.defineProperty(e,o,{enumerable:!0,get:t})},a.r=function(e){"undefined"!==typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},a.t=function(e,o){if(1&o&&(e=a(e)),8&o)return e;if(4&o&&"object"===typeof e&&e&&e.__esModule)return e;var t=Object.create(null);if(a.r(t),Object.defineProperty(t,"default",{enumerable:!0,value:e}),2&o&&"string"!=typeof e)for(var i in e)a.d(t,i,function(o){return e[o]}.bind(null,i));return t},a.n=function(e){var o=e&&e.__esModule?function(){return e["default"]}:function(){return e};return a.d(o,"a",o),o},a.o=function(e,o){return Object.prototype.hasOwnProperty.call(e,o)},a.p="/";var l=window["webpackJsonp"]=window["webpackJsonp"]||[],s=l.push.bind(l);l.push=o,l=l.slice();for(var c=0;c<l.length;c++)o(l[c]);var p=s;n.push([0,"chunk-vendors"]),t()})({0:function(e,o,t){e.exports=t("cd49")},"006a":function(e,o,t){e.exports=t.p+"img/michael-davis.70bdae6d.jpg"},"072b":function(e,o,t){e.exports=t.p+"img/health-ai.968d8f7f.png"},"0879":function(e,o,t){var i={"./anne-thessen.jpg":"e8a7","./bryan-laraway.jpg":"9d3a","./charles-borromeo.jpg":"477e","./chris-mungall.jpg":"2467","./craig-mcnamara.jpg":"4b84","./damian-smedley.jpg":"7430","./daniel-keith.jpg":"50ee","./david-osumi-sutherland.jpg":"b70c","./deepak-unni.jpg":"0f8c","./eric-douglass.jpg":"884b","./faisal-alquaddoomi.jpg":"31d7","./hannah-blau.jpg":"54e6","./harry-hochheiser.jpg":"7f12","./j-harry-caufield.jpg":"b28e","./jeremy-espino.jpg":"1b39","./jim-balhoff.jpg":"bfd6","./jules-jacobsen.jpg":"3daf","./julie-mcmurry.jpg":"4129","./justin-reese.jpg":"35c6","./kent-shefchek.jpg":"d071","./kevin-schaper.jpg":"3a38","./leigh-carmody.jpg":"216e","./lilly-winfree.jpg":"1b20","./matthew-brush.jpg":"1d6a","./melissa-haendel.jpg":"ffdb","./michael-davis.jpg":"006a","./michael-gargano.jpg":"afdf","./monica-munoz-torres.jpg":"5b1c","./nicolas-matentzoglu.jpg":"608a","./nicole-vasilevsky.jpg":"c1e2","./nicole-washington.jpg":"76f4","./nomi-harris.jpg":"1a05","./peter-robinson.jpg":"80d0","./pilar-cacheiro.jpg":"b82c","./sabrina-toro.jpg":"f89b","./sebastian-kohler.jpg":"b2e4","./seth-carbon.jpg":"209c","./shahim-essaid.jpg":"37ec","./sierra-taylor-moxon.jpg":"c581","./suzanna-lewis.jpg":"2833","./tim-putman.jpg":"1c11","./tudor-groza.jpg":"5346","./victoria-alexandra-soesanto.jpg":"c7d6","./vida-ravanmehr.jpg":"df54","./vincent-rubinetti.jpg":"db89","./zhou-joe-yuan.jpg":"cd5b"};function r(e){var o=n(e);return t(o)}function n(e){if(!t.o(i,e)){var o=new Error("Cannot find module '"+e+"'");throw o.code="MODULE_NOT_FOUND",o}return i[e]}r.keys=function(){return Object.keys(i)},r.resolve=n,e.exports=r,r.id="0879"},"0b8f":function(e,o){e.exports="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAPgAAABkCAMAAACcqw0BAAAA1VBMVEUAAADuLiTuLSPyHw7vLSTuLSTuKiHuLSPrLSfyLifuLSPtLSPvKyHuLSPvLSPrLB7vLSTuKiPuLSPuLSPtLCPsLCPuLSPuLSPuLSPuLSTuLCLuLCPuLSPvLSPsKyPuLSPuLSPuLSPuLCPtLCLtKSLuLSPvLSPuLCPuLCPuLSPuLSPvLSPuLSPuLCPuLSPuLSLuLSPuLSPuLCLuLCPuLCLvLSLuKyLuLSPvKyLuLSPuLCPuLCPuLCPuLSPvLSPvLiTxLiT3LyX5MCX1LyT/MCbzLiT8MCU8tyveAAAAP3RSTlMA/v0C+vQf8AsE9q8X544I7BPXkzIbZMXTqi13X0UQ3riHZzwN4oRxbJ2Zzn1XpCPayUpBTyc2symMW1O/vKwq8C0QAAANmElEQVR42u1c6WKqOBSOGERRFEHFHcV937VuDfv7P9KEsNiqbZ2Zdpyh8/26QMjtl3NOzhYEkY9B8YK0G1aO5RQIIT4jDhm+IxelxmtpXAZhQ+QzIKQamm5rJj+o5togVIg8BGRou3U8CkKEyKOAnWkChIj6w8RVlR/EQsT8rT1fASH0dquLILWZB6HBRZVZ+hospNA7odNNEBoEAp/vSld4bRRZ09YMNWCunbcJEBIEnAa1cu4K29X6cBi/yroWiDwZD4uZ+8T1bh3cRbvVm3KGZ+c6W8uAcCCQeDoGPoQIPeKILuVAOPAQ8Vja9LdAuQbCgYeIg5XEIs/MJyAceIx4VuG9vd3ugXDgMeLgRVa9ccMXEAo8SDwleKpuNZYgFHiQeLZBIy/Q6YNQ4EHi9QFvucTlkBj5o8RFL4hRpZBs648TdyVuJVcgFHjUxk+eI9ebISk9Pkg8P/fcmVEA4cBjxBOtjkscMmsQDjxGfLZnCHHESi0QDjxEPN7nKHcUVLIgHHiEeLawsBHxZWwjBkKCgPj+LqVoIhOPKXPbGyTVQld60sd3OyWZ2UQc8dBLSSkRhAZBBVUSu8136Dan0+lrY8RRKnK7KagZGkX/tLzMmhi2bnkOHI6GoWwoRNANSBvBB5Sq0RDxfrSFhAzxJUSsHydOoWLl4SA9u90A8G/XjkeJU3pnf9x8WVTPbFbranc6rJbjnwTAqU25XM4n7s+Qx882Px8mRR4FpRr0sPW5G0/ka+kRtHXDRh+W5hLxfEtJT6eDSqt9yzqbr1UGg6lYiP10wHBphqo3QOiqRU5Jk482OHJ7mWQYlmwJFP9RMpOqyBzDOE5DWN8wb5U6vPOM4bmfToYCVoxcvILMsKrjzy5dQwMu+tmPmNcVccFors9X4f1kJlFL7+aqYeFFVQ0krN4/VIYSY2mqA8ve/2xLOiBunSvVwjsovW5TnE53C5lBlq/uqlDN3Nfgl7Gg6qrr/pAqV+9Z6UxpQFvz9QiZylstz1UEw1Z9BdReP08Df75pmFp3d3M6Eow7re/peXa7Uw0q8PodMXMv/B3SGrEqMpBCei8aTBCv7UzCGkJXELv7vaofT1ICJDLt7MveCMRklFK3+05MlNmAtmp3KvfUIrdgKYevYbD0NfH4UIYI31J1mv+K+D9aiOhx/k6ny5UbT7VOckh1ozxkW/PBIQXAdjxM595qjiJRFl4VrSNW10ONtKMKQaNG5FQ8gU5J42WFg6Rff9OWfQZxkOgJhm9+i9SVMVQXhoE8zwBHg34ZgPakxGloP7sM6gkmohCiG/0NyHR1vERwvvI38wHtrKslD45t0OIgckzvdpN4AnGMJuWP5N9783wB6n6Uw3akI9HcNbRRxBKUYFBFtp0RXKOMrw6ShWkylby7qLGziRzr5iqYbLRP/iO2Cq7wLOKtV0/kKpskWhhoKUt5Wq5RyVo+Qxw670hNLRZ8UxE5h7YtV/LO44GFXzC4srdBnFjkaEtnWXdebah4UdhhGVzjScQTPdOzcsgvg6AF8+aNiEsbnftld1M7Lsg9Y7fygxbeUWVb6BEr6Qs6vuj04u6et2MRhXVbIqdMskN8hSKjO6eMnkQcrGno7+yHYDXKIm8TcauqLC4TZDGi2xPR/cBOo4pAbFomeU5mWcTMDGZfjxKNGeiOezNGExIFVTo6hTShcBshPo34tsEij7gS6PkeegaAWfkZWaxBq8RRG66dZloLE1+rPOENlgvHcUFxAwgGlmMoFE94t5cMng8yXXCLpxHf7BnVI973Q5Ke67yxn2rUooE988hdC27p2nDR0XNELWaEd8NZEth88QrXRcOxd7qXdxSoz2JbR50DuIsnEU8pvOWFt72MK90+p5EbNi+2MkENno+48TrTJExfhkQrzBGxhJ5Eq5Y5VzZRd9IFhTBXuuHIP9aVdWSzpePdlPZpxLMH3lNrgxAHiY1kEoFbnPjiFx4S1YXtjeJctzdh3WsRL0pZ6Ri6ykn9qDs+X2GdpdMEJQqytT1vIiSLrftFjKcRz/cYT+JqnzDKTnkvKbkc7s2sFqY/ZTFGRnV18s7iAOLLJM70YOeSq08gmcBuxOKpwki1KEZQ2uBDPMnGxcDGFdd2ZYtkbPDcugTtIwr5AhdJ2lMoquRSfJklORYZxm6bTfhmUXGXUhPB8syyqs5MN4mPpP004jUZ+kFr1dMAg5xvPa2ClKSchH5iaZ7cIsrAy9AX0zOtm2igvPHQ5YHmTjgaLqCpF/frj8OW5xEv6P5YlrieVRE6lOziIbDJMkk+CFEEm+R2qmG65gANM9I5kbwlwGFkuRNSug2FqfJp7eFZxPN7jziC8xr5qzXv2HMQzcSSFsJui9it4R4Lqx8X/nrhOH7/jna0PfBLuJCZNwqfVzKfRrw68pRYZcSNqwGOE7cWgdPNSjBCYZcsQERROtnw2gc5gty3YCmXb2eujpD5ZsFVZvUvvvV5CvEo1uLXYCTjuilFf9dszKwb0MLXQvLs5By0sx65fRG5Gz/sjDdX9ZhlkvWXcrD6RiXP4OJ1azkpKP1+ZUzQr1QKilI91lat3Eu+/WckHs0Nec3T9MiiTWy6YBMffvTEV5VMrOeI6/XnEKn0cIapTRkTeebRCXZ+f/zJ9ss62vE7+MZyudpkUlUqzeGg1JBGwnzOMRg002F4QZgXF+dGcjpIj3t9vAYPEc9ktiI0/IGyJ+ODI3GKXZOFyStz4r+5ZqxqYwvvzEC9xusRhoE3p57J+MLchDLjMtf74EHcNu/b9WwqH5ttXmqVUklWdV3XdE0zDMuysDuBkKYxd57jOJ5n3K9sLE3XueSF+Cfl3NpQuFQbzbNX6z8iR4l1Qry848iFXGkrghpB9C7xMmQoW+iWOMslV31XfjtzUONzJZty3fgM/BXEU9vCeJhsSEWZ4xxe0PucyjIwOVunBSkp9qqTyXLV2q6W68mx0BwmF1gBhpsL8cE2m7pCbLtq1frp0oKLWEFpvVPxszXHliOWVM23xhKDsJ7bfDMFHI+mjfr9V17H+fesInib/7gOfMwqC1o1Oz0wMF1tmCt/TtLx/HbdE6fJ3amI1RnztbCQVVXTbVtXIcONdoNmvzpZ11q5TT5br8czCaIc7ezsJYdXYFmOBsTV0bB7hWZ6mnzdCcjULXTpJKVnwckYxsDXmtAs0aZKUWpk1N2A8gmTMYppAdnUWUmByUh35194sWi2pYi8ifQOXomBjcicxm4DHkN9tjoe+pXm9NxBpm3bmK8Lw9AQhyVcGqS742qtnH+4k4LX6wqaYyukjXIBLW0vgusYJGTVDEQ0dn7MgPrevUlh+yqWAcjUJEKcBDqxOlaqgqRqeJlgD6/DHroravHTWfQLa85ga85uJs0RpJCBgS7Ne8jSjhnPh/1WO/HdTUMCLFSsJD5SJ4iCZqqTdtaxPOeqe62yg1jGKdGcba83B/nibjfqkOaawfZifhxPwJdW4BO0N9vJeNAQeJ6B1FXjXteZUalbbc1SWKWj0R8gjnSt0XozcVyZ2/4jzSy14ngt+oynBbAZI6KKJk2POLIQTSMiKmzfRBW3U4PyfCSzGN5zavFYqzoeTpPn02jO0SrWbBQwdkxa45OV43qVi2UTP/dRLTuaLt+3uYdeCKLBkftZVm0EERGoFHjHvmy9mQORztrc60m0x3bQfjGR1DvWWtvtNlferlarGt6FJ4XK/nXEGzY2Z92wVJ+zamga5IqnXWkwXmV+tj9OsVxjEr/2cztEul3s6eCx1Fz5nZdB4lIfG1cz0XLg04+QuhiLarPCqdHAUUZjMcICZijN1t/IOBgJaYbjJLGwzRPSP0UcqZZuUOxgVc/cGNC6CC3ECFXv0cvQ2eeRNiwn3ozpOIoeQGdfN4lL/EbaiAEgy3qf8zqIXIFIGrG8MC2U81ls0X8ndze/AmLkRrpXXZfvrW69Vu0rk1rWK7B2GRRBOhy/RN+OmciaRsqNyPEcu2oOXJBPQhtdn7xSg0NXQUCi67ZpUsLrvlc4rGvlb2gvlZKfYyp2e8uPzmVEr3poghlB1DxdvxrTa3Qgtu8IZGRpunr3XqI2kKFhoHuKpjoRGHZ9LM/JI6nxOk0rq+87IRLNxD9BOw4eRqzPYkVHwp3AO9c8jQRhJCW7tdvXeifHxd1QhzRxzpywOE+H4+oqBr4X0S8AHgPpoTlFiEgV3CJRT6Xy+VQq287cPounZoWzRSImAudfto6E13S633Jey2brbWzP/8bjY1EAVlPewP6bLf2Vqll2O5ng5DktOtiPxzjOPtZym03qX39Wur3esbrTJuKWf7kZEM/GZg7y9f/OL85EJwJSSVUq+d/5o78DPbxre42Bf71yfqN9JyqChUiEzo/BL0Jd4VQ3rUbD8P0W1ieospizW29v/duPLH8nerJflKIPdfBL4Pjvk+GVj+hkiH4Q6Sskyq9e7xihc3g+zfoaM3IGxq+U/iJMaD/TNse/KXQJ+tsGtd+AX4QxpxLiVoi+OX0Aic3Ja/qrp9+0sYG66H1Nr0uTXxS4AJCSoNPOsNDi8Js2NgBicydGNyjM+zeFqgDkOM2poUuFUP1s6dfIVnkjgnDrKCy/6fco2kdoah2x9Zv2c4LEy7QxqPwm/x0gFQe/E//KUvf/+B//4y3+AF8L9zdlK53BAAAAAElFTkSuQmCC"},"0bae":function(e,o,t){"use strict";t("9f37")},"0f8c":function(e,o,t){e.exports=t.p+"img/deepak-unni.3bd1817b.jpg"},1052:function(e,o,t){"use strict";t("51d9")},1352:function(e,o,t){e.exports=t.p+"img/ebi.3a8c2203.png"},"14a5":function(e,o,t){e.exports=t.p+"img/gwascatalog.75dd0d10.png"},1778:function(e){e.exports=JSON.parse('[{"id":"ecto"},{"id":"geno"},{"id":"hp"},{"id":"maxo"},{"id":"mondo"},{"id":"sepio"},{"id":"upheno"},{"id":"bfo"},{"id":"caro"},{"id":"chebi"},{"id":"cl"},{"id":"clo"},{"id":"eco"},{"id":"ero"},{"id":"fao"},{"id":"fbbt"},{"id":"go"},{"id":"hsapdv"},{"id":"iao"},{"id":"mpath"},{"id":"nbo"},{"id":"ncbitaxon"},{"id":"ncit"},{"id":"oba"},{"id":"pato"},{"id":"pco"},{"id":"po"},{"id":"pw"},{"id":"ro"},{"id":"so"},{"id":"stato"},{"id":"uberon"},{"id":"vt"},{"id":"wbbt"},{"id":"xco"},{"id":"zfa"}]')},1802:function(e,o,t){var i={"./berkeley.png":"d0b7","./charite.png":"1b56","./ebi.png":"1352","./garvan.png":"9993","./health-ai.png":"072b","./jackson.png":"e97b","./ohsu.png":"dd49","./oregon-state.png":"cd5a","./pittsburgh.png":"9392","./queen-mary.png":"ec4d","./renci.png":"a090"};function r(e){var o=n(e);return t(o)}function n(e){if(!t.o(i,e)){var o=new Error("Cannot find module '"+e+"'");throw o.code="MODULE_NOT_FOUND",o}return i[e]}r.keys=function(){return Object.keys(i)},r.resolve=n,e.exports=r,r.id="1802"},"18a5":function(e,o,t){},"19e4":function(e,o,t){},"1a05":function(e,o,t){e.exports=t.p+"img/nomi-harris.00db568b.jpg"},"1a7a":function(e,o,t){},"1b20":function(e,o,t){e.exports=t.p+"img/lilly-winfree.e043135c.jpg"},"1b39":function(e,o,t){e.exports=t.p+"img/jeremy-espino.2c510d44.jpg"},"1b56":function(e,o,t){e.exports=t.p+"img/charite.5201888e.png"},"1b63":function(e,o,t){},"1c11":function(e,o,t){e.exports=t.p+"img/tim-putman.3cb405e0.jpg"},"1d6a":function(e,o,t){e.exports=t.p+"img/matthew-brush.5860d609.jpg"},"1e31":function(e,o,t){e.exports=t.p+"img/loading.01db7180.svg"},"1f1c":function(e,o,t){e.exports=t.p+"img/udp.b0d0f092.png"},"209c":function(e,o,t){e.exports=t.p+"img/seth-carbon.910fe102.jpg"},"216e":function(e,o,t){e.exports=t.p+"img/leigh-carmody.b7a18b2f.jpg"},2467:function(e,o,t){e.exports=t.p+"img/chris-mungall.08ed17cc.jpg"},"24b8":function(e,o,t){},2634:function(e,o,t){e.exports=t.p+"img/orphanet.e4812613.png"},"26df":function(e){e.exports=JSON.parse('{"numFound":61,"docs":[{"id":"MONDO:0017310","id_std":"MONDO:0017310","id_eng":"MONDO:0017310","id_kw":"MONDO:0017310","prefix":"MONDO","label":["Marfan and Marfan-related disorder"],"label_std":["Marfan and Marfan-related disorder"],"label_eng":["Marfan and Marfan-related disorder"],"label_kw":["Marfan and Marfan-related disorder"],"edges":164,"has_phenotype":false,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"equivalent_curie":["ORPHA:284993","ORDO:284993","Orphanet:284993","UMLS:CN227112"],"equivalent_curie_std":["ORPHA:284993","ORDO:284993","Orphanet:284993","UMLS:CN227112"],"equivalent_curie_eng":["ORPHA:284993","ORDO:284993","Orphanet:284993","UMLS:CN227112"],"equivalent_curie_kw":["ORPHA:284993","ORDO:284993","Orphanet:284993","UMLS:CN227112"],"leaf":false,"_version_":1696524895947587600,"score":108.652695},{"id":"MONDO:0007947","id_std":"MONDO:0007947","id_eng":"MONDO:0007947","id_kw":"MONDO:0007947","prefix":"MONDO","label":["Marfan syndrome"],"label_std":["Marfan syndrome"],"label_eng":["Marfan syndrome"],"label_kw":["Marfan syndrome"],"definition":["Marfan syndrome is a disorder of the connective tissue. Connective tissue provides strength and flexibility to structures throughout the body such as bones, ligaments, muscles, walls of blood vessels, and heart valves. Marfan syndrome affects most organs and tissues, especially the skeleton, lungs, eyes, heart, and the large blood vessel that distributes blood from the heart to the rest of the body (the aorta). It is caused by mutations in the FBN1 gene, which provides instructions for making a protein called fibrillin-1. Marfan syndrome is inherited in an autosomal dominant pattern. At least 25% of cases are due to a new (de novo) mutation. Treatment is based on the signs and symptoms in each person."],"definition_std":["Marfan syndrome is a disorder of the connective tissue. Connective tissue provides strength and flexibility to structures throughout the body such as bones, ligaments, muscles, walls of blood vessels, and heart valves. Marfan syndrome affects most organs and tissues, especially the skeleton, lungs, eyes, heart, and the large blood vessel that distributes blood from the heart to the rest of the body (the aorta). It is caused by mutations in the FBN1 gene, which provides instructions for making a protein called fibrillin-1. Marfan syndrome is inherited in an autosomal dominant pattern. At least 25% of cases are due to a new (de novo) mutation. Treatment is based on the signs and symptoms in each person."],"definition_eng":["Marfan syndrome is a disorder of the connective tissue. Connective tissue provides strength and flexibility to structures throughout the body such as bones, ligaments, muscles, walls of blood vessels, and heart valves. Marfan syndrome affects most organs and tissues, especially the skeleton, lungs, eyes, heart, and the large blood vessel that distributes blood from the heart to the rest of the body (the aorta). It is caused by mutations in the FBN1 gene, which provides instructions for making a protein called fibrillin-1. Marfan syndrome is inherited in an autosomal dominant pattern. At least 25% of cases are due to a new (de novo) mutation. Treatment is based on the signs and symptoms in each person."],"definition_kw":["Marfan syndrome is a disorder of the connective tissue. Connective tissue provides strength and flexibility to structures throughout the body such as bones, ligaments, muscles, walls of blood vessels, and heart valves. Marfan syndrome affects most organs and tissues, especially the skeleton, lungs, eyes, heart, and the large blood vessel that distributes blood from the heart to the rest of the body (the aorta). It is caused by mutations in the FBN1 gene, which provides instructions for making a protein called fibrillin-1. Marfan syndrome is inherited in an autosomal dominant pattern. At least 25% of cases are due to a new (de novo) mutation. Treatment is based on the signs and symptoms in each person."],"edges":5475,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["Marfan\'s syndrome","MFS1","Marfan Syndrome, Type 1","Marfan syndrome type 1","MARFAN SYNDROME; MFS","MFS","Marfan syndrome","Marfan syndrome, type 1"],"synonym_std":["Marfan\'s syndrome","MFS1","Marfan Syndrome, Type 1","Marfan syndrome type 1","MARFAN SYNDROME; MFS","MFS","Marfan syndrome","Marfan syndrome, type 1"],"synonym_eng":["Marfan\'s syndrome","MFS1","Marfan Syndrome, Type 1","Marfan syndrome type 1","MARFAN SYNDROME; MFS","MFS","Marfan syndrome","Marfan syndrome, type 1"],"synonym_kw":["Marfan\'s syndrome","MFS1","Marfan Syndrome, Type 1","Marfan syndrome type 1","MARFAN SYNDROME; MFS","MFS","Marfan syndrome","Marfan syndrome, type 1"],"equivalent_curie":["UMLS:CN202883","COHD:258540","OMIM:154700","MIM:154700","SCTID:19346006","ORPHA:284963","ORDO:284963","Orphanet:284963","ORPHA:558","ORDO:558","Orphanet:558","DOID:14323","GARD:0006975","ICD9:759.82","NCIT:C34807","UMLS:C0024796","MESH:D008382"],"equivalent_curie_std":["UMLS:CN202883","COHD:258540","OMIM:154700","MIM:154700","SCTID:19346006","ORPHA:284963","ORDO:284963","Orphanet:284963","ORPHA:558","ORDO:558","Orphanet:558","DOID:14323","GARD:0006975","ICD9:759.82","NCIT:C34807","UMLS:C0024796","MESH:D008382"],"equivalent_curie_eng":["UMLS:CN202883","COHD:258540","OMIM:154700","MIM:154700","SCTID:19346006","ORPHA:284963","ORDO:284963","Orphanet:284963","ORPHA:558","ORDO:558","Orphanet:558","DOID:14323","GARD:0006975","ICD9:759.82","NCIT:C34807","UMLS:C0024796","MESH:D008382"],"equivalent_curie_kw":["UMLS:CN202883","COHD:258540","OMIM:154700","MIM:154700","SCTID:19346006","ORPHA:284963","ORDO:284963","Orphanet:284963","ORPHA:558","ORDO:558","Orphanet:558","DOID:14323","GARD:0006975","ICD9:759.82","NCIT:C34807","UMLS:C0024796","MESH:D008382"],"leaf":true,"_version_":1696524903676641300,"score":108.12949},{"id":"MONDO:0014831","id_std":"MONDO:0014831","id_eng":"MONDO:0014831","id_kw":"MONDO:0014831","prefix":"MONDO","label":["progeroid and marfanoid aspect-lipodystrophy syndrome"],"label_std":["progeroid and marfanoid aspect-lipodystrophy syndrome"],"label_eng":["progeroid and marfanoid aspect-lipodystrophy syndrome"],"label_kw":["progeroid and marfanoid aspect-lipodystrophy syndrome"],"edges":194,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["Marfanoid-Progeroid Syndrome","Marfanoid-progeroid syndrome","MFLS","MARFANOID-PROGEROID-LIPODYSTROPHY SYNDROME; MFLS","Marfan-Progeroid-Lipodystrophy Syndrome","Marfan-progeroid-lipodystrophy syndrome","Marfan Lipodystrophy Syndrome","Marfan lipodystrophy syndrome"],"synonym_std":["Marfanoid-Progeroid Syndrome","Marfanoid-progeroid syndrome","MFLS","MARFANOID-PROGEROID-LIPODYSTROPHY SYNDROME; MFLS","Marfan-Progeroid-Lipodystrophy Syndrome","Marfan-progeroid-lipodystrophy syndrome","Marfan Lipodystrophy Syndrome","Marfan lipodystrophy syndrome"],"synonym_eng":["Marfanoid-Progeroid Syndrome","Marfanoid-progeroid syndrome","MFLS","MARFANOID-PROGEROID-LIPODYSTROPHY SYNDROME; MFLS","Marfan-Progeroid-Lipodystrophy Syndrome","Marfan-progeroid-lipodystrophy syndrome","Marfan Lipodystrophy Syndrome","Marfan lipodystrophy syndrome"],"synonym_kw":["Marfanoid-Progeroid Syndrome","Marfanoid-progeroid syndrome","MFLS","MARFANOID-PROGEROID-LIPODYSTROPHY SYNDROME; MFLS","Marfan-Progeroid-Lipodystrophy Syndrome","Marfan-progeroid-lipodystrophy syndrome","Marfan Lipodystrophy Syndrome","Marfan lipodystrophy syndrome"],"equivalent_curie":["ORPHA:300382","ORDO:300382","Orphanet:300382","UMLS:C4310796","OMIM:616914","MIM:616914"],"equivalent_curie_std":["ORPHA:300382","ORDO:300382","Orphanet:300382","UMLS:C4310796","OMIM:616914","MIM:616914"],"equivalent_curie_eng":["ORPHA:300382","ORDO:300382","Orphanet:300382","UMLS:C4310796","OMIM:616914","MIM:616914"],"equivalent_curie_kw":["ORPHA:300382","ORDO:300382","Orphanet:300382","UMLS:C4310796","OMIM:616914","MIM:616914"],"leaf":true,"_version_":1696524889852215300,"score":103.42152},{"id":"MONDO:0017309","id_std":"MONDO:0017309","id_eng":"MONDO:0017309","id_kw":"MONDO:0017309","prefix":"MONDO","label":["neonatal Marfan syndrome","Marfan syndrome, neonatal"],"label_std":["neonatal Marfan syndrome","Marfan syndrome, neonatal"],"label_eng":["neonatal Marfan syndrome","Marfan syndrome, neonatal"],"label_kw":["neonatal Marfan syndrome","Marfan syndrome, neonatal"],"definition":["Neonatal Marfan syndrome is a rare, severe and life-threatening genetic disease, occuring during the neonatal period, characterized by classical Marfan syndrome manifestations in addition to facial dysmorphism (megalocornea, iridodonesis, ectopia lentis, crumpled ears, loose redundant skin giving a \'senile\' facial appearance), flexion joint contractures, pulmonary emphysema, and a severe, rapidly progressive cardiovascular disease (including ascending aortic dilatation and severe mitral and/or tricuspid valve insufficiency). Additionally, skeletal manifestations (arachnodactyly, dolichostenomelia, pectus deformities) are also associated."],"definition_std":["Neonatal Marfan syndrome is a rare, severe and life-threatening genetic disease, occuring during the neonatal period, characterized by classical Marfan syndrome manifestations in addition to facial dysmorphism (megalocornea, iridodonesis, ectopia lentis, crumpled ears, loose redundant skin giving a \'senile\' facial appearance), flexion joint contractures, pulmonary emphysema, and a severe, rapidly progressive cardiovascular disease (including ascending aortic dilatation and severe mitral and/or tricuspid valve insufficiency). Additionally, skeletal manifestations (arachnodactyly, dolichostenomelia, pectus deformities) are also associated."],"definition_eng":["Neonatal Marfan syndrome is a rare, severe and life-threatening genetic disease, occuring during the neonatal period, characterized by classical Marfan syndrome manifestations in addition to facial dysmorphism (megalocornea, iridodonesis, ectopia lentis, crumpled ears, loose redundant skin giving a \'senile\' facial appearance), flexion joint contractures, pulmonary emphysema, and a severe, rapidly progressive cardiovascular disease (including ascending aortic dilatation and severe mitral and/or tricuspid valve insufficiency). Additionally, skeletal manifestations (arachnodactyly, dolichostenomelia, pectus deformities) are also associated."],"definition_kw":["Neonatal Marfan syndrome is a rare, severe and life-threatening genetic disease, occuring during the neonatal period, characterized by classical Marfan syndrome manifestations in addition to facial dysmorphism (megalocornea, iridodonesis, ectopia lentis, crumpled ears, loose redundant skin giving a \'senile\' facial appearance), flexion joint contractures, pulmonary emphysema, and a severe, rapidly progressive cardiovascular disease (including ascending aortic dilatation and severe mitral and/or tricuspid valve insufficiency). Additionally, skeletal manifestations (arachnodactyly, dolichostenomelia, pectus deformities) are also associated."],"edges":207,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["neonatal MFS"],"synonym_std":["neonatal MFS"],"synonym_eng":["neonatal MFS"],"synonym_kw":["neonatal MFS"],"equivalent_curie":["SCTID:763839005","ORPHA:284979","ORDO:284979","Orphanet:284979","UMLS:CN202885","UMLS:CN536247"],"equivalent_curie_std":["SCTID:763839005","ORPHA:284979","ORDO:284979","Orphanet:284979","UMLS:CN202885","UMLS:CN536247"],"equivalent_curie_eng":["SCTID:763839005","ORPHA:284979","ORDO:284979","Orphanet:284979","UMLS:CN202885","UMLS:CN536247"],"equivalent_curie_kw":["SCTID:763839005","ORPHA:284979","ORDO:284979","Orphanet:284979","UMLS:CN202885","UMLS:CN536247"],"leaf":true,"_version_":1696524895942344700,"score":98.14615},{"id":"MONDO:0007948","id_std":"MONDO:0007948","id_eng":"MONDO:0007948","id_kw":"MONDO:0007948","prefix":"MONDO","label":["marfanoid hypermobility syndrome"],"label_std":["marfanoid hypermobility syndrome"],"label_eng":["marfanoid hypermobility syndrome"],"label_kw":["marfanoid hypermobility syndrome"],"edges":114,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["MARFANOID HYPERMOBILITY SYNDROME","marfanoid hypermobility syndrome"],"synonym_std":["MARFANOID HYPERMOBILITY SYNDROME","marfanoid hypermobility syndrome"],"synonym_eng":["MARFANOID HYPERMOBILITY SYNDROME","marfanoid hypermobility syndrome"],"synonym_kw":["MARFANOID HYPERMOBILITY SYNDROME","marfanoid hypermobility syndrome"],"equivalent_curie":["GARD:0003387","UMLS:C0268365","MESH:C531742","OMIM:154750","MIM:154750"],"equivalent_curie_std":["GARD:0003387","UMLS:C0268365","MESH:C531742","OMIM:154750","MIM:154750"],"equivalent_curie_eng":["GARD:0003387","UMLS:C0268365","MESH:C531742","OMIM:154750","MIM:154750"],"equivalent_curie_kw":["GARD:0003387","UMLS:C0268365","MESH:C531742","OMIM:154750","MIM:154750"],"leaf":true,"_version_":1696524903678738400,"score":88.514206},{"id":"MONDO:0009120","id_std":"MONDO:0009120","id_eng":"MONDO:0009120","id_kw":"MONDO:0009120","prefix":"MONDO","label":["marfanoid syndrome, De Silva type"],"label_std":["marfanoid syndrome, De Silva type"],"label_eng":["marfanoid syndrome, De Silva type"],"label_kw":["marfanoid syndrome, De Silva type"],"definition":["Marfanoid syndrome, da Silva type is characterized by the association of marfanoid habitus with visceral diverticula. It has been reported in four adults and two siblings from a consanguineous marriage in two different publications. Pediatric cases also presented with diaphragmatic hernia. Other connective tissue disorders with visceral diverticula have been reported previously, suggesting a relationship between these two conditions."],"definition_std":["Marfanoid syndrome, da Silva type is characterized by the association of marfanoid habitus with visceral diverticula. It has been reported in four adults and two siblings from a consanguineous marriage in two different publications. Pediatric cases also presented with diaphragmatic hernia. Other connective tissue disorders with visceral diverticula have been reported previously, suggesting a relationship between these two conditions."],"definition_eng":["Marfanoid syndrome, da Silva type is characterized by the association of marfanoid habitus with visceral diverticula. It has been reported in four adults and two siblings from a consanguineous marriage in two different publications. Pediatric cases also presented with diaphragmatic hernia. Other connective tissue disorders with visceral diverticula have been reported previously, suggesting a relationship between these two conditions."],"definition_kw":["Marfanoid syndrome, da Silva type is characterized by the association of marfanoid habitus with visceral diverticula. It has been reported in four adults and two siblings from a consanguineous marriage in two different publications. Pediatric cases also presented with diaphragmatic hernia. Other connective tissue disorders with visceral diverticula have been reported previously, suggesting a relationship between these two conditions."],"edges":83,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["diverticulosis of bowel, hernia, and retinal detachment","Marphanoid syndrome type De Silva","DIVERTICULOSIS OF BOWEL, HERNIA, AND RETINAL DETACHMENT"],"synonym_std":["diverticulosis of bowel, hernia, and retinal detachment","Marphanoid syndrome type De Silva","DIVERTICULOSIS OF BOWEL, HERNIA, AND RETINAL DETACHMENT"],"synonym_eng":["diverticulosis of bowel, hernia, and retinal detachment","Marphanoid syndrome type De Silva","DIVERTICULOSIS OF BOWEL, HERNIA, AND RETINAL DETACHMENT"],"synonym_kw":["diverticulosis of bowel, hernia, and retinal detachment","Marphanoid syndrome type De Silva","DIVERTICULOSIS OF BOWEL, HERNIA, AND RETINAL DETACHMENT"],"equivalent_curie":["ORPHA:2464","ORDO:2464","Orphanet:2464","GARD:0003401","MESH:C565619","UMLS:C1857227","OMIM:223330","MIM:223330"],"equivalent_curie_std":["ORPHA:2464","ORDO:2464","Orphanet:2464","GARD:0003401","MESH:C565619","UMLS:C1857227","OMIM:223330","MIM:223330"],"equivalent_curie_eng":["ORPHA:2464","ORDO:2464","Orphanet:2464","GARD:0003401","MESH:C565619","UMLS:C1857227","OMIM:223330","MIM:223330"],"equivalent_curie_kw":["ORPHA:2464","ORDO:2464","Orphanet:2464","GARD:0003401","MESH:C565619","UMLS:C1857227","OMIM:223330","MIM:223330"],"leaf":true,"_version_":1696524908805226500,"score":88.514206},{"id":"MONDO:0013574","id_std":"MONDO:0013574","id_eng":"MONDO:0013574","id_kw":"MONDO:0013574","prefix":"MONDO","label":["cutis laxa - Marfanoid syndrome"],"label_std":["cutis laxa - Marfanoid syndrome"],"label_eng":["cutis laxa - Marfanoid syndrome"],"label_kw":["cutis laxa - Marfanoid syndrome"],"edges":113,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["cutis laxa, neonatal, with MARFANOID phenotype","CUTIS LAXA, NEONATAL, WITH MARFANOID PHENOTYPE","cutis laxa - Marfanoid syndrome","Cutis Laxa-Marfanoid Syndrome"],"synonym_std":["cutis laxa, neonatal, with MARFANOID phenotype","CUTIS LAXA, NEONATAL, WITH MARFANOID PHENOTYPE","cutis laxa - Marfanoid syndrome","Cutis Laxa-Marfanoid Syndrome"],"synonym_eng":["cutis laxa, neonatal, with MARFANOID phenotype","CUTIS LAXA, NEONATAL, WITH MARFANOID PHENOTYPE","cutis laxa - Marfanoid syndrome","Cutis Laxa-Marfanoid Syndrome"],"synonym_kw":["cutis laxa, neonatal, with MARFANOID phenotype","CUTIS LAXA, NEONATAL, WITH MARFANOID PHENOTYPE","cutis laxa - Marfanoid syndrome","Cutis Laxa-Marfanoid Syndrome"],"equivalent_curie":["SCTID:254221009","ORPHA:171719","ORDO:171719","Orphanet:171719","OMIM:614100","MIM:614100","MESH:C563639","UMLS:C0432335"],"equivalent_curie_std":["SCTID:254221009","ORPHA:171719","ORDO:171719","Orphanet:171719","OMIM:614100","MIM:614100","MESH:C563639","UMLS:C0432335"],"equivalent_curie_eng":["SCTID:254221009","ORPHA:171719","ORDO:171719","Orphanet:171719","OMIM:614100","MIM:614100","MESH:C563639","UMLS:C0432335"],"equivalent_curie_kw":["SCTID:254221009","ORPHA:171719","ORDO:171719","Orphanet:171719","OMIM:614100","MIM:614100","MESH:C563639","UMLS:C0432335"],"leaf":true,"_version_":1696524887435247600,"score":88.514206},{"id":"MONDO:0009565","id_std":"MONDO:0009565","id_eng":"MONDO:0009565","id_kw":"MONDO:0009565","prefix":"MONDO","label":["microcephaly-glomerulonephritis-marfanoid habitus syndrome"],"label_std":["microcephaly-glomerulonephritis-marfanoid habitus syndrome"],"label_eng":["microcephaly-glomerulonephritis-marfanoid habitus syndrome"],"label_kw":["microcephaly-glomerulonephritis-marfanoid habitus syndrome"],"definition":["This syndrome is characterised by intellectual deficit, marfanoid habitus, microcephaly, and glomerulonephritis."],"definition_std":["This syndrome is characterised by intellectual deficit, marfanoid habitus, microcephaly, and glomerulonephritis."],"definition_eng":["This syndrome is characterised by intellectual deficit, marfanoid habitus, microcephaly, and glomerulonephritis."],"definition_kw":["This syndrome is characterised by intellectual deficit, marfanoid habitus, microcephaly, and glomerulonephritis."],"edges":163,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["MARFANOID HABITUS WITH MICROCEPHALY AND GLOMERULONEPHRITIS","MARFANOID habitus with microcephaly and glomerulonephritis","microcephaly glomerulonephritis Marfanoid habitus"],"synonym_std":["MARFANOID HABITUS WITH MICROCEPHALY AND GLOMERULONEPHRITIS","MARFANOID habitus with microcephaly and glomerulonephritis","microcephaly glomerulonephritis Marfanoid habitus"],"synonym_eng":["MARFANOID HABITUS WITH MICROCEPHALY AND GLOMERULONEPHRITIS","MARFANOID habitus with microcephaly and glomerulonephritis","microcephaly glomerulonephritis Marfanoid habitus"],"synonym_kw":["MARFANOID HABITUS WITH MICROCEPHALY AND GLOMERULONEPHRITIS","MARFANOID habitus with microcephaly and glomerulonephritis","microcephaly glomerulonephritis Marfanoid habitus"],"equivalent_curie":["UMLS:C1855348","OMIM:248760","MIM:248760","GARD:0003615","ORPHA:2172","ORDO:2172","Orphanet:2172","MESH:C565411"],"equivalent_curie_std":["UMLS:C1855348","OMIM:248760","MIM:248760","GARD:0003615","ORPHA:2172","ORDO:2172","Orphanet:2172","MESH:C565411"],"equivalent_curie_eng":["UMLS:C1855348","OMIM:248760","MIM:248760","GARD:0003615","ORPHA:2172","ORDO:2172","Orphanet:2172","MESH:C565411"],"equivalent_curie_kw":["UMLS:C1855348","OMIM:248760","MIM:248760","GARD:0003615","ORPHA:2172","ORDO:2172","Orphanet:2172","MESH:C565411"],"leaf":true,"_version_":1696524908473876500,"score":84.862335},{"id":"MONDO:0012171","id_std":"MONDO:0012171","id_eng":"MONDO:0012171","id_kw":"MONDO:0012171","prefix":"MONDO","label":["marfanoid habitus with situs inversus"],"label_std":["marfanoid habitus with situs inversus"],"label_eng":["marfanoid habitus with situs inversus"],"label_kw":["marfanoid habitus with situs inversus"],"edges":108,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["MARFANOID HABITUS WITH SITUS INVERSUS","marfanoid habitus with situs inversus"],"synonym_std":["MARFANOID HABITUS WITH SITUS INVERSUS","marfanoid habitus with situs inversus"],"synonym_eng":["MARFANOID HABITUS WITH SITUS INVERSUS","marfanoid habitus with situs inversus"],"synonym_kw":["MARFANOID HABITUS WITH SITUS INVERSUS","marfanoid habitus with situs inversus"],"equivalent_curie":["MESH:C563814","UMLS:C1836994","OMIM:609008","MIM:609008"],"equivalent_curie_std":["MESH:C563814","UMLS:C1836994","OMIM:609008","MIM:609008"],"equivalent_curie_eng":["MESH:C563814","UMLS:C1836994","OMIM:609008","MIM:609008"],"equivalent_curie_kw":["MESH:C563814","UMLS:C1836994","OMIM:609008","MIM:609008"],"leaf":true,"_version_":1696524884378648600,"score":84.862335},{"id":"MONDO:0009566","id_std":"MONDO:0009566","id_eng":"MONDO:0009566","id_kw":"MONDO:0009566","prefix":"MONDO","label":["marfanoid habitus-autosomal recessive intellectual disability syndrome"],"label_std":["marfanoid habitus-autosomal recessive intellectual disability syndrome"],"label_eng":["marfanoid habitus-autosomal recessive intellectual disability syndrome"],"label_kw":["marfanoid habitus-autosomal recessive intellectual disability syndrome"],"definition":["Marfanoid habitus B intellectual deficit, autosomal recessive is a very rare multiple congenital anomalies syndrome described in four sibs and characterized by intellectual deficit, flat face and some skeletelal features of Marfan syndrome such as tall stature, dolichostenomelia, arm span larger than height, arachnodactyly of hands and feet, little subcutaneous fat, muscle hypotonia and intellectual deficit."],"definition_std":["Marfanoid habitus B intellectual deficit, autosomal recessive is a very rare multiple congenital anomalies syndrome described in four sibs and characterized by intellectual deficit, flat face and some skeletelal features of Marfan syndrome such as tall stature, dolichostenomelia, arm span larger than height, arachnodactyly of hands and feet, little subcutaneous fat, muscle hypotonia and intellectual deficit."],"definition_eng":["Marfanoid habitus B intellectual deficit, autosomal recessive is a very rare multiple congenital anomalies syndrome described in four sibs and characterized by intellectual deficit, flat face and some skeletelal features of Marfan syndrome such as tall stature, dolichostenomelia, arm span larger than height, arachnodactyly of hands and feet, little subcutaneous fat, muscle hypotonia and intellectual deficit."],"definition_kw":["Marfanoid habitus B intellectual deficit, autosomal recessive is a very rare multiple congenital anomalies syndrome described in four sibs and characterized by intellectual deficit, flat face and some skeletelal features of Marfan syndrome such as tall stature, dolichostenomelia, arm span larger than height, arachnodactyly of hands and feet, little subcutaneous fat, muscle hypotonia and intellectual deficit."],"edges":249,"has_phenotype":true,"category":["disease"],"category_std":["disease"],"category_eng":["disease"],"category_kw":["disease"],"synonym":["MARFANOID MENTAL RETARDATION SYNDROME, AUTOSOMAL","MARFANOID mental retardation syndrome, autosomal","Marfanoid intellectual disability syndrome autosomal","Marfanoid mental retardation syndrome autosomal","MARFANOID intellectual disability syndrome, autosomal"],"synonym_std":["MARFANOID MENTAL RETARDATION SYNDROME, AUTOSOMAL","MARFANOID mental retardation syndrome, autosomal","Marfanoid intellectual disability syndrome autosomal","Marfanoid mental retardation syndrome autosomal","MARFANOID intellectual disability syndrome, autosomal"],"synonym_eng":["MARFANOID MENTAL RETARDATION SYNDROME, AUTOSOMAL","MARFANOID mental retardation syndrome, autosomal","Marfanoid intellectual disability syndrome autosomal","Marfanoid mental retardation syndrome autosomal","MARFANOID intellectual disability syndrome, autosomal"],"synonym_kw":["MARFANOID MENTAL RETARDATION SYNDROME, AUTOSOMAL","MARFANOID mental retardation syndrome, autosomal","Marfanoid intellectual disability syndrome autosomal","Marfanoid mental retardation syndrome autosomal","MARFANOID intellectual disability syndrome, autosomal"],"equivalent_curie":["ORPHA:2463","ORDO:2463","Orphanet:2463","MESH:C565410","OMIM:248770","MIM:248770","UMLS:C1855347","GARD:0003388"],"equivalent_curie_std":["ORPHA:2463","ORDO:2463","Orphanet:2463","MESH:C565410","OMIM:248770","MIM:248770","UMLS:C1855347","GARD:0003388"],"equivalent_curie_eng":["ORPHA:2463","ORDO:2463","Orphanet:2463","MESH:C565410","OMIM:248770","MIM:248770","UMLS:C1855347","GARD:0003388"],"equivalent_curie_kw":["ORPHA:2463","ORDO:2463","Orphanet:2463","MESH:C565410","OMIM:248770","MIM:248770","UMLS:C1855347","GARD:0003388"],"leaf":true,"_version_":1696524908458147800,"score":80.002235}],"facet_counts":{"category":{"disease":25,"publication":9,"anatomical entity":5,"cell":5,"gene":2,"sequence feature":2,"phenotype":1,"quality":1},"taxon_label":{"Gallus gallus":1,"Homo sapiens":1}},"highlighting":{"MONDO:0017310":{"highlight":"<em class=\\"hilite\\">Marfan</em> and <em class=\\"hilite\\">Marfan</em>-related disorder","match":"Marfan and Marfan-related disorder","has_highlight":true},"MONDO:0007947":{"highlight":"<em class=\\"hilite\\">Marfan</em> syndrome","match":"Marfan syndrome","has_highlight":true},"MONDO:0014831":{"highlight":"progeroid and <em class=\\"hilite\\">marfanoid</em> aspect-lipodystrophy syndrome","match":"progeroid and marfanoid aspect-lipodystrophy syndrome","has_highlight":true},"MONDO:0017309":{"highlight":"neonatal <em class=\\"hilite\\">Marfan</em> syndrome","match":"neonatal Marfan syndrome","has_highlight":true},"MONDO:0007948":{"highlight":"<em class=\\"hilite\\">marfanoid</em> hypermobility syndrome","match":"marfanoid hypermobility syndrome","has_highlight":true},"MONDO:0009120":{"highlight":"<em class=\\"hilite\\">marfanoid</em> syndrome, De Silva type","match":"marfanoid syndrome, De Silva type","has_highlight":true},"MONDO:0013574":{"highlight":"cutis laxa - <em class=\\"hilite\\">Marfanoid</em> syndrome","match":"cutis laxa - Marfanoid syndrome","has_highlight":true},"MONDO:0009565":{"highlight":"microcephaly-glomerulonephritis-<em class=\\"hilite\\">marfanoid</em> habitus syndrome","match":"microcephaly-glomerulonephritis-marfanoid habitus syndrome","has_highlight":true},"MONDO:0012171":{"highlight":"<em class=\\"hilite\\">marfanoid</em> habitus with situs inversus","match":"marfanoid habitus with situs inversus","has_highlight":true},"MONDO:0009566":{"highlight":"<em class=\\"hilite\\">marfanoid</em> habitus-autosomal recessive intellectual disability syndrome","match":"marfanoid habitus-autosomal recessive intellectual disability syndrome","has_highlight":true}}}')},2833:function(e,o,t){e.exports=t.p+"img/suzanna-lewis.a2e1c7b9.jpg"},"2a5a":function(e,o,t){"use strict";t("6b8b")},"2dc1":function(e,o,t){},"2f9c":function(e,o,t){e.exports=t.p+"img/mydrug.2399e82c.png"},"31d7":function(e,o,t){e.exports=t.p+"img/faisal-alquaddoomi.9df930a0.jpg"},3274:function(e,o,t){"use strict";t("adab")},"32bb":function(e,o,t){"use strict";t("64eb")},3305:function(e,o,t){},3467:function(e,o,t){"use strict";t("3305")},"34b6":function(e,o,t){e.exports=t.p+"img/category-cell-line.68e3150f.svg"},"35c6":function(e,o,t){e.exports=t.p+"img/justin-reese.785a4bed.jpg"},"37ec":function(e,o){e.exports="data:image/jpeg;base64,/9j/2wCEAAgGBgcGBQgHBwcJCQgKDBQNDAsLDBkSEw8UHRofHh0aHBwgJC4nICIsIxwcKDcpLDAxNDQ0Hyc5PTgyPC4zNDIBCQkJDAsMGA0NGDIhHCEyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMv/AABEIAIwAjAMBIgACEQEDEQH/xAGiAAABBQEBAQEBAQAAAAAAAAAAAQIDBAUGBwgJCgsQAAIBAwMCBAMFBQQEAAABfQECAwAEEQUSITFBBhNRYQcicRQygZGhCCNCscEVUtHwJDNicoIJChYXGBkaJSYnKCkqNDU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6g4SFhoeIiYqSk5SVlpeYmZqio6Slpqeoqaqys7S1tre4ubrCw8TFxsfIycrS09TV1tfY2drh4uPk5ebn6Onq8fLz9PX29/j5+gEAAwEBAQEBAQEBAQAAAAAAAAECAwQFBgcICQoLEQACAQIEBAMEBwUEBAABAncAAQIDEQQFITEGEkFRB2FxEyIygQgUQpGhscEJIzNS8BVictEKFiQ04SXxFxgZGiYnKCkqNTY3ODk6Q0RFRkdISUpTVFVWV1hZWmNkZWZnaGlqc3R1dnd4eXqCg4SFhoeIiYqSk5SVlpeYmZqio6Slpqeoqaqys7S1tre4ubrCw8TFxsfIycrS09TV1tfY2dri4+Tl5ufo6ery8/T19vf4+fr/2gAMAwEAAhEDEQA/AMBRT1FNAqQCvNPUHAU8CmgU8CgB6inAUgx1/E1y2r+KD5j2+nn7vDS4zz7f41UYuWwnJRWp0lzfW1kubmaOPPQFuT9BVP8A4SOxP+r3yH6bc/nXCFnuJC8kjyvnnLZY/nUsknlsBluOnHWtlSXUxdVnZnxCvaDntufGfyFSReIoJF5jZZB95WbgVxyu8ikh1yDgqfeliaXbyGYAgHHoKfskT7SR3UWtWz4zlTznPSrsFzDcqGhcEEdu1cGu8SJ5MhGcEA8n/OauRSvEysJhwQPlYgde/pSdJFKo+p22KMVkW+siRskARDAGOT+P61rI6yqHjIdGGQR3FYyi1uaxaewuKKXFLikUNpMU7FGBQBigVIopiipFpDHAU8U0U7opPYDmkBz/AIo1Q21uLSNyJJRlyOy+lchHgdTtGfTNXNVvhealLKvK5wDjIwOKit4RLIPkOW7A8/X0rqguWJzSblKyI4wOgAbPTK45qQq8kZwkp2jB7iteHSyjAgSc9gccfWrX9lTmMEI5Of4hnH40e1iX9Xn2MK0A8xUJxzjluv8AhWi7RR7fM+6D9GP4g5zVgaCZVIeMDd6KSfz61FF4ev5JsbHz2Lc5+tNVIvqS6M10K00ybfNQliTnf0P6f4VZ+0JcQvLHu81RkN049CO/f8q1YPB92MMcDPVPWpYfCs9uxOzg5II60vax7jWHn2M3TZB9okix98l1XOQAec/r/nv0en3KwSC1JU7uQF7cVyFxDPpl4AT5SYxvYcAZPXBzj2rRguMzRTJNHMF5BAxjHoKGlJEq8HqdrRQpDqGH8QznpS1zHSJSY+tOpKAMUVIBTRT1oAcBUV6/l2Nw391GI/KphTLpA9rMr9ChB+mKSBnmiuPMyASOgHTNdF4f0/LGeReWNYNqoMmcc9APauy0k4twPT0ravK0bInCwTndm5BaojA8VpwxxDHyVTtX3qAetaMSA1wNs9SyRMltCcHA/GrscESc7e1VEQj1qYvsxnPpVJktFjYo7DFNfYeNo5qEgnHX25pNh3Dk0EnM+MdHW70mWWNB5keWHrxXCac7m3wNpKMAV6Yr125USQuh/iGDmvL9Tsxb3m+MY3HDY4GRxn8sGuvDy0aOHFx1UjuLNt9nCf8AYFT1U0tidNhJOTtHNW6h7iWwlGKXFJikMx1p4pgp4oAeKHXMbD1BFAodlRTk9vTPFFxpX2PObXaWdumD1rrdJz9nDtxk8Vy3lmCaW3QE7X2huxFdBFPdQQoj2oEajHmRvkAe461pWV1oGG913Z1MEsCYJf2GOeatf2rYwY33Kr6Zrinv/s/DyNz/AAjiqL6paXDH5GkJGPnXdx+PSsY0b6nVKtbQ9Oj1S2ePKSI3ToaurcpJCZRjgZz6V5Vb3CbQI/lB/hycfrXYaXI1xpLTQyOY1HIEEhOPrik4W2KU01qdLHfxGME7c96cLy2P/LRAe+DXl9zdMGdPtTLGCSGB5+lWLF4hyLqXHXDHP9BVez01IdTXRHokoV/mRwRjtzmuB1m3J1CVQDjzCcemR/8AXFbljPNtAjkEiH361W3xXGvORzjBweOcEEfmBVU/dZhX96JpWsQgtYov7q4wfWpsUlLilcgSilpKAMYVItRinigB4rN1DP2yJgcYXj860gaq3cYeSNj0BNSzow7tMwLy2BvknAAGeR7+tdBBaLc2r8gAjGSM1iLvnaSdzwz7VXsMVuadLiMDPrj2qZt2RuoLmfmZVxpwSbzXhY98sMY/pUENlZRyFkgQOwIyBzXcIivHyR+NQ3EttBH8+zP+zQqr6A6Ce5yP9nReW0UceN42jBJwT39q9J8N2tqml+UE4VMcjmuYjzJ5dwQqxMcAHkkV1FnciKH93jDDBFJycnqV7NRWhwWsaFCmqTIC2A5dBngg8498HP8Ak1nW+hWkcwOZARnqQRz+Fd7qvlCTz3gEsRwHxyVPr9Kbapp8uGjHJ7KSMflWqqtKxi6CetijpOlvbrne8iNyC3X/AOvVKztootSmmkT7xBUZ4yzHn+tdb5cQXOwsPR23D9ax7mESXHmAZbzRnPpipUtxKleSTH0UE/MaKZzAKKKM0AYi1IKjBp4NADxTJk3r15U5FOBp39aRUZWaZj3sCxcpgBiCVHQH1qKC4aNR6Cr17bSvCTldiAtxyTWXGflPsc0rdzrUk3dGvJqot7fc5xxwKhtne9UzORyMhc1SuIcrHPgERkE56VRGtYm+zkPGD0wucjvg9hVxppxJnWcZWLc+p3KSCIRlgnBIbpWzZa6fJAJ575NYqTIVw8ZHAAPYD6U2CHy5FlGwpnJUt+n8qOSL2BVai3R0UesPeXDweW/kFceYTww71U+3jTLwYfMLkYPofeqhvobKPzXmWHI5JHGfb9arJfxauzCOM+X0LhcDnp17dePxq/Zp6mTrSi7NHbW+omSPg5BxTiuFeX++cgfpWNYRGC1QE9u/pW3K6m1gCHPGTWJc5NXIqKSlzVHMFLSUYoAwxTxUYNPBoAkBpwNMBpwNAxxAKkHoRg/SuZBMU0kR7EiulzXPahH/AKVKV6hiaRpTbWxIJRJavFnkjB9DUcFkskZinRMg8DGTmoILoW7biAUOAwPpU0zOJjdQ7xEw3EBeM/j1+taJe6Dlady7HZDvjtweQcVaWxQRk7F+Y54XIFZ6anBFs3g/Nxlhj/PNSW+pCO3VvJMjksWHoD/k/lTUWW60b7FqbSoLzZ9qQMEOQhOMn3FRWcAjvJlx+7YArhcAVctbvzGjijRd+cMenHf9anu4Et5gySGTcBnnkcd/zpO6Rm5Ju9rE8QDsB2A5qwoAXAUY6ACooU2R+55NS1kTJ3Y4UuabS0yRwpaaOadTA58Gng1EDTs0ASg04GoQ1LmkBNmse7w9xIRzzj8RWtbMJL6K3BHmurFR9B/iRWBYM0lrH5nEqgrIp6hh1B/GhrS5rS+KxVuYc5Hrwfar1nch4xA+Du4AZcjPqR/WiWPNQKGgYlOM9x1Bq4yVrMKkHe6N/wAmDdkjcVIUZAznnt6cVYWO1dhGmB1BO3gj/JrBju5Uh28b85LdM57/AF9KkW/lM2Ah2ZB9ATg/pTSv1F7S2ljfhtobZUnRD5uTuXoR9P0qOFDc3RbLPgkgHrx/9YZqktxPPlXPG7cAOMcY/LpRNd/ZtY0SIE+Y17GSB36jB/M03r7qM9dZM2hS5qISobi6gRv+Ped4SCckbWIH8qfmsmrOwJ3Vx+aUfhTRThQA4U7FIKdTA5gGnbqZQKYEmar3d9FaRlnI3Y4XuTVTUruWC3JjIBx6VhTMTKCfmLHknkmqhDm1ZEp2NfQNTmPiKKaY7pGVlRRxjvgfgDW74j03y2Gt2IzbTnFwg/gb+9j36H3+tcTA7R3aSoxV45FZSOxzXr+jkTTG3kRWhnj/AHiEcHIGa0qKzRMJNa9jg42EkefUVGy1Yu7ZLPUbi2h3CKORlUE54qI/ernlGzO6E+ZXGKoLY7+lXooAi5OM9BVKXhFYcH1FXLZi2MntSQSRcQCPJ46ZpfDdgdX8aW11Mn+jWWZySOMryMfjj86q3cjLbHBrbGdP8D3c1sSkk8i27NnkIFBwPTJdifetqKu7nLXdlY52LU2jvri7wD50rMy5xwzFuPzrXttUtbnAEm1zwFfg/h61yi/xfSnw/MCrcj3+tbSpqRzxm0duKeK5e1vri1ZI0fdGcfI/IH07106dq5pR5WbxlzEgp1NWn0DP/9k="},"397d":function(e,o,t){e.exports=t.p+"img/_member.05587ce6.jpg"},"3a38":function(e,o,t){e.exports=t.p+"img/kevin-schaper.40e685d5.jpg"},"3b32":function(e,o,t){e.exports=t.p+"img/sgd.b02349e1.png"},"3b48":function(e,o){e.exports="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH2QUHCjo7NefiBQAAAcZJREFUOMulk91qE0EUx3+zM0lM2SSNSWxQAwpK/LoSrSD4AH0D6ZP4BNIXsffihaAXufKjCqE3GhZUqhbTEJus6W53sx8zXqQUq8kK+r88Z85v5n/OHHH/Yc/wH1IAmw/amYdcLyHVhlo5dyK+vuFg/e2GvXHMo86Ql72DufljwLdRxLOui7MbEMUzV16gefxqn4vNU7h+ijELLABIaeHsBvT3Z6ArrSVGBwk5ZaGNQFkgRMYL6iXJ1VaRxrIijDS9zx5+qOlsu/hhSqkosy1IS3Drsk1/FJNqw5bj8bw7Iog04TRlZxDyxpkwOUzmAwCWbUWtpFipKIyBOJmZfvF+QhgZel9DNjtD3n05/LMHxsCW4/F9kvCxPyU6KhYCSksSpQRKCprVPE/fjikfWToGDNwp2588tIYffsL1VoE9N2GawJ12iSSFc/UcjUqOezcqVO1fAFFsePJ6zEpFYpBcOFNASsFgHNP94OEHCWurNZrV3PwxRonhfL1AIW9xtqo4Xc4TRJog0qy2bfwwpVFRi7+yXbRYu109kTDA0I3ZGYTkbYW0RPYYf5cAbl6yEUJgF1X2Mi2SZcHda7MGZgLWN5x/XuefQq+8WSNuxWsAAAAASUVORK5CYII="},"3daf":function(e,o,t){e.exports=t.p+"img/jules-jacobsen.c7c1a1a9.jpg"},"3dd4":function(e,o,t){e.exports=t.p+"img/hgnc.9844f37a.png"},4031:function(e,o,t){e.exports=t.p+"img/monochrom.e914b108.png"},4129:function(e,o,t){e.exports=t.p+"img/julie-mcmurry.b3ffcecd.jpg"},4354:function(e,o,t){e.exports=t.p+"img/string.4d073f1c.png"},"43fc":function(e,o,t){},"477e":function(e,o,t){e.exports=t.p+"img/charles-borromeo.39368a6b.jpg"},"499e":function(e,o,t){e.exports=t.p+"img/animalqtldb.6d83b9a2.png"},"4b84":function(e,o,t){e.exports=t.p+"img/craig-mcnamara.76bf767e.jpg"},"4c2a":function(e,o,t){e.exports=t.p+"img/category-pathway.fbd648e1.svg"},"4c96":function(e,o,t){e.exports=t.p+"img/flybase.e8046c12.png"},"50ee":function(e,o,t){e.exports=t.p+"img/daniel-keith.fae5889c.jpg"},"50fa":function(e,o,t){},"51d9":function(e,o,t){},5346:function(e,o,t){e.exports=t.p+"img/tudor-groza.65cbbad6.jpg"},"53c8":function(e,o,t){"use strict";t("1b63")},"54e6":function(e,o,t){e.exports=t.p+"img/hannah-blau.e514f6c6.jpg"},"54f9":function(e,o,t){"use strict";t("d025")},5522:function(e,o,t){e.exports=t.p+"img/category-anatomy.ca975b55.svg"},5762:function(e,o,t){},"58ab":function(e,o,t){e.exports=t.p+"img/ucscbands.3e9ea448.png"},"5adf":function(e,o,t){e.exports=t.p+"img/biogrid.c3ba63f2.png"},"5b1c":function(e,o,t){e.exports=t.p+"img/monica-munoz-torres.3219c8e4.jpg"},"5b33":function(e,o){e.exports="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHMAAAA9CAIAAAARc8GtAAAAAXNSR0IArs4c6QAAAAZ0Uk5TAIIAkQC0lV1BMQAAACF0RVh0U29mdHdhcmUAR3JhcGhpY0NvbnZlcnRlciAoSW50ZWwpd4f6GQAACkJJREFUeJzsm7+LHVUUx59V/gBrsbWzmEoQrG2UVIKQdgIWVjYStLCx0EIbYWpjoZVFEMUuBH/AECKErTaEBAwhP0zczZIXdwW/c7/vfd9559y5b957u0neusuXYd7M/XHu55577p07s6NPvrxwoqPQ6JlbcFx1/Mn+8tPX5z//kPrs4/dwbNIfrp+QXREoOJ498/Y7b70RhevQuffPtE1zQnaQtn79Hv5IpsTqjlFIfBR8jxVZjHS4oZR1WEtcngsxUNzcak/ILsBKstZhHVwrkqVQDhz/hOwMq7yPmHCs69oBdW5rAy6ZcpaDTsh2wmQlD5XruVBrT0DcnTif5fGE7AVBlMNSCrUa9UDpIgaF68Jq12f/a7Jw2D6sckk3WSlcKKWOIku4WDOsHHM3nqwirB3L1mctVvksndT2RBYutRrczSaLNjtXtR4XgwDxWc9VZ9i7FjdLyz6wlVfBm02WKy0ScQPZ+qwd+C68OpR8yojoo+dy2izA3WCydNjoX6DDE0s2RlWJUxxOmqZu036C9eIs3DZ1QFPXhc2HDSYr37TeSqw84ha90tJ3gZVkoWaa0sJ1wRe3qK6KuiovHp4XsnAHCQ2j2Mg2bU21U0+Mo94ybedTxkHtFg8kW1cVjp3bJl5KPB8oZiszdkN5t+xIyDpG7bTNM8+q68LR/rQnqeU4qfiTTbUgCFcd48i6xYN9/K0S2SnczLJB8WESN+p64SbO0mRFiifRlSIdd11SCc5D7UnsFd5Ft+mijQAuLJTjALHWdSW3xbnw9fVHvSgOzMjakRiVHYaOlFiUi1pWbbugwLaprQH28clFjCxc+qkcVgK7whq50MwM2fs74z7B4tsPx3cfPobw86/dJ7vjf6nu56OD7mRv/87f/+CoZBQS26Lw06lQb6e9fStU1GXB+c6YtdzbPbCvDLJk+xDb6WvGNFHOPtSpBNQIG6L9fWSf9KlD1n/XiBUUbk3uDsXq83rd290n37kI3tiRVKd4MjvyooKmfUiLmwzugdguEkAs2T+npck+B8qyfkK4cB/Cmkxr3dwyEZzr5lZ7985d6M/7jzkI4OacWoFJK1m7ws2+d9Dd6fKjRsnHgGxJGFKERb4TrLX8tMYtRDPbPYhgEK4DGYKsc8wyXzO7diWjz9TNq5DduvHg963bTv1jP68bd/ZcCRcvX1+2EGr71s6l9uq3P/z21Tc/f/TFdxJ+8iLualah22aDFRg1abJqm9puGmSxkqxCAYqdwL15jUFpFbKw9XT9qRqAMYefaN5SONBmZJyU0HQlvPz6WfjOUoUAmWjSDOi10x9I+InrREzKP178A57R5++4C7482l2bKv25sEtfRspuogbQm9fYeeg5FLU0WfQPmgEKbAbbg5+4OJjIGA6r7Cph9NKbA0uADXBwMkV2EDz1yrvIDr3w4qtWvAihfIEWa1csACEBi8LJebO/A6y4yFhhyeIKVsoaAV0QTz4LrU4WFtAInsDu4VzQMKRXIeIy3N9hg/rDokRT5bBVVUfKRMzaI1xcV3oLtyM7GkWyXaht2/mFwZidtEo0QKvIwlqMK+A1EA3HLEuQWw0ki1qsn9IGXEEL0Z5ueYDlbVoA8ARDGwZbZKoOZF38QWLGDYijCrPTufk36i7Oooq45FqLLBuGnqetdNshq10YTaw2+0CywiqmMAYDkA8LaeqYDAsukHlk44EeiS3c6LPMiHI4NGkhJ7H4voc7YVe278XJcKVo8OhAPgtxZCkmLHRbZNcoxugCJhW1kCwnPXorAcHF5Ke5LHMPezwHZRhAuJEsczG9HAg1ajcnrmcxi8ZCsLBbnSyJoFbYp4AAZIW88GjYITSc0wfGWUzojCHMi8Qoyjy5LQ4jYs1A3092Ale2cbo+b/bO6bCEm0aML2Ghz+YtdmRhhHVb+FGhbZz9yBELtUC2lxGXepqyuiAw25EYiHUOHJw3rRR7s1vbcER6+8GHfDYF9KV9dihZugAtQPv7bAV0zTxplTYe6LOIZVpLACvOp/PGKk8WbNrCjom9jqiq3a/03qHqG994ej6EaMCoRG8quC0zyunw0GVHXJmsVmnM283pa2FdLNfM6XgaTzZ/01SWVrL5bjs0suh/ui2vZ90WgVJ0kMDNEiWye/uYuOSw3aPaEWO9v5O1Le0N3niAxiLE43F2OjEuT1aLGKdIlomt28ZNgOiwuWiQ56UugUB51fC6nLJkJzG6W5AcQD1kV4oGfWSJSdG2a3+3Jz3JwsdZ67Bu/i34LGJLl7GqmQaD4yn4bA/ZoXnR3kOIBsTKkaKZFCzSBljKkqY4Dee01hnHorJksVBDeuZVkH0KWIfY1q+eaMAIoncwUY6su6WAUDcNL6IaOWxMb62PdcHx0SvlNEekhbaVlffZch7bVEcKiyG7TQMvxkU9ztLj+orqs37NFq6sNXvUfX4w4otubrNnhUfJ6LO6y3lcZJESFzmzx8SuKChb45A0MQu4lIU0mGb7SkAUWqFeq7zPClPUbogGFivCCMKr5jE+++rpAAvAbqWixDGWpYu6yzfBQ1qo9KnDDuxW7ExNw7127Qhzr8uW0NfMrG3u3NmT91l4U1pV7EfFGQwp+apusjOytw+7R9OdUDks0nOlMi0844/TWmZpIJcGPadbLIQ0beF2l8cJxmg+BFlT19wx2+uGyQHfZlo7ZQ/T5H0WY6GguOpyCbCQVjO6CJt2oOGwqK9QFGRvKTF3fJQm4diPRVmNwsuF7IsG7shkS4iRqswkKu+zuHH74bhPjiyWzTGN9miEeGFRtD6mwXr2lHn1gCwF2yjtXjtpxS2ySBwrzfZ6TFO2Ie+zWCdJSOR+OrL2rqStRe1PxzSRrKtLUuDmo3PWsHiRbxYkrN5hla0OP5mFudzR2abyJVepu5j32WzzpCFk0Qz7guTK9j1cKRcl66MYuMUXi7myhVl72N+R7JBmFmzLCoh7ycKUPjmyfcm4QVVVdbeXmlymXBRUqFE+y8VcwbyCItmBzSzY1qc82XKegWS3b+0QKE7WtB4ubx9AELXhthtJFiwKcmTLiYcXNUpvGfqkgECyWMytUF0keyi2RW0SWbitfQCBkHdNsohXR0QWypPFWCvIkS0nHl4UtDCx/SCDX0CB+PDquG2mrV78PCzbojaJLF/fOrgIC2VAEvrAvvWhz5Zts91wOGRhREGObDlxQXhOdWSH5Ipw6bz8Gu7i5es2Ma7wsxf7EZgWcLjoCnePFfpOCbKf3blaslqarHCMpl9yrE9WTR2YEYDcp1qz/ZemsR962s8UxVSrbGCKJbsvAV0VvBi7ZCjZ+HmslZ4R+YK6nLgsgFBRo7TbMkR0Rjrv3FZAVdO/rK+p58xXdd3/OKl3raxHu80Hex1kFxq5Clm5DK1ci6wpim8elxK/a6uqqrD/Mge0rpGlUKB9bizoqMgeuhC21szOsJgCQg2CUvrr/ufRVdFXo67jJGopU/NkL7VXN1Ft29oT/XwmypM90aHIkv0PAAD//wMAcbnpdyOXG0sAAAAASUVORK5CYII="},"5c21":function(e,o,t){"use strict";t("ea31")},"5caa":function(e,o,t){},"5cd7":function(e,o,t){e.exports=t.p+"img/category-unknown.5e2faa07.svg"},"5dac":function(e,o,t){e.exports=t.p+"img/genereviews.3d29edb4.png"},"5ed6":function(e,o,t){"use strict";t("aee8")},"608a":function(e,o,t){e.exports=t.p+"img/nicolas-matentzoglu.8db1b1b7.jpg"},6113:function(e,o,t){},"62eb":function(e,o,t){e.exports=t.p+"img/mychem.e62bb4c2.png"},"62f8":function(e,o,t){},"64af":function(e,o,t){e.exports=t.p+"img/category-gene.7111d2b7.svg"},"64eb":function(e,o,t){},6701:function(e,o,t){e.exports=t.p+"img/category-publication.759b17b4.svg"},"675f":function(e,o,t){e.exports=t.p+"img/monarch.e914b108.png"},"693c":function(e,o,t){e.exports=t.p+"img/category-phenotype.dff07e36.svg"},"6aa4":function(e,o,t){e.exports=t.p+"img/mmrrc.2468e4bd.png"},"6b8b":function(e,o,t){},"6da3":function(e,o,t){e.exports=t.p+"img/wormbase.ce41aaab.png"},"6e7f":function(e,o,t){e.exports=t.p+"img/omim.10f72a55.png"},"6ec3":function(e,o,t){},"6f32":function(e,o,t){var i={"./category-anatomy.svg":"5522","./category-case.svg":"75cc","./category-cell-line.svg":"34b6","./category-disease.svg":"fd9b","./category-function.svg":"aec5","./category-gene.svg":"64af","./category-genotype.svg":"7b75","./category-homolog.svg":"a3cd","./category-interaction.svg":"a427","./category-model.svg":"ac8f","./category-ortholog-disease.svg":"f05d","./category-ortholog-phenotype.svg":"e6ee","./category-pathway.svg":"4c2a","./category-phenotype.svg":"693c","./category-publication.svg":"6701","./category-unknown.svg":"5cd7","./category-variant.svg":"c7a0","./loading.svg":"1e31"};function r(e){var o=n(e);return t(o)}function n(e){if(!t.o(i,e)){var o=new Error("Cannot find module '"+e+"'");throw o.code="MODULE_NOT_FOUND",o}return i[e]}r.keys=function(){return Object.keys(i)},r.resolve=n,e.exports=r,r.id="6f32"},"71d9":function(e,o,t){},7430:function(e,o,t){e.exports=t.p+"img/damian-smedley.f9d46d2d.jpg"},"75cc":function(e,o,t){e.exports=t.p+"img/category-case.156826a1.svg"},"76f4":function(e,o,t){e.exports=t.p+"img/nicole-washington.b87984af.jpg"},"7b75":function(e,o,t){e.exports=t.p+"img/category-genotype.73698852.svg"},"7f12":function(e,o,t){e.exports=t.p+"img/harry-hochheiser.a2171b9d.jpg"},"7fb9":function(e,o,t){},"80d0":function(e,o,t){e.exports=t.p+"img/peter-robinson.95078722.jpg"},"83ef":function(e,o,t){},8670:function(e){e.exports=JSON.parse('{"@context":"http://obofoundry.github.io/registry/context.jsonld","ontologies":[{"activity_status":"active","browsers":[{"label":"BioPortal","title":"BioPortal Browser","url":"http://bioportal.bioontology.org/ontologies/BFO?p=classes"}],"contact":{"email":"phismith@buffalo.edu","github":"phismith","label":"Barry Smith","orcid":"0000-0003-1384-116X"},"depicted_by":"https://avatars2.githubusercontent.com/u/12972134?v=3&s=200","description":"The upper level ontology upon which OBO Foundry ontologies are built.","domain":"upper","homepage":"http://ifomis.org/bfo/","id":"bfo","in_foundry_order":1,"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"https://groups.google.com/forum/#!forum/bfo-discuss","ontology_purl":"http://purl.obolibrary.org/obo/bfo.owl","preferredPrefix":"BFO","products":[{"id":"bfo.owl","ontology_purl":"http://purl.obolibrary.org/obo/bfo.owl"},{"id":"bfo.obo","ontology_purl":"http://purl.obolibrary.org/obo/bfo.obo"}],"repository":"https://github.com/BFO-ontology/BFO","review":{"date":2016,"document":{"label":"PDF","link":"https://drive.google.com/open?id=0B81h9ah4tAM_RnNTRUZnVGRyWXM"}},"title":"Basic Formal Ontology","tracker":"https://github.com/BFO-ontology/BFO/issues","usages":[{"description":"BFO is imported by multiple OBO ontologies to standardize upper level structure","type":"owl_import","user":"http://obofoundry.org"}]},{"activity_status":"active","alternatePrefix":"ChEBI","browsers":[{"label":"CHEBI","title":"EBI CHEBI Browser","url":"http://www.ebi.ac.uk/chebi/chebiOntology.do?treeView=true&chebiId=CHEBI:24431#graphView"}],"build":{"infallible":1,"method":"obo2owl","source_url":"ftp://ftp.ebi.ac.uk/pub/databases/chebi/ontology/chebi.obo"},"contact":{"email":"amalik@ebi.ac.uk","github":"amalik01","label":"Adnan Malik","orcid":"0000-0001-8123-5351"},"depicted_by":"https://www.ebi.ac.uk/chebi/images/ChEBI_logo.png","description":"A structured classification of molecular entities of biological interest focusing on \'small\' chemical compounds.","domain":"biochemistry","homepage":"http://www.ebi.ac.uk/chebi","id":"chebi","in_foundry_order":1,"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/chebi.owl","page":"http://www.ebi.ac.uk/chebi/init.do?toolBarForward=userManual","preferredPrefix":"CHEBI","products":[{"id":"chebi.owl","ontology_purl":"http://purl.obolibrary.org/obo/chebi.owl"},{"id":"chebi.obo","ontology_purl":"http://purl.obolibrary.org/obo/chebi.obo"},{"id":"chebi.owl.gz","ontology_purl":"http://purl.obolibrary.org/obo/chebi.owl.gz","title":"chebi, compressed owl"},{"id":"chebi/chebi_lite.obo","ontology_purl":"http://purl.obolibrary.org/obo/chebi/chebi_lite.obo","title":"chebi_lite, no syns or xrefs"},{"id":"chebi/chebi_core.obo","ontology_purl":"http://purl.obolibrary.org/obo/chebi/chebi_core.obo","title":"chebi_core, no xrefs"}],"publications":[{"id":"http://europepmc.org/article/MED/26467479","title":"ChEBI in 2016: Improved services and an expanding collection of metabolites."}],"repository":"https://github.com/ebi-chebi/ChEBI","review":{"date":2010},"title":"Chemical Entities of Biological Interest","tracker":"https://github.com/ebi-chebi/ChEBI/issues","twitter":"chebit","usages":[{"description":"Rhea uses CHEBI to annotate reaction participants","examples":[{"description":"Query for all usages of CHEBI:29748 (chorismate)","url":"https://www.rhea-db.org/searchresults?q=CHEBI:29748"}],"user":"https://www.rhea-db.org/"},{"description":"ZFIN uses CHEBI to annotate experiments","examples":[{"description":"A curated zebrafish experiment involving exposure to (5Z,8Z,14Z)-11,12-dihydroxyicosatrienoic acid (CHEBI:63969)","url":"http://zfin.org/action/expression/experiment?id=ZDB-EXP-190627-10"}],"user":"http://zfin.org"}]},{"DO wiki":"http://diseaseontology.sourceforge.net/","activity_status":"active","browsers":[{"label":"DO","title":"DO Browser","url":"http://www.disease-ontology.org/"}],"build":{"infallible":1,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/DiseaseOntology/HumanDiseaseOntology/master/src/ontology/doid.obo"},"contact":{"email":"lynn.schriml@gmail.com","github":"lschriml","label":"Lynn Schriml","orcid":"0000-0001-8910-9851"},"description":"An ontology for describing the classification of human diseases organized by etiology.","domain":"disease","facebook":"https://www.facebook.com/diseaseontology","homepage":"http://www.disease-ontology.org","id":"doid","in_foundry_order":1,"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/doid.owl","preferredPrefix":"DOID","products":[{"id":"doid.owl","ontology_purl":"http://purl.obolibrary.org/obo/doid.owl","title":"Disease Ontology, OWL format. This file contains DO\'s is_a asserted hierarchy plus equivalent axioms to other OBO Foundry ontologies."},{"id":"doid.obo","ontology_purl":"http://purl.obolibrary.org/obo/doid.obo","title":"Disease Ontology, OBO format. This file omits the equivalent axioms."}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/?term=25348409","title":"Disease Ontology 2015 update: an expanded and updated database of human diseases for linking biomedical knowledge through disease data"}],"repository":"https://github.com/DiseaseOntology/HumanDiseaseOntology","review":{"date":2015,"document":{"label":"PDF","link":"https://drive.google.com/open?id=0B8vqEgF1N0NIZ082U2JETHlSTGs"}},"taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Human Disease Ontology","tracker":"https://github.com/DiseaseOntology/HumanDiseaseOntology/issues","twitter":"diseaseontology","usages":[{"description":"Alliance of Genome Resources - MGD, RGD, SGD, FlyBase, WormBase, ZFIN use DO","examples":[{"description":"Human diseases annotated to over 190,000 MOD genes, alleles, disease models and human genes","url":"https://www.alliancegenome.org/search?category=disease"},{"description":"The landing page for Coronavirus Infectious Disease","url":"https://www.alliancegenome.org/disease/DOID:0080599"}],"user":"https://www.alliancegenome.org"},{"description":"MGI disease model annotations use DO","examples":[{"description":"physical disorder","url":"http://www.informatics.jax.org/disease/DOID:0080015"}],"user":"http://www.informatics.jax.org/disease"},{"description":"Immune Epitope Database","examples":[{"description":"Aantibody and T cell epitopes associated with human diseases","url":"https://www.iedb.org"}],"user":"https://www.iedb.org"}]},{"activity_status":"active","browsers":[{"label":"AmiGO","title":"Gene Ontology AmiGO 2 Browser","url":"http://amigo.geneontology.org/amigo/term/GO:0008150#display-lineage-tab"}],"contact":{"email":"suzia@stanford.edu","github":"suzialeksander","label":"Suzi Aleksander","orcid":"0000-0001-6787-2901"},"dependencies":[{"id":"uberon","subset":"go/extensions/uberon_import.owl"},{"id":"cl","subset":"go/extensions/cl_import.owl"},{"id":"ncbitaxon","subset":"go/extensions/ncbitaxon_import.owl"},{"id":"ro","subset":"go/extensions/ro_import.owl"},{"connects":[{"id":"nifstd"},{"id":"go"}],"description":"Bridging axioms between nifstd and go","id":"go/extensions/go-bridge-to-nifstd.owl","publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24093723","title":"The Gene Ontology (GO) Cellular Component Ontology: integration with SAO (Subcellular Anatomy Ontology) and other recent developments."}],"title":"GO bridge to NIFSTD","type":"BridgeOntology"}],"depicted_by":"http://geneontology.org/assets/go-logo-icon.mini.png","description":"An ontology for describing the function of genes and gene products","domain":"biology","facebook":"https://www.facebook.com/Gene-Ontology-305908656519/","homepage":"http://geneontology.org/","id":"go","in_foundry_order":1,"integration_server":"http://build.berkeleybop.org/view/GO","label":"GO","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/go.owl","preferredPrefix":"GO","products":[{"description":"The main ontology in OWL. This is self contained and does not have connections to other OBO ontologies","id":"go.owl","ontology_purl":"http://purl.obolibrary.org/obo/go.owl","page":"http://geneontology.org/page/download-ontology","title":"GO (OWL edition)"},{"description":"Equivalent to go.owl, in obo format","id":"go.obo","ontology_purl":"http://purl.obolibrary.org/obo/go.obo","page":"http://geneontology.org/page/download-ontology","title":"GO (OBO Format edition)"},{"description":"Equivalent to go.owl, in obograph json format","id":"go.json","ontology_purl":"http://purl.obolibrary.org/obo/go.json","page":"https://github.com/geneontology/obographs/","title":"GO (JSON edition)"},{"description":"The main ontology plus axioms connecting to select external ontologies, with subsets of those ontologies","id":"go/extensions/go-plus.owl","ontology_purl":"http://purl.obolibrary.org/obo/go/extensions/go-plus.owl","page":"http://geneontology.org/page/download-ontology","title":"GO-Plus"},{"description":"The main ontology plus axioms connecting to select external ontologies, excluding the external ontologies themselves","id":"go/go-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/go/go-base.owl","page":"http://geneontology.org/page/download-ontology","title":"GO Base Module"},{"description":"As go-plus.owl, in obographs json format","id":"go/extensions/go-plus.json","ontology_purl":"http://purl.obolibrary.org/obo/go/extensions/go-plus.json","page":"https://github.com/geneontology/obographs/","title":"GO-Plus"},{"description":"Basic version of the GO, filtered such that the graph is guaranteed to be acyclic and annotations can be propagated up the graph. The relations included are is a, part of, regulates, negatively regulates and positively regulates. This version excludes relationships that cross the 3 GO hierarchies.","id":"go/go-basic.obo","ontology_purl":"http://purl.obolibrary.org/obo/go/go-basic.obo","page":"http://geneontology.org/page/download-ontology","title":"GO-Basic, Filtered, for use with legacy tools"},{"description":"As go-basic.obo, in json format","id":"go/go-basic.json","ontology_purl":"http://purl.obolibrary.org/obo/go/go-basic.json","page":"http://geneontology.org/page/download-ontology","title":"GO-Basic, Filtered, for use with legacy tools (JSON)"},{"description":"Classes added to ncbitaxon for groupings such as prokaryotes","id":"go/extensions/go-taxon-groupings.owl","ontology_purl":"http://purl.obolibrary.org/obo/go/extensions/go-taxon-groupings.owl","page":"http://geneontology.org/page/download-ontology","title":"GO Taxon Groupings"},{"description":"Equivalent to go.owl, but released daily. Note the snapshot release is not archived.","id":"go/snapshot/go.owl","ontology_purl":"http://purl.obolibrary.org/obo/go/snapshot/go.owl","page":"http://geneontology.org/page/download-ontology","title":"GO (OWL edition), daily snapshot release"},{"description":"Equivalent to go.owl, but released daily. Note the snapshot release is not archived.","id":"go/snapshot/go.obo","ontology_purl":"http://purl.obolibrary.org/obo/go/snapshot/go.obo","page":"http://geneontology.org/page/download-ontology","title":"GO (OBO Format edition), daily snapshot release"}],"repository":"https://github.com/geneontology/go-ontology","review":{"date":2010},"taxon":{"id":"NCBITaxon:1","label":"All life"},"title":"Gene Ontology","tracker":"https://github.com/geneontology/go-ontology/issues/","twitter":"news4go","usages":[{"description":"The GO ontology is used by the GO consortium for functional annotation of genes","examples":[{"description":"annotations to transmembrane transport","url":"http://amigo.geneontology.org/amigo/term/GO:0055085"}],"type":"annotation","user":"http://geneontology.org"},{"description":"Uniprot uses GO to show the function of proteins","examples":[{"description":"functional annotations of human Sonic hedgehog protein","url":"https://www.uniprot.org/uniprot/Q15465#function"}],"type":"annotation","user":"https://www.uniprot.org"},{"description":"Reactome annotates activities, pathways, and cellular localization using GO","examples":[{"description":"protein tyrosine kinase activity of an EGFR complex","url":"https://reactome.org/content/detail/R-HSA-177934"}],"type":"annotation","user":"https://reactome.org"},{"description":"The Alliance of Genome Resources uses GO for model organism gene function annotation","examples":[{"description":"Functional summary of C elegans nsy-1 gene","url":"https://www.alliancegenome.org/gene/WB:WBGene00003822#function---go-annotations"},{"description":"Gene Ontology Causal Activity Models for C elegans nsy-1 gene","url":"https://www.alliancegenome.org/gene/WB:WBGene00003822#pathways"}],"type":"annotation","user":"https://www.alliancegenome.org"},{"description":"Rhea uses GO to describe individual biochemical reactions","examples":[{"description":"Glutamine scyllo-inositol transaminase reaction and associated GO term","url":"https://www.rhea-db.org/rhea/22920"}],"type":"mapping","user":"https://www.rhea-db.org"}]},{"activity_status":"active","browsers":[{"label":"BioPortal","title":"BioPortal Browser","url":"http://bioportal.bioontology.org/ontologies/OBI?p=classes"}],"build":{"source_url":"http://purl.obofoundry.org/obo/obi/repository/trunk/src/ontology/branches/"},"contact":{"email":"bpeters@lji.org","github":"bpeters42","label":"Bjoern Peters","orcid":"0000-0002-8457-6693"},"depicted_by":"https://svn.code.sf.net/p/obi/code/trunk/web/htdocs/images/obi-lotext.png","description":"An integrated ontology for the description of life-science and clinical investigations","domain":"experiments","homepage":"http://obi-ontology.org","id":"obi","in_foundry_order":1,"integration_server":"http://build.berkeleybop.org/job/build-obi/","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"mailing_list":"http://groups.google.com/group/obi-users","ontology_purl":"http://purl.obolibrary.org/obo/obi.owl","preferredPrefix":"OBI","products":[{"description":"The full version of OBI in OWL format","id":"obi.owl","ontology_purl":"http://purl.obolibrary.org/obo/obi.owl","title":"OBI"},{"description":"The OBO-format version of OBI","id":"obi.obo","ontology_purl":"http://purl.obolibrary.org/obo/obi.obo","title":"OBI in OBO"},{"description":"A collection of important high-level terms and their relations from OBI and other ontologies","id":"obi/obi_core.owl","ontology_purl":"http://purl.obolibrary.org/obo/obi/obi_core.owl","title":"OBI Core"},{"description":"Base module for OBI","id":"obi/obi-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/obi/obi-base.owl","title":"OBI Base module"}],"repository":"https://github.com/obi-ontology/obi","review":{"date":2013,"document":{"label":"PDF","link":"https://drive.google.com/open?id=0B8vqEgF1N0NIMFlSM3RvdUxGTnc"}},"title":"Ontology for Biomedical Investigations","tracker":"http://purl.obolibrary.org/obo/obi/tracker","usages":[{"description":"PennTURBO accelerates the processes of finding and connecting key information from clinical records, via semantic modeling of the processes that generated the data. This makes the discovery of previously unappreciated relations between the data possible for research and for operational tasks.","examples":[{"description":"CYP2C1*2/*2 MI patient taking Clopidogrel","url":"https://github.com/PennTURBO/Turbo-Documentation/blob/master/IBI_CIC_TURBO_MAM_20190102.pdf"}],"user":"http://upibi.org/turbo/"},{"description":"The Evidence & Conclusion Ontology (ECO) describes types of scientific evidence within the biological research domain. ECO uses OBI to logically describe how the evidence arises from an investigation.","examples":[{"description":"rapid amplification of cDNA ends polymerase chain reaction evidence","url":"http://purl.obolibrary.org/obo/ECO_0001820"}],"user":"http://evidenceontology.org/"}]},{"activity_status":"active","browsers":[{"label":"BioPortal","title":"BioPortal Ontology Browser","url":"https://bioportal.bioontology.org/ontologies/PATO"}],"build":{"infallible":1,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/pato-ontology/pato/master/pato.obo"},"contact":{"email":"g.gkoutos@gmail.com","github":"gkoutos","label":"George Gkoutos","orcid":"0000-0002-2061-091X"},"description":"An ontology of phenotypic qualities (properties, attributes or characteristics)","domain":"phenotype","homepage":"https://github.com/pato-ontology/pato/","id":"pato","in_foundry_order":1,"jobs":[{"id":"https://travis-ci.org/pato-ontology/pato","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pato.owl","preferredPrefix":"PATO","products":[{"id":"pato.owl","ontology_purl":"http://purl.obolibrary.org/obo/pato.owl"},{"id":"pato.obo","ontology_purl":"http://purl.obolibrary.org/obo/pato.obo"},{"id":"pato.json","ontology_purl":"http://purl.obolibrary.org/obo/pato.json"},{"description":"Includes axioms linking to other ontologies, but no imports of those ontologies","id":"pato/pato-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/pato/pato-base.owl"}],"repository":"https://github.com/pato-ontology/pato","review":{"date":2010},"title":"Phenotype And Trait Ontology","tracker":"https://github.com/pato-ontology/pato/issues","usages":[{"description":"PATO is used by the Human Phenotype Ontology (HPO) for logical definitions of phenotypes that facilitate cross-species integration.","examples":[{"description":"An abnormality in a cellular process.","url":"https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0011017&viewMode=All&siblings=false"}],"publications":[{"id":"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6324074/","title":"Expansion of the Human Phenotype Ontology (HPO) knowledge base and resources"}],"type":"annotation","user":"https://hpo.jax.org/app/"}]},{"activity_status":"active","browsers":[{"label":"Planteome","title":"Planteome browser","url":"http://browser.planteome.org/amigo"}],"build":{"checkout":"git clone https://github.com/Planteome/plant-ontology.git","infallible":1,"method":"vcs","system":"git"},"contact":{"email":"jaiswalp@science.oregonstate.edu","github":"jaiswalp","label":"Pankaj Jaiswal","orcid":"0000-0002-1005-8383"},"description":"The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data.","domain":"anatomy and development","homepage":"http://browser.planteome.org/amigo","id":"po","in_foundry_order":1,"jobs":[{"id":"https://travis-ci.org/Planteome/plant-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/po.owl","page":"https://github.com/Planteome/plant-ontology","preferredPrefix":"PO","products":[{"id":"po.owl","ontology_purl":"http://purl.obolibrary.org/obo/po.owl"},{"id":"po.obo","ontology_purl":"http://purl.obolibrary.org/obo/po.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/23220694","title":"The plant ontology as a tool for comparative plant anatomy and genomic analyses."},{"id":"https://doi.org/10.1093/nar/gkx1152","title":"The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics."}],"repository":"https://github.com/Planteome/plant-ontology","review":{"date":2013,"document":{"label":"PDF","link":"https://drive.google.com/open?id=0B8vqEgF1N0NIV1o0N21UOHlLSmc"}},"taxon":{"id":"NCBITaxon:33090","label":"Viridiplantae"},"title":"Plant Ontology","tracker":"https://github.com/Planteome/plant-ontology/issues","twitter":"planteome","usages":[{"description":"Planteome uses PO to describe location of tissue expression for genes in viridiplantae","examples":[{"description":"Genes and proteins annotate to leaf","url":"http://browser.planteome.org/amigo/term/PO:0025034"}],"user":"http://planteome.org/"},{"description":"Maize CELL genomics DB uses PO to annotate images","examples":[{"description":"LhG4 Promoter Drivers","url":"http://maize.jcvi.org/cellgenomics/geneDB_list.php?filter=3"}],"user":"http://maize.jcvi.org/"},{"description":"MaizeGDB uses PO for annotation of genes","examples":[{"description":"Introduced in gene model set 5b in assembly version RefGen_v2.","url":"http://maizegdb.org/gene_center/gene/GRMZM5G863962"}],"user":"http://maizegdb.org/"},{"description":"Gramene uses PO for the annotation of plant genes","examples":[{"description":"Gramene annotations to leaf from Arabidopsis","url":"http://archive.gramene.org/db/ontology/search?id=PO:0025034"}],"user":"http://gramene.org/"}]},{"activity_status":"active","browsers":[{"label":"BioPortal","title":"BioPortal Browser","url":"http://bioportal.bioontology.org/ontologies/PR?p=classes&conceptid=http://purl.obolibrary.org/obo/PR_000000001"},{"label":"PRO","title":"PRO Home","url":"http://proconsortium.org"}],"build":{"infallible":0,"method":"obo2owl","oort_args":"--no-reasoner","source_url":"https://proconsortium.org/download/current/pro_nonreasoned.obo"},"contact":{"email":"dan5@georgetown.edu","github":"nataled","label":"Darren Natale","orcid":"0000-0001-5809-9523"},"depicted_by":"https://raw.githubusercontent.com/PROconsortium/logo/master/PROlogo_small.png","description":"An ontological representation of protein-related entities","development":{"id_policy":"https://pir17.georgetown.edu/confluence/display/PROWIKI/PRO+PURLs"},"documentation":"https://proconsortium.org/download/current/pro_readme.txt","domain":"proteins","homepage":"http://proconsortium.org","id":"pr","in_foundry_order":1,"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pr.owl","preferredPrefix":"PR","products":[{"description":"PRO after reasoning has been applied, OWL format.","id":"pr.owl","ontology_purl":"http://purl.obolibrary.org/obo/pr.owl","title":"pro_reasoned.owl"},{"description":"PRO after reasoning has been applied, OBO format.","id":"pr.obo","ontology_purl":"http://purl.obolibrary.org/obo/pr.obo","title":"pro_reasoned.obo"}],"publications":[{"id":"https://proconsortium.org/pro_dsmnt.shtml#publication","title":"Publications & Dissemination"}],"repository":"https://github.com/PROconsortium/PRoteinOntology","review":{"date":2010},"title":"PRotein Ontology (PRO)","tracker":"https://github.com/PROconsortium/PRoteinOntology/issues","usages":[{"description":"Colorado Richly Annotated Full-Text (CRAFT) Corpus; PRO is used for entity tagging and annotation","examples":[{"description":"Tagged entities (requires download)","url":"https://github.com/UCDenver-ccp/CRAFT/releases/tag/v4.0.1"}],"user":"https://github.com/UCDenver-ccp/CRAFT"},{"description":"Cell Ontology is a structured controlled vocabulary for cell types in animals; PRO is used for cell type definitions","examples":[{"description":"A B cell that is CD19-positive (uses the PRO term for non-species-specific CD19 molecule, PR:000001002)","url":"http://purl.obolibrary.org/obo/CL_0001201"}],"user":"http://www.obofoundry.org/ontology/cl.html"}]},{"activity_status":"active","build":{"infallible":0,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/xenopus-anatomy/xao/master/xenopus_anatomy.obo"},"contact":{"email":"esegerd3@gmail.com","github":"seger","label":"Erik Segerdell","orcid":"0000-0002-9611-1279"},"description":"XAO represents the anatomy and development of the African frogs Xenopus laevis and tropicalis.","domain":"anatomy","homepage":"http://www.xenbase.org/anatomy/xao.do?method=display","id":"xao","in_foundry_order":1,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/xao.owl","preferredPrefix":"XAO","products":[{"id":"xao.owl","ontology_purl":"http://purl.obolibrary.org/obo/xao.owl"},{"id":"xao.obo","ontology_purl":"http://purl.obolibrary.org/obo/xao.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/18817563","title":"An ontology for Xenopus anatomy and development."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24139024","title":"Enhanced XAO: the ontology of Xenopus anatomy and development underpins more accurate annotation of gene expression and queries on Xenbase."}],"repository":"https://github.com/xenopus-anatomy/xao","review":{"date":2010},"taxon":{"id":"NCBITaxon:8353","label":"Xenopus"},"title":"Xenopus Anatomy Ontology","tracker":"https://github.com/xenopus-anatomy/xao/issues","usages":[{"description":"Xenbase uses XAO to annotate gene expression.","examples":[{"description":"Xenopus genes expressed in the pronephric kidney.","url":"http://www.xenbase.org/anatomy/showanatomy.do?method=displayAnatomySummary&anatomyId=463"}],"user":"http://www.xenbase.org"}]},{"activity_status":"active","build":{"infallible":1,"method":"obo2owl","notes":"may be ready to switch to vcs soon","source_url":"https://raw.githubusercontent.com/cerivs/zebrafish-anatomical-ontology/master/src/zebrafish_anatomy.obo"},"contact":{"email":"van_slyke@zfin.org","github":"cerivs","label":"Ceri Van Slyke","orcid":"0000-0002-2244-7917"},"description":"A structured controlled vocabulary of the anatomy and development of the Zebrafish","domain":"anatomy","homepage":"https://wiki.zfin.org/display/general/Anatomy+Atlases+and+Resources","id":"zfa","in_foundry_order":1,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/zfa.owl","preferredPrefix":"ZFA","products":[{"id":"zfa.owl","ontology_purl":"http://purl.obolibrary.org/obo/zfa.owl"},{"id":"zfa.obo","ontology_purl":"http://purl.obolibrary.org/obo/zfa.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24568621","title":"The zebrafish anatomy and stage ontologies: representing the anatomy and development of Danio rerio."}],"repository":"https://github.com/cerivs/zebrafish-anatomical-ontology","review":{"date":2010},"taxon":{"id":"NCBITaxon:7954","label":"Danio"},"title":"Zebrafish anatomy and development ontology","tracker":"https://github.com/cerivs/zebrafish-anatomical-ontology/issues","usages":[{"description":"ZFIN uses ZFA to annotate gene expression and phenotype","examples":[{"description":"zebrafish genes expressed in hindbrain and genotypes with hindbrain phenotype","url":"http://zfin.org/ZFA:0000029"}],"user":"http://zfin.org"}]},{"activity_status":"active","build":{"checkout":"git clone  https://github.com/AgriculturalSemantics/agro.git","path":".","system":"git"},"contact":{"email":"m.a.laporte@cgiar.org","github":"marieALaporte","label":"Marie-Anglique Laporte","orcid":"0000-0002-8461-9745"},"dependencies":[{"id":"envo"},{"id":"go"},{"id":"foodon"},{"id":"ncbitaxon"},{"id":"pato"},{"id":"peco"},{"id":"po"},{"id":"to"},{"id":"ro"},{"id":"bfo"},{"id":"iao"},{"id":"obi"},{"id":"uo"},{"id":"xco"}],"description":"Ontology of agronomic practices, agronomic techniques, and agronomic variables used in agronomic experiments","domain":"agronomy","homepage":"https://github.com/AgriculturalSemantics/agro","id":"agro","jobs":[{"id":"https://travis-ci.org/AgriculturalSemantics/agro","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/agro.owl","preferredPrefix":"AGRO","products":[{"description":"Contains all AgrO terms and links to other relavent ontologies.","id":"agro.owl","ontology_purl":"http://purl.obolibrary.org/obo/agro.owl","title":"AgrO"}],"publications":[{"id":"http://ceur-ws.org/Vol-1747/IT205_ICBO2016.pdf","title":"Data-driven Agricultural Research for Development: A Need for Data Harmonization Via Semantics."}],"repository":"https://github.com/AgriculturalSemantics/agro","title":"Agronomy Ontology","tracker":"https://github.com/AgriculturalSemantics/agro/issues/","usages":[{"description":"AgroFIMS enables digital collection of agronomic data that is semantically described a priori with agronomic terms from AgrO.","user":"https://agrofims.org/about"},{"description":"AgrO is being used by GARDIAN to facilitate data search within publications and datasets for use in quantitative analyses.","user":"https://gardian.bigdata.cgiar.org/"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/insect-morphology/aism","path":".","system":"git"},"contact":{"email":"entiminae@gmail.com","github":"JCGiron","label":"Jennifer C. Girn","orcid":"0000-0002-0851-6883"},"dependencies":[{"id":"ro"},{"id":"uberon"},{"id":"caro"},{"id":"pato"},{"id":"bfo"},{"id":"bspo"}],"description":"The AISM contains terms used in insect biodiversity research for describing structures of the exoskeleton and the skeletomuscular system. It aims to serve as the basic backbone of generalized terms to be expanded with order-specific terminology.","domain":"anatomy","homepage":"https://github.com/insect-morphology/aism","id":"aism","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/aism.owl","preferredPrefix":"AISM","products":[{"id":"aism.owl","ontology_purl":"http://purl.obolibrary.org/obo/aism.owl"},{"id":"aism.obo","ontology_purl":"http://purl.obolibrary.org/obo/aism.obo"},{"id":"aism.json","ontology_purl":"http://purl.obolibrary.org/obo/aism.json"}],"repository":"https://github.com/insect-morphology/aism","title":"Ontology for the Anatomy of the Insect SkeletoMuscular system (AISM)","tracker":"https://github.com/insect-morphology/aism/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/EBISPOT/amphx_ontology.git","path":".","system":"git"},"contact":{"email":"hescriva@obs-banyuls.fr","github":"hescriva","label":"Hector Escriva","orcid":"0000-0001-7577-5028"},"dependencies":[{"id":"uberon"}],"description":"An ontology for the development and anatomy of Amphioxus (Branchiostoma lanceolatum).","domain":"anatomy","homepage":"https://github.com/EBISPOT/amphx_ontology","id":"amphx","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/amphx.owl","preferredPrefix":"AMPHX","products":[{"id":"amphx.owl","ontology_purl":"http://purl.obolibrary.org/obo/amphx.owl"},{"id":"amphx.obo","ontology_purl":"http://purl.obolibrary.org/obo/amphx.obo"}],"repository":"https://github.com/EBISPOT/amphx_ontology","title":"The Amphioxus Development and Anatomy Ontology","tracker":"https://github.com/EBISPOT/amphx_ontology/issues"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/obophenotype/ascomycete-phenotype-ontology/master/apo.obo"},"contact":{"email":"stacia@stanford.edu","github":"srengel","label":"Stacia R Engel","orcid":"0000-0001-5472-917X"},"description":"A structured controlled vocabulary for the phenotypes of Ascomycete fungi","domain":"phenotype","homepage":"http://www.yeastgenome.org/","id":"apo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/apo.owl","preferredPrefix":"APO","products":[{"id":"apo.owl","ontology_purl":"http://purl.obolibrary.org/obo/apo.owl"},{"id":"apo.obo","ontology_purl":"http://purl.obolibrary.org/obo/apo.obo"}],"repository":"https://github.com/obophenotype/ascomycete-phenotype-ontology","taxon":{"id":"NCBITaxon:4890","label":"Ascomycota"},"title":"Ascomycete phenotype ontology","tracker":"https://github.com/obophenotype/ascomycete-phenotype-ontology/issues"},{"activity_status":"active","contact":{"email":"MBrochhausen@uams.edu","github":"mbrochhausen","label":"Mathias Brochhausen","orcid":"0000-0003-1834-3856"},"description":"Defines terms and relations necessary for interoperation between epidemic models and public health application software that interface with these models","domain":"health","homepage":"https://github.com/ApolloDev/apollo-sv","id":"apollo_sv","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/apollo_sv.owl","preferredPrefix":"APOLLO_SV","products":[{"id":"apollo_sv.owl","ontology_purl":"http://purl.obolibrary.org/obo/apollo_sv.owl"}],"repository":"https://github.com/ApolloDev/apollo-sv","title":"Apollo Structured Vocabulary","tracker":"https://github.com/ApolloDev/apollo-sv/issues"},{"activity_status":"active","contact":{"email":"mcarthua@mcmaster.ca","github":"agmcarthur","label":"Andrew G. McArthur","orcid":"0000-0002-1142-3063"},"description":"Antibiotic resistance genes and mutations","homepage":"https://github.com/arpcard/aro","id":"aro","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"mailing_list":"https://mailman.mcmaster.ca/mailman/listinfo/card-l","ontology_purl":"http://purl.obolibrary.org/obo/aro.owl","preferredPrefix":"ARO","products":[{"id":"aro.owl","ontology_purl":"http://purl.obolibrary.org/obo/aro.owl"}],"repository":"https://github.com/arpcard/aro","title":"Antibiotic Resistance Ontology","tracker":"https://github.com/arpcard/aro/issues"},{"activity_status":"active","contact":{"email":"rlwalls2008@gmail.com","github":"ramonawalls","label":"Ramona Walls","orcid":"0000-0001-8815-0078"},"description":"An ontology to support the interoperability of biodiversity data, including data on museum collections, environmental/metagenomic samples, and ecological surveys.","domain":"biodiversity collections","homepage":"https://github.com/BiodiversityOntologies/bco","id":"bco","in_foundry":false,"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/bco.owl","preferredPrefix":"BCO","products":[{"id":"bco.owl","ontology_purl":"http://purl.obolibrary.org/obo/bco.owl"}],"repository":"https://github.com/BiodiversityOntologies/bco","title":"Biological Collections Ontology","tracker":"https://github.com/BiodiversityOntologies/bco/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/biological-spatial-ontology.git","infallible":1,"method":"vcs","path":"src/ontology","system":"git"},"contact":{"email":"cjmungall@lbl.gov","github":"cmungall","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"description":"An ontology for representing spatial concepts, anatomical axes, gradients, regions, planes, sides, and surfaces","domain":"anatomy","homepage":"https://github.com/obophenotype/biological-spatial-ontology","id":"bspo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/bspo.owl","preferredPrefix":"BSPO","products":[{"id":"bspo.owl","ontology_purl":"http://purl.obolibrary.org/obo/bspo.owl"},{"id":"bspo.obo","ontology_purl":"http://purl.obolibrary.org/obo/bspo.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/25140222","title":"Nose to tail, roots to shoots: spatial descriptors for phenotypic diversity in the Biological Spatial Ontology."}],"repository":"https://github.com/obophenotype/biological-spatial-ontology","title":"Biological Spatial Ontology","tracker":"https://github.com/obophenotype/biological-spatial-ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/BRENDA-Enzymes/BTO.git","path":".","system":"git"},"contact":{"email":"c.dudek@tu-braunschweig.de","github":"chdudek","label":"Christian-Alexander Dudek","orcid":"0000-0001-9117-7909"},"description":"A structured controlled vocabulary for the source of an enzyme comprising tissues, cell lines, cell types and cell cultures.","domain":"anatomy","homepage":"http://www.brenda-enzymes.org","id":"bto","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/bto.owl","page":"https://en.wikipedia.org/wiki/BRENDA_tissue_ontology","preferredPrefix":"BTO","products":[{"id":"bto.owl","ontology_purl":"http://purl.obolibrary.org/obo/bto.owl"},{"id":"bto.obo","ontology_purl":"http://purl.obolibrary.org/obo/bto.obo"},{"id":"bto.json","ontology_purl":"http://purl.obolibrary.org/obo/bto.json"}],"publications":[{"id":"https://www.ncbi.nlm.nih.gov/pubmed/21030441","title":"The BRENDA Tissue Ontology (BTO): the first all-integrating ontology of all organisms for enzyme sources"}],"repository":"https://github.com/BRENDA-Enzymes/BTO","title":"BRENDA tissue / enzyme source","tracker":"https://github.com/BRENDA-Enzymes/BTO/issues"},{"activity_status":"active","build":{"method":"obo2owl","notes":"moving to owl soon","source_url":"http://obo.cvs.sourceforge.net/*checkout*/obo/obo/ontology/anatomy/caro/caro.obo"},"contact":{"email":"haendel@ohsu.edu","github":"mellybelly","label":"Melissa Haendel","orcid":"0000-0001-9114-8737"},"description":"An upper level ontology to facilitate interoperability between existing anatomy ontologies for different species","domain":"anatomy","homepage":"https://github.com/obophenotype/caro/","id":"caro","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/caro.owl","preferredPrefix":"CARO","products":[{"id":"caro.owl","ontology_purl":"http://purl.obolibrary.org/obo/caro.owl"}],"repository":"https://github.com/obophenotype/caro","title":"Common Anatomy Reference Ontology","tracker":"https://github.com/obophenotype/caro/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/cdao.owl"},"contact":{"email":"balhoff@renci.org","github":"balhoff","label":"Jim Balhoff","orcid":"0000-0002-8688-6599"},"description":"a formalization of concepts and relations relevant to evolutionary comparative analysis","homepage":"https://github.com/evoinfo/cdao","id":"cdao","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cdao.owl","preferredPrefix":"CDAO","products":[{"id":"cdao.owl","ontology_purl":"http://purl.obolibrary.org/obo/cdao.owl"}],"repository":"https://github.com/evoinfo/cdao","title":"Comparative Data Analysis Ontology","tracker":"https://github.com/evoinfo/cdao/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/Southern-Cross-Plant-Science/cdno.git","path":".","system":"git"},"contact":{"email":"l.andres.hernandez.18@student.scu.edu.au","github":"LilyAndres","label":"Liliana Andres Hernandez","orcid":"0000-0002-7696-731X"},"dependencies":[{"id":"ro"},{"id":"bfo"},{"id":"pato"},{"id":"chebi"},{"id":"envo"}],"description":"CDNO provides structured terminologies to describe nutritional attributes of material entities that contribute to human diet.","domain":"diet, metabolomics and nutrition","homepage":"https://github.com/Southern-Cross-Plant-Science/cdno","id":"cdno","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cdno.owl","preferredPrefix":"CDNO","products":[{"id":"cdno.owl","name":"Compositional Dietary Nutrition Ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/cdno.owl"},{"id":"cdno.obo","name":"Compositional Dietary Nutrition Ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/cdno.obo"}],"repository":"https://github.com/Southern-Cross-Plant-Science/cdno","title":"Compositional Dietary Nutrition Ontology","tracker":"https://github.com/Southern-Cross-Plant-Science/cdno/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://raw.githubusercontent.com/semanticchemistry/semanticchemistry/master/ontology/cheminf.owl"},"contact":{"email":"egon.willighagen@gmail.com","github":"egonw","label":"Egon Willighagen","orcid":"0000-0001-7542-0286"},"description":"Includes terms for the descriptors commonly used in cheminformatics software applications and the algorithms which generate them.","domain":"biochemistry","homepage":"https://github.com/semanticchemistry/semanticchemistry","id":"cheminf","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"https://groups.google.com/forum/#!forum/cheminf-ontology","ontology_purl":"http://purl.obolibrary.org/obo/cheminf.owl","preferredPrefix":"CHEMINF","products":[{"id":"cheminf.owl","ontology_purl":"http://purl.obolibrary.org/obo/cheminf.owl"}],"repository":"https://github.com/semanticchemistry/semanticchemistry","title":"Chemical Information Ontology","tracker":"https://github.com/semanticchemistry/semanticchemistry/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/chiro.git","path":".","system":"git"},"contact":{"email":"vasilevs@ohsu.edu","github":"nicolevasilevsky","label":"Nicole Vasilevsky","orcid":"0000-0001-5208-3432"},"dependencies":[{"id":"chebi"},{"id":"mp"},{"id":"hp"},{"id":"go"},{"id":"pr"},{"id":"uberon"},{"id":"ncbitaxon"}],"description":"CHEBI provides a distinct role hierarchy. Chemicals in the structural hierarchy are connected via a \'has role\' relation. CHIRO provides links from these roles to useful other classes in other ontologies. This will allow direct connection between chemical structures (small molecules, drugs) and what they do. This could be formalized using \'capable of\', in the same way Uberon and the Cell Ontology link structures to processes.","domain":"biochemistry","homepage":"https://github.com/obophenotype/chiro","id":"chiro","jobs":[{"id":"https://travis-ci.org/obophenotype/chiro","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/chiro.owl","preferredPrefix":"CHIRO","products":[{"id":"chiro.owl","name":"CHEBI Integrated Role Ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/chiro.owl"},{"id":"chiro.obo","name":"CHEBI Integrated Role Ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/chiro.obo"}],"repository":"https://github.com/obophenotype/chiro","title":"CHEBI Integrated Role Ontology","tracker":"https://github.com/obophenotype/chiro/issues"},{"activity_status":"active","contact":{"email":"batchelorc@rsc.org","github":"batchelorc","label":"Colin Batchelor","orcid":"0000-0001-5985-7429"},"description":"CHMO, the chemical methods ontology, describes methods used to","domain":"health","homepage":"https://github.com/rsc-ontologies/rsc-cmo","id":"chmo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"chemistry-ontologies@googlegroups.com","ontology_purl":"http://purl.obolibrary.org/obo/chmo.owl","preferredPrefix":"CHMO","products":[{"id":"chmo.owl","ontology_purl":"http://purl.obolibrary.org/obo/chmo.owl"}],"repository":"https://github.com/rsc-ontologies/rsc-cmo","title":"Chemical Methods Ontology","tracker":"https://github.com/rsc-ontologies/rsc-cmo/issues"},{"activity_status":"active","contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqun Oliver He","orcid":"0000-0001-9189-9661"},"description":"The Coronavirus Infectious Disease Ontology (CIDO) aims to ontologically represent and standardize various aspects of coronavirus infectious diseases, including their etiology, transmission, epidemiology, pathogenesis, diagnosis, prevention, and treatment.","homepage":"https://github.com/cido-ontology/cido","id":"cido","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"cido-discuss@googlegroups.com","ontology_purl":"http://purl.obolibrary.org/obo/cido.owl","preferredPrefix":"CIDO","products":[{"id":"cido.owl","ontology_purl":"http://purl.obolibrary.org/obo/cido.owl"}],"repository":"https://github.com/cido-ontology/cido","title":"Coronavirus Infectious Disease Ontology","tracker":"https://github.com/cido-ontology/cido/issues"},{"activity_status":"active","contact":{"email":"frederic.bastian@unil.ch","github":"fbastian","label":"Frdric Bastian","orcid":"0000-0002-9415-5104"},"description":"An ontology to capture confidence information about annotations.","homepage":"https://github.com/BgeeDB/confidence-information-ontology","id":"cio","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cio.owl","preferredPrefix":"CIO","products":[{"id":"cio.owl","ontology_purl":"http://purl.obolibrary.org/obo/cio.owl"},{"id":"cio.obo","ontology_purl":"http://purl.obolibrary.org/obo/cio.obo"}],"repository":"https://github.com/BgeeDB/confidence-information-ontology","title":"Confidence Information Ontology","tracker":"https://github.com/BgeeDB/confidence-information-ontology"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/cell-ontology.git","email_cc":"cl_edit@googlegroups.com","infallible":1,"method":"vcs","system":"git"},"canonical":"cl.owl","contact":{"email":"addiehl@buffalo.edu","github":"addiehl","label":"Alexander Diehl","orcid":"0000-0001-9990-8331"},"dependencies":[{"id":"uberon"},{"id":"go"},{"id":"ro"},{"id":"chebi"},{"id":"pato"}],"description":"The Cell Ontology is a structured controlled vocabulary for cell types in animals.","domain":"cells","homepage":"https://obophenotype.github.io/cell-ontology/","id":"cl","integration_server":"http://build.berkeleybop.org/job/build-cl/","jobs":[{"id":"https://travis-ci.org/obophenotype/cell-ontology","type":"travis-ci"}],"label":"Cell Ontology","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"https://groups.google.com/g/cl_edit","ontology_purl":"http://purl.obolibrary.org/obo/cl.owl","preferredPrefix":"CL","products":[{"description":"Complete ontology, plus inter-ontology axioms, and imports modules","format":"owl-rdf/xml","id":"cl.owl","is_canonical":true,"ontology_purl":"http://purl.obolibrary.org/obo/cl.owl","title":"Main CL OWL edition","uses":["uberon","chebi","go","pr","pato"]},{"derived_from":"cl.owl","description":"Complete ontology, plus inter-ontology axioms, and imports modules merged in","format":"obo","id":"cl.obo","ontology_purl":"http://purl.obolibrary.org/obo/cl.obo","title":"CL obo format edition"},{"description":"Basic version, no inter-ontology axioms","format":"obo","id":"cl/cl-basic.obo","ontology_purl":"http://purl.obolibrary.org/obo/cl/cl-basic.obo","title":"Basic CL"},{"description":"complete CL but with no imports or external axioms","id":"cl/cl-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/cl/cl-base.owl","title":"CL base module"}],"publications":[{"id":"https://www.ncbi.nlm.nih.gov/pubmed/27377652","title":"The Cell Ontology 2016: enhanced content, modularization, and ontology interoperability."}],"repository":"https://github.com/obophenotype/cell-ontology","taxon":{"id":"NCBITaxon:33208","label":"Metazoa"},"title":"Cell Ontology","tracker":"https://github.com/obophenotype/cell-ontology/issues","usages":[{"description":"The National Human Genome Research Institute (NHGRI) launched a public research consortium named ENCODE, the Encyclopedia Of DNA Elements, in September 2003, to carry out a project to identify all functional elements in the human genome sequence. The ENCODE DCC users Uberon to annotate samples","publications":[{"id":"https://doi.org/10.1093/database/bav010","title":"Ontology application and use at the ENCODE DCC"}],"seeAlso":"https://www.biosharing.org/biodbcore-000034","type":"annotation","user":"https://www.encodeproject.org/"},{"description":"FANTOM5 is using Uberon and CL to annotate samples allowing for transcriptome analyses with cell-type and tissue-level specificity.","examples":[{"description":"FANTOM5 samples annotated to neuron","url":"http://fantom.gsc.riken.jp/5/sstar/CL:0000540"}],"type":"annotation","user":"http://fantom5-collaboration.gsc.riken.jp/"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/luis-gonzalez-m/Collembola.git","path":".","system":"git"},"contact":{"email":"lagonzalezmo@unal.edu.co","github":"luis-gonzalez-m","label":"Luis Gonzlez-Montaa","orcid":"0000-0002-9136-9932"},"dependencies":[{"id":"ro"}],"description":"CLAO is an ontology of anatomical terms employed in morphological descriptions for the Class Collembola (Arthropoda: Hexapoda).","domain":"anatomy","homepage":"https://github.com/luis-gonzalez-m/Collembola","id":"clao","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/clao.owl","preferredPrefix":"CLAO","products":[{"id":"clao.owl","ontology_purl":"http://purl.obolibrary.org/obo/clao.owl"},{"id":"clao.obo","ontology_purl":"http://purl.obolibrary.org/obo/clao.obo"}],"repository":"https://github.com/luis-gonzalez-m/Collembola","title":"Collembola Anatomy Ontology","tracker":"https://github.com/luis-gonzalez-m/Collembola/issues"},{"activity_status":"active","contact":{"email":"siiraa@umich.edu","github":"siiraa","label":"Sirarat Sarntivijai","orcid":"0000-0002-2548-641X"},"dependencies":[{"id":"uberon"},{"id":"cl"},{"id":"doid"},{"id":"ncbitaxon"}],"description":"An ontology to standardize and integrate cell line information and to support computer-assisted reasoning.","homepage":"http://www.clo-ontology.org","id":"clo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/clo.owl","preferredPrefix":"CLO","products":[{"id":"clo.owl","ontology_purl":"http://purl.obolibrary.org/obo/clo.owl"}],"repository":"https://github.com/CLO-Ontology/CLO","title":"Cell Line Ontology","tracker":"https://github.com/CLO-Ontology/CLO/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/EBISPOT/clyh_ontology.git","path":".","system":"git"},"contact":{"email":"lucas.leclere@obs-vlfr.fr","github":"L-Leclere","label":"Lucas Leclere","orcid":"0000-0002-7440-0467"},"dependencies":[{"id":"uberon"},{"id":"ro"},{"id":"iao"}],"description":"The Clytia hemisphaerica Development and Anatomy Ontology (CLYH) describes the anatomical and developmental features of the Clytia hemisphaerica life cycle.","domain":"anatomy","homepage":"https://github.com/EBISPOT/clyh_ontology","id":"clyh","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/clyh.owl","preferredPrefix":"CLYH","products":[{"id":"clyh.owl","ontology_purl":"http://purl.obolibrary.org/obo/clyh.owl"},{"id":"clyh.obo","ontology_purl":"http://purl.obolibrary.org/obo/clyh.obo"}],"repository":"https://github.com/EBISPOT/clyh_ontology","title":"Clytia hemisphaerica Development and Anatomy Ontology","tracker":"https://github.com/EBISPOT/clyh_ontology/issues"},{"activity_status":"active","browsers":[{"label":"RGD","title":"RGD Ontology Browser","url":"http://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=CMO:0000000"}],"build":{"method":"obo2owl","source_url":"https://download.rgd.mcw.edu/ontology/clinical_measurement/clinical_measurement.obo"},"contact":{"email":"jrsmith@mcw.edu","github":"jrsjrs","label":"Jennifer Smith","orcid":"0000-0002-6443-9376"},"description":"Morphological and physiological measurement records generated from clinical and model organism research and health programs.","domain":"clinical","homepage":"http://rgd.mcw.edu/rgdweb/ontology/search.html","id":"cmo","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cmo.owl","page":"https://download.rgd.mcw.edu/ontology/clinical_measurement/","preferredPrefix":"CMO","products":[{"id":"cmo.owl","ontology_purl":"http://purl.obolibrary.org/obo/cmo.owl"},{"id":"cmo.obo","ontology_purl":"http://purl.obolibrary.org/obo/cmo.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/22654893","title":"Three ontologies to define phenotype measurement data."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24103152","title":"The clinical measurement, measurement method and experimental condition ontologies: expansion, improvements and new applications."}],"repository":"https://github.com/rat-genome-database/CMO-Clinical-Measurement-Ontology","title":"Clinical measurement ontology","tracker":"https://github.com/rat-genome-database/CMO-Clinical-Measurement-Ontology/issues"},{"activity_status":"active","contact":{"email":"bpeters@lji.org","github":"bpeters42","label":"Bjoern Peters","orcid":"0000-0002-8457-6693"},"description":"COB brings together key terms from a wide range of OBO projects to improve interoperability.","domain":"upper","homepage":"https://github.com/OBOFoundry/COB","id":"cob","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cob.owl","preferredPrefix":"COB","products":[{"description":"Core Ontology for Biology and Biomedicine, main ontology","id":"cob.owl","ontology_purl":"http://purl.obolibrary.org/obo/cob.owl","title":"COB"},{"description":"base module for COB","id":"cob/cob-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/cob/cob-base.owl","title":"COB base module"},{"id":"cob/cob-to-external.owl","ontology_purl":"http://purl.obolibrary.org/obo/cob/cob-to-external.owl","title":"COB to external","type":"BridgeOntology"},{"description":"demo of COB including subsets of other ontologies (Experimental, for demo purposes only)","id":"cob/products/demo-cob.owl","ontology_purl":"http://purl.obolibrary.org/obo/cob/products/demo-cob.owl","status":"alpha","title":"COB demo ontology (experimental)"}],"repository":"https://github.com/OBOFoundry/COB","title":"Core Ontology for Biology and Biomedicine","tracker":"https://github.com/OBOFoundry/COB/issues"},{"activity_status":"active","contact":{"email":"entiminae@gmail.com","github":"JCGiron","label":"Jennifer C. Giron","orcid":"0000-0002-0851-6883"},"dependencies":[{"id":"aism"},{"id":"ro"},{"id":"uberon"},{"id":"caro"},{"id":"pato"},{"id":"bfo"},{"id":"bspo"}],"description":"The Coleoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of beetles in biodiversity research. It has been built using the Ontology Develoment Kit, with the Ontology for the Anatomy of the Insect Skeleto-Muscular system (AISM) as a backbone.","domain":"Insect Anatomy.","homepage":"https://github.com/insect-morphology/colao","id":"colao","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/colao.owl","preferredPrefix":"COLAO","products":[{"id":"colao.owl","ontology_purl":"http://purl.obolibrary.org/obo/colao.owl"},{"id":"colao.obo","ontology_purl":"http://purl.obolibrary.org/obo/colao.obo"}],"repository":"https://github.com/insect-morphology/colao","title":"Coleoptera Anatomy Ontology (COLAO)","tracker":"https://github.com/insect-morphology/colao/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/data2health/contributor-role-ontology.git","path":"src/ontology","system":"git"},"contact":{"email":"whimar@ohsu.edu","github":"marijane","label":"Marijane White","orcid":"0000-0001-5059-4132"},"description":"A classification of the diverse roles performed in the work leading to a published research output in the sciences. Its purpose to provide transparency in contributions to scholarly published work, to enable improved systems of attribution, credit, and accountability.","domain":"scholarly contribution roles","homepage":"https://github.com/data2health/contributor-role-ontology","id":"cro","layout":"ontology_detail","license":{"label":"CC BY 2.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/2.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cro.owl","preferredPrefix":"CRO","products":[{"id":"cro.owl","ontology_purl":"http://purl.obolibrary.org/obo/cro.owl","title":"CRO"}],"repository":"https://github.com/data2health/contributor-role-ontology","title":"Contributor Role Ontology","tracker":"https://github.com/data2health/contributor-role-ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/ctenophore-ontology.git","method":"vcs","path":".","system":"git"},"contact":{"email":"cjmungall@lbl.gov","github":"cmungall","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"dependencies":[{"id":"uberon"},{"id":"ro"}],"description":"An anatomical and developmental ontology for ctenophores (Comb Jellies)","domain":"anatomy","homepage":"https://github.com/obophenotype/ctenophore-ontology","id":"cteno","jobs":[{"id":"https://travis-ci.org/obophenotype/ctenophore-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cteno.owl","preferredPrefix":"CTENO","products":[{"id":"cteno.owl","ontology_purl":"http://purl.obolibrary.org/obo/cteno.owl"}],"repository":"https://github.com/obophenotype/ctenophore-ontology","taxon":{"id":"NCBITaxon:10197","label":"Ctenophore"},"title":"Ctenophore Ontology","tracker":"https://github.com/obophenotype/ctenophore-ontology/issues"},{"activity_status":"active","contact":{"email":"alpha.tom.kodamullil@scai.fraunhofer.de","github":"akodamullil","label":"Dr. Alpha Tom Kodamullil","orcid":"0000-0001-9896-3531"},"description":"The core Ontology of Clinical Trials (CTO) will serve as a structured resource integrating basic terms and concepts in the context of clinical trials. Thereby covering clinicaltrails.gov. CoreCTO will serve as a basic ontology to generate extended versions for specific applications such as annotation of variables in study documents from clinical trials.","homepage":"https://github.com/ClinicalTrialOntology/CTO/","id":"cto","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cto.owl","preferredPrefix":"CTO","products":[{"id":"cto.owl","ontology_purl":"http://purl.obolibrary.org/obo/cto.owl"}],"repository":"https://github.com/ClinicalTrialOntology/CTO","title":"CTO: Core Ontology of Clinical Trials","tracker":"https://github.com/ClinicalTrialOntology/CTO/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/cvdo.owl"},"contact":{"email":"paul.fabry@usherbrooke.ca","github":"pfabry","label":"Paul Fabry","orcid":"0000-0002-3336-2476"},"description":"An ontology to describe entities related to cardiovascular diseases","domain":"health","homepage":"https://github.com/OpenLHS/CVDO","id":"cvdo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/cvdo.owl","preferredPrefix":"CVDO","products":[{"id":"cvdo.owl","ontology_purl":"http://purl.obolibrary.org/obo/cvdo.owl"}],"repository":"https://github.com/OpenLHS/CVDO","title":"Cardiovascular Disease Ontology","tracker":"https://github.com/OpenLHS/CVDO/issues"},{"activity_status":"active","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/dictyBase/migration-data/master/ontologies/dicty_anatomy.obo"},"contact":{"email":"pfey@northwestern.edu","github":"pfey03","label":"Petra Fey","orcid":"0000-0002-4532-2703"},"description":"A structured controlled vocabulary of the anatomy of the slime-mold Dictyostelium discoideum","domain":"anatomy","homepage":"http://dictybase.org/","id":"ddanat","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ddanat.owl","preferredPrefix":"DDANAT","products":[{"id":"ddanat.owl","ontology_purl":"http://purl.obolibrary.org/obo/ddanat.owl"},{"id":"ddanat.obo","ontology_purl":"http://purl.obolibrary.org/obo/ddanat.obo"}],"repository":"https://github.com/dictyBase/migration-data","taxon":{"id":"NCBITaxon:44689","label":"Dictyostelium discoideum"},"title":"Dictyostelium discoideum anatomy","tracker":"https://github.com/dictyBase/migration-data/issues","twitter":"dictybase"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/dicty-phenotype-ontology.git","path":".","system":"git"},"contact":{"email":"pfey@northwestern.edu","github":"pfey03","label":"Petra Fey","orcid":"0000-0002-4532-2703"},"description":"A structured controlled vocabulary of phenotypes of the slime-mould <i>Dictyostelium discoideum</i>.","domain":"anatomy","homepage":"http://dictybase.org/","id":"ddpheno","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ddpheno.owl","preferredPrefix":"DDPHENO","products":[{"id":"ddpheno.owl","ontology_purl":"http://purl.obolibrary.org/obo/ddpheno.owl"},{"id":"ddpheno.obo","ontology_purl":"http://purl.obolibrary.org/obo/ddpheno.obo"}],"repository":"https://github.com/obophenotype/dicty-phenotype-ontology","taxon":{"id":"NCBITaxon:44689","label":"Dictyostelium discoideum"},"title":"Dictyostelium discoideum phenotype ontology","tracker":"https://github.com/obophenotype/dicty-phenotype-ontology/issues","twitter":"dictybase"},{"activity_status":"active","contact":{"email":"mbrochhausen@gmail.com","github":"mbrochhausen","label":"Mathias Brochhausen","orcid":"0000-0003-1834-3856"},"description":"The Potential Drug-drug Interaction and Potential Drug-drug Interaction Evidence Ontology","homepage":"https://github.com/DIDEO/DIDEO","id":"dideo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/dideo.owl","preferredPrefix":"DIDEO","products":[{"id":"dideo.owl","ontology_purl":"http://purl.obolibrary.org/obo/dideo.owl"}],"repository":"https://github.com/DIDEO/DIDEO","title":"Drug-drug Interaction and Drug-drug Interaction Evidence Ontology","tracker":"https://github.com/DIDEO/DIDEO/issues"},{"activity_status":"active","browsers":[{"label":"DO","title":"DO Browser","url":"http://www.disease-ontology.org/"}],"build":{"source_url":"https://github.com/DiseaseOntology/DiseaseDriversOntology/tree/main/src/ontology/disdriv.owl"},"contact":{"email":"lynn.schriml@gmail.com","github":"lschriml","label":"Lynn Schriml","orcid":"0000-0001-8910-9851"},"description":"Ontology for drivers and triggers of human diseases, built to classify ExO ontology exposure stressors. An application ontology. Built in collaboration with EnvO, ExO, ECTO and ChEBI.","domain":"disease","homepage":"http://www.disease-ontology.org","id":"disdriv","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/disdriv.owl","preferredPrefix":"DISDRIV","products":[{"id":"disdriv.owl","ontology_purl":"http://purl.obolibrary.org/obo/disdriv.owl"}],"repository":"https://github.com/DiseaseOntology/DiseaseDriversOntology","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Disease Drivers Ontology","tracker":"https://github.com/DiseaseOntology/DiseaseDriversOntology/issues","twitter":"diseaseontology","usages":[{"description":"Human Disease Ontology","examples":[{"description":"fetal alcohol syndrome, has exposure stressor some alcohol","url":"https://www.disease-ontology.org/?id=DOID:0050665"}],"user":"https://www.disease-ontology.org"}]},{"activity_status":"active","browsers":[{"label":"FB","title":"FlyBase Browser","url":"http://flybase.org/.bin/cvreport.html?cvterm=FBcv:0000347"}],"build":{"checkout":"git clone https://github.com/FlyBase/drosophila-phenotype-ontolog.git","path":".","system":"git"},"contact":{"email":"cp390@cam.ac.uk","github":"Clare72","label":"Clare Pilgrim","orcid":"0000-0002-1373-1705"},"description":"An ontology of commonly encountered and/or high level Drosophila phenotypes.","domain":"phenotype","homepage":"http://purl.obolibrary.org/obo/fbcv","id":"dpo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/dpo.owl","preferredPrefix":"FBcv","products":[{"id":"dpo.owl","ontology_purl":"http://purl.obolibrary.org/obo/dpo.owl"},{"id":"dpo.obo","ontology_purl":"http://purl.obolibrary.org/obo/dpo.obo"},{"id":"dpo.json","ontology_purl":"http://purl.obolibrary.org/obo/dpo.json"}],"publications":[{"id":"https://doi.org/10.1186/2041-1480-4-30","title":"The Drosophila phenotype ontology."}],"repository":"https://github.com/FlyBase/drosophila-phenotype-ontology","taxon":{"id":"NCBITaxon:7227","label":"Drosophila"},"title":"Drosophila Phenotype Ontology","tracker":"https://github.com/FlyBase/drosophila-phenotype-ontology/issues","usages":[{"description":"FlyBase uses dpo for phenotype data annotation in Drosophila","examples":[{"description":"alleles and constructs annotated to pupal lethal in FlyBase","url":"http://flybase.org/cgi-bin/cvreport.html?rel=is_a&id=FBcv:0002030"}],"user":"http://flybase.org"}]},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/dron.owl"},"contact":{"email":"hoganwr@gmail.com","github":"hoganwr","label":"William Hogan","orcid":"0000-0002-9881-1017"},"description":"An ontology to support comparative effectiveness researchers studying claims data.","domain":"health","homepage":"https://github.com/ufbmi/dron","id":"dron","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/dron.owl","preferredPrefix":"DRON","products":[{"id":"dron.owl","ontology_purl":"http://purl.obolibrary.org/obo/dron.owl"}],"repository":"https://github.com/ufbmi/dron","title":"The Drug Ontology","tracker":"https://github.com/ufbmi/dron/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/EBISPOT/duo.git","path":".","system":"git"},"contact":{"email":"mcourtot@gmail.com","github":"mcourtot","label":"Melanie Courtot","orcid":"0000-0002-9551-6370"},"dependencies":[{"id":"iao"},{"id":"bfo"}],"description":"DUO is an ontology which represent data use conditions.","homepage":"https://github.com/EBISPOT/DUO","id":"duo","jobs":[{"id":"https://travis-ci.org/EBISPOT/DUO","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/duo.owl","preferredPrefix":"DUO","products":[{"id":"duo.owl","ontology_purl":"http://purl.obolibrary.org/obo/duo.owl"}],"repository":"https://github.com/EBISPOT/DUO","title":"Data Use Ontology","tracker":"https://github.com/EBISPOT/DUO/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/echinoderm-ontology/ecao_ontology.git","path":".","system":"git"},"contact":{"email":"ettensohn@cmu.edu","github":"ettensohn","label":"Charles Ettensohn","orcid":"0000-0002-3625-0955"},"dependencies":[{"id":"uberon"},{"id":"ro"},{"id":"cl"}],"description":"An ontology for the development and anatomy of the different species of the phylum Echinodermata (NCBITaxon:7586).","domain":"anatomy","homepage":"https://github.com/echinoderm-ontology/ecao_ontology","id":"ecao","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ecao.owl","preferredPrefix":"ECAO","products":[{"id":"ecao.owl","ontology_purl":"http://purl.obolibrary.org/obo/ecao.owl"},{"id":"ecao.obo","ontology_purl":"http://purl.obolibrary.org/obo/ecao.obo"}],"repository":"https://github.com/echinoderm-ontology/ecao_ontology","title":"The Echinoderm Anatomy and Development Ontology","tracker":"https://github.com/echinoderm-ontology/ecao_ontology/issues"},{"activity_status":"active","build":{"infallible":1,"method":"obo2owl","notes":"switch to vcs","source_url":"https://raw.githubusercontent.com/evidenceontology/evidenceontology/master/eco.obo"},"contact":{"email":"mgiglio@som.umaryland.edu","github":"mgiglio99","label":"Michelle Giglio","orcid":"0000-0001-7628-5565"},"description":"An ontology for experimental and other evidence statements.","domain":"experiments","funded_by":["http://www.nsf.gov/awardsearch/showAward?AWD_ID=1458400"],"homepage":"https://github.com/evidenceontology/evidenceontology/","id":"eco","jobs":[{"id":"https://travis-ci.org/evidenceontology/evidenceontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/eco.owl","preferredPrefix":"ECO","products":[{"id":"eco.owl","ontology_purl":"http://purl.obolibrary.org/obo/eco.owl"},{"id":"eco.obo","ontology_purl":"http://purl.obolibrary.org/obo/eco.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/30407590","title":"ECO, the Evidence & Conclusion Ontology: community standard for evidence information."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/25052702","title":"Standardized description of scientific evidence using the Evidence Ontology (ECO)"}],"repository":"https://github.com/evidenceontology/evidenceontology","title":"Evidence ontology","tracker":"https://github.com/evidenceontology/evidenceontology/issues","usages":[{"description":"ECO is used by the GO consortium for evidence on GO associations","examples":[{"description":"annotations to transmembrane transport","url":"http://amigo.geneontology.org/amigo/term/GO:0055085"}],"type":"annotation","user":"http://geneontology.org"},{"description":"ECO is used by the Monarch Initiative for evidence types for disease to phenotype annotations.","examples":[{"description":"Parkinsonism: Characteristic neurologic anomaly resulting form degeneration of dopamine-generating cells in the substantia nigra, a region of the midbrain, characterized clinically by shaking, rigidity, slowness of movement and difficulty with walking and gait.","url":"https://monarchinitiative.org/phenotype/HP%3A0001300#disease"}],"publications":[{"id":"https://academic.oup.com/nar/article/45/D1/D712/2605791","title":"The Monarch Initiative: an integrative data and analytic platform connecting phenotypes to genotypes across species"}],"type":"annotation","user":"https://monarchinitiative.org/"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/EcologicalSemantics/ecocore.git","path":".","system":"git"},"contact":{"email":"p.buttigieg@gmail.com","github":"pbuttigieg","label":"Pier Luigi Buttigieg","orcid":"0000-0002-4366-3088"},"dependencies":[{"id":"pco"},{"id":"ro"},{"id":"bfo"},{"id":"pato"},{"id":"envo"},{"id":"chebi"},{"id":"go"},{"id":"uberon"},{"id":"po"},{"id":"iao"}],"description":"Ecocore is a community ontology for the concise and controlled description of ecological traits of organisms.","domain":"ecological functions, ecological interactions","homepage":"https://github.com/EcologicalSemantics/ecocore","id":"ecocore","jobs":[{"id":"https://travis-ci.org/EcologicalSemantics/ecocore","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ecocore.owl","preferredPrefix":"ECOCORE","products":[{"id":"ecocore.owl","ontology_purl":"http://purl.obolibrary.org/obo/ecocore.owl"},{"id":"ecocore.obo","ontology_purl":"http://purl.obolibrary.org/obo/ecocore.obo"}],"repository":"https://github.com/EcologicalSemantics/ecocore","title":"An ontology of core ecological entities","tracker":"https://github.com/EcologicalSemantics/ecocore/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/EnvironmentOntology/environmental-exposure-ontology.git","path":".","system":"git"},"contact":{"email":"annethessen@gmail.com","github":"diatomsRcool","label":"Anne Thessen","orcid":"0000-0002-2908-3327"},"dependencies":[{"id":"chebi"},{"id":"envo"},{"id":"exo"},{"id":"go"},{"id":"iao"},{"id":"maxo"},{"id":"nbo"},{"id":"ncit"},{"id":"ncbitaxon"},{"id":"pato"},{"id":"ro"},{"id":"uberon"},{"id":"xco"}],"description":"ECTO describes exposures to experimental treatments of plants and model organisms (e.g. exposures to modification of diet, lighting levels, temperature); exposures of humans or any other organisms to stressors through a variety of routes, for purposes of public health, environmental monitoring etc, stimuli, natural and experimental, any kind of environmental condition or change in condition that can be experienced by an organism or population of organisms on earth. The scope is very general and can include for example plant treatment regimens, as well as human clinical exposures (although these may better be handled by a more specialized ontology).","domain":"environment","homepage":"https://github.com/EnvironmentOntology/environmental-exposure-ontology","id":"ecto","jobs":[{"id":"https://travis-ci.org/EnvironmentOntology/environmental-exposure-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ecto.owl","preferredPrefix":"ECTO","products":[{"id":"ecto.owl","name":"Environmental conditions, treatments and exposures ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/ecto.owl"},{"id":"ecto.obo","name":"Environmental conditions, treatments and exposures ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/ecto.obo"},{"id":"ecto.json","name":"Environmental conditions, treatments and exposures ontology additional release in OBOJSon format","ontology_purl":"http://purl.obolibrary.org/obo/ecto.json"},{"id":"ecto/ecto-base.owl","name":"Environmental conditions, treatments and exposures ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/ecto/ecto-base.owl"},{"id":"ecto/ecto-base.obo","name":"Environmental conditions, treatments and exposures ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/ecto/ecto-base.obo"},{"id":"ecto/ecto-base.json","name":"Environmental conditions, treatments and exposures ontology additional release in OBOJSon format","ontology_purl":"http://purl.obolibrary.org/obo/ecto/ecto-base.json"}],"repository":"https://github.com/EnvironmentOntology/environmental-exposure-ontology","title":"Environmental conditions, treatments and exposures ontology","tracker":"https://github.com/EnvironmentOntology/environmental-exposure-ontology/issues"},{"activity_status":"active","build":{"method":"obo2owl","notes":"new url soon","source_url":"ftp://ftp.hgu.mrc.ac.uk/pub/MouseAtlas/Anatomy/EMAPA.obo"},"contact":{"email":"Terry.Hayamizu@jax.org","github":"tfhayamizu","label":"Terry Hayamizu","orcid":"0000-0002-0956-8634"},"description":"An ontology for mouse anatomy covering embryonic development and postnatal stages.","domain":"anatomy","homepage":"http://www.informatics.jax.org/expression.shtml","id":"emapa","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/emapa.owl","preferredPrefix":"EMAPA","products":[{"id":"emapa.owl","ontology_purl":"http://purl.obolibrary.org/obo/emapa.owl"},{"id":"emapa.obo","ontology_purl":"http://purl.obolibrary.org/obo/emapa.obo"}],"repository":"https://github.com/obophenotype/mouse-anatomy-ontology","taxon":{"id":"NCBITaxon:10088","label":"Mus"},"title":"Mouse Developmental Anatomy Ontology","tracker":"https://github.com/obophenotype/mouse-anatomy-ontology/issues","usages":[{"description":"GXD","seeAlso":"https://www.biosharing.org/biodbcore-000659","user":"http://www.informatics.jax.org/expression.shtml"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/EnvironmentOntology/envo.git","email_cc":"cjmungall@lbl.gov","infallible":1,"method":"vcs","path":".","system":"git"},"contact":{"email":"p.buttigieg@gmail.com","github":"pbuttigieg","label":"Pier Luigi Buttigieg","orcid":"0000-0002-4366-3088"},"dependencies":[{"id":"uberon"},{"id":"pco"},{"id":"ro"},{"id":"chebi"},{"id":"ncbitaxon"},{"id":"foodon"},{"id":"po"},{"id":"go"}],"description":"Ontology of environmental features and habitats","domain":"environment","homepage":"http://environmentontology.org/","id":"envo","jobs":[{"id":"https://travis-ci.org/EnvironmentOntology/envo","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/envo.owl","page":"https://github.com/EnvironmentOntology/envo","preferredPrefix":"ENVO","products":[{"id":"envo.owl","ontology_purl":"http://purl.obolibrary.org/obo/envo.owl","title":"main ENVO OWL release"},{"id":"envo.json","ontology_purl":"http://purl.obolibrary.org/obo/envo.json","title":"ENVO in obographs JSON format"},{"id":"envo.obo","ontology_purl":"http://purl.obolibrary.org/obo/envo.obo","title":"ENVO in OBO Format. May be lossy"},{"id":"envo/subsets/envo-basic.obo","ontology_purl":"http://purl.obolibrary.org/obo/envo/subsets/envo-basic.obo","title":"OBO-Basic edition of ENVO"},{"id":"envo/subsets/envoEmpo.owl","ontology_purl":"http://purl.obolibrary.org/obo/envo/subsets/envoEmpo.owl","title":"Earth Microbiome Project subset"},{"homepage":"http://environmentontology.org/downloads","id":"envo/subsets/EnvO-Lite-GSC.obo","ontology_purl":"http://purl.obolibrary.org/obo/envo/subsets/EnvO-Lite-GSC.obo","title":"GSC Lite subset of ENVO"}],"publications":[{"id":"http://www.dx.doi.org/10.1186/2041-1480-4-43","title":"The environment ontology: contextualising biological and biomedical entities"},{"id":"https://doi.org/10.1186/s13326-016-0097-6","title":"The environment ontology in 2016: bridging domains with increased scope, semantic density, and interoperation"}],"repository":"https://github.com/EnvironmentOntology/envo","title":"Environment Ontology","tracker":"https://github.com/EnvironmentOntology/envo/issues/","usages":[{"description":"describing species habitats","examples":[{"description":"Pseudobarbus burchelli (Tradou Redfin) is a species of bony fishes in the family Cyprinidae. They are associated with freshwater habitat. Individuals can grow to 13.5 cm. They have sexual reproduction.","url":"http://eol.org/pages/211700/data"}],"type":"data-annotation","user":"http://eol.org"},{"description":"describing stomach contents","type":"data-annotation","user":"http://globalbioticinteractions.org"},{"description":"annotating datasets in data repositories","seeAlso":"http://blogs.nature.com/scientificdata/2015/12/17/isa-explorer/","type":"dataset-description","user":"http://www.nature.com/sdata/"},{"description":"Samples collected during Tara Oceans expedition are annotated with ENVO","examples":[{"description":"Sample collected during the Tara Oceans expedition (2009-2013) at station TARA_004 (latitudeN=36.5533, longitudeE=-6.5669)","url":"https://www.ebi.ac.uk/metagenomics/projects/ERP001736/samples/ERS487899"}],"user":"http://oceans.taraexpeditions.org/en/"},{"description":"Annotation of habitats of microbes","examples":[{"description":"Annotation of habitat of Pseudovibrio sp. FO-BEG1 to marine environment","url":"https://www.ncbi.nlm.nih.gov/nuccore/NC_016642"}],"user":"https://www.ncbi.nlm.nih.gov/"},{"description":"Annotation and semantic search over microbial data sets","examples":[{"description":"Example metadata of a sample of marine water near Lisboa, taken as part of the Ocean Sampling Day Project (https://www.microb3.eu/osd.html). ENVO is used for the fields environmental feature, material, and biome.","url":"https://www.planetmicrobe.org/project/#/samples/200"}],"user":"https://www.planetmicrobe.org/project/"}]},{"activity_status":"active","contact":{"email":"alpha.tom.kodamullil@scai.fraunhofer.de","github":"akodamullil","label":"Alpha Tom Kodamullil","orcid":"0000-0001-9896-3531"},"dependencies":[{"id":"BFO"}],"description":"A application driven Epilepsy Ontology with official terms from the ILAE.","domain":"disease","homepage":"https://github.com/SCAI-BIO/EpilepsyOntology","id":"epio","label":"Epilepsy Ontology","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/epio.owl","preferredPrefix":"EPIO","products":[{"id":"epio.owl","ontology_purl":"http://purl.obolibrary.org/obo/epio.owl"},{"id":"EPIO_merged.owl","ontology_purl":"http://purl.obolibrary.org/obo/EPIO_merged.owl"}],"repository":"https://github.com/SCAI-BIO/EpilepsyOntology","title":"Epilepsy Ontology","tracker":"https://github.com/SCAI-BIO/EpilepsyOntology/issues"},{"activity_status":"active","contact":{"email":"stoeckrt@pennmedicine.upenn.edu","github":"cstoeckert","label":"Chris Stoeckert","orcid":"0000-0002-5714-991X"},"depicted_by":"https://raw.githubusercontent.com/EuPathDB/communitysite/master/assets/images/VEuPathDB-logo-s.png","description":"An ontology is developed to support Eukaryotic Pathogen, Host & Vector Genomics Resource (VEuPathDB; https://veupathdb.org).","domain":"functional genomics, population biology, clinical epidemiology, and microbiomes","homepage":"https://github.com/VEuPathDB-ontology/VEuPathDB-ontology","id":"eupath","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/eupath.owl","preferredPrefix":"EUPATH","products":[{"id":"eupath.owl","ontology_purl":"http://purl.obolibrary.org/obo/eupath.owl"}],"repository":"https://github.com/VEuPathDB-ontology/VEuPathDB-ontology","title":"VEuPathDB ontology","tracker":"https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues","usages":[{"description":"The VEuPathDB ontology is used in the VEuPathDB (Eukaryotic Pathogen, Vector & Host Informatics Resources) covers both functional genomics and population biology.","type":"annotation and query","user":"https://veupathdb.org"},{"description":"The VEuPathDB ontology is used in the clinical epidemiology resources.","type":"annotation and query","user":"https://clinepidb.org"},{"description":"The VEuPathDB ontology is used in the MicrobiomeDB, a systems biology platform for integrating, mining and analyzing microbiome experiments.","type":"annotation and query","user":"https://microbiomedb.org"}]},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/CTDbase/exposure-ontology/master/src/ontology/exo.obo"},"contact":{"email":"cjgrondin@ncsu.edu","github":"cjgrondin","label":"Cynthia Grondin","orcid":"0000-0002-4642-1738"},"description":"Vocabularies for describing exposure data to inform understanding of environmental health.","domain":"health","homepage":"http://ctdbase.org/help/exposureHelp.jsp","id":"exo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/exo.owl","page":"http://ctdbase.org/downloads/#exposures","preferredPrefix":"ExO","products":[{"id":"exo.owl","ontology_purl":"http://purl.obolibrary.org/obo/exo.owl"},{"id":"exo.obo","ontology_purl":"http://purl.obolibrary.org/obo/exo.obo"}],"repository":"https://github.com/CTDbase/exposure-ontology","title":"Exposure ontology","tracker":"https://github.com/CTDbase/exposure-ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/fungal-anatomy-ontology.git","infallible":1,"method":"vcs","system":"git"},"contact":{"email":"vw253@cam.ac.uk","github":"ValWood","label":"Val Wood","orcid":"0000-0001-6330-7526"},"description":"A structured controlled vocabulary for the anatomy of fungi.","domain":"anatomy","homepage":"https://github.com/obophenotype/fungal-anatomy-ontology/","id":"fao","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fao.owl","preferredPrefix":"FAO","products":[{"id":"fao.owl","ontology_purl":"http://purl.obolibrary.org/obo/fao.owl"},{"id":"fao.obo","ontology_purl":"http://purl.obolibrary.org/obo/fao.obo"}],"repository":"https://github.com/obophenotype/fungal-anatomy-ontology","taxon":{"id":"NCBITaxon:4751","label":"Fungal"},"title":"Fungal gross anatomy","tracker":"https://github.com/obophenotype/fungal-anatomy-ontology/issues"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/dosumis/fbbi/master/src/ontology/fbbi.obo"},"contact":{"email":"wawong@gmail.com","github":"wawong","label":"Willy Wong","orcid":"0000-0002-8841-5870"},"description":"A structured controlled vocabulary of sample preparation, visualization and imaging methods used in biomedical research.","domain":"imaging experiments","homepage":"http://cellimagelibrary.org/","id":"fbbi","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fbbi.owl","preferredPrefix":"FBbi","products":[{"id":"fbbi.owl","ontology_purl":"http://purl.obolibrary.org/obo/fbbi.owl"}],"repository":"https://github.com/CRBS/Biological_Imaging_Methods_Ontology","title":"Biological Imaging Methods Ontology","tracker":"https://github.com/CRBS/Biological_Imaging_Methods_Ontology/issues"},{"activity_status":"active","browsers":[{"label":"FB","title":"FlyBase Browser","url":"http://flybase.org/.bin/cvreport.html?cvterm=FBbt:10000000"},{"label":"VFB","title":"Virtual Fly Brain","url":"http://www.virtualflybrain.org/site/stacks/index.htm?add=FBbt:00007401"},{"label":"BioPortal","title":"BioPortal Browser","url":"http://bioportal.bioontology.org/ontologies/FB-BT?p=classes"}],"build":{"checkout":"git clone https://github.com/FlyBase/drosophila-anatomy-developmental-ontology.git","path":".","system":"git"},"contact":{"email":"cp390@cam.ac.uk","github":"Clare72","label":"Clare Pilgrim","orcid":"0000-0002-1373-1705"},"description":"An ontology representing the gross anatomy of Drosophila melanogaster.","domain":"Drosophilid anatomy","homepage":"http://purl.obolibrary.org/obo/fbbt","id":"fbbt","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fbbt.owl","preferredPrefix":"FBbt","products":[{"id":"fbbt.owl","ontology_purl":"http://purl.obolibrary.org/obo/fbbt.owl"},{"id":"fbbt.obo","ontology_purl":"http://purl.obolibrary.org/obo/fbbt.obo"},{"id":"fbbt.json","ontology_purl":"http://purl.obolibrary.org/obo/fbbt.json"},{"id":"fbbt/fbbt-simple.owl","ontology_purl":"http://purl.obolibrary.org/obo/fbbt/fbbt-simple.owl"},{"id":"fbbt/fbbt-simple.obo","ontology_purl":"http://purl.obolibrary.org/obo/fbbt/fbbt-simple.obo"}],"publications":[{"id":"https://doi.org/10.1186/2041-1480-4-32","title":"The Drosophila anatomy ontology. [Journal of Biomedical Semantics"},{"id":"https://doi.org/10.1093/bioinformatics/bts113","title":"A strategy for building neuroanatomy ontologies"},{"id":"https://doi.org/10.1093/bioinformatics/btr677","title":"The Virtual Fly Brain Browser and Query Interface"},{"id":"https://doi.org/10.1093/nar/gkj068","title":"FlyBase: anatomical data, images and queries"}],"repository":"https://github.com/FlyBase/drosophila-anatomy-developmental-ontology","taxon":{"id":"NCBITaxon:7227","label":"Drosophila"},"title":"Drosophila gross anatomy","tracker":"http://purl.obolibrary.org/obo/fbbt/tracker","usages":[{"description":"VFB uses FBbt to annotate brain images","examples":[{"description":"Ring neuron R2 in VFB","url":"http://www.virtualflybrain.org/site/stacks/index.htm?id=FBbt_00003651"},{"description":"genes expressed in ring neuron R2 in VFB","url":"http://www.virtualflybrain.org/do/gene_list.html?action=geneex&id=FBbt:00003651"}],"user":"http://www.virtualflybrain.org/"},{"description":"Flybase uses FBbt for expression and phenotype data annotation in Drosophila","examples":[{"description":"alleles, constructs and insertions annotated to neuron in FlyBase","url":"http://flybase.org/cgi-bin/cvreport.html?rel=is_a&id=FBbt:00005106"}],"user":"http://flybase.org"}]},{"activity_status":"active","browsers":[{"label":"FB","title":"FlyBase Browser","url":"http://flybase.org/.bin/cvreport.html?cvterm=FBcv:0000013"}],"build":{"checkout":"git clone https://github.com/FlyBase/flybase-controlled-vocabulary.git","path":".","system":"git"},"contact":{"email":"cp390@cam.ac.uk","github":"Clare72","label":"Clare Pilgrim","orcid":"0000-0002-1373-1705"},"description":"A structured controlled vocabulary used for various aspects of annotation by FlyBase.","domain":"organisms","homepage":"http://purl.obolibrary.org/obo/fbcv","id":"fbcv","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fbcv.owl","preferredPrefix":"FBcv","products":[{"id":"fbcv.owl","ontology_purl":"http://purl.obolibrary.org/obo/fbcv.owl"},{"id":"fbcv.obo","ontology_purl":"http://purl.obolibrary.org/obo/fbcv.obo"},{"id":"fbcv.json","ontology_purl":"http://purl.obolibrary.org/obo/fbcv.json"}],"repository":"https://github.com/FlyBase/flybase-controlled-vocabulary","title":"FlyBase Controlled Vocabulary","tracker":"https://github.com/FlyBase/flybase-controlled-vocabulary/issues","usages":[{"description":"FlyBase uses FBcv for phenotype data annotation in Drosophila","examples":[{"description":"alleles and constructs annotated to bang sensitive in FlyBase","url":"http://flybase.org/cgi-bin/cvreport.html?rel=is_a&id=FBcv:0000391"}],"user":"http://flybase.org"}]},{"activity_status":"active","browsers":[{"label":"FB","title":"FlyBase Browser","url":"http://flybase.org/.bin/cvreport.html?cvterm=FBdv:00007008"}],"build":{"checkout":"git clone https://github.com/FlyBase/drosophila-developmental-ontology.git","path":".","system":"git"},"contact":{"email":"cp390@cam.ac.uk","github":"Clare72","label":"Clare Pilgrim","orcid":"0000-0002-1373-1705"},"description":"A structured controlled vocabulary of the development of Drosophila melanogaster.","domain":"development","homepage":"http://purl.obolibrary.org/obo/fbdv","id":"fbdv","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fbdv.owl","preferredPrefix":"FBdv","products":[{"id":"fbdv.owl","ontology_purl":"http://purl.obolibrary.org/obo/fbdv.owl"},{"id":"fbdv.obo","ontology_purl":"http://purl.obolibrary.org/obo/fbdv.obo"},{"id":"fbdv.json","ontology_purl":"http://purl.obolibrary.org/obo/fbdv.json"},{"id":"fbdv/fbdv-simple.owl","ontology_purl":"http://purl.obolibrary.org/obo/fbdv/fbdv-simple.owl"},{"id":"fbdv/fbdv-simple.obo","ontology_purl":"http://purl.obolibrary.org/obo/fbdv/fbdv-simple.obo"}],"repository":"https://github.com/FlyBase/drosophila-developmental-ontology","taxon":{"id":"NCBITaxon:7227","label":"Drosophila"},"title":"Drosophila development","tracker":"http://purl.obolibrary.org/obo/fbdv/tracker"},{"activity_status":"active","contact":{"email":"georgeta.bordea@u-bordeaux.fr","github":"getbordea","label":"Georgeta Bordea","orcid":"0000-0001-9921-8234"},"description":"Food-Drug interactions automatically extracted from scientific literature","domain":"diet, metabolomics and nutrition","homepage":"https://gitub.u-bordeaux.fr/erias/fideo","id":"fideo","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fideo.owl","preferredPrefix":"FIDEO","products":[{"id":"fideo.owl","ontology_purl":"http://purl.obolibrary.org/obo/fideo.owl"}],"repository":"https://github.com/getbordea/fideo","title":"Food Interactions with Drugs Evidence Ontology","tracker":"https://github.com/getbordea/fideo/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://github.com/flora-phenotype-ontology/flopoontology/raw/master/ontology/flopo.owl"},"contact":{"email":"robert.hoehndorf@kaust.edu.sa","github":"leechuck","label":"Robert Hoehndorf","orcid":"0000-0001-8149-5890"},"description":"Traits and phenotypes of flowering plants occurring in digitized Floras","domain":"phenotype","homepage":"https://github.com/flora-phenotype-ontology/flopoontology","id":"flopo","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/flopo.owl","preferredPrefix":"FLOPO","products":[{"id":"flopo.owl","ontology_purl":"http://purl.obolibrary.org/obo/flopo.owl"}],"repository":"https://github.com/flora-phenotype-ontology/flopoontology","taxon":{"id":"NCBITaxon:33090","label":"Viridiplantae"},"title":"Flora Phenotype Ontology","tracker":"https://github.com/flora-phenotype-ontology/flopoontology/issues"},{"activity_status":"active","contact":{"email":"polcaes@gmail.com","github":"pcastellanoescuder","label":"Pol Castellano Escuder","orcid":"0000-0001-6466-877X"},"dependencies":[{"id":"chebi"},{"id":"foodon"}],"description":"FOBI (Food-Biomarker Ontology) is an ontology to represent food intake data and associate it with metabolomic data","domain":"metabolomics and nutrition","homepage":"https://github.com/pcastellanoescuder/FoodBiomarkerOntology","id":"fobi","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fobi.owl","preferredPrefix":"FOBI","products":[{"format":"owl-rdf/xml","id":"fobi.owl","ontology_purl":"http://purl.obolibrary.org/obo/fobi.owl","title":"FOBI is an ontology to represent food intake data and associate it with metabolomic data"}],"repository":"https://github.com/pcastellanoescuder/FoodBiomarkerOntology","title":"Food-Biomarker Ontology","tracker":"https://github.com/pcastellanoescuder/FoodBiomarkerOntology/issues"},{"activity_status":"active","contact":{"email":"damion_dooley@sfu.ca","github":"ddooley","label":"Damion Dooley","orcid":"0000-0002-8844-9165"},"dependencies":[{"id":"uberon"},{"id":"ro"},{"id":"eo"},{"id":"chebi"},{"id":"ncbitaxon"},{"id":"bfo"},{"id":"envo"},{"id":"obi"}],"description":"A broadly scoped ontology representing entities which bear a food role. It encompasses materials in natural ecosystems and agriculture that are consumed by humans and domesticated animals. This includes any generic (unbranded) raw or processed food material found in processing plants, markets, stores or food distribution points. FoodOn also imports nutritional component and dietary pattern terms from other OBO Foundry ontologies to support interoperability in diet and nutrition research","domain":"food","homepage":"https://foodon.org/","id":"foodon","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/foodon.owl","preferredPrefix":"FOODON","products":[{"format":"owl-rdf/xml","id":"foodon.owl","ontology_purl":"http://purl.obolibrary.org/obo/foodon.owl","title":"FoodOn ontology with import file references and over 9,000 food products"},{"format":"owl-rdf/xml","id":"foodon_core.owl","ontology_purl":"http://purl.obolibrary.org/obo/foodon_core.owl","title":"FoodOn core ontology (currently the same as foodon.owl)"}],"repository":"https://github.com/FoodOntology/foodon","title":"Food Ontology","tracker":"https://github.com/FoodOntology/foodon/issues/"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/futres/fovt.git","path":".","system":"git"},"contact":{"email":"meghan.balk@gmail.com","github":"megbalk","label":"Meghan Balk","orcid":"0000-0003-2699-3066"},"dependencies":[{"id":"bco"},{"id":"oba"},{"id":"ro"},{"id":"bfo"},{"id":"pato"},{"id":"bspo"},{"id":"iao"},{"id":"uberon"}],"description":"FuTRES Ontology of Vertebrate Traits is an application ontology used to convert vertebrate trait data in spreadsheet to triples. FOVT leverages the BioCollections Ontology (BCO) to link observations of individual specimens to their trait values. Traits are defined in the Ontology of Biological Attributes (OBA).","domain":"vertebrate traits","homepage":"https://github.com/futres/fovt","id":"fovt","jobs":[{"id":"https://travis-ci.org/futres/fovt","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fovt.owl","preferredPrefix":"FOVT","products":[{"id":"fovt.owl","name":"FuTRES Ontology of Vertebrate Traits main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/fovt.owl"},{"id":"fovt.obo","name":"FuTRES Ontology of Vertebrate Traits additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/fovt.obo"},{"id":"fovt/fovt-base.owl","name":"FuTRES Ontology of Vertebrate Traits main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/fovt/fovt-base.owl"},{"id":"fovt/fovt-base.obo","name":"FuTRES Ontology of Vertebrate Traits additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/fovt/fovt-base.obo"}],"repository":"https://github.com/futres/fovt","title":"FuTRES Ontology of Vertebrate Traits","tracker":"https://github.com/futres/fovt/issues"},{"activity_status":"active","build":{"infallible":1,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/pombase/fypo/master/release/fypo.owl"},"contact":{"email":"vw253@cam.ac.uk","github":"ValWood","label":"Val Wood","orcid":"0000-0001-6330-7526"},"depicted_by":"https://github.com/pombase/website/blob/master/src/assets/FYPO_logo_tiny.png","description":"FYPO is a formal ontology of phenotypes observed in fission yeast.","domain":"phenotype","homepage":"https://github.com/pombase/fypo","id":"fypo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fypo.owl","preferredPrefix":"FYPO","products":[{"id":"fypo.owl","ontology_purl":"http://purl.obolibrary.org/obo/fypo.owl"},{"id":"fypo.obo","ontology_purl":"http://purl.obolibrary.org/obo/fypo.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/23658422","title":"FYPO: The Fission Yeast Phenotype Ontology."}],"repository":"https://github.com/pombase/fypo","taxon":{"id":"NCBITaxon:4896","label":"S. pombe"},"title":"Fission Yeast Phenotype Ontology","tracker":"https://github.com/pombase/fypo/issues","usages":[{"description":"Pombase uses fypo for phenotype data annotation in fission yeast","examples":[{"description":"genotypes annotated to abnormal mitotic cell cycle in fission yeast","url":"https://www.pombase.org/term/FYPO:0000059"}],"user":"https://www.pombase.org"}]},{"activity_status":"active","contact":{"email":"rbca.jackson@gmail.com","github":"beckyjackson","label":"Rebecca Jackson","orcid":"0000-0003-4871-5569"},"description":"An ontology to represent genomics cohort attributes","domain":"cohort studies","homepage":"https://github.com/IHCC-cohorts/GECKO","id":"gecko","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/gecko.owl","preferredPrefix":"GECKO","products":[{"id":"gecko.owl","ontology_purl":"http://purl.obolibrary.org/obo/gecko.owl"}],"repository":"https://github.com/IHCC-cohorts/GECKO","title":"Genomics Cohorts Knowledge Ontology","tracker":"https://github.com/IHCC-cohorts/GECKO/issues","usages":[{"description":"IHCC uses GECKO to standardize data from various cohorts for the IHCC cohort browser","user":"https://ihccglobal.org/"}]},{"activity_status":"active","contact":{"email":"damion_dooley@sfu.ca","github":"ddooley","label":"Damion Dooley","orcid":"0000-0002-8844-9165"},"dependencies":[{"id":"uberon"},{"id":"po"},{"id":"ro"},{"id":"chebi"},{"id":"ncbitaxon"}],"description":"The Genomic Epidemiology Ontology (GenEpiO) covers vocabulary necessary to identify, document and research foodborne pathogens and associated outbreaks.","domain":"health","homepage":"http://genepio.org/","id":"genepio","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/genepio.owl","page":"https://github.com/GenEpiO/genepio","preferredPrefix":"GENEPIO","products":[{"homepage":"http://genepio.github.io/genepio/","id":"genepio.owl","ontology_purl":"http://purl.obolibrary.org/obo/genepio.owl"}],"repository":"https://github.com/GenEpiO/genepio","title":"Genomic Epidemiology Ontology","tracker":"https://github.com/GenEpiO/genepio/issues/"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/monarch-initiative/GENO-ontology.git","path":"src/ontology","system":"git"},"contact":{"email":"mhb120@gmail.com","github":"mbrush","label":"Matthew Brush","orcid":"0000-0002-1048-5019"},"description":"An integrated ontology for representing the genetic variations described in genotypes, and their causal relationships to phenotype and diseases.","domain":"genotype-to-phenotype associations","homepage":"https://github.com/monarch-initiative/GENO-ontology/","id":"geno","layout":"ontology_detail","license":{"label":"CC BY-SA 2.0","logo":"https://mirrors.creativecommons.org/presskit/buttons/80x15/png/by-sa.png","url":"https://creativecommons.org/licenses/by-sa/2.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/geno.owl","preferredPrefix":"GENO","products":[{"id":"geno.owl","ontology_purl":"http://purl.obolibrary.org/obo/geno.owl","title":"GENO"}],"repository":"https://github.com/monarch-initiative/GENO-ontology","title":"Genotype Ontology","tracker":"https://github.com/monarch-initiative/GENO-ontology/issues"},{"activity_status":"active","contact":{"email":"hoganwr@gmail.com","github":"hoganwr","label":"Bill Hogan","orcid":"0000-0002-9881-1017"},"description":"An ontology of geographical entities","homepage":"https://github.com/ufbmi/geographical-entity-ontology/wiki","id":"geo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/geo.owl","preferredPrefix":"GEO","products":[{"id":"geo.owl","ontology_purl":"http://purl.obolibrary.org/obo/geo.owl"}],"repository":"https://github.com/ufbmi/geographical-entity-ontology","title":"Geographical Entity Ontology","tracker":"https://github.com/ufbmi/geographical-entity-ontology/issues"},{"activity_status":"active","browsers":[{"label":"Structure Browser","title":"GNOme Glycan Structure Browser","url":"https://gnome.glyomics.org/StructureBrowser.html?HexNAc=4&Hex=5&dHex=1&NeuAc=2"},{"label":"Composition Browser","title":"GNOme Glycan Composition Browser","url":"https://gnome.glyomics.org/CompositionBrowser.html?HexNAc=4&Hex=5&dHex=1&NeuAc=2"}],"build":{"checkout":"git clone https://github.com/glygen-glycan-data/GNOme.git","path":".","system":"git"},"contact":{"email":"nje5@georgetown.edu","github":"edwardsnj","label":"Nathan Edwards","orcid":"0000-0001-5168-3196"},"description":"GlyTouCan provides stable accessions for glycans described at varyious degrees of characterization, including compositions (no linkage) and topologies (no carbon bond positions or anomeric configurations). GNOme organizes these stable accessions for interative browsing, for text-based searching, and for automated reasoning with well-defined characterization levels.","domain":"glycan structure","homepage":"https://gnome.glyomics.org/","id":"gno","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/gno.owl","preferredPrefix":"GNO","products":[{"description":"Glycan Naming and Subsumption Ontology, OWL format (primary)","id":"gno.owl","ontology_purl":"http://purl.obolibrary.org/obo/gno.owl"},{"description":"Glycan Naming and Subsumption Ontology, OBO format (automated conversion from OWL)","id":"gno.obo","ontology_purl":"http://purl.obolibrary.org/obo/gno.obo"},{"description":"Glycan Naming and Subsumption Ontology, JSON format (automated conversion from OWL)","id":"gno.json","ontology_purl":"http://purl.obolibrary.org/obo/gno.json"}],"repository":"https://github.com/glygen-glycan-data/GNOme","title":"Glycan Naming and Subsumption Ontology (GNOme)","tracker":"https://github.com/glygen-glycan-data/GNOme/issues","usages":[{"description":"GlyGen - Computational and Informatics Resources for Glycoscience","examples":[{"description":"GNOme attributes and related glycans on glycan pages","url":"https://www.glygen.org/glycan/G00028MO"}],"user":"https://www.glygen.org/"},{"description":"PRO - Protein Ontology","examples":[{"description":"example of PRO use of GNO terms","url":"http://purl.obolibrary.org/obo/PR_000059585"}],"user":"https://proconsortium.org/"},{"description":"ChEBI - Chemical Entities of Biological Interest","examples":[{"description":"example of ChEBI use of GNO terms","url":"http://purl.obolibrary.org/obo/CHEBI_167503"}],"user":"https://www.ebi.ac.uk/chebi/init.do"}]},{"activity_status":"active","contact":{"email":"danielle.welter@uni.lu","github":"daniwelter","label":"Danielle Welter","orcid":"0000-0003-1058-2668"},"description":"The Human Ancestry Ontology (HANCESTRO) provides a systematic description of the ancestry concepts used in the NHGRI-EBI Catalog of published genome-wide association studies.","domain":"ancestry","homepage":"https://github.com/EBISPOT/ancestro","id":"hancestro","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/hancestro.owl","preferredPrefix":"HANCESTRO","products":[{"description":"The full version of HANCESTRO in OWL format","id":"hancestro.owl","ontology_purl":"http://purl.obolibrary.org/obo/hancestro.owl","title":"HANCESTRO"},{"description":"Version of HANCESTRO with BFO upper hierarchy for easier integration with other ontologies","id":"hancestro_bfo.owl","ontology_purl":"http://purl.obolibrary.org/obo/hancestro_bfo.owl","title":"HANCESTRO BFO"}],"repository":"https://github.com/EBISPOT/ancestro","title":"Human Ancestry Ontology","tracker":"https://github.com/EBISPOT/ancestro/issues","usages":[{"description":"The Experimental Factor Ontology (EFO) provides a systematic description of many experimental variables available in EBI databases, and for external projects such as the NHGRI GWAS catalogue. It combines parts of several biological ontologies, such as anatomy, disease and chemical compounds.","examples":[{"description":"Population category defined using ancestry informative markers (AIMs) based on genetic/genomic data","url":"https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHANCESTRO_0004&viewMode=All&siblings=false"}],"user":"http://www.ebi.ac.uk/efo"},{"description":"The Genomic Epidemiology Ontology (GenEpiO) covers vocabulary necessary to identify, document and research foodborne pathogens and associated outbreaks.","examples":[{"description":"Population category defined using ancestry informative markers (AIMs) based on genetic/genomic data","url":"https://www.ebi.ac.uk/ols/ontologies/genepio/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHANCESTRO_0004&viewMode=All&siblings=false"}],"user":"https://genepio.org/"},{"description":"FoodOn (http://foodon.org) is a consortium-driven project to build a comprehensive and easily accessible global farm-to-fork ontology about food, that accurately and consistently describes foods commonly known in cultures from around the world.","examples":[{"description":"Population category defined using ancestry informative markers (AIMs) based on genetic/genomic data","url":"https://www.ebi.ac.uk/ols/ontologies/foodon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHANCESTRO_0004&viewMode=All&siblings=false"}],"user":"http://foodon.org"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/hymao/hao.git","system":"git"},"contact":{"email":"diapriid@gmail.com","github":"mjy","label":"Matt Yoder","orcid":"0000-0002-5640-5491"},"description":"A structured controlled vocabulary of the anatomy of the Hymenoptera (bees, wasps, and ants)","domain":"anatomy","homepage":"http://hymao.org","id":"hao","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/hao.owl","preferredPrefix":"HAO","products":[{"id":"hao.owl","ontology_purl":"http://purl.obolibrary.org/obo/hao.owl"},{"id":"hao.obo","ontology_purl":"http://purl.obolibrary.org/obo/hao.obo"},{"id":"hao/depictions.owl","ontology_purl":"http://purl.obolibrary.org/obo/hao/depictions.owl"}],"repository":"https://github.com/hymao/hao","taxon":{"id":"NCBITaxon:7399","label":"Hymenoptera"},"title":"Hymenoptera Anatomy Ontology","tracker":"https://github.com/hymao/hao/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/BgeeDB/homology-ontology.git","method":"vcs","path":"src/ontology","system":"git"},"contact":{"email":"bgee@sib.swiss","github":"fbastian","label":"Frederic Bastian","orcid":"0000-0002-9415-5104"},"description":"This ontology represents concepts related to homology, as well as other concepts used to describe similarity and non-homology.","homepage":"https://github.com/BgeeDB/homology-ontology","id":"hom","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/hom.owl","preferredPrefix":"HOM","products":[{"id":"hom.owl","ontology_purl":"http://purl.obolibrary.org/obo/hom.owl"}],"repository":"https://github.com/BgeeDB/homology-ontology","title":"Homology Ontology","tracker":"https://github.com/BgeeDB/homology-ontology/issues"},{"activity_status":"active","build":{"infallible":1,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/hsapdv/hsapdv.obo"},"contact":{"email":"frederic.bastian@unil.ch","github":"fbastian","label":"Frdric Bastian","orcid":"0000-0002-9415-5104"},"description":"Life cycle stages for Human","homepage":"https://github.com/obophenotype/developmental-stage-ontologies/wiki/HsapDv","id":"hsapdv","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/hsapdv.owl","page":"https://github.com/obophenotype/developmental-stage-ontologies","preferredPrefix":"HsapDv","products":[{"id":"hsapdv.owl","ontology_purl":"http://purl.obolibrary.org/obo/hsapdv.owl"},{"id":"hsapdv.obo","ontology_purl":"http://purl.obolibrary.org/obo/hsapdv.obo"}],"repository":"https://github.com/obophenotype/developmental-stage-ontologies","title":"Human Developmental Stages","tracker":"https://github.com/obophenotype/developmental-stage-ontologies/issues"},{"activity_status":"active","contact":{"email":"fernanda.dorea@sva.se","github":"nandadorea","label":"Fernanda Dorea","orcid":"0000-0001-8638-8525"},"dependencies":[{"id":"uberon"},{"id":"bfo"},{"id":"ro"},{"id":"obi"},{"id":"ncbitaxon"}],"description":"The health Surveillance Ontology (HSO) focuses on \\"surveillance system level data\\", that is, data outputs from surveillance activities, such as number of samples collected, cases observed, etc. It aims to support One-Health surveillance, covering animal health, public health and food safety surveillance.","domain":"health","homepage":"https://w3id.org/hso","id":"hso","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/hso.owl","page":"https://github.com/SVA-SE/HSO","preferredPrefix":"HSO","products":[{"homepage":"https://w3id.org/hso","id":"hso.owl","ontology_purl":"http://purl.obolibrary.org/obo/hso.owl"}],"repository":"https://github.com/SVA-SE/HSO","title":"Health Surveillance Ontology","tracker":"https://github.com/SVA-SE/HSO/issues/"},{"activity_status":"active","contact":{"email":"aellenhicks@gmail.com","github":"aellenhicks","label":"Amanda Hicks","orcid":"0000-0002-1795-5570"},"description":"An ontology for representing clinical data about hypertension, intended to support classification of patients according to various diagnostic guidelines","homepage":"https://github.com/aellenhicks/htn_owl","id":"htn","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/htn.owl","preferredPrefix":"HTN","products":[{"id":"htn.owl","ontology_purl":"http://purl.obolibrary.org/obo/htn.owl","title":"HTN"}],"repository":"https://github.com/aellenhicks/htn_owl","source_url":"https://raw.githubusercontent.com/aellenhicks/htn_owl/master/htn.owl","title":"Hypertension Ontology","tracker":"https://github.com/aellenhicks/htn_owl/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/iao.owl"},"contact":{"email":"jiezheng@pennmedicine.upenn.edu","github":"zhengj2007","label":"Jie Zheng","orcid":"0000-0002-2999-0103"},"depicted_by":"https://avatars0.githubusercontent.com/u/13591168?v=3&s=200","description":"An ontology of information entities.","domain":"information","homepage":"https://github.com/information-artifact-ontology/IAO/","id":"iao","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/iao.owl","preferredPrefix":"IAO","products":[{"id":"iao.owl","ontology_purl":"http://purl.obolibrary.org/obo/iao.owl"},{"id":"iao/ontology-metadata.owl","ontology_purl":"http://purl.obolibrary.org/obo/iao/ontology-metadata.owl","page":"https://github.com/information-artifact-ontology/IAO/wiki/OntologyMetadata","title":"IAO ontology metadata"},{"id":"iao/dev/iao.owl","ontology_purl":"http://purl.obolibrary.org/obo/iao/dev/iao.owl","title":"IAO dev"},{"contact":{"email":"mbrochhausen@gmail.com","label":"Mathias Brochhausen"},"description":"An ontology based on a theory of document acts describing what people can do with documents","id":"iao/d-acts.owl","ontology_purl":"http://purl.obolibrary.org/obo/iao/d-acts.owl","title":"ontology of document acts"}],"repository":"https://github.com/information-artifact-ontology/IAO","title":"Information Artifact Ontology","tracker":"https://github.com/information-artifact-ontology/IAO/issues","usages":[{"description":"IAO is used widely by multiple OBO ontologies for information representation.","type":"owl_import","user":"http://obofoundry.org"}]},{"activity_status":"active","contact":{"email":"liumeng94@sjtu.edu.cn","github":"Lemon-Liu","label":"Meng LIU","orcid":"0000-0003-3781-6962"},"description":"ICEO is an integrated biological ontology for the description of bacterial integrative and conjugative elements (ICEs).","homepage":"https://github.com/ontoice/ICEO","id":"iceo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/iceo.owl","preferredPrefix":"ICEO","products":[{"id":"iceo.owl","ontology_purl":"http://purl.obolibrary.org/obo/iceo.owl"}],"repository":"https://github.com/ontoice/ICEO","title":"Integrative and Conjugative Element Ontology","tracker":"https://github.com/ontoice/ICEO/issues"},{"activity_status":"active","contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqun Oliver He","orcid":"0000-0001-9189-9661"},"description":"An ontology of clinical informed consents","domain":"informed consent","homepage":"https://github.com/ICO-ontology/ICO","id":"ico","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ico.owl","preferredPrefix":"ICO","products":[{"id":"ico.owl","ontology_purl":"http://purl.obolibrary.org/obo/ico.owl"}],"repository":"https://github.com/ICO-ontology/ICO","title":"Informed Consent Ontology","tracker":"https://github.com/ICO-ontology/ICO/issues","usages":[{"description":"tracking informed consent in the kidney precision medicine project that has over 20 institutes involved.","user":"http://kpmp.org"}]},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/ido.owl"},"contact":{"email":"Lindsay.Cowell@utsouthwestern.edu","github":"lgcowell","label":"Lindsay Cowell","orcid":"0000-0003-1617-8244"},"description":"A set of interoperable ontologies that will together provide coverage of the infectious disease domain. IDO core is the upper-level ontology that hosts terms of general relevance across the domain, while extension ontologies host terms to specific to a particular part of the domain.","domain":"health","homepage":"http://www.bioontology.org/wiki/index.php/Infectious_Disease_Ontology","id":"ido","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ido.owl","preferredPrefix":"IDO","products":[{"id":"ido.owl","ontology_purl":"http://purl.obolibrary.org/obo/ido.owl"}],"repository":"https://github.com/infectious-disease-ontology/infectious-disease-ontology","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Infectious Disease Ontology","tracker":"https://github.com/infectious-disease-ontology/infectious-disease-ontology/issues"},{"activity_status":"active","contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqun Oliver He","orcid":"0000-0001-9189-9661"},"description":"An ontology of interactions and interaction networks","homepage":"https://github.com/INO-ontology/ino","id":"ino","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ino.owl","preferredPrefix":"INO","products":[{"id":"ino.owl","ontology_purl":"http://purl.obolibrary.org/obo/ino.owl"}],"repository":"https://github.com/INO-ontology/ino","title":"Interaction Network Ontology","tracker":"https://github.com/INO-ontology/ino/issues"},{"activity_status":"active","contact":{"email":"paul.fabry@usherbrooke.ca","github":"pfabry","label":"Paul Fabry","orcid":"0000-0002-3336-2476"},"dependencies":[{"id":"iao"},{"id":"obi"},{"id":"ogms"},{"id":"omiabis"},{"id":"omrse"},{"id":"opmi"}],"description":"LABO is an ontology of informational entities formalizing clinical laboratory tests prescriptions and reporting documents.","domain":"clinical documentation","homepage":"https://github.com/OpenLHS/LABO","id":"labo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/labo.owl","preferredPrefix":"LABO","products":[{"id":"labo.owl","ontology_purl":"http://purl.obolibrary.org/obo/labo.owl"}],"releases":"https://github.com/OpenLHS/LABO/releases/","repository":"https://github.com/OpenLHS/LABO","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"clinical LABoratory Ontology","tracker":"https://github.com/OpenLHS/LABO/issues","usages":[{"description":"LABO is a part of a core ontological model of medical knowledge in the PARS3 data platform.","type":"database architecture","user":"https://griis.ca/"}]},{"activity_status":"active","contact":{"email":"lagonzalezmo@unal.edu.co","github":"luis-gonzalez-m","label":"Luis A. Gonzalez-Montana","orcid":"0000-0002-9136-9932"},"dependencies":[{"id":"aism"},{"id":"ro"},{"id":"uberon"},{"id":"caro"},{"id":"pato"},{"id":"bfo"},{"id":"bspo"}],"description":"The Lepidoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of moths and butterflies in biodiversity research. LEPAO is developed in part by BIOfid (The Specialised Information Service Biodiversity Research).","domain":"Insect Anatomy","homepage":"https://github.com/insect-morphology/lepao","id":"lepao","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/lepao.owl","preferredPrefix":"LEPAO","products":[{"id":"lepao.owl","ontology_purl":"http://purl.obolibrary.org/obo/lepao.owl"},{"id":"lepao.obo","ontology_purl":"http://purl.obolibrary.org/obo/lepao.obo"}],"repository":"https://github.com/insect-morphology/lepao","title":"Lepidoptera Anatomy Ontology","tracker":"https://github.com/insect-morphology/lepao/issues"},{"activity_status":"active","build":{"infallible":1,"insert_ontology_id":true,"method":"obo2owl","source_url":"ftp://ftp.informatics.jax.org/pub/reports/adult_mouse_anatomy.obo"},"contact":{"email":"Terry.Hayamizu@jax.org","github":"tfhayamizu","label":"Terry Hayamizu","orcid":"0000-0002-0956-8634"},"description":"A structured controlled vocabulary of the adult anatomy of the mouse (Mus).","domain":"anatomy","homepage":"https://github.com/obophenotype/mouse-anatomy-ontology","id":"ma","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ma.owl","page":"http://www.informatics.jax.org/searches/AMA_form.shtml","preferredPrefix":"MA","products":[{"id":"ma.owl","ontology_purl":"http://purl.obolibrary.org/obo/ma.owl"},{"id":"ma.obo","ontology_purl":"http://purl.obolibrary.org/obo/ma.obo"}],"repository":"https://github.com/obophenotype/mouse-anatomy-ontology","taxon":{"id":"NCBITaxon:10088","label":"Mus"},"title":"Mouse adult gross anatomy","tracker":"https://github.com/obophenotype/mouse-anatomy-ontology/issues","usages":[{"description":"GXD","seeAlso":"https://www.biosharing.org/biodbcore-000659","user":"http://www.informatics.jax.org/expression.shtml"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/monarch-initiative/MAxO.git","path":".","system":"git"},"contact":{"email":"Leigh.Carmody@jax.org","github":"LCCarmody","label":"Leigh Carmody","orcid":"0000-0001-7941-2961"},"dependencies":[{"id":"iao"},{"id":"go"},{"id":"ro"},{"id":"uberon"},{"id":"foodon"},{"id":"chebi"}],"description":"Medical Action Ontology is an ontology...","domain":"medical","homepage":"https://github.com/monarch-initiative/MAxO","id":"maxo","jobs":[{"id":"https://travis-ci.org/monarch-initiative/MAxO","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/maxo.owl","preferredPrefix":"MAXO","products":[{"id":"maxo.owl","name":"Medical Action Ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/maxo.owl"},{"id":"maxo.obo","name":"Medical Action Ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/maxo.obo"},{"id":"maxo.json","name":"Medical Action Ontology additional release in OBOJSon format","ontology_purl":"http://purl.obolibrary.org/obo/maxo.json"},{"id":"maxo/maxo-base.owl","name":"Medical Action Ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/maxo/maxo-base.owl"},{"id":"maxo/maxo-base.obo","name":"Medical Action Ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/maxo/maxo-base.obo"},{"id":"maxo/maxo-base.json","name":"Medical Action Ontology additional release in OBOJSon format","ontology_purl":"http://purl.obolibrary.org/obo/maxo/maxo-base.json"}],"repository":"https://github.com/monarch-initiative/MAxO","title":"Medical Action Ontology","tracker":"https://github.com/monarch-initiative/MAxO/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/microbial-conditions-ontology/microbial-conditions-ontology.git","path":".","system":"git"},"contact":{"email":"citlalli.mejiaalmonte@gmail.com","github":"citmejia","label":"Citlalli Meja-Almonte","orcid":"0000-0002-0142-5591"},"dependencies":[{"id":"bfo"},{"id":"chebi"},{"id":"cl"},{"id":"clo"},{"id":"micro"},{"id":"ncbitaxon"},{"id":"peco"},{"id":"ncit"},{"id":"obi"},{"id":"omit"},{"id":"omp"},{"id":"pato"},{"id":"uberon"},{"id":"zeco"}],"description":"Microbial Conditions Ontology is an ontology...","domain":"experimental conditions","homepage":"https://github.com/microbial-conditions-ontology/microbial-conditions-ontology","id":"mco","jobs":[{"id":"https://travis-ci.org/microbial-conditions-ontology/microbial-conditions-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mco.owl","preferredPrefix":"MCO","products":[{"id":"mco.owl","ontology_purl":"http://purl.obolibrary.org/obo/mco.owl"},{"id":"mco.obo","ontology_purl":"http://purl.obolibrary.org/obo/mco.obo"}],"repository":"https://github.com/microbial-conditions-ontology/microbial-conditions-ontology","title":"Microbial Conditions Ontology","tracker":"https://github.com/microbial-conditions-ontology/microbial-conditions-ontology/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/mf.owl"},"contact":{"email":"janna.hastings@gmail.com","github":"jannahastings","label":"Janna Hastings","orcid":"0000-0002-3469-4923"},"description":"The Mental Functioning Ontology is an overarching ontology for all aspects of mental functioning.","domain":"health","homepage":"https://github.com/jannahastings/mental-functioning-ontology","id":"mf","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mf.owl","preferredPrefix":"MF","products":[{"id":"mf.owl","ontology_purl":"http://purl.obolibrary.org/obo/mf.owl"}],"repository":"https://github.com/jannahastings/mental-functioning-ontology","title":"Mental Functioning Ontology","tracker":"https://github.com/jannahastings/mental-functioning-ontology/issues"},{"activity_status":"active","contact":{"email":"druzinsk@uic.edu","github":"RDruzinsky","label":"Robert Druzinsky","orcid":"0000-0002-1572-1316"},"dependencies":[{"id":"uberon"}],"description":"The Mammalian Feeding Muscle Ontology is an antomy ontology for the muscles of the head and neck that participate in feeding, swallowing, and other oral-pharyngeal behaviors.","domain":"anatomy","homepage":"https://github.com/rdruzinsky/feedontology","id":"mfmo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mfmo.owl","preferredPrefix":"MFMO","products":[{"id":"mfmo.owl","ontology_purl":"http://purl.obolibrary.org/obo/mfmo.owl"}],"repository":"https://github.com/RDruzinsky/feedontology","taxon":{"id":"NCBITaxon:40674","label":"Mammalian"},"title":"Mammalian Feeding Muscle Ontology","tracker":"https://github.com/RDruzinsky/feedontology/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/mfoem.owl"},"contact":{"email":"janna.hastings@gmail.com","github":"jannahastings","label":"Janna Hastings","orcid":"0000-0002-3469-4923"},"description":"An ontology of affective phenomena such as emotions, moods, appraisals and subjective feelings.","domain":"health","homepage":"https://github.com/jannahastings/emotion-ontology","id":"mfoem","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mfoem.owl","preferredPrefix":"MFOEM","products":[{"id":"mfoem.owl","ontology_purl":"http://purl.obolibrary.org/obo/mfoem.owl"}],"repository":"https://github.com/jannahastings/emotion-ontology","title":"Emotion Ontology","tracker":"https://github.com/jannahastings/emotion-ontology/issues"},{"activity_status":"active","contact":{"email":"janna.hastings@gmail.com","github":"jannahastings","label":"Janna Hastings","orcid":"0000-0002-3469-4923"},"description":"An ontology to describe and classify mental diseases such as schizophrenia, annotated with DSM-IV and ICD codes where applicable","domain":"health","homepage":"https://github.com/jannahastings/mental-functioning-ontology","id":"mfomd","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mfomd.owl","preferredPrefix":"MFOMD","products":[{"id":"mfomd.owl","ontology_purl":"http://purl.obolibrary.org/obo/mfomd.owl"}],"repository":"https://github.com/jannahastings/mental-functioning-ontology","title":"Mental Disease Ontology","tracker":"https://github.com/jannahastings/mental-functioning-ontology/issues"},{"activity_status":"active","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/HUPO-PSI/psi-mi-CV/master/psi-mi.obo"},"contact":{"email":"pporras@ebi.ac.uk","github":"pporrasebi","label":"Pablo Porras Milln","orcid":"0000-0002-8429-8793"},"description":"A structured controlled vocabulary for the annotation of experiments concerned with protein-protein interactions.","domain":"experiments","homepage":"https://github.com/HUPO-PSI/psi-mi-CV","id":"mi","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mi.owl","page":"https://github.com/HUPO-PSI/psi-mi-CV","preferredPrefix":"MI","products":[{"id":"mi.owl","ontology_purl":"http://purl.obolibrary.org/obo/mi.owl"},{"id":"mi.obo","ontology_purl":"http://purl.obolibrary.org/obo/mi.obo"}],"repository":"https://github.com/HUPO-PSI/psi-mi-CV","title":"Molecular Interactions Controlled Vocabulary","tracker":"https://github.com/HUPO-PSI/psi-mi-CV/issues"},{"activity_status":"active","biosharing":"https://fairsharing.org/FAIRsharing.ca48xs","contact":{"email":"hilmar.lapp@duke.edu","github":"hlapp","label":"Hilmar Lapp","orcid":"0000-0001-9107-0714"},"description":"An application ontology to formalize annotation of phylogenetic data.","homepage":"http://www.evoio.org/wiki/MIAPA","id":"miapa","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"mailing_list":"http://groups.google.com/group/miapa-discuss","ontology_purl":"http://purl.obolibrary.org/obo/miapa.owl","preferredPrefix":"MIAPA","products":[{"id":"miapa.owl","ontology_purl":"http://purl.obolibrary.org/obo/miapa.owl"}],"releases":"https://github.com/evoinfo/miapa/releases","repository":"https://github.com/evoinfo/miapa","title":"MIAPA Ontology","tracker":"https://github.com/evoinfo/miapa/issues"},{"activity_status":"active","contact":{"email":"carrine.blank@umontana.edu","github":"carrineblank","label":"Carrine Blank","orcid":"0000-0002-2100-6351"},"description":"An ontology of prokaryotic phenotypic and metabolic characters","domain":"phenotype","homepage":"https://github.com/carrineblank/MicrO","id":"micro","layout":"ontology_detail","license":{"label":"CC BY 2.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/2.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/micro.owl","preferredPrefix":"MICRO","products":[{"id":"micro.owl","ontology_purl":"http://purl.obolibrary.org/obo/micro.owl"}],"repository":"https://github.com/carrineblank/MicrO","title":"Ontology of Prokaryotic Phenotypic and Metabolic Characters","tracker":"https://github.com/carrineblank/MicrO/issues"},{"activity_status":"active","browsers":[{"label":"RGD","title":"RGD Ontology Browser","url":"http://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MMO:0000000"}],"build":{"method":"obo2owl","source_url":"https://download.rgd.mcw.edu/ontology/measurement_method/measurement_method.obo"},"contact":{"email":"jrsmith@mcw.edu","github":"jrsjrs","label":"Jennifer Smith","orcid":"0000-0002-6443-9376"},"description":"A representation of the variety of methods used to make clinical and phenotype measurements. ","domain":"clinical","homepage":"https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MMO:0000000","id":"mmo","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mmo.owl","page":"https://download.rgd.mcw.edu/ontology/measurement_method/","preferredPrefix":"MMO","products":[{"id":"mmo.owl","ontology_purl":"http://purl.obolibrary.org/obo/mmo.owl"},{"id":"mmo.obo","ontology_purl":"http://purl.obolibrary.org/obo/mmo.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/22654893","title":"Three ontologies to define phenotype measurement data."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24103152","title":"The clinical measurement, measurement method and experimental condition ontologies: expansion, improvements and new applications."}],"repository":"https://github.com/rat-genome-database/MMO-Measurement-Method-Ontology","title":"Measurement method ontology","tracker":"https://github.com/rat-genome-database/MMO-Measurement-Method-Ontology/issues"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/mmusdv/mmusdv.obo"},"contact":{"email":"frederic.bastian@unil.ch","github":"fbastian","label":"Frdric Bastian","orcid":"0000-0002-9415-5104"},"description":"Life cycle stages for Mus Musculus","homepage":"https://github.com/obophenotype/developmental-stage-ontologies/wiki/MmusDv","id":"mmusdv","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mmusdv.owl","page":"https://github.com/obophenotype/developmental-stage-ontologies","preferredPrefix":"MmusDv","products":[{"id":"mmusdv.owl","ontology_purl":"http://purl.obolibrary.org/obo/mmusdv.owl"},{"id":"mmusdv.obo","ontology_purl":"http://purl.obolibrary.org/obo/mmusdv.obo"}],"repository":"https://github.com/obophenotype/developmental-stage-ontologies","title":"Mouse Developmental Stages","tracker":"https://github.com/obophenotype/developmental-stage-ontologies/issues"},{"activity_status":"active","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/HUPO-PSI/psi-mod-CV/master/PSI-MOD.obo"},"contact":{"email":"pierre-alain.binz@chuv.ch","github":"pabinz","label":"Pierre-Alain Binz","orcid":"0000-0002-0045-7698"},"description":"PSI-MOD is an ontology consisting of terms that describe protein chemical modifications","domain":"proteins","homepage":"http://www.psidev.info/MOD","id":"mod","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mod.owl","preferredPrefix":"MOD","products":[{"description":"PSI-MOD Ontology, OWL format","id":"mod.owl","ontology_purl":"http://purl.obolibrary.org/obo/mod.owl","title":"PSI-MOD.owl"},{"description":"PSI-MOD Ontology, OBO format","id":"mod.obo","ontology_purl":"http://purl.obolibrary.org/obo/mod.obo","title":"PSI-MOD.obo"}],"publications":[{"id":"https://pubmed.ncbi.nlm.nih.gov/18688235/","title":"The PSI-MOD community standard for representation of protein modification data"}],"repository":"https://github.com/HUPO-PSI/psi-mod-CV","title":"Protein modification","tracker":"https://github.com/HUPO-PSI/psi-mod-CV/issues"},{"activity_status":"active","browsers":[{"label":"Monarch","title":"Monarch Initiative Disease Browser","url":"https://monarchinitiative.org/disease/MONDO:0019609"}],"canonical":"mondo.owl","contact":{"email":"nicole@tislab.org","github":"nicolevasilevsky","label":"Nicole Vasilevsky","orcid":"0000-0001-5208-3432"},"depicted_by":"https://raw.githubusercontent.com/monarch-initiative/mondo/master/docs/images/mondo_logo_black-stacked-small.png","description":"A semi-automatically constructed ontology that merges in multiple disease resources to yield a coherent merged ontology.","domain":"disease","homepage":"https://monarch-initiative.github.io/mondo","id":"mondo","label":"Mondo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"https://groups.google.com/group/mondo-users","ontology_purl":"http://purl.obolibrary.org/obo/mondo.owl","preferredPrefix":"MONDO","products":[{"description":"Complete ontology. Uses MONDO IDs. Imports merged. The original mondo.owl without merged imports and with equivalence axioms can now be obtained from the release pages and is called mondo-with-equivalents.","format":"owl-rdf/xml","id":"mondo.owl","is_canonical":true,"ontology_purl":"http://purl.obolibrary.org/obo/mondo.owl","title":"Main OWL edition"},{"derived_from":"mondo.owl","description":"As OWL. xrefs can be used as proxy for equivalence. Uses Mondo IDs.","format":"obo","id":"mondo.obo","ontology_purl":"http://purl.obolibrary.org/obo/mondo.obo","title":"obo-format edition"},{"derived_from":"mondo.owl","description":"Equivalent to the OWL edition.","format":"obo","id":"mondo.json","ontology_purl":"http://purl.obolibrary.org/obo/mondo.json","title":"json edition"}],"repository":"https://github.com/monarch-initiative/mondo","taxon":{"id":"NCBITaxon:33208","label":"Metazoa"},"title":"Mondo Disease Ontology","tracker":"https://github.com/monarch-initiative/mondo/issues","usages":[{"description":"Mondo is used by the Monarch Initiative for disease annotations.","examples":[{"description":"Parkinsonism: Characteristic neurologic anomaly resulting form degeneration of dopamine-generating cells in the substantia nigra, a region of the midbrain, characterized clinically by shaking, rigidity, slowness of movement and difficulty with walking and gait.","url":"https://monarchinitiative.org/phenotype/HP:0001300#disease"}],"publications":[{"id":"https://academic.oup.com/nar/article/45/D1/D712/2605791","title":"The Monarch Initiative: an integrative data and analytic platform connecting phenotypes to genotypes across species "}],"type":"annotation","user":"https://monarchinitiative.org/"},{"description":"Mondo is used by the ClinGen for disease curations.","examples":[{"description":"SCN1A is an autosomal dominant mutation in Dravet syndrome.","url":"https://search.clinicalgenome.org/kb/conditions/MONDO_0011794"}],"type":"annotation","user":"https://www.clinicalgenome.org/"},{"description":"Mondo is used by the Kids First Data Resource Portal for disease annotations. Note, a login is needed to access the portal and view the Mondo-curated data.","type":"annotation","user":"https://portal.kidsfirstdrc.org/"},{"description":"Mondo is used by the Ancestory for some disease curations.","examples":[{"description":"Some term names in ancestory.com are curated with Mondo, for ease of use.","url":"https://support.ancestry.com/s/article/Disease-Condition-Catalog-Powered-by-MONDO"}],"type":"annotation","user":"https://www.ancestry.com/"},{"description":"Mondo is used by the Human Cell Atlas for some disease annotations.","examples":[{"description":"Disease status specimen is normal, type 2 diabetes mellitus.","url":"https://data.humancellatlas.org/explore/projects?filter=%5B%7B%22facetName%22:%22organ%22,%22terms%22:%5B%22kidney%22%5D%7D,%7B%22facetName%22:%22donorDisease%22,%22terms%22:%5B%22acoustic%20neuroma%22,%22acute%20kidney%20tubular%20necrosis%22%5D%7D%5D&catalog=dcp1"}],"type":"annotation","user":"https://www.humancellatlas.org/"}]},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://raw.githubusercontent.com/rsc-ontologies/rxno/master/mop.owl"},"contact":{"email":"batchelorc@rsc.org","github":"batchelorc","label":"Colin Batchelor","orcid":"0000-0001-5985-7429"},"description":"Processes at the molecular level","homepage":"https://github.com/rsc-ontologies/rxno","id":"mop","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"mailing_list":"chemistry-ontologies@googlegroups.com","ontology_purl":"http://purl.obolibrary.org/obo/mop.owl","preferredPrefix":"MOP","products":[{"id":"mop.owl","ontology_purl":"http://purl.obolibrary.org/obo/mop.owl","title":"Molecular Process Ontology"}],"repository":"https://github.com/rsc-ontologies/rxno","title":"Molecular Process Ontology","tracker":"https://github.com/rsc-ontologies/rxno/issues"},{"activity_status":"active","browsers":[{"label":"MGI","title":"MGI MP Browser","url":"http://www.informatics.jax.org/searches/MP_form.shtml"},{"label":"RGD","title":"RGD MP Browser","url":"https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MP:0000001"},{"label":"Monarch","title":"Monarch Phenotype Page","url":"http://monarchinitiative.org/phenotype/MP:0000001"}],"build":{"checkout":"git clone https://github.com/mgijax/mammalian-phenotype-ontology.git","path":".","system":"git"},"contact":{"email":"drsbello@gmail.com","github":"sbello","label":"Sue Bello","orcid":"0000-0003-4606-0597"},"description":"Standard terms for annotating mammalian phenotypic data.","domain":"phenotype","homepage":"http://www.informatics.jax.org/searches/MP_form.shtml","id":"mp","jobs":[{"id":"http://build.berkeleybop.org/job/build-mp-edit","type":"DryRunBuild"}],"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"https://groups.google.com/forum/#!forum/phenotype-ontologies-editors","ontology_purl":"http://purl.obolibrary.org/obo/mp.owl","page":"https://github.com/mgijax/mammalian-phenotype-ontology","preferredPrefix":"MP","products":[{"description":"The main ontology in OWL. Contains all MP terms and links to other OBO ontologies.","id":"mp.owl","ontology_purl":"http://purl.obolibrary.org/obo/mp.owl","page":"https://github.com/mgijax/mammalian-phenotype-ontology/releases/tag/current","title":"MP (OWL edition)"},{"description":"A direct translation of the MP (OWL edition) into OBO format.","id":"mp.obo","ontology_purl":"http://purl.obolibrary.org/obo/mp.obo","page":"https://github.com/mgijax/mammalian-phenotype-ontology/releases/tag/current","title":"MP (OBO edition)"},{"description":"For a description of the format see https://github.com/geneontology/obographs.","id":"mp.json","ontology_purl":"http://purl.obolibrary.org/obo/mp.json","page":"https://github.com/mgijax/mammalian-phenotype-ontology/releases/tag/current","title":"MP (obographs JSON edition)"},{"description":"The main ontology plus axioms connecting to select external ontologies, excluding axioms from the the external ontologies themselves.","id":"mp/mp-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/mp/mp-base.owl","page":"https://github.com/mgijax/mammalian-phenotype-ontology/releases/tag/current","title":"MP Base Module"}],"repository":"https://github.com/mgijax/mammalian-phenotype-ontology","taxon":{"id":"NCBITaxon:40674","label":"Mammalia"},"title":"Mammalian Phenotype Ontology","tracker":"https://github.com/mgijax/mammalian-phenotype-ontology/issues","usages":[{"description":"MGI annotates phenotypes of mouse models using the MP","examples":[{"description":"Term browser page for embryonic lethality showing information about the term including definition, placement in the MP hierarchy, and link to mouse models annotated to this term or any of its decendants","url":"http://www.informatics.jax.org/vocab/mp_ontology/MP:0008762"}],"user":"http://www.informatics.jax.org/vocab/mp_ontology"},{"description":"RGD annotates phenotypes associated with rat genes and alleles using the MP","examples":[{"description":"Term browser page for embryonic lethality showing information about the term including definition, placement in the MP hierarchy, and link to annotations to this term or any of its decendants","url":"https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MP:0008762"}],"user":"https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MP:0000001"},{"description":"IMPC annotates abnormal phenotypes of mice carrying null alleles found following the application of a standardised set of physiological tests","examples":[{"description":"All IMPC alleles that have been annotated to the MP term \'decreased memory-marker CD4-positive NK T cell number\'.","url":"https://www.mousephenotype.org/data/phenotypes/MP:0013522"}],"user":"https://www.mousephenotype.org/"}]},{"activity_status":"active","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/PaulNSchofield/mpath/master/mpath.obo"},"contact":{"email":"pns12@hermes.cam.ac.uk","github":"PaulNSchofield","label":"Paul Schofield","orcid":"0000-0002-5111-7263"},"description":"A structured controlled vocabulary of mutant and transgenic mouse pathology phenotypes","domain":"health","homepage":"http://www.pathbase.net","id":"mpath","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mpath.owl","preferredPrefix":"MPATH","products":[{"id":"mpath.owl","ontology_purl":"http://purl.obolibrary.org/obo/mpath.owl"}],"repository":"https://github.com/PaulNSchofield/mpath","taxon":{"id":"NCBITaxon:10088","label":"Mus"},"title":"Mouse pathology ontology","tracker":"https://github.com/PaulNSchofield/mpath/issues"},{"activity_status":"active","contact":{"email":"mbrochhausen@uams.edu","github":"mbrochhausen","label":"Mathias Brochhausen","orcid":"0000-0003-1834-3856"},"description":"An ontology of minimum information regarding potential drug-drug interaction information.","domain":"health","homepage":"https://github.com/MPIO-Developers/MPIO","id":"mpio","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mpio.owl","preferredPrefix":"MPIO","products":[{"id":"mpio.owl","ontology_purl":"http://purl.obolibrary.org/obo/mpio.owl"}],"repository":"https://github.com/MPIO-Developers/MPIO","title":"Minimum PDDI Information Ontology","tracker":"https://github.com/MPIO-Developers/MPIO/issues"},{"activity_status":"active","contact":{"email":"bpeters@lji.org","github":"bpeters42","label":"Bjoern Peters","orcid":"0000-0002-8457-6693"},"description":"An ontology for Major Histocompatibility Complex (MHC) restriction in experiments","domain":"Major Histocompatibility Complex","homepage":"https://github.com/IEDB/MRO","id":"mro","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/mro.owl","preferredPrefix":"MRO","products":[{"id":"mro.owl","ontology_purl":"http://purl.obolibrary.org/obo/mro.owl"}],"repository":"https://github.com/IEDB/MRO","title":"MHC Restriction Ontology","tracker":"https://github.com/IEDB/MRO/issues","usages":[{"description":"MRO is used by the The Immune Epitope Database (IEDB) annotations","examples":[{"description":"Epitope ID: 59611 based on reference 1003499","url":"https://www.iedb.org/assay/1357035"}],"publications":[{"id":"https://doi.org/10.1093/nar/gky1006","title":"The Immune Epitope Database (IEDB): 2018 update"}],"type":"annotation","user":"https://www.iedb.org/"}]},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo"},"contact":{"email":"gerhard.mayer@rub.de","github":"germa","label":"Gerhard Mayer","orcid":"0000-0002-1767-2343"},"createdWith":"http://oboedit.org","dependencies":[{"id":"pato"},{"id":"uo"}],"description":"A structured controlled vocabulary for the annotation of experiments concerned with proteomics mass spectrometry.","domain":"MS experiments","homepage":"http://www.psidev.info/groups/controlled-vocabularies","id":"ms","integration_server":"https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master","label":"MS","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"mailing_list":"psidev-ms-vocab@lists.sourceforge.net","ontology_purl":"http://purl.obolibrary.org/obo/ms.owl","page":"http://www.psidev.info/groups/controlled-vocabularies","preferredPrefix":"MS","products":[{"id":"ms.obo","ontology_purl":"http://purl.obolibrary.org/obo/ms.obo"},{"id":"ms.owl","ontology_purl":"http://purl.obolibrary.org/obo/ms.owl"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/23482073","title":"The HUPO proteomics standards initiative- mass spectrometry controlled vocabulary."}],"repository":"https://github.com/HUPO-PSI/psi-ms-CV","title":"Mass spectrometry ontology","tracker":"https://github.com/HUPO-PSI/psi-ms-CV/issues"},{"activity_status":"active","browsers":[{"label":"BioPortal","title":"BioPortal Ontology Browser","url":"https://bioportal.bioontology.org/ontologies/NBO"}],"build":{"method":"owl2obo","source_url":"https://raw.githubusercontent.com/obo-behavior/behavior-ontology/master/nbo.owl"},"contact":{"email":"g.gkoutos@bham.ac.uk","github":"gkoutos","label":"George Gkoutos","orcid":"0000-0002-2061-091X"},"description":"An ontology of human and animal behaviours and behavioural phenotypes","domain":"behavior","homepage":"https://github.com/obo-behavior/behavior-ontology/","id":"nbo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/nbo.owl","preferredPrefix":"NBO","products":[{"id":"nbo.owl","ontology_purl":"http://purl.obolibrary.org/obo/nbo.owl"}],"repository":"https://github.com/obo-behavior/behavior-ontology","title":"Neuro Behavior Ontology","tracker":"https://github.com/obo-behavior/behavior-ontology/issues"},{"activity_status":"active","browsers":[{"label":"NCBI","title":"NCBI Taxonomy Browser","url":"http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi"}],"build":{"infallible":1,"method":"archive","path":"archive","source_url":"http://build.berkeleybop.org/job/build-ncbitaxon/lastSuccessfulBuild/artifact/*zip*/archive.zip"},"contact":{"email":"frederic.bastian@unil.ch","github":"fbastian","label":"Frederic Bastian","orcid":"0000-0002-9415-5104"},"createdWith":"https://github.com/owlcollab/owltools/","description":"An ontology representation of the NCBI organismal taxonomy","domain":"taxonomy","homepage":"https://github.com/obophenotype/ncbitaxon","id":"ncbitaxon","jobs":[{"id":"http://build.berkeleybop.org/job/build-ncbitaxon/","type":"ReleaseBuild"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ncbitaxon.owl","page":"http://www.ncbi.nlm.nih.gov/taxonomy","preferredPrefix":"NCBITaxon","products":[{"id":"ncbitaxon.owl","ontology_purl":"http://purl.obolibrary.org/obo/ncbitaxon.owl","title":"Main release"},{"id":"ncbitaxon.obo","ontology_purl":"http://purl.obolibrary.org/obo/ncbitaxon.obo","title":"OBO Format version of Main release"},{"id":"ncbitaxon.json","ontology_purl":"http://purl.obolibrary.org/obo/ncbitaxon.json","title":"OBOGraphs JSON version of Main release"},{"id":"ncbitaxon/subsets/taxslim.owl","ontology_purl":"http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl","page":"https://github.com/obophenotype/ncbitaxon/blob/master/subsets/README.md","title":"taxslim"},{"id":"ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl","ontology_purl":"http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl","page":"https://github.com/obophenotype/ncbitaxon/blob/master/subsets/README.md","title":"taxslim disjointness axioms"}],"repository":"https://github.com/obophenotype/ncbitaxon","source":"http://www.ncbi.nlm.nih.gov/taxonomy","title":"NCBI organismal classification","tracker":"https://github.com/obophenotype/ncbitaxon/issues","wasDerivedFrom":"ftp://ftp.ebi.ac.uk/pub/databases/taxonomy/taxonomy.dat"},{"activity_status":"active","contact":{"email":"haendel@ohsu.edu","github":"mellybelly","label":"Melissa Haendel","orcid":"0000-0001-9114-8737"},"description":"NCI Thesaurus (NCIt)is a reference terminology that includes broad coverage of the cancer domain, including cancer related diseases, findings and abnormalities. The NCIt OBO Edition aims to increase integration of the NCIt with OBO Library ontologies. NCIt OBO Edition releases should be considered experimental.","homepage":"https://github.com/NCI-Thesaurus/thesaurus-obo-edition","id":"ncit","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ncit.owl","preferredPrefix":"NCIT","products":[{"description":"A direct transformation of the standard NCIt content using OBO-style term and ontology IRIs and annotation properties.","id":"ncit.owl","ontology_purl":"http://purl.obolibrary.org/obo/ncit.owl","title":"NCIt OBO Edition OWL format"},{"id":"ncit.obo","ontology_purl":"http://purl.obolibrary.org/obo/ncit.obo","title":"NCIt OBO Edition OBO format"},{"description":"This version replaces NCIt terms with direct references to terms from other domain-specific OBO Library ontologies (e.g. cell types, cellular components, anatomy), supporting cross-ontology reasoning. The current release incorporates CL (cell types) and Uberon (anatomy).","id":"ncit/ncit-plus.owl","mireots_from":["cl","uberon"],"ontology_purl":"http://purl.obolibrary.org/obo/ncit/ncit-plus.owl","title":"NCIt Plus"},{"description":"This is a subset extracted from NCIt Plus, based on the [NCIt Neoplasm Core value set](https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/Neoplasm/About_Core.html) as a starting point.","id":"ncit/neoplasm-core.owl","ontology_purl":"http://purl.obolibrary.org/obo/ncit/neoplasm-core.owl","title":"NCIt Plus Neoplasm Core"}],"repository":"https://github.com/NCI-Thesaurus/thesaurus-obo-edition","title":"NCI Thesaurus OBO Edition","tracker":"https://github.com/NCI-Thesaurus/thesaurus-obo-edition/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/ncro/prebuild/ncro.owl"},"contact":{"email":"huang@southalabama.edu","github":"Huang-OMIT","label":"Jingshan Huang","orcid":"0000-0003-2408-2883"},"description":"An ontology for non-coding RNA, both of biological origin, and engineered.","domain":"experiments","homepage":"http://omnisearch.soc.southalabama.edu/w/index.php/Ontology","id":"ncro","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"mailing_list":"ncro-devel@googlegroups.com, ncro-discuss@googlegroups.com","ontology_purl":"http://purl.obolibrary.org/obo/ncro.owl","preferredPrefix":"NCRO","products":[{"id":"ncro/dev/ncro.owl","ontology_purl":"http://purl.obolibrary.org/obo/ncro/dev/ncro.owl","title":"NCRO development version"}],"repository":"https://github.com/OmniSearch/ncro","title":"Non-Coding RNA Ontology","tracker":"https://github.com/OmniSearch/ncro/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/SpeciesFileGroup/nomen.git","system":"git"},"canonical":"nomen.owl","contact":{"email":"diapriid@gmail.com","github":"mjy","label":"Matt Yoder","orcid":"0000-0002-5640-5491"},"description":"NOMEN is a nomenclatural ontology for biological names (not concepts).  It encodes the goverened rules of nomenclature.","domain":"biological nomenclature","funded_by":["NSF-ABI-1356381","Species File Group"],"homepage":"https://github.com/SpeciesFileGroup/nomen","id":"nomen","label":"NOMEN","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"mailing_list":"https://groups.google.com/forum/#!forum/nomen-discuss","ontology_purl":"http://purl.obolibrary.org/obo/nomen.owl","preferredPrefix":"NOMEN","products":[{"description":"core ontology","id":"nomen.owl","is_canonical":true,"ontology_purl":"http://purl.obolibrary.org/obo/nomen.owl","title":"NOMEN","type":"owl:Ontology"}],"repository":"https://github.com/SpeciesFileGroup/nomen","title":"NOMEN - A nomenclatural ontology for biological names","tracker":"https://github.com/SpeciesFileGroup/nomen/issues","type":"owl:Ontology","usages":[{"description":"TaxonWorks is an integrated web-based workbench for taxonomists and biodiversity scientists.","seeAlso":"https://github.com/SpeciesFileGroup/taxonworks","type":"application","user":"https://taxonworks.org"}]},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://raw.githubusercontent.com/OAE-ontology/OAE/master/src/oae_merged.owl"},"contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqunh He","orcid":"0000-0001-9189-9661"},"description":"A biomedical ontology in the domain of adverse events","domain":"adverse events, health","homepage":"https://github.com/OAE-ontology/OAE/","id":"oae","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/oae.owl","preferredPrefix":"OAE","products":[{"id":"oae.owl","ontology_purl":"http://purl.obolibrary.org/obo/oae.owl"}],"repository":"https://github.com/OAE-ontology/OAE","title":"Ontology of Adverse Events","tracker":"https://github.com/OAE-ontology/OAE/issues"},{"activity_status":"active","contact":{"email":"mjyoder@illinois.edu","github":"mjy","label":"Matt Yoder","orcid":"0000-0002-5640-5491"},"description":"OArCS is an ontology describing the Arthropod ciruclatory system.","domain":"anatomy","homepage":"https://github.com/aszool/oarcs","id":"oarcs","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/oarcs.owl","preferredPrefix":"OARCS","products":[{"id":"oarcs.owl","ontology_purl":"http://purl.obolibrary.org/obo/oarcs.owl"}],"repository":"https://github.com/aszool/oarcs","title":"Ontology of Arthropod Circulatory Systems","tracker":"https://github.com/aszool/oarcs/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/bio-attribute-ontology.git","method":"vcs","path":".","system":"git"},"contact":{"email":"cjmungall@lbl.gov","github":"cmungall","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"description":"A collection of biological attributes (traits) covering all kingdoms of life.","domain":"phenotype","homepage":"https://github.com/obophenotype/bio-attribute-ontology","id":"oba","jobs":[{"id":"https://travis-ci.org/obophenotype/bio-attribute-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/oba.owl","page":"http://wiki.geneontology.org/index.php/Extensions/x-attribute","preferredPrefix":"OBA","products":[{"id":"oba.owl","ontology_purl":"http://purl.obolibrary.org/obo/oba.owl"},{"id":"oba.obo","ontology_purl":"http://purl.obolibrary.org/obo/oba.obo"},{"id":"oba/subsets/oba-basic.obo","ontology_purl":"http://purl.obolibrary.org/obo/oba/subsets/oba-basic.obo"}],"repository":"https://github.com/obophenotype/bio-attribute-ontology","title":"Ontology of Biological Attributes","tracker":"https://github.com/obophenotype/bio-attribute-ontology/issues"},{"activity_status":"active","contact":{"email":"jiezheng@pennmedicine.upenn.edu","github":"zhengj2007","label":"Jie Zheng","orcid":"0000-0002-2999-0103"},"description":"A biomedical ontology in the domain of biological and clinical statistics.","domain":"statistics","homepage":"https://github.com/obcs/obcs","id":"obcs","in_foundry":false,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/obcs.owl","preferredPrefix":"OBCS","products":[{"id":"obcs.owl","ontology_purl":"http://purl.obolibrary.org/obo/obcs.owl"}],"repository":"https://github.com/obcs/obcs","title":"Ontology of Biological and Clinical Statistics","tracker":"https://github.com/obcs/obcs/issues"},{"activity_status":"active","contact":{"email":"jiezheng@pennmedicine.upenn.edu","github":"zhengj2007","label":"Jie Zheng","orcid":"0000-0002-2999-0103"},"description":"An ontology built for annotation and modeling of biobank repository and biobanking administration","domain":"biobanking, specimens, bio-repository, biocuration","homepage":"https://github.com/biobanking/biobanking","id":"obib","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/obib.owl","preferredPrefix":"OBIB","products":[{"id":"obib.owl","ontology_purl":"http://purl.obolibrary.org/obo/obib.owl"}],"repository":"https://github.com/biobanking/biobanking","title":"Ontology for Biobanking","tracker":"https://github.com/biobanking/biobanking/issues","usages":[{"description":"TURBO ontology supporting the PennTURBO project.","examples":[{"description":"Blood draw time","url":"http://purl.obolibrary.org/obo/OBIB_0000079"}],"user":"https://github.com/PennTURBO/turbo-ontology"},{"description":"The Minimum Information About BIobank data Sharing (MIABIS) aims to standardize data elements used to describe biobanks, research on samples and associated data. General attributes to describe biobanks, sample collections and studies at an aggregated/metadata level are defined in MIABIS Core 2.0 (Merino-Martinez et al., 2016).","user":"https://github.com/MIABIS/miabis"},{"description":"The National Cancer Institute Biorepositories and Biospecimen Research Branch (BBRB) is an international leader in research and policy activities related to biospecimen collection, processing, and storage, also known as biobanking.","user":"https://biospecimens.cancer.gov/resources/vocabularies.asp"}]},{"activity_status":"active","contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqun Oliver He","orcid":"0000-0001-9189-9661"},"description":"A formal ontology of genes and genomes of biological organisms.","homepage":"https://bitbucket.org/hegroup/ogg","id":"ogg","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ogg.owl","preferredPrefix":"OGG","products":[{"id":"ogg.owl","ontology_purl":"http://purl.obolibrary.org/obo/ogg.owl"}],"repository":"https://bitbucket.org/hegroup/ogg","title":"The Ontology of Genes and Genomes","tracker":"https://bitbucket.org/hegroup/ogg/issues/"},{"activity_status":"active","contact":{"email":"baeverma@jcvi.org","github":"BAevermann","label":"Brian Aevermann","orcid":"0000-0003-1346-1327"},"depicted_by":"https://avatars2.githubusercontent.com/u/12973154?s=200&v=4","description":"An ontology for representing treatment of disease and diagnosis and on carcinomas and other pathological entities","domain":"medicine","homepage":"https://github.com/OGMS/ogms","id":"ogms","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ogms.owl","preferredPrefix":"OGMS","products":[{"id":"ogms.owl","ontology_purl":"http://purl.obolibrary.org/obo/ogms.owl"}],"repository":"https://github.com/OGMS/ogms","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Ontology for General Medical Science","tracker":"https://github.com/OGMS/ogms/issues"},{"activity_status":"active","contact":{"email":"linikujp@gmail.com","github":"linikujp","label":"Asiyah Yu Lin","orcid":"0000-0002-5379-5359"},"description":"An application ontology to represent genetic susceptibility to a specific disease, adverse event, or a pathological process.","homepage":"https://github.com/linikujp/OGSF","id":"ogsf","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ogsf.owl","preferredPrefix":"OGSF","products":[{"id":"ogsf.owl","ontology_purl":"http://purl.obolibrary.org/obo/ogsf.owl"}],"repository":"https://github.com/linikujp/OGSF","title":"Ontology of Genetic Susceptibility Factor","tracker":"https://github.com/linikujp/OGSF/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/ohd.owl"},"contact":{"email":"wdduncan@gmail.com","github":"wdduncan","label":"Bill Duncan","orcid":"0000-0001-9625-1899"},"description":"The Oral Health and Disease Ontology was created, initially, to represent the content of dental practice health records.","domain":"health","homepage":"https://purl.obolibrary.org/obo/ohd/home","id":"ohd","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ohd.owl","preferredPrefix":"OHD","products":[{"id":"ohd.owl","ontology_purl":"http://purl.obolibrary.org/obo/ohd.owl"},{"id":"ohd/dev/ohd.owl","ontology_purl":"http://purl.obolibrary.org/obo/ohd/dev/ohd.owl","title":"OHD dev"}],"repository":"https://github.com/oral-health-and-disease-ontologies/ohd-ontology","title":"Oral Health and Disease Ontology","tracker":"https://github.com/oral-health-and-disease-ontologies/ohd-ontology/issues"},{"activity_status":"active","contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqun Oliver He","orcid":"0000-0001-9189-9661"},"description":"The Ontology of Host-Microbiome Interactions aims to ontologically represent and standardize various entities and relations related to microbiomes, microbiome host organisms (e.g., human and mouse), and the interactions between the hosts and microbiomes at different conditions.","homepage":"https://github.com/ohmi-ontology/ohmi","id":"ohmi","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"http://groups.google.com/group/ohmi-discuss","ontology_purl":"http://purl.obolibrary.org/obo/ohmi.owl","preferredPrefix":"OHMI","products":[{"id":"ohmi.owl","ontology_purl":"http://purl.obolibrary.org/obo/ohmi.owl"}],"repository":"https://github.com/ohmi-ontology/ohmi","title":"Ontology of Host-Microbiome Interactions","tracker":"https://github.com/ohmi-ontology/ohmi/issues"},{"activity_status":"active","contact":{"email":"edong@umich.edu","github":"e4ong1031","label":"Edison Ong","orcid":"0000-0002-5159-414X"},"description":"OHPI is a community-driven ontology of host-pathogen interactions (OHPI) and represents the virulence factors (VFs) and how the mutants of VFs in the Victors database become less virulence inside a host organism or host cells. It is developed to represent manually curated HPI knowledge available in the PHIDIAS resource.","homepage":"https://github.com/OHPI/ohpi","id":"ohpi","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"http://groups.google.com/group/ohpi-discuss","ontology_purl":"http://purl.obolibrary.org/obo/ohpi.owl","preferredPrefix":"OHPI","products":[{"id":"ohpi.owl","ontology_purl":"http://purl.obolibrary.org/obo/ohpi.owl"}],"repository":"https://github.com/OHPI/ohpi","title":"Ontology of Host Pathogen Interactions","tracker":"https://github.com/OHPI/ohpi/issues"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/olatdv/olatdv.obo"},"contact":{"email":"frederic.bastian@unil.ch","github":"fbastian","label":"Frdric Bastian","orcid":"0000-0002-9415-5104"},"description":"Life cycle stages for Medaka","homepage":"https://github.com/obophenotype/developmental-stage-ontologies/wiki/OlatDv","id":"olatdv","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/olatdv.owl","page":"https://github.com/obophenotype/developmental-stage-ontologies","preferredPrefix":"OlatDv","products":[{"id":"olatdv.obo","ontology_purl":"http://purl.obolibrary.org/obo/olatdv.obo"},{"id":"olatdv.owl","ontology_purl":"http://purl.obolibrary.org/obo/olatdv.owl"}],"repository":"https://github.com/obophenotype/developmental-stage-ontologies","title":"Medaka Developmental Stages","tracker":"https://github.com/obophenotype/developmental-stage-ontologies/issues"},{"activity_status":"active","contact":{"email":"huang@southalabama.edu","github":"Huang-OMIT","label":"Jingshan Huang","orcid":"0000-0003-2408-2883"},"description":"Ontology to establish data exchange standards and common data elements in the microRNA (miR) domain","homepage":"http://omit.cis.usouthal.edu/","id":"omit","in_foundry":false,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/omit.owl","page":"http://omit.cis.usouthal.edu/","preferredPrefix":"OMIT","products":[{"id":"omit.owl","ontology_purl":"http://purl.obolibrary.org/obo/omit.owl"}],"repository":"https://github.com/OmniSearch/omit","title":"Ontology for MIRNA Target","tracker":"https://github.com/OmniSearch/omit/issues"},{"activity_status":"active","contact":{"email":"cjmungall@lbl.gov","github":"cmungall","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"description":"An ontology specifies terms that are used to annotate ontology terms for all OBO ontologies. The ontology was developed as part of Information Artifact Ontology (IAO).","domain":"ontology term annotation","homepage":"https://github.com/information-artifact-ontology/ontology-metadata","id":"omo","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/omo.owl","preferredPrefix":"OMO","products":[{"id":"omo.owl","ontology_purl":"http://purl.obolibrary.org/obo/omo.owl"}],"repository":"https://github.com/information-artifact-ontology/ontology-metadata","title":"OBO Metadata Ontology","tracker":"https://github.com/information-artifact-ontology/ontology-metadata/issues","usages":[{"description":"OMO is imported by multiple OBO ontologies for ontology term annotations.","type":"owl_import","user":"http://obofoundry.org"}]},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/microbialphenotypes/OMP-ontology-files/master/omp.obo"},"contact":{"email":"jimhu@tamu.edu","github":"jimhu-tamu","label":"James C. Hu","orcid":"0000-0001-9016-2684"},"description":"An ontology of phenotypes covering microbes","domain":"phenotype","homepage":"http://microbialphenotypes.org","id":"omp","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/omp.owl","preferredPrefix":"OMP","products":[{"id":"omp.owl","ontology_purl":"http://purl.obolibrary.org/obo/omp.owl"},{"id":"omp.obo","ontology_purl":"http://purl.obolibrary.org/obo/omp.obo"}],"repository":"https://github.com/microbialphenotypes/OMP-ontology","title":"Ontology of Microbial Phenotypes","tracker":"https://github.com/microbialphenotypes/OMP-ontology/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://github.com/ufbmi/OMRSE"},"contact":{"email":"hoganwr@gmail.com","github":"hoganwr","label":"Bill Hogan","orcid":"0000-0002-9881-1017"},"description":"This ontology covers the domain of social entities that are related to health care, such as demographic information and the roles of various individuals and organizations.","domain":"medicine","homepage":"https://github.com/ufbmi/OMRSE/wiki/OMRSE-Overview","id":"omrse","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/omrse.owl","preferredPrefix":"OMRSE","products":[{"id":"omrse.owl","ontology_purl":"http://purl.obolibrary.org/obo/omrse.owl"}],"repository":"https://github.com/ufbmi/OMRSE","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Ontology of Medically Related Social Entities","tracker":"https://github.com/ufbmi/OMRSE/issues"},{"activity_status":"active","contact":{"email":"chenyangnutrition@gmail.com","github":"cyang0128","label":"Chen Yang","orcid":"0000-0001-9202-5309"},"dependencies":[{"id":"ons"},{"id":"obi"},{"id":"foodon"}],"description":"An ontology to standardize research output of nutritional epidemiologic studies.","domain":"nutritional epidemiology, observational studies, dietary surveys","homepage":"https://github.com/cyang0128/Nutritional-epidemiologic-ontologies","id":"one","label":"Ontology for Nutritional Epidemiology","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/one.owl","page":"https://github.com/cyang0128/Nutritional-epidemiologic-ontologies","preferredPrefix":"ONE","products":[{"id":"one.owl","ontology_purl":"http://purl.obolibrary.org/obo/one.owl"}],"repository":"https://github.com/cyang0128/Nutritional-epidemiologic-ontologies","title":"Ontology for Nutritional Epidemiology","tracker":"https://github.com/cyang0128/Nutritional-epidemiologic-ontologies/issues"},{"activity_status":"active","contact":{"email":"francesco.vitali@ibba.cnr.it","github":"FrancescoVit","label":"Francesco Vitali","orcid":"0000-0001-9125-4337"},"dependencies":[{"id":"uberon"},{"id":"ro"},{"id":"chebi"},{"id":"ncbitaxon"},{"id":"bfo"},{"id":"envo"},{"id":"obi"},{"id":"foodon"}],"description":"An ontology for description of concepts in the nutritional studies domain.","domain":"nutrition, nutritional studies, nutrition professionals","homepage":"https://github.com/enpadasi/Ontology-for-Nutritional-Studies","id":"ons","label":"Ontology for Nutritional Studies","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ons.owl","page":"https://github.com/enpadasi/Ontology-for-Nutritional-Studies","preferredPrefix":"ONS","products":[{"id":"ons.owl","ontology_purl":"http://purl.obolibrary.org/obo/ons.owl","title":"ONS latest release"}],"repository":"https://github.com/enpadasi/Ontology-for-Nutritional-Studies","title":"Ontology for Nutritional Studies","tracker":"https://github.com/enpadasi/Ontology-for-Nutritional-Studies/issues"},{"activity_status":"active","contact":{"email":"fortuna@ebd.csic.es","github":"miguelfortuna","label":"Miguel A. Fortuna","orcid":"0000-0002-8374-1941"},"dependencies":[{"id":"fbcv"},{"id":"gsso"},{"id":"stato"},{"id":"ro"},{"id":"ncit"}],"description":"OntoAvida develops an integrated vocabulary for the description of the most widely-used computational approach for studying evolution using digital organisms (i.e., self-replicating computer programs that evolve within a user-defined computational environment).","domain":"digital evolution, a branch of Artificial Life.","homepage":"https://gitlab.com/fortunalab/ontoavida","id":"ontoavida","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ontoavida.owl","preferredPrefix":"ONTOAVIDA","products":[{"id":"ontoavida.owl","ontology_purl":"http://purl.obolibrary.org/obo/ontoavida.owl"},{"id":"ontoavida.obo","ontology_purl":"http://purl.obolibrary.org/obo/ontoavida.obo"}],"repository":"https://gitlab.com/fortunalab/ontoavida","title":"OntoAvida: ontology for Avida digital evolution platform","tracker":"https://gitlab.com/fortunalab/ontoavida/-/issues"},{"activity_status":"active","browsers":[{"label":"BioPortal","title":"BioPortal Browser","url":"https://bioportal.bioontology.org/ontologies/ONTONEO"}],"build":{"source_url":"http://purl.obolibrary.org/obo/ontoneo/ontoneo.owl"},"contact":{"email":"fernanda.farinelli@gmail.com","github":"FernandaFarinelli","label":"Fernanda Farinelli","orcid":"0000-0003-2338-8872"},"description":"The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.","domain":"biomedical, health","homepage":"ontoneo.com","id":"ontoneo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"mailing_list":"http://groups.google.com/group/ontoneo-discuss","ontology_purl":"http://purl.obolibrary.org/obo/ontoneo.owl","preferredPrefix":"ONTONEO","products":[{"id":"ontoneo.owl","ontology_purl":"http://purl.obolibrary.org/obo/ontoneo.owl"}],"repository":"https://github.com/ontoneo-project/Ontoneo","title":"Obstetric and Neonatal Ontology","tracker":"https://github.com/ontoneo-project/Ontoneo/issues"},{"activity_status":"active","contact":{"email":"mbrochhausen@gmail.com","github":"mbrochhausen","label":"Mathias Brochhausen","orcid":"0000-0003-1834-3856"},"description":"An ontology built for representating the organizational components of trauma centers and trauma systems.","domain":"health","homepage":"https://github.com/OOSTT/OOSTT","id":"oostt","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/oostt.owl","preferredPrefix":"OOSTT","products":[{"id":"oostt.owl","ontology_purl":"http://purl.obolibrary.org/obo/oostt.owl"}],"repository":"https://github.com/OOSTT/OOSTT","title":"Ontology of Organizational Structures of Trauma centers and Trauma systems","tracker":"https://github.com/OOSTT/OOSTT/issues"},{"activity_status":"active","contact":{"email":"jiezheng@pennmedicine.upenn.edu","github":"zhengj2007","label":"Jie Zheng","orcid":"0000-0002-2999-0103"},"description":"A reference ontology for parasite life cycle stages.","domain":"life cycle stage, parasite organism","homepage":"https://github.com/OPL-ontology/OPL","id":"opl","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/opl.owl","preferredPrefix":"OPL","products":[{"id":"opl.owl","ontology_purl":"http://purl.obolibrary.org/obo/opl.owl"}],"repository":"https://github.com/OPL-ontology/OPL","title":"Ontology for Parasite LifeCycle","tracker":"https://github.com/OPL-ontology/OPL/issues","usages":[{"description":"The ontology for parasite lifecycle is used in the VEuPathDB (Eukaryotic Pathogen, Vector & Host Informatics Resources) for parasite life cycle annotation.","type":"annotation and query","user":"https://veupathdb.org"},{"description":"The ontology for parasite lifecycle is used in the GeneDB for parasite life cycle annotation.","type":"annotation and query","user":"https://www.genedb.org"}]},{"activity_status":"active","contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqun Oliver He","orcid":"0000-0001-9189-9661"},"description":"The Ontology of Precision Medicine and Investigation (OPMI) aims to ontologically represent and standardize various entities and relations associated with precision medicine and related investigations at different conditions.","homepage":"https://github.com/OPMI/opmi","id":"opmi","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"mailing_list":"http://groups.google.com/group/opmi-discuss","ontology_purl":"http://purl.obolibrary.org/obo/opmi.owl","preferredPrefix":"OPMI","products":[{"id":"opmi.owl","ontology_purl":"http://purl.obolibrary.org/obo/opmi.owl"}],"repository":"https://github.com/OPMI/opmi","title":"Ontology of Precision Medicine and Investigation","tracker":"https://github.com/OPMI/opmi/issues"},{"activity_status":"active","contact":{"email":"safisher@upenn.edu","github":"safisher","label":"Stephen Fisher","orcid":"0000-0001-8034-7685"},"description":"An application ontology designed to annotate next-generation sequencing experiments performed on RNA.","domain":"experiments","homepage":"http://kim.bio.upenn.edu/software/ornaseq.shtml","id":"ornaseq","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ornaseq.owl","preferredPrefix":"ORNASEQ","products":[{"id":"ornaseq.owl","ontology_purl":"http://purl.obolibrary.org/obo/ornaseq.owl"}],"repository":"https://github.com/safisher/ornaseq","title":"Ontology of RNA Sequencing","tracker":"https://github.com/safisher/ornaseq/issues"},{"activity_status":"active","contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqunh He","orcid":"0000-0001-9189-9661"},"description":"A biomedical ontology in the domain of vaccine adverse events.","homepage":"http://www.violinet.org/ovae/","id":"ovae","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ovae.owl","preferredPrefix":"OVAE","products":[{"id":"ovae.owl","ontology_purl":"http://purl.obolibrary.org/obo/ovae.owl"}],"repository":"https://github.com/OVAE-Ontology/ovae","title":"Ontology of Vaccine Adverse Events","tracker":"https://github.com/OVAE-Ontology/ovae/issues"},{"activity_status":"active","contact":{"email":"davidos@ebi.ac.uk","github":"dosumis","label":"David Osumi-Sutherland","orcid":"0000-0002-7073-9172"},"dependencies":[{"id":"pr"},{"id":"go"},{"id":"uberon"},{"id":"ro"},{"id":"pato"},{"id":"ncbitaxon"},{"id":"bfo"},{"id":"cl"},{"id":"omo"},{"id":"nbo"},{"id":"chebi"},{"id":"so"}],"description":"Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties.","domain":"phenotype","homepage":"https://github.com/obophenotype/provisional_cell_ontology","id":"pcl","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pcl.owl","preferredPrefix":"PCL","products":[{"id":"pcl.owl","ontology_purl":"http://purl.obolibrary.org/obo/pcl.owl"},{"id":"pcl.obo","ontology_purl":"http://purl.obolibrary.org/obo/pcl.obo"},{"id":"pcl.json","ontology_purl":"http://purl.obolibrary.org/obo/pcl.json"},{"id":"pcl-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/pcl-base.owl"},{"id":"pcl-base.obo","ontology_purl":"http://purl.obolibrary.org/obo/pcl-base.obo"},{"id":"pcl-base.json","ontology_purl":"http://purl.obolibrary.org/obo/pcl-base.json"},{"id":"pcl-full.owl","ontology_purl":"http://purl.obolibrary.org/obo/pcl-full.owl"},{"id":"pcl-full.obo","ontology_purl":"http://purl.obolibrary.org/obo/pcl-full.obo"},{"id":"pcl-full.json","ontology_purl":"http://purl.obolibrary.org/obo/pcl-full.json"},{"id":"pcl-simple.owl","ontology_purl":"http://purl.obolibrary.org/obo/pcl-simple.owl"},{"id":"pcl-simple.obo","ontology_purl":"http://purl.obolibrary.org/obo/pcl-simple.obo"},{"id":"pcl-simple.json","ontology_purl":"http://purl.obolibrary.org/obo/pcl-simple.json"}],"repository":"https://github.com/obophenotype/provisional_cell_ontology","title":"Provisional Cell Ontology","tracker":"https://github.com/obophenotype/provisional_cell_ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/PopulationAndCommunityOntology/pco.git","path":".","system":"git"},"contact":{"email":"rlwalls2008@gmail.com","github":"ramonawalls","label":"Ramona Walls","orcid":"0000-0001-8815-0078"},"dependencies":[{"id":"ro"},{"id":"bfo"},{"id":"pato"},{"id":"envo"},{"id":"go"},{"id":"iao"},{"id":"bfo"},{"id":"ncbitaxon"},{"id":"caro"}],"description":"An ontology about groups of interacting organisms such as populations and communities","domain":"collections of organisms","homepage":"https://github.com/PopulationAndCommunityOntology/pco","id":"pco","jobs":[{"id":"https://travis-ci.org/PopulationAndCommunityOntology/pco","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pco.owl","preferredPrefix":"PCO","products":[{"id":"pco.owl","ontology_purl":"http://purl.obolibrary.org/obo/pco.owl"}],"repository":"https://github.com/PopulationAndCommunityOntology/pco","title":"Population and Community Ontology","tracker":"https://github.com/PopulationAndCommunityOntology/pco/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/pdro.owl"},"contact":{"email":"paul.fabry@usherbrooke.ca","github":"pfabry","label":"Paul Fabry","orcid":"0000-0002-3336-2476"},"description":"An ontology to describe entities related to prescription of drugs","domain":"health","homepage":"https://github.com/OpenLHS/PDRO","id":"pdro","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pdro.owl","preferredPrefix":"PDRO","products":[{"id":"pdro.owl","ontology_purl":"http://purl.obolibrary.org/obo/pdro.owl"}],"repository":"https://github.com/OpenLHS/PDRO","title":"The Prescription of Drugs Ontology","tracker":"https://github.com/OpenLHS/PDRO/issues"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/pdumdv/pdumdv.obo"},"contact":{"email":"frederic.bastian@unil.ch","github":"fbastian","label":"Frdric Bastian","orcid":"0000-0002-9415-5104"},"description":"Life cycle stages for Platynereis dumerilii","homepage":"https://github.com/obophenotype/developmental-stage-ontologies/wiki/PdumDv","id":"pdumdv","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pdumdv.owl","page":"https://github.com/obophenotype/developmental-stage-ontologies","preferredPrefix":"PdumDv","products":[{"id":"pdumdv.owl","ontology_purl":"http://purl.obolibrary.org/obo/pdumdv.owl"},{"id":"pdumdv.obo","ontology_purl":"http://purl.obolibrary.org/obo/pdumdv.obo"}],"repository":"https://github.com/obophenotype/developmental-stage-ontologies","title":"Platynereis Developmental Stages","tracker":"https://github.com/obophenotype/developmental-stage-ontologies/issues"},{"activity_status":"active","contact":{"email":"jaiswalp@science.oregonstate.edu","github":"jaiswalp","label":"Pankaj Jaiswal","orcid":"0000-0002-1005-8383"},"description":"A structured, controlled vocabulary which describes the treatments, growing conditions, and/or study types used in plant biology experiments.","domain":"experimental conditions","homepage":"http://planteome.org/","id":"peco","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/peco.owl","page":"http://browser.planteome.org/amigo/term/PECO:0007359","preferredPrefix":"PECO","products":[{"id":"peco.owl","ontology_purl":"http://purl.obolibrary.org/obo/peco.owl"},{"id":"peco.obo","ontology_purl":"http://purl.obolibrary.org/obo/peco.obo"}],"publications":[{"id":"https://doi.org/10.1093/nar/gkx1152","title":"The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics."}],"repository":"https://github.com/Planteome/plant-experimental-conditions-ontology","title":"Plant Experimental Conditions Ontology","tracker":"https://github.com/Planteome/plant-experimental-conditions-ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/PHI-base/phipo.git","path":".","system":"git"},"contact":{"email":"alayne.cuzick@rothamsted.ac.uk","github":"CuzickA","label":"Alayne Cuzick","orcid":"0000-0001-8941-3984"},"dependencies":[{"id":"pato"}],"description":"PHIPO is a formal ontology of species-neutral phenotypes observed in pathogen-host interactions.","domain":"phenotype","homepage":"https://github.com/PHI-base/phipo","id":"phipo","jobs":[{"id":"https://travis-ci.org/PHI-base/phipo","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/phipo.owl","preferredPrefix":"PHIPO","products":[{"id":"phipo.owl","ontology_purl":"http://purl.obolibrary.org/obo/phipo.owl"},{"id":"phipo.obo","ontology_purl":"http://purl.obolibrary.org/obo/phipo.obo"}],"repository":"https://github.com/PHI-base/phipo","title":"Pathogen Host Interaction Phenotype Ontology","tracker":"https://github.com/PHI-base/phipo/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/planaria-ontology.git","path":".","system":"git"},"contact":{"email":"smr@stowers.org","github":"srobb1","label":"Sofia Robb","orcid":"0000-0002-3528-5267"},"dependencies":[{"id":"ro"},{"id":"uberon"}],"description":"PLANA, the planarian anatomy ontology, encompasses the anatomy and life cycle stages for both __Schmidtea mediterranea__ biotypes.","domain":"anatomy","homepage":"https://github.com/obophenotype/planaria-ontology","id":"plana","jobs":[{"id":"https://travis-ci.org/obophenotype/planaria-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/plana.owl","preferredPrefix":"PLANA","products":[{"id":"plana.owl","ontology_purl":"http://purl.obolibrary.org/obo/plana.owl"},{"id":"plana.obo","ontology_purl":"http://purl.obolibrary.org/obo/plana.obo"}],"repository":"https://github.com/obophenotype/planaria-ontology","title":"planaria-ontology","tracker":"https://github.com/obophenotype/planaria-ontology/issues","usages":[{"description":"Planosphere\'s PAGE database uses PLANA to annotate gene expression locations","examples":[{"description":"The user can get an overview of the genes expressed in the planarian epidermis","url":"https://planosphere.stowers.org/ontology/PLANA_0000034"}],"user":"https://planosphere.stowers.org/"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/planarian-phenotype-ontology.git","path":".","system":"git"},"contact":{"email":"smr@stowers.org","github":"srobb1","label":"Sofia Robb","orcid":"0000-0002-3528-5267"},"dependencies":[{"id":"ro"},{"id":"pato"},{"id":"plana"},{"id":"go"}],"description":"Planarian Phenotype Ontology is an ontology of phenotypes observed in the planarian Schmidtea mediterranea.","domain":"phenotype","homepage":"https://github.com/obophenotype/planarian-phenotype-ontology","id":"planp","jobs":[{"id":"https://travis-ci.org/obophenotype/planarian-phenotype-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/planp.owl","preferredPrefix":"PLANP","products":[{"id":"planp.owl","ontology_purl":"http://purl.obolibrary.org/obo/planp.owl"},{"id":"planp.obo","ontology_purl":"http://purl.obolibrary.org/obo/planp.obo"}],"repository":"https://github.com/obophenotype/planarian-phenotype-ontology","title":"Planarian Phenotype Ontology","tracker":"https://github.com/obophenotype/planarian-phenotype-ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/porifera-ontology.git","infallible":1,"method":"vcs","path":".","system":"git"},"contact":{"email":"robert.thacker@stonybrook.edu","github":"bobthacker","label":"Bob Thacker","orcid":"0000-0002-9654-0073"},"dependencies":[{"id":"uberon"},{"id":"ro"}],"description":"An ontology covering the anatomy of the taxon Porifera (sponges)","domain":"anatomy","homepage":"https://github.com/obophenotype/porifera-ontology","id":"poro","jobs":[{"id":"https://travis-ci.org/obophenotype/porifera-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/poro.owl","preferredPrefix":"PORO","products":[{"id":"poro.owl","ontology_purl":"http://purl.obolibrary.org/obo/poro.owl"},{"id":"poro.obo","ontology_purl":"http://purl.obolibrary.org/obo/poro.obo"}],"publications":[{"id":"https://doi.org/10.1186/2041-1480-5-39","title":"The Porifera Ontology (PORO): enhancing sponge systematics with an anatomy ontology"}],"repository":"https://github.com/obophenotype/porifera-ontology","taxon":{"id":"NCBITaxon:6040","label":"Porifera"},"title":"Porifera Ontology","tracker":"https://github.com/obophenotype/porifera-ontology/issues"},{"activity_status":"active","contact":{"email":"rlwalls2008@gmail.com","github":"ramonawalls","label":"Ramona Walls","orcid":"0000-0001-8815-0078"},"description":"An ontology for describing the phenology of individual plants and populations of plants, and for integrating plant phenological data across sources and scales.","domain":"plant phenotypes","homepage":"https://github.com/PlantPhenoOntology/PPO","id":"ppo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"mailing_list":"ppo-discuss@googlegroups.com","ontology_purl":"http://purl.obolibrary.org/obo/ppo.owl","preferredPrefix":"PPO","products":[{"id":"ppo.owl","ontology_purl":"http://purl.obolibrary.org/obo/ppo.owl"}],"repository":"https://github.com/PlantPhenoOntology/PPO","taxon":{"id":"NCBITaxon:33090","label":"Viridiplantae"},"title":"Plant Phenology Ontology","tracker":"https://github.com/PlantPhenoOntology/PPO/issues"},{"activity_status":"active","build":{"checkout":"https://github.com/Display-Lab/psdo.git","path":".","system":"git"},"contact":{"email":"zachll@umich.edu","github":"zachll","label":"Zach Landis-Lewis","orcid":"0000-0002-9117-9338"},"dependencies":[{"id":"stato"},{"id":"iao"},{"id":"bfo"},{"id":"ro"}],"description":"Ontology to reproducibly study visualizations of clinical performance","domain":"learning systems","homepage":"https://github.com/Display-Lab/psdo","id":"psdo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/psdo.owl","preferredPrefix":"PSDO","products":[{"id":"psdo.owl","ontology_purl":"http://purl.obolibrary.org/obo/psdo.owl"}],"repository":"https://github.com/Display-Lab/psdo","title":"Performance Summary Display Ontology","tracker":"https://github.com/Display-Lab/psdo/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/Planteome/plant-stress-ontology.git","path":".","system":"git"},"contact":{"email":"cooperl@science.oregonstate.edu","github":"cooperl09","label":"Laurel Cooper","orcid":"0000-0002-6379-8932"},"dependencies":[{"id":"ro"}],"description":"The Plant Stress Ontology describes...","domain":"plant disease and abiotic stress","homepage":"https://github.com/Planteome/plant-stress-ontology","id":"pso","jobs":[{"id":"https://travis-ci.org/Planteome/plant-stress-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pso.owl","preferredPrefix":"PSO","products":[{"id":"pso.owl","ontology_purl":"http://purl.obolibrary.org/obo/pso.owl"},{"id":"pso.obo","ontology_purl":"http://purl.obolibrary.org/obo/pso.obo"}],"repository":"https://github.com/Planteome/plant-stress-ontology","title":"Plant Stress Ontology","tracker":"https://github.com/Planteome/plant-stress-ontology/issues"},{"activity_status":"active","browsers":[{"label":"RGD","title":"RGD Ontology Browser","url":"http://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=PW:0000001"}],"build":{"method":"obo2owl","source_url":"https://download.rgd.mcw.edu/ontology/pathway/pathway.obo"},"contact":{"email":"gthayman@mcw.edu","github":"gthayman","label":"G. Thomas Hayman","orcid":"0000-0002-9553-7227"},"description":"A controlled vocabulary for annotating gene products to pathways.","domain":"biological process","homepage":"http://rgd.mcw.edu/rgdweb/ontology/search.html","id":"pw","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/pw.owl","page":"https://download.rgd.mcw.edu/ontology/pathway/","preferredPrefix":"PW","products":[{"id":"pw.owl","ontology_purl":"http://purl.obolibrary.org/obo/pw.owl"},{"id":"pw.obo","ontology_purl":"http://purl.obolibrary.org/obo/pw.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/21478484","title":"The Rat Genome Database pathway portal."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24499703","title":"The pathway ontology - updates and applications."}],"repository":"https://github.com/rat-genome-database/PW-Pathway-Ontology","title":"Pathway ontology","tracker":"https://github.com/rat-genome-database/PW-Pathway-Ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/Radiobiology-Informatics-Consortium/RBO.git","path":".","system":"git"},"contact":{"email":"daniel.c.berrios@nasa.gov","github":"DanBerrios","label":"Daniel C. Berrios","orcid":"0000-0003-4312-9552"},"dependencies":[{"id":"ro"},{"id":"bfo"},{"id":"pato"},{"id":"chmo"},{"id":"envo"},{"id":"obi"},{"id":"uo"}],"description":"RBO is an ontology for the effects of radiation on biota in terrestrial and space environments.","domain":"radiation biology, the study of the effects of radiation on biological systems","homepage":"https://github.com/Radiobiology-Informatics-Consortium/RBO","id":"rbo","jobs":[{"id":"https://travis-ci.org/DanBerrios/RBO","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/rbo.owl","preferredPrefix":"RBO","products":[{"id":"rbo.owl","name":"Radiation Biology Ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/rbo.owl"},{"id":"rbo.obo","name":"Radiation Biology Ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/rbo.obo"},{"id":"rbo.json","name":"Radiation Biology Ontology additional release in OBOJSon format","ontology_purl":"http://purl.obolibrary.org/obo/rbo.json"},{"id":"rbo/rbo-base.owl","name":"Radiation Biology Ontology main release in OWL format","ontology_purl":"http://purl.obolibrary.org/obo/rbo/rbo-base.owl"},{"id":"rbo/rbo-base.obo","name":"Radiation Biology Ontology additional release in OBO format","ontology_purl":"http://purl.obolibrary.org/obo/rbo/rbo-base.obo"},{"id":"rbo/rbo-base.json","name":"Radiation Biology Ontology additional release in OBOJSon format","ontology_purl":"http://purl.obolibrary.org/obo/rbo/rbo-base.json"}],"repository":"https://github.com/Radiobiology-Informatics-Consortium/RBO","title":"Radiation Biology Ontology","tracker":"https://github.com/Radiobiology-Informatics-Consortium/RBO/issues"},{"aberowl_id":"relo","activity_status":"active","build":{"checkout":"git clone https://github.com/oborel/obo-relations.git","infallible":1,"method":"vcs","system":"git"},"canonical":"ro.owl","contact":{"email":"cjmungall@lbl.gov","github":"cmungall","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"description":"Relationship types shared across multiple ontologies","documentation":"https://oborel.github.io/obo-relations/","domain":"relations","homepage":"https://oborel.github.io/","id":"ro","jobs":[{"id":"https://travis-ci.org/oborel/obo-relations","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"mailing_list":"https://groups.google.com/forum/#!forum/obo-relations","ontology_purl":"http://purl.obolibrary.org/obo/ro.owl","preferredPrefix":"RO","products":[{"description":"Canonical edition","id":"ro.owl","ontology_purl":"http://purl.obolibrary.org/obo/ro.owl","title":"Relation Ontology"},{"description":"The obo edition is less expressive than the OWL, and has imports merged in","id":"ro.obo","ontology_purl":"http://purl.obolibrary.org/obo/ro.obo","title":"Relation Ontology in obo format"},{"id":"ro.json","ontology_purl":"http://purl.obolibrary.org/obo/ro.json","title":"Relation Ontology in obojson format"},{"description":"Minimal subset intended to work with BFO-classes","id":"ro/core.owl","ontology_purl":"http://purl.obolibrary.org/obo/ro/core.owl","page":"https://github.com/oborel/obo-relations/wiki/ROCore","title":"RO Core relations"},{"description":"Axioms defined within RO and to be used in imports for other ontologies","id":"ro/ro-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/ro/ro-base.owl","page":"https://github.com/INCATools/ontology-development-kit/issues/50","title":"RO base ontology"},{"description":"For use in ecology and environmental science","id":"ro/subsets/ro-interaction.owl","ontology_purl":"http://purl.obolibrary.org/obo/ro/subsets/ro-interaction.owl","title":"Interaction relations"},{"id":"ro/subsets/ro-eco.owl","ontology_purl":"http://purl.obolibrary.org/obo/ro/subsets/ro-eco.owl","title":"Ecology subset"},{"description":"For use in neuroscience","id":"ro/subsets/ro-neuro.owl","ontology_purl":"http://purl.obolibrary.org/obo/ro/subsets/ro-neuro.owl","page":"http://bioinformatics.oxfordjournals.org/content/28/9/1262.long","title":"Neuroscience subset"}],"repository":"https://github.com/oborel/obo-relations","title":"Relation Ontology","tracker":"https://github.com/oborel/obo-relations/issues","usages":[{"description":"RO is used for annotation extensions in the GO and GO Causal Activity Models.","examples":[{"description":"wg_biogenesis_FlyBase","url":"http://model.geneontology.org/56d1143000003402"}],"publications":[{"id":"https://doi.org/10.1186/1471-2105-15-155","title":"A method for increasing expressivity of Gene Ontology annotations using a compositional approach"}],"type":"annotation","user":"http://geneontology.org"}]},{"activity_status":"active","browsers":[{"label":"RGD","title":"RGD Ontology Browser","url":"http://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=RS:0000457"}],"build":{"method":"obo2owl","source_url":"https://download.rgd.mcw.edu/ontology/rat_strain/rat_strain.obo"},"contact":{"email":"sjwang@mcw.edu","github":"shurjenw","label":"Shur-Jen Wang","orcid":"0000-0001-5256-8683"},"description":"Ontology of rat strains","homepage":"http://rgd.mcw.edu/rgdweb/search/strains.html","id":"rs","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/rs.owl","page":"https://download.rgd.mcw.edu/ontology/rat_strain/","preferredPrefix":"RS","products":[{"id":"rs.owl","ontology_purl":"http://purl.obolibrary.org/obo/rs.owl"},{"id":"rs.obo","ontology_purl":"http://purl.obolibrary.org/obo/rs.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24267899","title":"Rat Strain Ontology: structured controlled vocabulary designed to facilitate access to strain data at RGD."}],"repository":"https://github.com/rat-genome-database/RS-Rat-Strain-Ontology","taxon":{"id":"NCBITaxon:10114","label":"Rattus"},"title":"Rat Strain Ontology","tracker":"https://github.com/rat-genome-database/RS-Rat-Strain-Ontology/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://raw.githubusercontent.com/rsc-ontologies/rxno/master/rxno.owl"},"contact":{"email":"batchelorc@rsc.org","github":"batchelorc","label":"Colin Batchelor","orcid":"0000-0001-5985-7429"},"description":"Connects organic name reactions to their roles in an organic synthesis and to processes in MOP","homepage":"https://github.com/rsc-ontologies/rxno","id":"rxno","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"mailing_list":"chemistry-ontologies@googlegroups.com","ontology_purl":"http://purl.obolibrary.org/obo/rxno.owl","preferredPrefix":"RXNO","products":[{"id":"rxno.owl","ontology_purl":"http://purl.obolibrary.org/obo/rxno.owl","title":"Name Reaction Ontology"}],"repository":"https://github.com/rsc-ontologies/rxno","title":"Name Reaction Ontology","tracker":"https://github.com/rsc-ontologies/rxno/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/monarch-initiative/SEPIO-ontology.git","path":"src/ontology","system":"git"},"contact":{"email":"mhb120@gmail.com","github":"mbrush","label":"Matthew Brush","orcid":"0000-0002-1048-5019"},"description":"An ontology for representing the provenance of scientific claims and the evidence that supports them.","domain":"scientific claims, evidence","homepage":"https://github.com/monarch-initiative/SEPIO-ontology","id":"sepio","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/sepio.owl","preferredPrefix":"SEPIO","products":[{"id":"sepio.owl","ontology_purl":"http://purl.obolibrary.org/obo/sepio.owl","title":"SEPIO"}],"repository":"https://github.com/monarch-initiative/SEPIO-ontology","title":"Scientific Evidence and Provenance Information Ontology","tracker":"https://github.com/monarch-initiative/SEPIO-ontology/issues"},{"activity_status":"active","build":{"method":"obo2owl","notes":"SWITCH","source_url":"https://raw.githubusercontent.com/The-Sequence-Ontology/SO-Ontologies/master/so.obo"},"contact":{"email":"keilbeck@genetics.utah.edu","github":"keilbeck","label":"Karen Eilbeck","orcid":"0000-0002-0831-6427"},"depicted_by":"http://sequenceontology.org/img/so_icon.png","description":"A structured controlled vocabulary for sequence annotation, for the exchange of annotation data and for the description of sequence objects in databases.","domain":"biological sequence","homepage":"https://github.com/The-Sequence-Ontology/SO-Ontologies","id":"so","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"mailing_list":"https://sourceforge.net/p/song/mailman/song-devel/","ontology_purl":"http://purl.obolibrary.org/obo/so.owl","page":"https://en.wikipedia.org/wiki/Sequence_Ontology","preferredPrefix":"SO","products":[{"id":"so.owl","ontology_purl":"http://purl.obolibrary.org/obo/so.owl"},{"id":"so.obo","ontology_purl":"http://purl.obolibrary.org/obo/so.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/15892872","title":"The Sequence Ontology: a tool for the unification of genome annotations."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/20226267","title":"Evolution of the Sequence Ontology terms and relationships."}],"repository":"https://github.com/The-Sequence-Ontology/SO-Ontologies","title":"Sequence types and features ontology","tracker":"https://github.com/The-Sequence-Ontology/SO-Ontologies/issues"},{"activity_status":"active","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/obophenotype/spider-ontology/master/spider_comparative_biology.obo"},"contact":{"email":"ramirez@macn.gov.ar","github":"martinjramirez","label":"Martin Ramirez","orcid":"0000-0002-0358-0130"},"description":"An ontology for spider comparative biology including anatomical parts (e.g. leg, claw), behavior (e.g. courtship, combing) and products (i.g. silk, web, borrow).","domain":"anatomy","homepage":"http://research.amnh.org/atol/files/","id":"spd","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/spd.owl","preferredPrefix":"SPD","products":[{"id":"spd.owl","ontology_purl":"http://purl.obolibrary.org/obo/spd.owl"}],"repository":"https://github.com/obophenotype/spider-ontology","taxon":{"id":"NCBITaxon:6893","label":"spiders"},"title":"Spider Ontology","tracker":"https://github.com/obophenotype/spider-ontology/issues"},{"activity_status":"active","contact":{"email":"alejandra.gonzalez.beltran@gmail.com","github":"agbeltran","label":"Alejandra Gonzalez-Beltran","orcid":"0000-0003-3499-8262"},"depicted_by":"https://raw.githubusercontent.com/ISA-tools/stato/dev/images/stato-logo-3.png","description":"STATO is a general-purpose STATistics Ontology. Its aim is to provide coverage for processes such as statistical tests, their conditions of application, and information needed or resulting from statistical methods, such as probability distributions, variables, spread and variation metrics. STATO also covers aspects of experimental design and description of plots and graphical representations commonly used to provide visual cues of data distribution or layout and to assist review of the results.","domain":"statistics","homepage":"http://stato-ontology.org/","id":"stato","in_foundry":false,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/stato.owl","preferredPrefix":"STATO","products":[{"id":"stato.owl","ontology_purl":"http://purl.obolibrary.org/obo/stato.owl"}],"repository":"https://github.com/ISA-tools/stato","title":"The Statistical Methods Ontology","tracker":"https://github.com/ISA-tools/stato/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://raw.githubusercontent.com/allysonlister/swo/master/release/swo_inferred.owl"},"contact":{"email":"allyson.lister@oerc.ox.ac.uk","github":"allysonlister","label":"Allyson Lister","orcid":"0000-0002-7702-4495"},"description":"The Software Ontology (SWO) is a resource for describing software tools, their types, tasks, versions, provenance and associated data. It contains detailed information on licensing and formats as well as software applications themselves, mainly (but not limited) to the bioinformatics community.","domain":"software","homepage":"https://github.com/allysonlister/swo","id":"swo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/swo.owl","preferredPrefix":"SWO","products":[{"id":"swo.owl","ontology_purl":"http://purl.obolibrary.org/obo/swo.owl"}],"repository":"https://github.com/allysonlister/swo","title":"Software ontology","tracker":"https://github.com/allysonlister/swo/issues"},{"activity_status":"active","build":{"infallible":1,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/DiseaseOntology/SymptomOntology/master/symp.obo"},"contact":{"email":"lynn.schriml@gmail.com","github":"lschriml","label":"Lynn Schriml","orcid":"0000-0001-8910-9851"},"description":"An ontology of disease symptoms, with symptoms encompasing perceived changes in function, sensations or appearance reported by a patient indicative of a disease.","domain":"disease","homepage":"http://symptomontologywiki.igs.umaryland.edu/mediawiki/index.php/Main_Page","id":"symp","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/symp.owl","preferredPrefix":"SYMP","products":[{"id":"symp.owl","ontology_purl":"http://purl.obolibrary.org/obo/symp.owl"},{"id":"symp.obo","ontology_purl":"http://purl.obolibrary.org/obo/symp.obo"}],"repository":"https://github.com/DiseaseOntology/SymptomOntology","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Symptom Ontology","tracker":"https://github.com/DiseaseOntology/SymptomOntology/issues","usages":[{"description":"Symptoms of human diseases in the DO","examples":[{"description":"symptoms of human diseases","url":"http://www.disease-ontology.org/?id=DOID:0060164"}],"user":"http://www.disease-ontology.org"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/phenoscape/taxrank.git","path":".","system":"git"},"contact":{"email":"balhoff@renci.org","github":"balhoff","label":"Jim Balhoff","orcid":"0000-0002-8688-6599"},"description":"A vocabulary of taxonomic ranks (species, family, phylum, etc)","domain":"taxonomy","homepage":"https://github.com/phenoscape/taxrank","id":"taxrank","jobs":[{"id":"https://travis-ci.org/phenoscape/taxrank","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/taxrank.owl","preferredPrefix":"TAXRANK","products":[{"id":"taxrank.owl","ontology_purl":"http://purl.obolibrary.org/obo/taxrank.owl"},{"id":"taxrank.obo","ontology_purl":"http://purl.obolibrary.org/obo/taxrank.obo"}],"repository":"https://github.com/phenoscape/taxrank","title":"Taxonomic rank vocabulary","tracker":"https://github.com/phenoscape/taxrank/issues"},{"activity_status":"active","browsers":[{"label":"Planteome","title":"Planteome browser","url":"http://browser.planteome.org/amigo/term/TO:0000387#display-lineage-tab"}],"build":{"checkout":"git clone https://github.com/Planteome/plant-trait-ontology.git","infallible":1,"method":"vcs","path":".","system":"git"},"contact":{"email":"jaiswalp@science.oregonstate.edu","github":"jaiswalp","label":"Pankaj Jaiswal","orcid":"0000-0002-1005-8383"},"description":"A controlled vocabulary to describe phenotypic traits in plants.","domain":"phenotype","homepage":"http://browser.planteome.org/amigo","id":"to","jobs":[{"id":"https://travis-ci.org/Planteome/plant-trait-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/to.owl","page":"http://browser.planteome.org/amigo/term/TO:0000387#display-lineage-tab","preferredPrefix":"TO","products":[{"id":"to.owl","ontology_purl":"http://purl.obolibrary.org/obo/to.owl"},{"id":"to.obo","ontology_purl":"http://purl.obolibrary.org/obo/to.obo"}],"publications":[{"id":"https://doi.org/10.1093/nar/gkx1152","title":"The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics."}],"repository":"https://github.com/Planteome/plant-trait-ontology","taxon":{"id":"NCBITaxon:33090","label":"Viridiplantae"},"title":"Plant Trait Ontology","tracker":"https://github.com/Planteome/plant-trait-ontology/issues","usages":[{"description":"Planteome uses TO to describe traits for genes and germplasm","examples":[{"description":"Genes and proteins annotated to submergence tolerance, including SUB1","url":"http://browser.planteome.org/amigo/term/TO:0000286"}],"user":"http://planteome.org/"},{"description":"Gramene uses PO for the annotation of plant genes and QTLs","examples":[{"description":"Gramene annotations to submergence tolerance","url":"http://archive.gramene.org/db/ontology/search?id=TO:0000286"}],"user":"http://gramene.org/"}]},{"activity_status":"active","build":{"infallible":1,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/DiseaseOntology/PathogenTransmissionOntology/master/src/ontology/trans.obo"},"contact":{"email":"lynn.schriml@gmail.com","github":"lschriml","label":"Lynn Schriml","orcid":"0000-0001-8910-9851"},"description":"An ontology representing the disease transmission process during which the pathogen is transmitted directly or indirectly from its natural reservoir, a susceptible host or source to a new host.","domain":"disease","homepage":"https://github.com/DiseaseOntology/PathogenTransmissionOntology","id":"trans","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/trans.owl","preferredPrefix":"TRANS","products":[{"id":"trans.owl","ontology_purl":"http://purl.obolibrary.org/obo/trans.owl"},{"id":"trans.obo","ontology_purl":"http://purl.obolibrary.org/obo/trans.obo"}],"repository":"https://github.com/DiseaseOntology/PathogenTransmissionOntology","title":"Pathogen Transmission Ontology","tracker":"https://github.com/DiseaseOntology/PathogenTransmissionOntology/issues","usages":[{"description":"Methods of trnasmission of human diseases in the DO","examples":[{"description":"methods of transmission of human diseases","url":"http://www.disease-ontology.org/?id=DOID:12365"}],"user":"http://www.disease-ontology.org"}]},{"activity_status":"active","contact":{"email":"balhoff@renci.org","github":"balhoff","label":"Jim Balhoff","orcid":"0000-0002-8688-6599"},"description":"An ontology covering the taxonomy of teleosts (bony fish)","domain":"taxonomy","homepage":"https://github.com/phenoscape/teleost-taxonomy-ontology","id":"tto","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/tto.owl","preferredPrefix":"TTO","products":[{"id":"tto.obo","ontology_purl":"http://purl.obolibrary.org/obo/tto.obo"},{"id":"tto.owl","ontology_purl":"http://purl.obolibrary.org/obo/tto.owl"}],"repository":"https://github.com/phenoscape/teleost-taxonomy-ontology","taxon":{"id":"NCBITaxon:32443","label":"Teleostei"},"title":"Teleost taxonomy ontology","tracker":"https://github.com/phenoscape/teleost-taxonomy-ontology/issues"},{"activity_status":"active","browsers":[{"label":"BioPortal","title":"BioPortal Browser","url":"http://bioportal.bioontology.org/ontologies/TXPO?p=classes"},{"label":"TOXPILOT","title":"TOXPILOT","url":"https://toxpilot.nibiohn.go.jp/"}],"contact":{"email":"yuki.yamagata@riken.jp","github":"yuki-yamagata","label":"Yuki Yamagata","orcid":"0000-0002-9673-1283"},"description":"TOXic Process Ontology (TXPO) systematizes a wide variety of terms involving toxicity courses and processes. The first version of TXPO focuses on liver toxicity.","domain":"toxicity","homepage":"https://toxpilot.nibiohn.go.jp/","id":"txpo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/txpo.owl","preferredPrefix":"TXPO","products":[{"id":"txpo.owl","ontology_purl":"http://purl.obolibrary.org/obo/txpo.owl"}],"repository":"https://github.com/txpo-ontology/TXPO","title":"Toxic Process Ontology","tracker":"https://github.com/txpo-ontology/TXPO/issues"},{"activity_status":"active","biosharing":"https://www.biosharing.org/bsg-000016","browsers":[{"label":"RGD","title":"Gene Ontology AmiGO 2 Browser","url":"http://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=UBERON:0001062"},{"label":"AmiGO (SUBSET)","title":"Gene Ontology AmiGO 2 Browser","url":"http://amigo.geneontology.org/amigo/term/UBERON:0001062#display-lineage-tab"},{"label":"Bgee (gene expression)","title":"Bgee gene expression queries","url":"http://bgee.org/?page=gene"},{"label":"FANTOM5","title":"FANTOM5 Data Portal","url":"http://fantom.gsc.riken.jp/5/sstar/UBERON:0001890"},{"label":"KnowledgeSpace","title":"INCF KnowledgeSpace Portal","url":"https://knowledge-space.org/index.php/pages/view/UBERON:0000061"}],"build":{"checkout":"svn --ignore-externals co http://svn.code.sf.net/p/obo/svn/uberon/trunk","email_cc":"cjmungall@lbl.gov","infallible":1,"method":"vcs","system":"svn"},"canonical":"uberon.owl","contact":{"email":"cjmungall@lbl.gov","github":"cmungall","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"dependencies":[{"id":"go","subset":"uberon/go_import.owl"},{"id":"cl","subset":"uberon/cl_import.owl"},{"id":"chebi","subset":"uberon/chebi_import.owl"},{"id":"pr","subset":"uberon/pr_import.owl"}],"depicted_by":"http://uberon.github.io/images/u-logo.jpg","description":"An integrated cross-species anatomy ontology covering animals and bridging multiple species-specific ontologies","domain":"anatomy","exampleClass":"UBERON_0002046","funded_by":["NIH R24OD011883","NIH R01HG004838","NIH P41HG002273","NSF DEB-0956049"],"homepage":"http://uberon.org","id":"uberon","label":"Uberon","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"mailing_list":"https://lists.sourceforge.net/lists/listinfo/obo-anatomy","ontology_purl":"http://purl.obolibrary.org/obo/uberon.owl","page":"http://en.wikipedia.org/wiki/Uberon","preferredPrefix":"UBERON","products":[{"description":"core ontology","id":"uberon.owl","is_canonical":true,"ontology_purl":"http://purl.obolibrary.org/obo/uberon.owl","title":"Uberon","type":"owl:Ontology"},{"description":"Axioms defined within Uberon and to be used in imports for other ontologies","id":"uberon/uberon-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/uberon/uberon-base.owl","page":"https://github.com/INCATools/ontology-development-kit/issues/50","title":"Uberon base ontology"},{"description":"Uberon extended","id":"uberon/ext.owl","mireots_from":"cl","ontology_purl":"http://purl.obolibrary.org/obo/uberon/ext.owl","title":"Uberon edition that includes subsets of other ontologies and axioms connecting to them","type":"owl:Ontology"},{"description":"Uberon edition that excludes external ontologies and most relations","format":"obo","id":"uberon/basic.obo","ontology_purl":"http://purl.obolibrary.org/obo/uberon/basic.obo","title":"Uberon basic","type":"obo-basic-ontology"},{"connects":[{"id":"uberon"},{"id":"zfa"}],"description":"Taxonomic equivalence axioms connecting zebrafish-specific classes to generic uberon counterparts","id":"uberon/bridge/uberon-bridge-to-zfa.owl","ontology_purl":"http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-zfa.owl","page":"https://github.com/obophenotype/uberon/wiki/inter-anatomy-ontology-bridge-ontologies","title":"Uberon bridge to ZFA","type":"BridgeOntology"},{"connects":[{"id":"uberon"},{"id":"ma"}],"description":"Taxonomic equivalence axioms connecting adult mouse specific classes to generic uberon counterparts","id":"uberon/bridge/uberon-bridge-to-ma.owl","ontology_purl":"http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-ma.owl","page":"https://github.com/obophenotype/uberon/wiki/inter-anatomy-ontology-bridge-ontologies","title":"Uberon bridge to MA","type":"BridgeOntology"},{"description":"Extended uberon plus all metazoan ontologies","id":"uberon/composite-metazoan.owl","ontology_purl":"http://purl.obolibrary.org/obo/uberon/composite-metazoan.owl","page":"https://github.com/obophenotype/uberon/wiki/Multi-species-composite-ontologies","taxon":"Metazoa","title":"Uberon composite metazoan ontology","type":"MergedOntology"},{"id":"uberon/composite-vertebrate.owl","mireots_from":["zfa","xao","fbbt","wbbt","ma","fma","emapa","ehdaa2"],"ontology_purl":"http://purl.obolibrary.org/obo/uberon/composite-vertebrate.owl","page":"https://github.com/obophenotype/uberon/wiki/Multi-species-composite-ontologies","taxon":"Metazoa","title":"Uberon composite vertebrate ontology","type":"MergedOntology"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/22293552","title":"Uberon, an integrative multi-species anatomy ontology"},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/25009735","title":"Unification of multi-species vertebrate anatomy ontologies for comparative biology in Uberon"}],"redirects":[{"match":"releases/","url":"http://svn.code.sf.net/p/obo/svn/uberon/releases/"},{"match":"","url":"http://berkeleybop.org/ontologies/uberon/"}],"releases":"http://purl.obolibrary.org/obo/uberon/releases/","repository":"https://github.com/obophenotype/uberon","slack":"https://obo-communitygroup.slack.com/archives/C01CR698CF2","taxon":{"id":"NCBITaxon:33208","label":"Metazoa"},"title":"Uberon multi-species anatomy ontology","tracker":"https://github.com/obophenotype/uberon/issues","twitter":"uberanat","type":"owl:Ontology","usages":[{"description":"Bgee is a database to retrieve and compare gene expression patterns between animal species. Bgee in using Uberon to annotate the site of expression, and Bgee curators one the major contributors to the ontology.","examples":[{"description":"Uberon terms used to annotate expression of human hemoglobin subunit beta","url":"http://bgee.org/?page=gene&gene_id=ENSG00000244734"}],"seeAlso":"https://www.biosharing.org/biodbcore-000228","type":"annotation","user":"http://bgee.org/"},{"description":"The National Human Genome Research Institute (NHGRI) launched a public research consortium named ENCODE, the Encyclopedia Of DNA Elements, in September 2003, to carry out a project to identify all functional elements in the human genome sequence. The ENCODE DCC users Uberon to annotate samples","publications":[{"id":"https://doi.org/10.1093/database/bav010","title":"Ontology application and use at the ENCODE DCC"}],"seeAlso":"https://www.biosharing.org/biodbcore-000034","type":"annotation","user":"https://www.encodeproject.org/"},{"description":"FANTOM5 is using Uberon and CL to annotate samples allowing for transcriptome analyses with cell-type and tissue-level specificity.","examples":[{"description":"FANTOM5 samples annotated to telencephalon or its parts","url":"http://fantom.gsc.riken.jp/5/sstar/UBERON:0001893"}],"type":"annotation","user":"http://fantom5-collaboration.gsc.riken.jp/"},{"description":"Querying expression and phenotype data","type":"query","user":"https://monarchinitiative.org/"},{"description":"GO Database is used for querying for functional annotations relevant to a tissue","examples":[{"description":"GO annotations relevant to the uberon class for brain","url":"http://amigo.geneontology.org/amigo/term/UBERON:0000955"}],"type":"query","user":"https://geneontology.org/"},{"description":"The Phenoscape project is both a major driver of and contributor to Uberon, contibuting thousands of terms. The teleost (bony fishes) component of Uberon was derived from the Teleost Anatomy Ontology, developed by the Phenoscape group. Most of the high level design of the skeletal system comes from the Vertebrate Skeletal Anatomy Ontology (VSAO), also created by the Phenoscape group. Phenoscape curators continue to extend the ontology, covering a wide variety of tetrapod structures, with an emphasis on the appendicular system.","user":"http://phenoscape.org"},{"description":"Searchable collection of neuroscience data and ontology for neuroscience","type":"Database","user":"https://neuinfo.org/"},{"description":"cooperative data platform to be used by diverse communities in making data more FAIR.","type":"Database","user":"https://scicrunch.org/"},{"description":"SCPortalen","publications":[{"id":"https://doi.org/10.1093/nar/gkx949","title":"SCPortalen: human and mouse single-cell centric database"}],"type":"Database","user":"http://single-cell.clst.riken.jp/"},{"description":"ChEMBL uses Uberon to describe organ/tissue information in assays","publications":[{"id":"https://doi.org/10.1093/nar/gky1075","title":"ChEMBL: towards direct deposition of bioassay data"}],"type":"Database","user":"https://www.ebi.ac.uk/chembl/"}],"wikidata_template":"https://en.wikipedia.org/wiki/Template:Uberon"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/bio-ontology-research-group/unit-ontology/master/unit.obo"},"contact":{"email":"g.gkoutos@gmail.com","github":"gkoutos","label":"George Gkoutos","orcid":"0000-0002-2061-091X"},"description":"Metrical units for use in conjunction with PATO","domain":"phenotype","homepage":"https://github.com/bio-ontology-research-group/unit-ontology","id":"uo","in_foundry":false,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/uo.owl","preferredPrefix":"UO","products":[{"id":"uo.owl","ontology_purl":"http://purl.obolibrary.org/obo/uo.owl"},{"id":"uo.obo","ontology_purl":"http://purl.obolibrary.org/obo/uo.obo"}],"repository":"https://github.com/bio-ontology-research-group/unit-ontology","title":"Units of measurement ontology","tracker":"https://github.com/bio-ontology-research-group/unit-ontology/issues"},{"activity_status":"active","build":{"method":"archive","path":"archive/ontology","source_url":"http://build.berkeleybop.org/job/build-pheno-ontologies/lastSuccessfulBuild/artifact/*zip*/archive.zip"},"contact":{"email":"vasilevs@ohsu.edu","github":"nicolevasilevsky","label":"Nicole Vasilevsky","orcid":"0000-0001-5208-3432"},"description":"The uPheno ontology integrates multiple phenotype ontologies into a unified cross-species phenotype ontology.","homepage":"https://github.com/obophenotype/upheno","id":"upheno","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"mailing_list":"https://groups.google.com/forum/#!forum/phenotype-ontologies-editors","ontology_purl":"http://purl.obolibrary.org/obo/upheno.owl","preferredPrefix":"UPHENO","products":[{"description":"uPheno 1 is no longer actively maintained, please start using uPheno 2 (see below).","id":"upheno.owl","ontology_purl":"http://purl.obolibrary.org/obo/upheno.owl","page":"https://github.com/obophenotype/upheno","title":"uPheno 1 (inactive)"},{"description":"No longer actively maintained.","id":"upheno/mp-hp-view.owl","ontology_purl":"http://purl.obolibrary.org/obo/upheno/mp-hp-view.owl","page":"https://github.com/obophenotype/upheno/tree/master/hp-mp","title":"uPheno MP-HP equivalence axioms"},{"description":"The new version of uPheno, along with species independent phenotypes amd additional phenotype relations. The ontology is still in Beta status, but we recommend users to migrate their infrastructures to uPheno 2 as uPheno 1 is no longer actively maintained.","id":"upheno/v2/upheno.owl","ontology_purl":"http://purl.obolibrary.org/obo/upheno/v2/upheno.owl","page":"https://github.com/obophenotype/upheno-dev","title":"uPheno 2"}],"repository":"https://github.com/obophenotype/upheno","title":"Unified phenotype ontology (uPheno)","tracker":"https://github.com/obophenotype/upheno/issues","usages":[{"description":"uPheno is used by the Monarch Initiative for cross-species inference.","examples":[{"description":"Characteristic neurologic anomaly resulting form degeneration of dopamine-generating cells in the substantia nigra, a region of the midbrain, characterized clinically by shaking, rigidity, slowness of movement and difficulty with walking and gait.","url":"https://monarchinitiative.org/phenotype/HP:0001300#disease"}],"publications":[{"id":"https://academic.oup.com/nar/article/45/D1/D712/2605791","title":"The Monarch Initiative: an integrative data and analytic platform connecting phenotypes to genotypes across species "}],"type":"analysis","user":"https://monarchinitiative.org/"}]},{"activity_status":"active","build":{"source_url":"https://raw.githubusercontent.com/vaccineontology/VO/master/src/VO_merged.owl"},"contact":{"email":"yongqunh@med.umich.edu","github":"yongqunh","label":"Yongqunh He","orcid":"0000-0001-9189-9661"},"description":"VO is a biomedical ontology in the domain of vaccine and vaccination.","domain":"health","homepage":"http://www.violinet.org/vaccineontology","id":"vo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/vo.owl","preferredPrefix":"VO","products":[{"id":"vo.owl","ontology_purl":"http://purl.obolibrary.org/obo/vo.owl"}],"repository":"https://github.com/vaccineontology/VO","title":"Vaccine Ontology","tracker":"https://github.com/vaccineontology/VO/issues"},{"activity_status":"active","build":{"checkout":"svn co http://phenotype-ontologies.googlecode.com/svn/trunk/src/ontology/vt","method":"vcs","system":"svn"},"contact":{"email":"caripark@iastate.edu","github":"caripark","label":"Carissa Park","orcid":"0000-0002-2346-5201"},"description":"An ontology of traits covering vertebrates","homepage":"https://github.com/AnimalGenome/vertebrate-trait-ontology","id":"vt","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/vt.owl","preferredPrefix":"VT","products":[{"id":"vt.owl","ontology_purl":"http://purl.obolibrary.org/obo/vt.owl"}],"repository":"https://github.com/AnimalGenome/vertebrate-trait-ontology","title":"Vertebrate trait ontology","tracker":"https://github.com/AnimalGenome/vertebrate-trait-ontology/issues","usages":[{"description":"The Animal Quantitative Trait Loci (QTL) Database (Animal QTLdb) annotates trait mapping data for livestock animals using the VTO","examples":[{"description":"Links to cattle QTL associated with the VTO term gastrointestinal system morphology trait or its descendants","url":"https://www.animalgenome.org/cgi-bin/QTLdb/BT/traitsrch?tword=Gastrointestinal%20tract%20weight"}],"user":"https://www.animalgenome.org/cgi-bin/QTLdb/index"},{"description":"The Rat Genome Database (RGD) uses the VTO to annotate rat QTL","examples":[{"description":"Annotations of rat QTL associated with the VTO term cholesterol amount or its descendants","url":"https://rgd.mcw.edu/rgdweb/ontology/annot.html?acc_id=VT:0003947&species=Rat"}],"user":"https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=VT:0000001"},{"description":"The Mouse Phenome Database (MPD) uses the VTO to annotate mouse strain traits","examples":[{"description":"Studies in the MPD database that have measurements related to the VTO term spleen size trait or its descendants","url":"https://phenome.jax.org/ontologies/VT:0002224"}],"user":"https://phenome.jax.org/ontologies/navigate/VT:0000001"}]},{"activity_status":"active","contact":{"email":"balhoff@renci.org","github":"balhoff","label":"Jim Balhoff","orcid":"0000-0002-8688-6599"},"description":"Comprehensive hierarchy of extinct and extant vertebrate taxa.","homepage":"https://github.com/phenoscape/vertebrate-taxonomy-ontology","id":"vto","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/vto.owl","preferredPrefix":"VTO","products":[{"id":"vto.owl","ontology_purl":"http://purl.obolibrary.org/obo/vto.owl"},{"id":"vto.obo","ontology_purl":"http://purl.obolibrary.org/obo/vto.obo"}],"redirects":[{"match":"","url":"http://vertebrate-taxonomy-ontology.s3-website-us-west-2.amazonaws.com/releases/"}],"repository":"https://github.com/phenoscape/vertebrate-taxonomy-ontology","title":"Vertebrate Taxonomy Ontology","tracker":"https://github.com/phenoscape/vertebrate-taxonomy-ontology/issues","usages":[{"description":"Phenoscape uses VTO to annotate systematics data","user":"http://phenoscape.org"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/c-elegans-gross-anatomy-ontology.git","path":".","system":"git"},"contact":{"email":"raymond@caltech.edu","github":"raymond91125","label":"Raymond Lee","orcid":"0000-0002-8151-7479"},"description":"A structured controlled vocabulary of the anatomy of <i>Caenorhabditis elegans</i>.","domain":"anatomy","homepage":"https://github.com/obophenotype/c-elegans-gross-anatomy-ontology","id":"wbbt","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/wbbt.owl","preferredPrefix":"WBbt","products":[{"id":"wbbt.owl","ontology_purl":"http://purl.obolibrary.org/obo/wbbt.owl"},{"id":"wbbt.obo","ontology_purl":"http://purl.obolibrary.org/obo/wbbt.obo"}],"publications":[{"id":"https://pubmed.ncbi.nlm.nih.gov/18629098/","title":"Building a cell and anatomy ontology of Caenorhabditis elegans"}],"repository":"https://github.com/obophenotype/c-elegans-gross-anatomy-ontology","taxon":{"id":"NCBITaxon:6237","label":"Caenorhabditis"},"title":"C. elegans Gross Anatomy Ontology","tracker":"https://github.com/obophenotype/c-elegans-gross-anatomy-ontology/issues","usages":[{"description":"WormBase uses WBbt to curate anatomical expression patterns and anatomy function annotations, and to allow search and indexing on the WormBase site","examples":[{"description":"Expression for gene daf-16 with WormBase ID WBGene00000912","url":"http://www.wormbase.org/db/get?name=WBGene00000912;class=Gene;widget=expression"}],"publications":[{"id":"https://academic.oup.com/nar/article/48/D1/D762/5603222","title":"WormBase: a modern Model Organism Information Resource"}],"type":"annotation","user":"https://www.wormbase.org/"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/c-elegans-development-ontology.git","path":".","system":"git"},"contact":{"email":"cgrove@caltech.edu","github":"chris-grove","label":"Chris Grove","orcid":"0000-0001-9076-6015"},"description":"A structured controlled vocabulary of the development of <i>Caenorhabditis elegans</i>.","domain":"developemental life stage","homepage":"https://github.com/obophenotype/c-elegans-development-ontology","id":"wbls","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/wbls.owl","preferredPrefix":"WBls","products":[{"id":"wbls.owl","ontology_purl":"http://purl.obolibrary.org/obo/wbls.owl"},{"id":"wbls.obo","ontology_purl":"http://purl.obolibrary.org/obo/wbls.obo"}],"publications":[{"id":"https://academic.oup.com/nar/article/48/D1/D762/5603222","title":"WormBase: a modern Model Organism Information Resource"}],"repository":"https://github.com/obophenotype/c-elegans-development-ontology","taxon":{"id":"NCBITaxon:6237","label":"Caenorhabditis"},"title":"C. elegans development ontology","tracker":"https://github.com/obophenotype/c-elegans-development-ontology/issues","usages":[{"description":"WormBase uses WBls to curate temporal expression patterns, and to allow search and indexing on the WormBase site","examples":[{"description":"Expression for daf-16 gene with WormBase ID WBGene00000912.","url":"http://www.wormbase.org/db/get?name=WBGene00000912;class=Gene;widget=expression"}],"publications":[{"id":"https://academic.oup.com/nar/article/48/D1/D762/5603222","title":"WormBase: a modern Model Organism Information Resource"}],"type":"annotation","user":"https://www.wormbase.org/"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/c-elegans-phenotype-ontology.git","path":".","system":"git"},"contact":{"email":"cgrove@caltech.edu","github":"chris-grove","label":"Chris Grove","orcid":"0000-0001-9076-6015"},"description":"A structured controlled vocabulary of <i>Caenorhabditis elegans</i> phenotypes","domain":"phenotype","homepage":"https://github.com/obophenotype/c-elegans-phenotype-ontology","id":"wbphenotype","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/wbphenotype.owl","preferredPrefix":"WBPhenotype","products":[{"id":"wbphenotype.owl","ontology_purl":"http://purl.obolibrary.org/obo/wbphenotype.owl"},{"id":"wbphenotype.obo","ontology_purl":"http://purl.obolibrary.org/obo/wbphenotype.obo"},{"id":"wbphenotype/wbphenotype-base.owl","ontology_purl":"http://purl.obolibrary.org/obo/wbphenotype/wbphenotype-base.owl"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/?term=21261995","title":"Worm Phenotype Ontology: integrating phenotype data within and beyond the C. elegans community."}],"repository":"https://github.com/obophenotype/c-elegans-phenotype-ontology","taxon":{"id":"NCBITaxon:6237","label":"Caenorhabditis"},"title":"C. elegans phenotype","tracker":"https://github.com/obophenotype/c-elegans-phenotype-ontology/issues","usages":[{"description":"WormBase uses WBPhenotype to curate worm phenotypes, and to allow search and indexing on the WormBase site","examples":[{"description":"Expression for daf-16 gene with WormBase ID WBGene00000912.","url":"http://www.wormbase.org/db/get?name=WBGene00000912;class=Gene;widget=expression"}],"publications":[{"id":"https://academic.oup.com/nar/article/48/D1/D762/5603222","title":"WormBase: a modern Model Organism Information Resource"}],"type":"annotation","user":"https://www.wormbase.org/"},{"description":"Monarch integrates phenotype annotations from sources such as WormBase, and allows for querying using the WBPhenotype ontology.","examples":[{"description":"Egg long: The fertilized oocytes have a greater than standard length measured end to end compared to control.","url":"https://monarchinitiative.org/phenotype/WBPhenotype%3A0000370"}],"publications":[{"id":"https://academic.oup.com/nar/article/45/D1/D712/2605791","title":"The Monarch Initiative: an integrative data and analytic platform connecting phenotypes to genotypes across species "}],"type":"annotation","user":"https://monarchinitiative.org/"}]},{"activity_status":"active","browsers":[{"label":"RGD","title":"RGD Ontology Browser","url":"http://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=XCO:0000000"}],"build":{"method":"obo2owl","source_url":"https://download.rgd.mcw.edu/ontology/experimental_condition/experimental_condition.obo"},"contact":{"email":"jrsmith@mcw.edu","github":"jrsjrs","label":"Jennifer Smith","orcid":"0000-0002-6443-9376"},"description":"Conditions under which physiological and morphological measurements are made both in the clinic and in studies involving humans or model organisms.","domain":"clinical","homepage":"https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=XCO:0000000","id":"xco","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/xco.owl","page":"https://download.rgd.mcw.edu/ontology/experimental_condition/","preferredPrefix":"XCO","products":[{"id":"xco.owl","ontology_purl":"http://purl.obolibrary.org/obo/xco.owl"},{"id":"xco.obo","ontology_purl":"http://purl.obolibrary.org/obo/xco.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/22654893","title":"Three ontologies to define phenotype measurement data."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24103152","title":"The clinical measurement, measurement method and experimental condition ontologies: expansion, improvements and new applications."}],"repository":"https://github.com/rat-genome-database/XCO-experimental-condition-ontology","title":"Experimental condition ontology","tracker":"https://github.com/rat-genome-database/XCO-experimental-condition-ontology/issues"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo"},"contact":{"email":"gerhard.mayer@rub.de","github":"germa","label":"Gerhard Mayer","orcid":"0000-0002-1767-2343"},"createdWith":"http://oboedit.org","description":"A structured controlled vocabulary for cross-linking reagents used with proteomics mass spectrometry.","domain":"MS cross-linker reagents","homepage":"http://www.psidev.info/groups/controlled-vocabularies","id":"xlmod","integration_server":"https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/tree/master/cv","label":"xlmod","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"mailing_list":"psidev-ms-vocab@lists.sourceforge.net","ontology_purl":"http://purl.obolibrary.org/obo/xlmod.owl","page":"http://www.psidev.info/groups/controlled-vocabularies","preferredPrefix":"XLMOD","products":[{"id":"xlmod.obo","ontology_purl":"http://purl.obolibrary.org/obo/xlmod.obo"}],"repository":"https://github.com/HUPO-PSI/mzIdentML","title":"HUPO-PSI cross-linking and derivatization reagents controlled vocabulary","tracker":"https://github.com/HUPO-PSI/mzIdentML/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/xenopus-phenotype-ontology.git","path":".","system":"git"},"contact":{"email":"esegerd3@gmail.com","github":"seger","label":"Erik Segerdell","orcid":"0000-0002-9611-1279"},"dependencies":[{"id":"iao"},{"id":"go"},{"id":"ro"},{"id":"pato"},{"id":"bfo"},{"id":"chebi"},{"id":"cl"},{"id":"xao"}],"description":"XPO represents anatomical, cellular, and gene function phenotypes occurring throughout the development of the African frogs Xenopus laevis and tropicalis.","domain":"phenotype","homepage":"https://github.com/obophenotype/xenopus-phenotype-ontology","id":"xpo","jobs":[{"id":"https://travis-ci.org/obophenotype/xenopus-phenotype-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/xpo.owl","preferredPrefix":"XPO","products":[{"id":"xpo.owl","ontology_purl":"http://purl.obolibrary.org/obo/xpo.owl"},{"id":"xpo.obo","ontology_purl":"http://purl.obolibrary.org/obo/xpo.obo"}],"repository":"https://github.com/obophenotype/xenopus-phenotype-ontology","taxon":{"id":"NCBITaxon:8353","label":"Xenopus"},"title":"Xenopus Phenotype Ontology","tracker":"https://github.com/obophenotype/xenopus-phenotype-ontology/issues"},{"activity_status":"active","build":{"method":"obo2owl","source_url":"https://raw.githubusercontent.com/ybradford/zebrafish-experimental-conditions-ontology/master/zeco.obo"},"contact":{"email":"ybradford@zfin.org","github":"ybradford","label":"Yvonne Bradford","orcid":"0000-0002-9900-7880"},"description":"Experimental conditions applied to zebrafish, developed to facilitate experiment condition annotation at ZFIN","domain":"environment","homepage":"https://github.com/ybradford/zebrafish-experimental-conditions-ontology","id":"zeco","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/zeco.owl","preferredPrefix":"ZECO","products":[{"id":"zeco.obo","ontology_purl":"http://purl.obolibrary.org/obo/zeco.obo"},{"id":"zeco.owl","ontology_purl":"http://purl.obolibrary.org/obo/zeco.owl"},{"id":"zeco.json","ontology_purl":"http://purl.obolibrary.org/obo/zeco.json"}],"repository":"https://github.com/ybradford/zebrafish-experimental-conditions-ontology","taxon":{"id":"NCBITaxon:7954","label":"Danio"},"title":"Zebrafish Experimental Conditions Ontology","tracker":"https://github.com/ybradford/zebrafish-experimental-conditions-ontology/issues"},{"activity_status":"active","build":{"infallible":1,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/obophenotype/developmental-stage-ontologies/master/src/zfs/zfs.obo"},"contact":{"email":"van_slyke@zfin.org","github":"cerivs","label":"Ceri Van Slyke","orcid":"0000-0002-2244-7917"},"description":"Developmental stages of the Zebrafish","domain":"anatomy","homepage":"https://wiki.zfin.org/display/general/Anatomy+Atlases+and+Resources","id":"zfs","in_foundry":false,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/zfs.owl","page":"https://github.com/obophenotype/developmental-stage-ontologies/wiki/ZFS","preferredPrefix":"ZFS","products":[{"id":"zfs.owl","ontology_purl":"http://purl.obolibrary.org/obo/zfs.owl"},{"id":"zfs.obo","ontology_purl":"http://purl.obolibrary.org/obo/zfs.obo"}],"repository":"https://github.com/cerivs/zebrafish-anatomical-ontology","taxon":{"id":"NCBITaxon:7954","label":"Danio"},"title":"Zebrafish developmental stages ontology","tracker":"https://github.com/cerivs/zebrafish-anatomical-ontology/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/obophenotype/zebrafish-phenotype-ontology.git","path":".","system":"git"},"contact":{"email":"ybradford@zfin.org","github":"ybradford","label":"Yvonne Bradford","orcid":"0000-0002-9900-7880"},"dependencies":[{"id":"go"},{"id":"ro"},{"id":"pato"},{"id":"bspo"},{"id":"zfa"},{"id":"bfo"},{"id":"uberon"}],"description":"The Zebrafish Phenotype Ontology formally defines all phenotypes of the Zebrafish model organism.","domain":"phenotype","homepage":"https://github.com/obophenotype/zebrafish-phenotype-ontology","id":"zp","jobs":[{"id":"https://travis-ci.org/obophenotype/zebrafish-phenotype-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/zp.owl","preferredPrefix":"ZP","products":[{"id":"zp.owl","ontology_purl":"http://purl.obolibrary.org/obo/zp.owl"},{"id":"zp.obo","ontology_purl":"http://purl.obolibrary.org/obo/zp.obo"}],"repository":"https://github.com/obophenotype/zebrafish-phenotype-ontology","title":"Zebrafish Phenotype Ontology","tracker":"https://github.com/obophenotype/zebrafish-phenotype-ontology/issues","usages":[{"description":"Monarch integrates phenotype annotations from sources such as ZFIIN, and allows for querying using the ZP ontology.","examples":[{"description":"adaxial cell absent, abnormal: Abnormal(ly) absent (of) adaxial cell.","url":"https://monarchinitiative.org/phenotype/ZP:0005692"}],"publications":[{"id":"https://academic.oup.com/nar/article/45/D1/D712/2605791","title":"The Monarch Initiative: an integrative data and analytic platform connecting phenotypes to genotypes across species"}],"type":"annotation","user":"https://monarchinitiative.org/"}]},{"activity_status":"active","build":{"checkout":"git clone https://github.com/Superraptor/GSSO.git","path":".","system":"git"},"contact":{"email":"kronkcj@mail.uc.edu","github":"Superraptor","label":"Clair Kronk","orcid":"0000-0001-8397-8810"},"description":"The Gender, Sex, and Sexual Orientation (GSSO) ontology has terms for annotating interdisciplinary information concerning gender, sex, and sexual orientation for primary usage in the biomedical and adjacent sciences.","domain":"organisms","homepage":"https://gsso.research.cchmc.org/","id":"gsso","layout":"ontology_detail","license":{"label":"Apache 2.0 License","url":"http://www.apache.org/licenses/LICENSE-2.0"},"ontology_purl":"http://purl.obolibrary.org/obo/gsso.owl","preferredPrefix":"GSSO","products":[{"id":"gsso.owl","ontology_purl":"http://purl.obolibrary.org/obo/gsso.owl"},{"id":"gsso.obo","ontology_purl":"http://purl.obolibrary.org/obo/gsso.obo"},{"id":"gsso.json","ontology_purl":"http://purl.obolibrary.org/obo/gsso.json"}],"repository":"https://github.com/Superraptor/GSSO","title":"Gender, Sex, and Sexual Orientation (GSSO) ontology","tracker":"https://github.com/Superraptor/GSSO/issues"},{"activity_status":"active","alternativePrefix":"HPO","browsers":[{"label":"HPO","title":"JAX HPO Browser","url":"https://hpo.jax.org/app/"},{"label":"Monarch","title":"Monarch Phenotype Page","url":"http://monarchinitiative.org/phenotype/HP:0000118"}],"build":{"infallible":1,"method":"archive","path":"archive/hp","source_url":"http://compbio.charite.de/hudson/job/hpo/lastSuccessfulBuild/artifact/*zip*/archive.zip"},"contact":{"email":"dr.sebastian.koehler@gmail.com","github":"drseb","label":"Sebastian Koehler","orcid":"0000-0002-5316-1399"},"description":"A structured and controlled vocabulary for the phenotypic features encountered in human hereditary and other disease.","domain":"phenotype","homepage":"http://www.human-phenotype-ontology.org/","id":"hp","jobs":[{"id":"https://travis-ci.org/obophenotype/human-phenotype-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"hpo","url":"https://hpo.jax.org/app/license"},"mailing_list":"https://groups.io/g/human-phenotype-ontology","ontology_purl":"http://purl.obolibrary.org/obo/hp.owl","preferredPrefix":"HP","products":[{"id":"hp.owl","ontology_purl":"http://purl.obolibrary.org/obo/hp.owl"},{"id":"hp.obo","ontology_purl":"http://purl.obolibrary.org/obo/hp.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/18950739","title":"The Human Phenotype Ontology: a tool for annotating and analyzing human hereditary disease."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/26119816","title":"The Human Phenotype Ontology: Semantic Unification of Common and Rare Disease."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/24217912","title":"The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data."},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/30476213","title":"Expansion of the Human Phenotype Ontology (HPO) knowledge base and resources."}],"repository":"https://github.com/obophenotype/human-phenotype-ontology","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Human Phenotype Ontology","tracker":"https://github.com/obophenotype/human-phenotype-ontology/issues/","twitter":"hp_ontology","usages":[{"description":"HPO is used by the Monarch Initiative for phenotype annotations.","examples":[{"url":"https://monarchinitiative.org/phenotype/HP:0001300"}],"reference":"https://academic.oup.com/nar/article/45/D1/D712/2605791","type":"annotation","user":"https://monarchinitiative.org/"}]},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://raw.githubusercontent.com/SED-ML/KiSAO/deploy/kisao.owl"},"contact":{"email":"karr@mssm.edu","github":"jonrkarr","label":"Jonathan Karr","orcid":"0000-0002-2605-5080"},"description":"A classification of algorithms for simulating biology and their outputs","domain":"algorithms","homepage":"http://co.mbine.org/standards/kisao","id":"kisao","layout":"ontology_detail","license":{"label":"Artistic License 2.0","url":"http://opensource.org/licenses/Artistic-2.0"},"ontology_purl":"http://purl.obolibrary.org/obo/kisao.owl","preferredPrefix":"KISAO","products":[{"id":"kisao.owl","ontology_purl":"http://purl.obolibrary.org/obo/kisao.owl"}],"repository":"https://github.com/SED-ML/KiSAO","title":"Kinetic Simulation Algorithm Ontology","tracker":"https://github.com/SED-ML/KiSAO/issues"},{"activity_status":"active","build":{"method":"owl2obo","source_url":"https://svn.code.sf.net/p/mamo-ontology/code/tags/latest/mamo-xml.owl"},"contact":{"email":"nicolas@ascistance.co.uk","github":"gambardella","label":"Nicolas Gambardella","orcid":"0000-0002-6309-7327"},"description":"The Mathematical Modelling Ontology (MAMO) is a classification of the types of mathematical models used mostly in the life sciences, their variables, relationships and other relevant features.","homepage":"http://sourceforge.net/p/mamo-ontology/wiki/Home/","id":"mamo","layout":"ontology_detail","license":{"label":"Artistic License 2.0","url":"http://opensource.org/licenses/Artistic-2.0"},"ontology_purl":"http://purl.obolibrary.org/obo/mamo.owl","preferredPrefix":"MAMO","products":[{"id":"mamo.owl","ontology_purl":"http://purl.obolibrary.org/obo/mamo.owl"}],"repository":"http://sourceforge.net/p/mamo-ontology","title":"Mathematical modeling ontology","tracker":"http://sourceforge.net/p/mamo-ontology/tickets/"},{"activity_status":"active","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"http://www.ebi.ac.uk/sbo/exports/Main/SBO_OBO.obo"},"contact":{"email":"sheriff@ebi.ac.uk","github":"rsmsheriff","label":"Rahuman Sheriff","orcid":"0000-0003-0705-9809"},"description":"Terms commonly used in Systems Biology, and in particular in computational modeling.","domain":"biochemistry","homepage":"http://www.ebi.ac.uk/sbo/","id":"sbo","layout":"ontology_detail","license":{"label":"Artistic License 2.0","url":"http://opensource.org/licenses/Artistic-2.0"},"ontology_purl":"http://purl.obolibrary.org/obo/sbo.owl","preferredPrefix":"SBO","products":[{"id":"sbo.owl","ontology_purl":"http://purl.obolibrary.org/obo/sbo.owl"}],"repository":"https://github.com/EBI-BioModels/SBO","title":"Systems Biology Ontology","tracker":"https://github.com/EBI-BioModels/SBO/issues"},{"activity_status":"active","build":{"checkout":"git clone https://github.com/scdodev/scdo-ontology.git","path":".","system":"git"},"contact":{"email":"giant.plankton@gmail.com","github":"JadeHotchkiss","label":"Jade Hotchkiss","orcid":"0000-0002-2193-0704"},"dependencies":[{"id":"apollo_sv"},{"id":"aro"},{"id":"chebi"},{"id":"chmo"},{"id":"cmo"},{"id":"doid"},{"id":"dron"},{"id":"duo"},{"id":"envo"},{"id":"eupath"},{"id":"exo"},{"id":"gaz"},{"id":"gsso"},{"id":"hp"},{"id":"hsapdv"},{"id":"ico"},{"id":"ido"},{"id":"idomal"},{"id":"mp"},{"id":"nbo"},{"id":"ncit"},{"id":"obi"},{"id":"ogms"},{"id":"opmi"},{"id":"pr"},{"id":"sbo"},{"id":"stato"},{"id":"symp"},{"id":"uo"},{"id":"vo"},{"id":"vt"}],"description":"An ontology for the standardization of terminology and integration of knowledge about Sickle Cell Disease.","domain":"disease","homepage":"https://scdontology.h3abionet.org/","id":"scdo","layout":"ontology_detail","license":{"label":"GPL-3.0","url":"https://www.gnu.org/licenses/gpl-3.0.en.html"},"ontology_purl":"http://purl.obolibrary.org/obo/scdo.owl","preferredPrefix":"SCDO","products":[{"id":"scdo.owl","ontology_purl":"http://purl.obolibrary.org/obo/scdo.owl"},{"id":"scdo.obo","ontology_purl":"http://purl.obolibrary.org/obo/scdo.obo"}],"repository":"https://github.com/scdodev/scdo-ontology","title":"Sickle Cell Disease Ontology","tracker":"https://github.com/scdodev/scdo-ontology/issues"},{"activity_status":"orphaned","build":{"infallible":1,"insert_ontology_id":true,"method":"obo2owl","source_url":"ftp://ftp.ebi.ac.uk/pub/databases/chebi/ontology/fix.obo"},"contact":{"email":null,"label":"chEBI"},"description":"An ontology of physico-chemical methods and properties.","homepage":"http://www.ebi.ac.uk/chebi","id":"fix","in_foundry":false,"layout":"ontology_detail","ontology_purl":"http://purl.obolibrary.org/obo/fix.owl","products":[{"id":"fix.owl","ontology_purl":"http://purl.obolibrary.org/obo/fix.owl"},{"id":"fix.obo","ontology_purl":"http://purl.obolibrary.org/obo/fix.obo"}],"title":"Physico-chemical methods and properties"},{"activity_status":"orphaned","description":"An ontology that formalizes the genomic element by defining an upper class genetic interval","homepage":"https://code.google.com/p/ontology-for-genetic-interval/","id":"ogi","layout":"ontology_detail","ontology_purl":"http://purl.obolibrary.org/obo/ogi.owl","products":[{"id":"ogi.owl","ontology_purl":"http://purl.obolibrary.org/obo/ogi.owl"}],"title":"Ontology for genetic interval","tracker":"https://code.google.com/p/ontology-for-genetic-interval/issues/list"},{"activity_status":"orphaned","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"http://obo.cvs.sourceforge.net/*checkout*/obo/obo/ontology/physicochemical/rex.obo"},"description":"An ontology of physico-chemical processes, i.e. physico-chemical changes occurring in course of time.","id":"rex","layout":"ontology_detail","ontology_purl":"http://purl.obolibrary.org/obo/rex.owl","products":[{"id":"rex.owl","ontology_purl":"http://purl.obolibrary.org/obo/rex.owl"}],"title":"Physico-chemical process"},{"activity_status":"orphaned","build":{"checkout":"git clone https://github.com/obophenotype/sibo.git","method":"vcs","system":"git"},"contact":{"email":"cjmungall@lbl.gov","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"description":"Social Behavior in insects","domain":"behavior","homepage":"https://github.com/obophenotype/sibo","id":"sibo","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/sibo.owl","products":[{"id":"sibo.owl","ontology_purl":"http://purl.obolibrary.org/obo/sibo.owl"},{"id":"sibo.obo","ontology_purl":"http://purl.obolibrary.org/obo/sibo.obo"}],"repository":"https://github.com/obophenotype/sibo","title":"Social Insect Behavior Ontology","tracker":"https://github.com/obophenotype/sibo/issues"},{"activity_status":"orphaned","build":{"method":"obo2owl","source_url":"http://variationontology.org/vario_download/vario.obo"},"contact":{"email":"mauno.vihinen@med.lu.se","label":"Mauno Vihinen"},"description":"Variation Ontology, VariO, is an ontology for standardized, systematic description of effects, consequences and mechanisms of variations.","homepage":"http://variationontology.org","id":"vario","layout":"ontology_detail","ontology_purl":"http://purl.obolibrary.org/obo/vario.owl","preferredPrefix":"VariO","products":[{"id":"vario.owl","ontology_purl":"http://purl.obolibrary.org/obo/vario.owl"},{"id":"vario.obo","ontology_purl":"http://purl.obolibrary.org/obo/vario.obo"}],"title":"Variation Ontology"},{"activity_status":"inactive","build":{"checkout":"git clone https://github.com/obophenotype/cephalopod-ontology.git","method":"vcs","path":"src/ontology","system":"git"},"contact":{"email":"cjmungall@lbl.gov","github":"cmungall","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"description":"An anatomical and developmental ontology for cephalopods","domain":"anatomy","homepage":"https://github.com/obophenotype/cephalopod-ontology","id":"ceph","jobs":[{"id":"https://travis-ci.org/obophenotype/cephalopod-ontology","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ceph.owl","products":[{"id":"ceph.owl","ontology_purl":"http://purl.obolibrary.org/obo/ceph.owl","title":"main version"},{"id":"ceph.obo","ontology_purl":"http://purl.obolibrary.org/obo/ceph.obo","title":"oboformat edition"}],"repository":"https://github.com/obophenotype/cephalopod-ontology","taxon":{"id":"NCBITaxon:6605","label":"Cephalopod"},"title":"Cephalopod Ontology","tracker":"https://github.com/obophenotype/cephalopod-ontology/issues"},{"activity_status":"inactive","build":{"method":"archive","path":"archive","source_url":"http://build.berkeleybop.org/job/build-gaz/lastSuccessfulBuild/artifact/*zip*/archive.zip"},"contact":{"email":"lschriml@som.umaryland.edu","github":"lschriml","label":"Lynn Schriml","orcid":"0000-0001-8910-9851"},"description":"A gazetteer constructed on ontological principles","homepage":"http://environmentontology.github.io/gaz/","id":"gaz","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"mailing_list":"https://groups.google.com/forum/#!forum/obo-gazetteer","ontology_purl":"http://purl.obolibrary.org/obo/gaz.owl","products":[{"id":"gaz.owl","ontology_purl":"http://purl.obolibrary.org/obo/gaz.owl"},{"id":"gaz.obo","ontology_purl":"http://purl.obolibrary.org/obo/gaz.obo"}],"repository":"https://github.com/EnvironmentOntology/gaz","title":"Gazetteer","tracker":"https://github.com/EnvironmentOntology/gaz/issues"},{"activity_status":"inactive","build":{"checkout":"git clone https://github.com/geneontology/unipathway.git","path":".","system":"git"},"contact":{"email":"Anne.Morgat@sib.swiss","github":"amorgat","label":"Anne Morgat","orcid":"0000-0002-1216-2969"},"dependencies":[{"id":"ro"}],"description":"A manually curated resource for the representation and annotation of metabolic pathways","domain":"pathways","homepage":"https://github.com/geneontology/unipathway","id":"upa","jobs":[{"id":"https://travis-ci.org/geneontology/unipathway","type":"travis-ci"}],"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/upa.owl","products":[{"id":"upa.owl","ontology_purl":"http://purl.obolibrary.org/obo/upa.owl"},{"id":"upa.obo","ontology_purl":"http://purl.obolibrary.org/obo/upa.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/22102589","title":"UniPathway: a resource for the exploration and annotation of metabolic pathways"}],"repository":"https://github.com/geneontology/unipathway","title":"Unipathway","tracker":"https://github.com/geneontology/unipathway/issues"},{"activity_status":"inactive","build":{"checkout":"git clone https://github.com/obophenotype/human-developmental-anatomy-ontology.git","method":"vcs","path":"src/ontology","system":"git"},"contact":{"email":"J.Bard@ed.ac.uk","label":"Jonathan Bard"},"description":"AEO is an ontology of anatomical structures that expands CARO, the Common Anatomy Reference Ontology","domain":"anatomy","homepage":"https://github.com/obophenotype/human-developmental-anatomy-ontology/","id":"aeo","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/aeo.owl","preferredPrefix":"AEO","products":[{"id":"aeo.owl","ontology_purl":"http://purl.obolibrary.org/obo/aeo.owl"}],"repository":"https://github.com/obophenotype/human-developmental-anatomy-ontology","title":"Anatomical Entity Ontology","tracker":"https://github.com/obophenotype/human-developmental-anatomy-ontology/issues"},{"activity_status":"inactive","build":{"checkout":"git clone https://github.com/obophenotype/human-developmental-anatomy-ontology.git","method":"vcs","path":"src/ontology","system":"git"},"contact":{"email":"J.Bard@ed.ac.uk","label":"Jonathan Bard"},"dependencies":[{"id":"aeo"},{"id":"caro"},{"id":"cl"}],"description":"A structured controlled vocabulary of stage-specific anatomical structures of the developing human.","domain":"anatomy","homepage":"https://github.com/obophenotype/human-developmental-anatomy-ontology","id":"ehdaa2","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ehdaa2.owl","products":[{"id":"ehdaa2.owl","ontology_purl":"http://purl.obolibrary.org/obo/ehdaa2.owl"},{"id":"ehdaa2.obo","ontology_purl":"http://purl.obolibrary.org/obo/ehdaa2.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/22973865","title":"A new ontology (structured hierarchy) of human developmental anatomy for the first 7 weeks (Carnegie stages 1-20)."}],"repository":"https://github.com/obophenotype/human-developmental-anatomy-ontology","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Human developmental anatomy, abstract","tracker":"https://github.com/obophenotype/human-developmental-anatomy-ontology/issues"},{"activity_status":"inactive","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/ero.owl"},"contact":{"email":"Marc_Ciriello@hms.harvard.edu","label":"Marc Ciriello"},"description":"An ontology of research resources such as instruments. protocols, reagents, animal models and biospecimens.","documentation":"https://open.med.harvard.edu/wiki/display/eaglei/Ontology","domain":"resources","homepage":"https://open.med.harvard.edu/wiki/display/eaglei/Ontology","id":"ero","layout":"ontology_detail","license":{"label":"CC BY 2.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/2.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/ero.owl","products":[{"id":"ero.owl","ontology_purl":"http://purl.obolibrary.org/obo/ero.owl"}],"title":"eagle-i resource ontology"},{"activity_status":"inactive","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"http://svn.code.sf.net/p/obo/svn/fma-conversion/trunk/fma2_obo.obo"},"contact":{"email":"mejino@u.washington.edu","label":"Onard Mejino"},"description":"This is currently a slimmed down version of FMA","domain":"anatomy","homepage":"http://si.washington.edu/projects/fma","id":"fma","layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/fma.owl","page":"http://en.wikipedia.org/wiki/Foundational_Model_of_Anatomy","preferredPrefix":"FMA","products":[{"id":"fma.owl","ontology_purl":"http://purl.obolibrary.org/obo/fma.owl"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/18688289","title":"Translating the Foundational Model of Anatomy into OWL"},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/18360535","title":"The foundational model of anatomy in OWL: Experience and perspectives"},{"id":"http://www.ncbi.nlm.nih.gov/pubmed/16779026","title":"Challenges in converting frame-based ontology into OWL: the Foundational Model of Anatomy case-study"}],"taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Foundational Model of Anatomy Ontology (subset)","tracker":"https://sourceforge.net/p/obo/foundational-model-of-anatomy-fma-requests/"},{"activity_status":"inactive","contact":{"email":"topalis@imbb.forth.gr","label":"Pantelis Topalis"},"description":"An application ontology to cover all aspects of malaria as well as the intervention attempts to control it.","domain":"health","homepage":"https://www.vectorbase.org/ontology-browser","id":"idomal","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/idomal.owl","products":[{"id":"idomal.owl","ontology_purl":"http://purl.obolibrary.org/obo/idomal.owl"},{"id":"idomal.obo","ontology_purl":"http://purl.obolibrary.org/obo/idomal.obo"}],"repository":"https://github.com/VEuPathDB-ontology/IDOMAL","title":"Malaria Ontology"},{"activity_status":"inactive","contact":{"email":"louis@imbb.forth.gr","label":"Christos (Kitsos) Louis"},"description":"Application ontology for entities related to insecticide resistance in mosquitos","domain":"environment","id":"miro","layout":"ontology_detail","ontology_purl":"http://purl.obolibrary.org/obo/miro.owl","products":[{"id":"miro.owl","ontology_purl":"http://purl.obolibrary.org/obo/miro.owl"},{"id":"miro.obo","ontology_purl":"http://purl.obolibrary.org/obo/miro.obo"}],"repository":"https://github.com/VEuPathDB-ontology/MIRO","taxon":{"id":"NCBITaxon:44484","label":"Anopheles"},"title":"Mosquito insecticide resistance","validate":false},{"activity_status":"inactive","browsers":[{"label":"RNAO","title":"RNA Ontology jOWL Browser","url":"http://bgsu-rna.github.io/rnao/"}],"build":{"checkout":"git clone https://github.com/BGSU-RNA/rnao.git","method":"vcs","system":"git"},"contact":{"email":"BatchelorC@rsc.org","label":"Colin Batchelor","orcid":"0000-0001-5985-7429"},"description":"Controlled vocabulary pertaining to RNA function and based on RNA sequences, secondary and three-dimensional structures.","domain":"molecular structure","homepage":"https://github.com/bgsu-rna/rnao","id":"rnao","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/rnao.owl","products":[{"id":"rnao.owl","ontology_purl":"http://purl.obolibrary.org/obo/rnao.owl"},{"id":"rnao.obo","ontology_purl":"http://purl.obolibrary.org/obo/rnao.obo"}],"repository":"https://github.com/BGSU-RNA/rnao","title":"RNA ontology","tracker":"https://github.com/BGSU-RNA/rnao/issues"},{"activity_status":"inactive","contact":{"email":"dsonensh@odu.edu","label":"Daniel Sonenshine"},"description":"The anatomy of the Tick, <i>Families: Ixodidae, Argassidae</i>","domain":"anatomy","homepage":"https://www.vectorbase.org/ontology-browser","id":"tads","layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/tads.owl","products":[{"id":"tads.owl","ontology_purl":"http://purl.obolibrary.org/obo/tads.owl"},{"id":"tads.obo","ontology_purl":"http://purl.obolibrary.org/obo/tads.obo"}],"repository":"https://github.com/VEuPathDB-ontology/TADS","taxon":{"id":"NCBITaxon:6939","label":"Ixodidae"},"title":"Tick Anatomy Ontology"},{"activity_status":"inactive","contact":{"email":"topalis@imbb.forth.gr","label":"Pantelis Topalis"},"description":"A structured controlled vocabulary of the anatomy of mosquitoes.","domain":"anatomy","homepage":"https://www.vectorbase.org/ontology-browser","id":"tgma","layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"ontology_purl":"http://purl.obolibrary.org/obo/tgma.owl","products":[{"id":"tgma.owl","ontology_purl":"http://purl.obolibrary.org/obo/tgma.owl"},{"id":"tgma.obo","ontology_purl":"http://purl.obolibrary.org/obo/tgma.obo"}],"repository":"https://github.com/VEuPathDB-ontology/TGMA","taxon":{"id":"NCBITaxon:44484","label":"Anopheles"},"title":"Mosquito gross anatomy ontology"},{"activity_status":"inactive","contact":{"email":"jiezheng@pennmedicine.upenn.edu","github":"zhengj2007","label":"Jie Zheng","orcid":"0000-0002-2999-0103"},"description":"An application ontology built for beta cell genomics studies.","homepage":"https://github.com/obi-bcgo/bcgo","id":"bcgo","in_foundry":false,"is_obsolete":true,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"products":[{"id":"bcgo.owl","ontology_purl":"http://purl.obolibrary.org/obo/bcgo.owl"}],"repository":"https://github.com/obi-bcgo/bcgo","title":"Beta Cell Genomics Ontology","tracker":"https://github.com/obi-bcgo/bcgo/issues"},{"activity_status":"inactive","build":{"method":"obo2owl","source_url":"https://github.com/Planteome/plant-environment-ontology/blob/master/plant-environment-ontology.obo.owl"},"contact":{"email":"jaiswalp@science.oregonstate.edu","github":"jaiswalp","label":"Pankaj Jaiswal","orcid":"0000-0002-1005-8383"},"description":"A structured, controlled vocabulary which describes the treatments, growing conditions, and/or study types used in plant biology experiments.","domain":"environment","homepage":"http://planteome.org/","id":"eo","is_obsolete":true,"layout":"ontology_detail","license":{"label":"CC BY 4.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/4.0/"},"page":"http://browser.planteome.org/amigo/term/EO:0007359","products":[{"id":"eo.owl","ontology_purl":"http://purl.obolibrary.org/obo/eo.owl"},{"id":"eo.obo","ontology_purl":"http://purl.obolibrary.org/obo/eo.obo"}],"publications":[{"id":"http://www.ncbi.nlm.nih.gov/pubmed/22847540","title":"Ontologies as integrative tools for plant science."}],"replaced_by":"peco","repository":"https://github.com/Planteome/plant-environment-ontology","title":"Plant Environment Ontology","tracker":"https://github.com/Planteome/plant-environment-ontology/issues","usages":[{"description":"Planteome uses EO to describe traits for genes and germplasm","examples":[{"description":"Genes and proteins annotated to cold temperature regiment","url":"http://browser.planteome.org/amigo/term/EO:0007174"}],"user":"http://planteome.org/"},{"description":"Gramene uses EO for the annotation of plant genes and QTLs","examples":[{"description":"Gramene annotations to cold temperature regiment","url":"http://archive.gramene.org/db/ontology/search?id=EO:0007174"}],"user":"http://gramene.org/"}]},{"activity_status":"inactive","contact":{"email":"david.c.blackburn@gmail.com","label":"David Blackburn"},"homepage":"http://github.com/seger/aao","id":"aao","is_obsolete":true,"layout":"ontology_detail","replaced_by":"uberon","taxon":{"id":"NCBITaxon:8292","label":"Amphibia"},"title":"Amphibian gross anatomy"},{"activity_status":"inactive","contact":{"email":"adw_geeks@umich.edu","label":"Animal Diversity Web technical staff"},"homepage":"http://www.animaldiversity.org","id":"adw","is_obsolete":true,"layout":"ontology_detail","title":"Animal natural history and life history"},{"activity_status":"inactive","contact":{"email":"david.c.blackburn@gmail.com","label":"David Blackburn"},"homepage":"http://www.amphibanat.org","id":"ato","is_obsolete":true,"layout":"ontology_detail","taxon":{"id":"NCBITaxon:8292","label":"Amphibia"},"title":"Amphibian taxonomy"},{"activity_status":"inactive","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"http://4dx.embl.de/4DXpress_4d/edocs/bilateria_mrca.obo"},"contact":{"email":"henrich@embl.de","label":"Thorsten Heinrich"},"domain":"anatomy","homepage":"http://4dx.embl.de/4DXpress","id":"bila","is_obsolete":true,"layout":"ontology_detail","products":[{"id":"bila.owl","ontology_purl":"http://purl.obolibrary.org/obo/bila.owl"}],"replaced_by":"uberon","taxon":{"id":"NCBITaxon:33213","label":"Bilateria"},"title":"Bilateria anatomy"},{"activity_status":"inactive","contact":{"email":"vlee@ebi.ac.uk","label":"Vivian Lee"},"homepage":"http://www.ebi.ac.uk/Rebholz-srv/GRO/GRO.html","id":"bootstrep","is_obsolete":true,"layout":"ontology_detail","replaced_by":"molecular_function","title":"Gene Regulation Ontology"},{"activity_status":"inactive","contact":{"email":"engelsta@ohsu.edu","label":"Mark Engelstad","orcid":"0000-0001-5889-4463"},"homepage":"https://code.google.com/p/craniomaxillofacial-ontology/","id":"cmf","is_obsolete":true,"layout":"ontology_detail","title":"CranioMaxilloFacial ontology"},{"activity_status":"inactive","contact":{"email":"Lindsay.Cowell@utsouthwestern.edu","label":"Lindsay Cowell","orcid":"0000-0003-1617-8244"},"homepage":"http://www.dukeontologygroup.org/Projects.html","id":"dc_cl","is_obsolete":true,"layout":"ontology_detail","replaced_by":"cl","taxon":{"id":"all","label":null},"title":"Dendritic cell"},{"activity_status":"inactive","contact":{"email":"J.Bard@ed.ac.uk","label":"Jonathan Bard"},"homepage":"http://genex.hgu.mrc.ac.uk/","id":"ehda","is_obsolete":true,"layout":"ontology_detail","replaced_by":"ehdaa2","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Human developmental anatomy, timed version"},{"activity_status":"inactive","contact":{"email":"J.Bard@ed.ac.uk","label":"Jonathan Bard"},"homepage":null,"id":"ehdaa","is_obsolete":true,"layout":"ontology_detail","replaced_by":"ehdaa2","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Human developmental anatomy, abstract version"},{"activity_status":"inactive","contact":{"email":"evoc@sanbi.ac.za","label":"eVOC mailing list"},"homepage":"http://www.evocontology.org/","id":"ev","is_obsolete":true,"layout":"ontology_detail","title":"eVOC (Expressed Sequence Annotation for Humans)"},{"activity_status":"inactive","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/flu.owl"},"contact":{"email":"burkesquires@gmail.com","label":"Burke Squires"},"domain":"health","homepage":"http://purl.obolibrary.org/obo/flu/","id":"flu","in_foundry":false,"is_obsolete":true,"layout":"ontology_detail","products":[{"id":"flu.owl","ontology_purl":"http://purl.obolibrary.org/obo/flu.owl"}],"title":"Influenza Ontology","tracker":"http://purl.obolibrary.org/obo/flu/tracker"},{"activity_status":"inactive","contact":{"email":"po-discuss@plantontology.org","label":"Plant Ontology Administrators"},"homepage":"http://www.gramene.org/plant_ontology/","id":"gro","is_obsolete":true,"layout":"ontology_detail","title":"Cereal Plant Gross Anatomy"},{"activity_status":"inactive","contact":{"email":"peteremidford@yahoo.com","label":"Peter Midford","orcid":"0000-0001-6512-3296"},"homepage":"http://www.mesquiteproject.org/ontology/Habronattus/index.html","id":"habronattus","is_obsolete":true,"layout":"ontology_detail","title":"Habronattus courtship"},{"activity_status":"inactive","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"https://web.archive.org/web/20131127090937/http://www.inoh.org/ontologies/MoleculeRoleOntology.obo"},"contact":{"email":"curator@inoh.org","label":"INOH curators"},"homepage":"http://www.inoh.org","id":"imr","is_obsolete":true,"layout":"ontology_detail","title":"Molecule role (INOH Protein name/family name ontology)"},{"activity_status":"inactive","contact":{"email":"interhelp@ebi.ac.uk","label":"InterPro Help"},"homepage":"http://www.ebi.ac.uk/interpro/index.html","id":"ipr","is_obsolete":true,"layout":"ontology_detail","title":"Protein Domains"},{"activity_status":"inactive","contact":{"email":"peteremidford@yahoo.com","label":"Peter Midford","orcid":"0000-0001-6512-3296"},"homepage":"http://www.mesquiteproject.org/ontology/Loggerhead/index.html","id":"loggerhead","is_obsolete":true,"layout":"ontology_detail","title":"Loggerhead nesting"},{"activity_status":"inactive","contact":{"email":"julie@igbmc.u-strasbg.fr","label":"Julie Thompson"},"homepage":"http://www-igbmc.u-strasbg.fr/BioInfo/MAO/mao.html","id":"mao","is_obsolete":true,"layout":"ontology_detail","title":"Multiple alignment"},{"activity_status":"inactive","contact":{"email":"j.bard@ed.ac.uk","label":"Jonathan Bard"},"homepage":null,"id":"mat","is_obsolete":true,"layout":"ontology_detail","title":"Minimal anatomical terminology"},{"activity_status":"inactive","contact":{"email":"smtifahim@gmail.com","label":"Fahim Imam"},"homepage":"http://neuinfo.org/","id":"nif_dysfunction","is_obsolete":true,"layout":"ontology_detail","replaced_by":"doid","title":"NIF Dysfunction"},{"activity_status":"inactive","contact":{"email":"smtifahim@gmail.com","label":"Fahim Imam"},"homepage":"http://neuinfo.org/","id":"nif_grossanatomy","is_obsolete":true,"layout":"ontology_detail","replaced_by":"uberon","title":"NIF Gross Anatomy"},{"activity_status":"inactive","contact":{"email":"cjmungall@lbl.gov","label":"Chris Mungall","orcid":"0000-0002-6601-2165"},"homepage":"http://www.obofoundry.org/ro","id":"obo_rel","is_obsolete":true,"layout":"ontology_detail","replaced_by":"ro","title":"OBO relationship types (legacy)"},{"activity_status":"inactive","contact":{"email":"jaiswalp@science.oregonstate.edu","github":"jaiswalp","label":"Pankaj Jaiswal","orcid":"0000-0002-1005-8383"},"homepage":"http://www.plantontology.org","id":"pao","is_obsolete":true,"layout":"ontology_detail","taxon":{"id":"NCBITaxon:33090","label":"Viridiplantae"},"title":"Plant Anatomy Ontology"},{"activity_status":"inactive","contact":{"email":"henrich@embl.de","label":"Thorsten Heinrich"},"homepage":"http://4dx.embl.de/platy","id":"pd_st","is_obsolete":true,"layout":"ontology_detail","replaced_by":"pdumdv","taxon":{"id":"NCBITaxon:6358","label":"Platynereis"},"title":"Platynereis stage ontology"},{"activity_status":"inactive","contact":{"email":"po-discuss@plantontology.org","label":"Plant Ontology Administrators"},"homepage":"http://www.plantontology.org","id":"pgdso","is_obsolete":true,"layout":"ontology_detail","taxon":{"id":"NCBITaxon:33090","label":"Viridiplantae"},"title":"Plant Growth and Development Stage"},{"activity_status":"inactive","contact":{"email":"mb4@sanger.ac.uk","label":"Matt Berriman"},"homepage":"http://www.sanger.ac.uk/Users/mb4/PLO/","id":"plo","is_obsolete":true,"layout":"ontology_detail","title":"Plasmodium life cycle"},{"activity_status":"inactive","contact":{"email":"satya30@uga.edu","label":"Satya S. Sahoo"},"homepage":"http://lsdis.cs.uga.edu/projects/glycomics/propreo/","id":"propreo","is_obsolete":true,"layout":"ontology_detail","title":"Proteomics data and process provenance"},{"activity_status":"inactive","contact":{"email":"slarson@ncmir.ucsd.edu","label":"Stephen Larson"},"homepage":"http://ccdb.ucsd.edu/CCDBWebSite/sao.html","id":"sao","is_obsolete":true,"layout":"ontology_detail","replaced_by":"go","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Subcellular anatomy ontology"},{"activity_status":"inactive","contact":{"email":"Adrien.Coulet@loria.fr","label":"Adrien Coulet"},"homepage":"http://www.loria.fr/~coulet/sopharm2.0_description.php","id":"sopharm","is_obsolete":true,"layout":"ontology_detail","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Suggested Ontology for Pharmacogenomics"},{"activity_status":"inactive","contact":{"email":"pierre.sprumont@unifr.ch","label":"Pierre Sprumont"},"homepage":null,"id":"tahe","is_obsolete":true,"layout":"ontology_detail","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Terminology of Anatomy of Human Embryology"},{"activity_status":"inactive","contact":{"email":"pierre.sprumont@unifr.ch","label":"Pierre Sprumont"},"homepage":null,"id":"tahh","is_obsolete":true,"layout":"ontology_detail","taxon":{"id":"NCBITaxon:9606","label":"Homo sapiens"},"title":"Terminology of Anatomy of Human Histology"},{"activity_status":"inactive","contact":{"email":"cherry@genome.stanford.edu","label":"Mike Cherry","orcid":"0000-0001-9163-5180"},"homepage":"http://www.yeastgenome.org/","id":"ypo","is_obsolete":true,"layout":"ontology_detail","replaced_by":"apo","taxon":{"id":"NCBITaxon:4932","label":"Saccharomyces cerevisiae"},"title":"Yeast phenotypes"},{"activity_status":"inactive","contact":{"email":"Leszek@missouri.edu","label":"Leszek Vincent","orcid":"0000-0002-9316-2919"},"homepage":"http://www.maizemap.org/","id":"zea","is_obsolete":true,"layout":"ontology_detail","taxon":{"id":"NCBITaxon:4575","label":"Zea"},"title":"Maize gross anatomy"},{"activity_status":"inactive","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/aero.owl"},"contact":{"email":"mcourtot@gmail.com","label":"Melanie Courtot","orcid":"0000-0002-9551-6370"},"description":"The Adverse Event Reporting Ontology (AERO) is an ontology aimed at supporting clinicians at the time of data entry, increasing quality and accuracy of reported adverse events","domain":"health","homepage":"http://purl.obolibrary.org/obo/aero","id":"aero","is_obsolete":true,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"products":[{"id":"aero.owl","ontology_purl":"http://purl.obolibrary.org/obo/aero.owl"}],"title":"Adverse Event Reporting Ontology"},{"activity_status":"inactive","contact":{"email":"maria.herrero@kcl.ac.uk","label":"Maria Herrero"},"description":"A formal represention for drug-drug interactions knowledge.","domain":"health","homepage":"http://labda.inf.uc3m.es/doku.php?id=es:labda_dinto","id":"dinto","is_obsolete":true,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"https://creativecommons.org/licenses/by/3.0/"},"products":[{"id":"dinto.owl","ontology_purl":"http://purl.obolibrary.org/obo/dinto.owl"}],"publications":[{"id":"http://pubs.acs.org/doi/10.1021/acs.jcim.5b00119","title":"DINTO: Using OWL Ontologies and SWRL Rules to Infer DrugDrug Interactions and Their Mechanisms."}],"repository":"https://github.com/labda/DINTO","title":"The Drug-Drug Interactions Ontology","tracker":"https://github.com/labda/DINTO/issues"},{"activity_status":"inactive","build":{"insert_ontology_id":true,"method":"obo2owl","notes":"new url soon","source_url":"ftp://ftp.hgu.mrc.ac.uk/pub/MouseAtlas/Anatomy/EMAP_combined.obo"},"contact":{"email":"Terry.Hayamizu@jax.org","github":"tfhayamizu","label":"Terry Hayamizu","orcid":"0000-0002-0956-8634"},"description":"A structured controlled vocabulary of stage-specific anatomical structures of the mouse (Mus).","domain":"anatomy","homepage":"http://emouseatlas.org","id":"emap","is_obsolete":true,"layout":"ontology_detail","page":"https://www.emouseatlas.org/emap/about/what_is_emap.html","products":[{"id":"emap.owl","ontology_purl":"http://purl.obolibrary.org/obo/emap.owl"}],"replaced_by":"emapa","taxon":{"id":"NCBITaxon:10088","label":"Mus"},"title":"Mouse gross anatomy and development, timed","used_by":[{"label":"GXD","url":"https://www.biosharing.org/biodbcore-000659"}]},{"activity_status":"inactive","build":{"method":"owl2obo","source_url":"http://purl.obolibrary.org/obo/epo.owl"},"description":"An ontology designed to support the semantic annotation of epidemiology resources","homepage":"https://code.google.com/p/epidemiology-ontology/","id":"epo","in_foundry":false,"is_obsolete":true,"layout":"ontology_detail","products":[{"id":"epo.owl","ontology_purl":"http://purl.obolibrary.org/obo/epo.owl"}],"title":"Epidemiology Ontology"},{"activity_status":"inactive","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"http://obo.cvs.sourceforge.net/*checkout*/obo/obo/ontology/taxonomy/fly_taxonomy.obo"},"contact":{"email":"cp390@cam.ac.uk","github":"Clare72","label":"Clare Pilgrim","orcid":"0000-0002-1373-1705"},"description":"The taxonomy of the family <i>Drosophilidae</i> (largely after Baechli) and of other taxa referred to in FlyBase.","domain":"taxonomy","homepage":"http://www.flybase.org/","id":"fbsp","is_obsolete":true,"layout":"ontology_detail","products":[{"id":"fbsp.owl","ontology_purl":"http://purl.obolibrary.org/obo/fbsp.owl"}],"taxon":{"id":"NCBITaxon:7227","label":"Drosophila"},"title":"Fly taxonomy"},{"activity_status":"inactive","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"http://download.baderlab.org/INOH/ontologies/EventOntology_172.obo"},"homepage":"http://www.inoh.org","id":"iev","is_obsolete":true,"layout":"ontology_detail","title":"Event (INOH pathway ontology)"},{"activity_status":"inactive","build":{"method":"owl2obo","source_url":"http://www.lipidprofiles.com/LipidOntology"},"contact":{"email":"bakerc@unb.ca","label":"Christipher Baker"},"description":"An ontology representation of the LIPIDMAPS nomenclature classification.","domain":"lipids","id":"lipro","in_foundry":false,"is_obsolete":true,"layout":"ontology_detail","title":"Lipid Ontology"},{"activity_status":"inactive","contact":{"email":"Thorsten.Henrich@embl-heidelberg.de","label":"Thorsten Henrich"},"description":"A structured controlled vocabulary of the anatomy and development of the Japanese medaka fish, <i>Oryzias latipes</i>.","domain":"anatomy","id":"mfo","is_obsolete":true,"layout":"ontology_detail","products":[{"id":"mfo.owl","ontology_purl":"http://purl.obolibrary.org/obo/mfo.owl"}],"replaced_by":"olatdv","taxon":{"id":"NCBITaxon:8089","label":"Oryzias"},"title":"Medaka fish anatomy and development"},{"activity_status":"inactive","contact":{"email":"topalis@imbb.forth.gr","label":"Pantelis Topalis"},"description":"An application ontology for use with miRNA databases.","homepage":"http://code.google.com/p/mirna-ontology/","id":"mirnao","is_obsolete":true,"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"https://creativecommons.org/publicdomain/zero/1.0/"},"products":[{"id":"mirnao.owl","ontology_purl":"http://purl.obolibrary.org/obo/mirnao.owl"}],"title":"microRNA Ontology"},{"activity_status":"inactive","contact":{"email":"stoeckrt@pcbi.upenn.edu","label":"Chris Stoeckert","orcid":"0000-0002-5714-991X"},"description":"A standardized description of a microarray experiment in support of MAGE v.1.","domain":"experiments","homepage":"http://mged.sourceforge.net/ontologies/MGEDontology.php","id":"mo","is_obsolete":true,"layout":"ontology_detail","page":"http://mged.sourceforge.net/software/downloads.php","products":[{"id":"mo.owl","ontology_purl":"http://purl.obolibrary.org/obo/mo.owl"}],"replaced_by":"obi","title":"Microarray experimental conditions"},{"activity_status":"inactive","contact":{"email":"smtifahim@gmail.com","label":"Fahim Imam"},"description":"Neuronal cell types","homepage":"http://neuinfo.org/","id":"nif_cell","is_obsolete":true,"layout":"ontology_detail","replaced_by":"cl","title":"NIF Cell"},{"activity_status":"inactive","build":{"method":"owl2obo","source_url":"https://msi-workgroups.svn.sourceforge.net/svnroot/msi-workgroups/ontology/NMR.owl"},"contact":{"email":"schober@imbi.uni-freiburg.de","label":"Schober Daniel"},"description":"Descriptors relevant to the experimental conditions of the Nuclear Magnetic Resonance (NMR) component in a metabolomics investigation.","domain":"experiments","homepage":"http://msi-ontology.sourceforge.net/","id":"nmr","is_obsolete":true,"layout":"ontology_detail","page":"http://msi-ontology.sourceforge.net/ontology/NMR.owlDocument.doc","products":[{"id":"nmr.owl","ontology_purl":"http://purl.obolibrary.org/obo/nmr.owl"}],"title":"NMR-instrument specific component of metabolomics investigations"},{"activity_status":"inactive","contact":{"email":"mbrochhausen@gmail.com","label":"Mathias Brochhausen","orcid":"0000-0003-1834-3856"},"description":"An ontological version of MIABIS (Minimum Information About BIobank data Sharing)","domain":"health","homepage":"https://github.com/OMIABIS/omiabis-dev","id":"omiabis","is_obsolete":true,"layout":"ontology_detail","license":{"label":"CC BY 3.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png","url":"http://creativecommons.org/licenses/by/3.0/"},"products":[{"id":"omiabis.owl","ontology_purl":"http://purl.obolibrary.org/obo/omiabis.owl"}],"repository":"https://github.com/OMIABIS/omiabis-dev","title":"Ontologized MIABIS","tracker":"https://github.com/OMIABIS/omiabis-dev/issues"},{"activity_status":"inactive","contact":{"email":"john.garavelli@ebi.ac.uk","label":"John Garavelli"},"description":"For the description of covalent bonds in proteins.","domain":"proteins","homepage":"http://www.ebi.ac.uk/RESID/","id":"resid","is_obsolete":true,"layout":"ontology_detail","title":"Protein covalent bond"},{"activity_status":"inactive","build":{"insert_ontology_id":true,"method":"obo2owl","source_url":"https://raw.githubusercontent.com/HUPO-PSI/gelml/master/CV/sep.obo"},"contact":{"email":"psidev-gps-dev@lists.sourceforge.net","label":"SEP developers via the PSI and MSI mailing lists"},"description":"A structured controlled vocabulary for the annotation of sample processing and separation techniques in scientific experiments.","domain":"provenance","homepage":"http://psidev.info/index.php?q=node/312","id":"sep","is_obsolete":true,"layout":"ontology_detail","page":"http://psidev.info/index.php?q=node/312","products":[{"id":"sep.owl","ontology_purl":"http://purl.obolibrary.org/obo/sep.owl"}],"title":"Sample processing and separation techniques"},{"activity_status":"inactive","build":{"method":"obo2owl","source_url":"http://purl.obolibrary.org/obo/tao.obo"},"contact":{"email":"wasila.dahdul@usd.edu","label":"Wasila Dahdul","orcid":"0000-0003-3162-7490"},"description":"Multispecies fish anatomy ontology. Originally seeded from ZFA, but intended to cover terms relevant to other taxa","domain":"anatomy","homepage":"http://wiki.phenoscape.org/wiki/Teleost_Anatomy_Ontology","id":"tao","is_obsolete":true,"layout":"ontology_detail","license":{"label":"CC0 1.0","logo":"http://mirrors.creativecommons.org/presskit/buttons/80x15/png/cc-zero.png","url":"http://creativecommons.org/publicdomain/zero/1.0/"},"products":[{"id":"tao.owl","ontology_purl":"http://purl.obolibrary.org/obo/tao.owl"}],"publications":[{"id":"https://www.ncbi.nlm.nih.gov/pubmed/20547776","title":"The teleost anatomy ontology: anatomical representation for the genomics age"}],"replaced_by":"uberon","taxon":{"id":"NCBITaxon:32443","label":"Teleostei"},"title":"Teleost Anatomy Ontology"},{"activity_status":"inactive","id":"vhog","is_obsolete":true,"layout":"ontology_detail","products":[{"id":"vhog.owl","ontology_purl":"http://purl.obolibrary.org/obo/vhog.owl"}],"replaced_by":"uberon","title":"Vertebrate Homologous Ontology Group Ontology"},{"activity_status":"inactive","build":{"method":"obo2owl","source_url":"http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/tags/vocab-releases/VSAO/vsao.obo"},"contact":{"email":"wasila.dahdul@usd.edu","label":"Wasila Dahdul","orcid":"0000-0003-3162-7490"},"description":"Vertebrate skeletal anatomy ontology.","domain":"anatomy","homepage":"https://www.nescent.org/phenoscape/Main_Page","id":"vsao","is_obsolete":true,"layout":"ontology_detail","page":"https://www.phenoscape.org/wiki/Skeletal_Anatomy_Jamboree","products":[{"id":"vsao.owl","ontology_purl":"http://purl.obolibrary.org/obo/vsao.owl"}],"replaced_by":"uberon","taxon":{"id":"NCBITaxon:7742","label":"Vertebrata"},"title":"Vertebrate Skeletal Anatomy Ontology-"}]}')},"867a":function(e,o,t){},8684:function(e,o,t){"use strict";t("7fb9")},8697:function(e,o,t){"use strict";t("975b")},"884b":function(e,o,t){e.exports=t.p+"img/eric-douglass.bbbfb3f5.jpg"},"89c5":function(e,o,t){},"8af4":function(e,o,t){},"8c9a":function(e,o,t){"use strict";t("6113")},9184:function(e,o,t){e.exports=t.p+"img/rgd.41c5460d.png"},9223:function(e,o,t){"use strict";t("19e4")},9318:function(e,o,t){e.exports=t.p+"img/ncbi.350aa411.png"},9392:function(e,o,t){e.exports=t.p+"img/pittsburgh.18cf7bd5.png"},"940f":function(e,o,t){},"975b":function(e,o,t){},9808:function(e,o,t){e.exports=t.p+"img/decipher.0e5a9173.png"},9918:function(e,o,t){},"993b":function(e){e.exports=JSON.parse('[{"year":2021,"list":[{"title":"Knowledge Beacons: Web services for data harvesting of distributed biomedical knowledge","authors":"Hannestad LM, Danck V, Godden M, Suen IW, Huellas-Bruskiewicz KC, Good BM, Mungall CJ, Bruskiewich RM","journal":"PLoS One","issue":"16(3):e0231916","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/33755673/)"},{"title":"The Human Phenotype Ontology in 2021","authors":"Khler S, Gargano M, Matentzoglu N, Carmody LC, Lewis-Smith D, Vasilevsky NA, Danis D, Balagura G, Baynam G, Brower AM, Callahan TJ, Chute CG, Est JL, Galer PD, Ganesan S, Griese M, Haimel M, Pazmandi J, Hanauer M, Harris NL, Hartnett MJ, Hastreiter M, Hauck F, He Y, Jeske T, Kearney H, Kindle G, Klein C, Knoflach K, Krause R, Lagorce D, McMurry JA, Miller JA, Munoz-Torres MC, Peters RL, Rapp CK, Rath AM, Rind SA, Rosenberg AZ, Segal MM, Seidel MG, Smedley D, Talmy T, Thomas Y, Wiafe SA, Xian J, Yksel Z, Helbig I, Mungall CJ, Haendel MA, Robinson PN","journal":"Nucleic Acids Research","issue":"49(D1):D1207-D1217","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/33264411/)"},{"title":"PhenoTagger: A Hybrid Method for Phenotype Concept Recognition using Human Phenotype Ontology","authors":"Luo L, Yan S, Lai PT, Veltri D, Oler A, Xirasagar S, Ghosh R, Similuk M, Robinson PN, Lu Z","journal":"Bioinformatics","issue":"btab019","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/33471061/)"},{"title":"KG-COVID-19: A Framework to Produce Customized Knowledge Graphs for COVID-19 Response","authors":"Reese JT, Unni D, Callahan TJ, Cappelletti L, Ravanmehr V, Carbon S, Shefchek KA, Good BM, Balhoff JP, Fontana T, Blau H, Matentzoglu N, Harris NL, Munoz-Torres MC, Haendel MA, Robinson PN, Joachimiak MP, Mungall CJ","journal":"Patterns (N Y)","issue":"2(1):100155","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/33196056/)"},{"title":"Is authorship sufficient for today\'s collaborative research? A call for contributor roles","authors":"Vasilevsky NA, Hosseini M, Teplitzky S, Ilik V, Mohammadi E, Schneider J, Kern B, Colomb J, Edmunds SC, Gutzman K, Himmelstein DS, White M, Smith B, O\'Keefe L, Haendel M, Holmes KL","journal":"Account Res","issue":"28(1):23-43","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32602379/)"}]},{"year":2020,"list":[{"title":"Significantly different clinical phenotypes associated with mutations in synthesis and transamidase+remodeling glycosylphosphatidylinositol (GPI)-anchor biosynthesis genes","authors":"Carmody LC, Blau H, Danis D, Zhang XA, Gourdine JP, Vasilevsky N, Krawitz P, Thompson MD, Robinson PN","journal":"Orphanet J Rare Dis","issue":"15(1):40","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32019583/)"},{"title":"Phenotate: crowdsourcing phenotype annotations as exercises in undergraduate classes","authors":"Chang WH, Mashouri P, Lozano AX, Johnstone B, Husic M, Olry A, Maiella S, Balci TB, Sawyer SL, Robinson PN, Rath A, Brudno M","journal":"Genet Med","issue":"22(8):1391-1400","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32366968/)"},{"title":"An Improved Phenotype-Driven Tool for Rare Mendelian Variant Prioritization: Benchmarking Exomiser on Real Patient Whole-Exome Data","authors":"Cipriani V, Pontikos N, Arno G, Sergouniotis PI, Lenassi E, Thawong P, Danis D, Michaelides M, Webster AR, Moore AT, Robinson PN, Jacobsen JOB, Smedley D","journal":"Genes (Basel)","issue":"11(4):460","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32340307/)"},{"title":"Structuring, reuse and analysis of electronic dental data using the Oral Health and Disease Ontology","authors":"Duncan WD, Thyvalikakath T, Haendel M, Torniai C, Hernandez P, Song M, Acharya A, Caplan DJ, Schleyer T, Ruttenberg A","journal":"J Biomed Semantics","issue":"11(1):8","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32819435/)"},{"title":"Supplementation of the ESID registry working definitions for the clinical diagnosis of inborn errors of immunity with encoded human phenotype ontology (HPO) terms","authors":"Gasteiger LM, Robinson PN, Pazmandi J, Boztug K, Seppnen MRJ, Seidel MG; Registry Working Party of the European Society for Immunodeficiencies (ESID)","journal":"J Allergy Clin Immunol Pract","issue":"8(5):1778","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32389282/)"},{"title":"How many rare diseases are there?","authors":"Haendel M, Vasilevsky N, Unni D, Bologa C, Harris N, Rehm H, Hamosh A, Baynam G, Groza T, McMurry J, Dawkins H, Rath A, Thaxon C, Bocci G, Joachimiak MP, Khler S, Robinson PN, Mungall C, Oprea TI","journal":"Nat Rev Drug Discov","issue":"19(2):77-78","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32020066/)"},{"title":"HBA-DEALS: accurate and simultaneous identification of differential expression and splicing using hierarchical Bayesian analysis","authors":"Karlebach G, Hansen P, Veiga DF, Steinhaus R, Danis D, Li S, Anczukow O, Robinson PN","journal":"Genome Biol","issue":"21(1):171","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32660516/)"},{"title":"Modelling kidney disease using ontology: insights from the Kidney Precision Medicine Project","authors":"Ong E, Wang LL, Schaub J, O\'Toole JF, Steck B, Rosenberg AZ, Dowd F, Hansen J, Barisoni L, Jain S, de Boer IH, Valerius MT, Waikar SS, Park C, Crawford DC, Alexandrov T, Anderton CR, Stoeckert C, Weng C, Diehl AD, Mungall CJ, Haendel M, Robinson PN, Himmelfarb J, Iyengar R, Kretzler M, Mooney S, He Y","journal":"Nat Rev Nephrol","issue":"16(11):686-696","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32939051/)"},{"title":"parSMURF, a high-performance computing tool for the genome-wide detection of pathogenic variants","authors":"Petrini A, Mesiti M, Schubach M, Frasca M, Danis D, Re M, Grossi G, Cappelletti L, Castrignan T, Robinson PN, Valentini G","journal":"Gigascience","issue":"9(5):giaa052","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32444882/)"},{"title":"Ontologies, Knowledge Representation, and Machine Learning for Translational Research: Recent Contributions","authors":"Robinson PN, Haendel MA","journal":"Yearb Med Inform","issue":"29(1):159-162","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32823310/)"},{"title":"Interpretable Clinical Genomics with a Likelihood Ratio Paradigm","authors":"Robinson PN, Ravanmehr V, Jacobsen JOB, Danis D, Zhang XA, Carmody LC, Gargano MA, Thaxton CL, Karlebach G, Reese J, Holtgrewe M, Khler S, McMurry JA, Haendel MA, Smedley D","journal":"Am J Hum Genet","issue":"107(3):403-417","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32755546/)"},{"title":"The case for open science: rare diseases","authors":"Rubinstein YR, Robinson PN, Gahl WA, Avillach P, Baynam G, Cederroth H, Goodwin RM, Groft SC, Hansson MG, Harris NL, Huser V, Mascalzoni D, McMurry JA, Might M, Nellaker C, Mons B, Paltoo DN, Pevsner J, Posada M, Rockett-Frase AP, Roos M, Rubinstein TB, Taruscio D, van Enckevort E, Haendel MA","journal":"JAMIA Open","issue":"3(3):472-486","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/33426479/)"},{"title":"The Monarch Initiative in 2019: an integrative data and analytic platform connecting phenotypes to genotypes across species","authors":"Shefchek KA, Harris NL, Gargano M, Matentzoglu N, Unni D, Brush M, Keith D, Conlin T, Vasilevsky N, Zhang XA, Balhoff JP, Babb L, Bello SM, Blau H, Bradford Y, Carbon S, Carmody L, Chan LE, Cipriani V, Cuzick A, Della Rocca M, Dunn N, Essaid S, Fey P, Grove C, Gourdine JP, Hamosh A, Harris M, Helbig I, Hoatlin M, Joachimiak M, Jupp S, Lett KB, Lewis SE, McNamara C, Pendlington ZM, Pilgrim C, Putman T, Ravanmehr V, Reese J, Riggs E, Robb S, Roncaglia P, Seager J, Segerdell E, Similuk M, Storm AL, Thaxon C, Thessen A, Jacobsen JOB, McMurry JA, Groza T, Khler S, Smedley D, Robinson PN, Mungall CJ, Haendel MA, Munoz-Torres MC, Osumi-Sutherland D","journal":"Nucleic Acids Res","issue":"48(D1):D704-D715","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31701156)"},{"title":"A phenome-wide association study of 26 mendelian genes reveals phenotypic expressivity of common and rare variants within the general population","authors":"Tcheandjieu C, Aguirre M, Gustafsson S, Saha P, Potiny P, Haendel M, Ingelsson E, Rivas MA, Priest JR","journal":"PLoS Genet","issue":"16(11):e1008802","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/33226994/)"},{"title":"Transforming the study of organisms: Phenomic data models and knowledge bases","authors":"Thessen AE, Walls RL, Vogt L, Singer J, Warren R, Buttigieg PL, Balhoff JP, Mungall CJ, McGuinness DL, Stucky BJ, Yoder MJ, Haendel MA","journal":"PLoS Comput Biol","issue":"16(11):e1008376","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/33232313/)"},{"title":"A harmonized meta-knowledgebase of clinical interpretations of somatic genomic variants in cancer","authors":"Wagner AH, Walsh B, Mayfield G, Tamborero D, Sonkin D, Krysiak K, Deu-Pons J, Duren RP, Gao J, McMurry J, Patterson S, Del Vecchio Fitz C, Pitel BA, Sezerman OU, Ellrott K, Warner JL, Rieke DT, Aittokallio T, Cerami E, Ritter DI, Schriml LM, Freimuth RR, Haendel M, Raca G, Madhavan S, Baudis M, Beckmann JS, Dienstmann R, Chakravarty D, Li XS, Mockus S, Elemento O, Schultz N, Lopez-Bigas N, Lawler M, Goecks J, Griffith M, Griffith OL, Margolin AA","journal":"Nat Genet","issue":"52(4):448-457","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/32246132/)"}]},{"year":2019,"list":[{"title":"Clinical Data: Sources and Types, Regulatory Constraints, Applications","authors":"Ahalt SC, Chute CG, Fecho K, Glusman G, Hadlock J, Taylor CO, Pfaff ER, Robinson PN, Solbrig H, Ta C, Tatonetti N, Weng C; Biomedical Data Translator Consortium","journal":"Clin Transl Sci","issue":"12(4):329-333","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/31074176/)"},{"title":"Assessment of Bones Deficient in Fibrillin-1 Microfibrils Reveals Pronounced Sex Differences","authors":"Altinbas L, Bormann N, Lehmann D, Jeuthe S, Wulsten D, Kornak U, Robinson PN, Wildemann B, Kararigas G","journal":"Int J Mol Sci","issue":"20(23):6059","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/31805661)"},{"title":"New models for human disease from the International Mouse Phenotyping Consortium","authors":"Cacheiro P, Haendel MA, Smedley D; International Mouse Phenotyping Consortium and the Monarch Initiative","journal":"Mamm Genome","issue":"30(5-6):143-150","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31127358)"},{"title":"An analysis and metric of reusable data licensing practices for biomedical resources","authors":"Carbon S, Champieux R, McMurry JA, Winfree L, Wyatt LR, Haendel MA","journal":"PLoS One","issue":"14(3):e0213090","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/30917137)"},{"title":"Representing glycophenotypes: semantic unification of glycobiology resources for disease discovery","authors":"Gourdine JF, Brush MH, Vasilevsky NA, Shefchek K, Khler S, Matentzoglu N, Munoz-Torres MC, McMurry JA, Zhang XA, Robinson PN, Haendel MA","journal":"Database (Oxford)","issue":"baz114","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31735951)"},{"title":"ROBOT: A Tool for Automating Ontology Workflows","authors":"Jackson RC, Balhoff JP, Douglass E, Harris NL, Mungall CJ, Overton JA","journal":"BMC Bioinformatics","issue":"20(1):407","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31357927)"},{"title":"Expansion of the Human Phenotype Ontology (HPO) knowledge base and resources","authors":"Khler S, Carmody L, Vasilevsky N, Jacobsen JOB, Danis D, Gourdine JP, Gargano M, Harris NL, Matentzoglu N, McMurry JA, Osumi-Sutherland D, Cipriani V, Balhoff JP, Conlin T, Blau H, Baynam G, Palmer R, Gratian D, Dawkins H, Segal M, Jansen AC, Muaz A, Chang WH, Bergerson J, Laulederkind SJF, Yksel Z, Beltran S, Freeman AF, Sergouniotis PI, Durkin D, Storm AL, Hanauer M, Brudno M, Bello SM, Sincan M, Rageth K, Wheeler MT, Oegema R, Lourghi H, Della Rocca MG, Thompson R, Castellanos F, Priest J, Cunningham-Rundles C, Hegde A, Lovering RC, Hajek C, Olry A, Notarangelo L, Similuk M, Zhang XA, Gmez-Andrs D, Lochmller H, Dollfus H, Rosenzweig S, Marwaha S, Rath A, Sullivan K, Smith C, Milner JD, Leroux D, Boerkoel CF, Klion A, Carter MC, Groza T, Smedley D, Haendel MA, Mungall C, Robinson PN","journal":"Nucleic Acids Res","issue":"47(D1):D1018-D1027","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/30476213)"},{"title":"Encoding Clinical Data with the Human Phenotype Ontology for Computational Differential Diagnostics","authors":"Khler S, ien NC, Buske OJ, Groza T, Jacobsen JOB, McNamara C, Vasilevsky N, Carmody LC, Gourdine JP, Gargano M, McMurry JA, Danis D, Mungall CJ, Smedley D, Haendel M, Robinson PN","journal":"Curr Protoc Hum Gene","issue":"103(1):e92","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31479590)"},{"title":"Enabling Global Clinical Collaborations on Identifiable Patient Data: The Minerva Initiative","authors":"Nellker C, Alkuraya FS, Baynam G, Bernier RA, Bernier FPJ, Boulanger V, Brudno M, Brunner HG, Clayton-Smith J, Cogn B, Dawkins HJS, deVries BBA, Douzgou S, Dudding-Byth T, Eichler EE, Ferlaino M, Fieggen K, Firth HV, FitzPatrick DR, Gration D, Groza T, Haendel M, Hallowell N, Hamosh A, Hehir-Kwa J, Hitz MP, Hughes M, Kini U, Kleefstra T, Kooy RF, Krawitz P, Kry S, Lees M, Lyon GJ, Lyonnet S, Marcadier JL, Meyn S, Moslerov V, Politei JM, Poulton CC, Raymond FL, Reijnders MRF, Robinson PN, Romano C, Rose CM, Sainsbury DCG, Schofield L, Sutton VR, Turnovec M, Van Dijck A, Van Esch H, Wilkie AOM; Minerva Consortium","journal":"Front Genet","issue":"10:611","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31417602)"},{"title":"An ontological foundation for ocular phenotypes and rare eye diseases","authors":"Sergouniotis PI, Maxime E, Leroux D, Olry A, Thompson R, Rath A, Robinson PN, Dollfus H; ERN-EYE Ontology Study Group","journal":"Orphanet J Rare Dis","issue":"14(1):8","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/30626441)"},{"title":"The Sickle Cell Disease Ontology: enabling universal sickle cell-based knowledge representation","authors":"Sickle Cell Disease Ontology Working Group","journal":"Database (Oxford)","issue":"","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31769834)"},{"title":"The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution","authors":"Taylor DM, Aronow BJ, Tan K, Bernt K, Salomonis N, Greene CS, Frolova A, Henrickson SE, Wells A, Pei L, Jaiswal JK, Whitsett J, Hamilton KE, MacParland SA, Kelsen J, Heuckeroth RO, Potter SS, Vella LA, Terry NA, Ghanem LR, Kennedy BC, Helbig I, Sullivan KE, Castelo-Soccio L, Kreigstein A, Herse F, Nawijn MC, Koppelman GH, Haendel M, Harris NL, Rokita JL, Zhang Y, Regev A, Rozenblatt-Rosen O, Rood JE, Tickle TL, Vento-Tormo R, Alimohamed S, Lek M, Mar JC, Loomes KM, Barrett DM, Uapinyoying P, Beggs AH, Agrawal PB, Chen YW, Muir AB, Garmire LX, Snapper SB, Nazarian J, Seeholzer SH, Fazelinia H, Singh LN, Faryabi RB, Raman P, Dawany N, Xie HM, Devkota B, Diskin SJ, Anderson SA, Rappaport EF, Peranteau W, Wikenheiser-Brokamp KA, Teichmann S, Wallace D, Peng T, Ding YY, Kim MS, Xing Y, Kong SW, Bnnemann CG, Mandl KD, White PS","journal":"Dev Cell","issue":"49(1):10-29","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/30930166)"},{"title":"The Plant Ontology Facilitates Comparisons of Plant Development Stages Across Species","authors":"Walls RL, Cooper L, Elser J, Gandolfo MA, Mungall CJ, Smith B, Stevenson DW, Jaiswal P","journal":"Front Plant Sci","issue":"10:631","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31214208)"},{"title":"Semantic integration of clinical laboratory tests from electronic health records for deep phenotyping and biomarker discovery","authors":"Zhang XA, Yates A, Vasilevsky N, Gourdine JP, Callahan TJ, Carmody LC, Danis D, Joachimiak MP, Ravanmehr V, Pfaff ER, Champion J, Robasky K, Xu H, Fecho K, Walton NA, Zhu RL, Ramsdill J, Mungall CJ, Khler S, Haendel MA, McDonald CJ, Vreeman DJ, Peden DB, Bennett TD, Feinstein JA, Martin B, Stefanski AL, Hunter LE, Chute CG, Robinson PN","journal":"NPJ Digit Med","issue":"2:32","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/31119199)"}]},{"year":2018,"list":[{"title":"Classification, Ontology, and Precision Medicine (2018)","authors":"Haendel MA, Chute CG, Robinson PN 2018","journal":"N Engl J Med","issue":"379(15):1452-1462","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/30304648)"},{"title":"Harmonising phenomics information for a better Interoperability in the rare disease field","authors":"Maiella S, Olry A, Hanauer M, Lanneau V, Lourghi H, Donadille B, Rodwell C, Khler S, Seelow D, Jupp S, Parkinson H, Groza T, Brudno M, Robinson PN, Rath A","journal":"Eur J Med Genet","issue":"61(11):706-714","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/30304648)"},{"title":"Classification, Ontology, and Precision Medicine","authors":"Muoz-Fuentes V, Cacheiro P, Meehan TF, Aguilar-Pimentel JA, Brown SDM, Flenniken AM, Flicek P, Galli A, Mashhadi HH, Hrabe de Angelis M, Kim JK, Lloyd KCK, McKerlie C, Morgan H, Murray SA, Nutter LMJ, Reilly PT, Seavitt JR, Seong JK, Simon M, Wardle-Jones H, Mallon AM, Smedley D, Parkinson HE; IMPC consortium","journal":"Conserv Genet","issue":"19(4):995-1005","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/30100824)"},{"title":"Plain-language medical vocabulary for precision diagnosis","authors":"Vasilevsky NA, Foster ED, Engelstad ME, Carmody L, Might M, Chambers C, Dawkins HJS, Lewis J, Della Rocca MG, Snyder M, Boerkoel CF, Rath A, Terry SF, Kent A, Searle B, Baynam G, Jones E, Gavin P, Bamshad M, Chong J, Groza T, Adams D, Resnick AC, Heath AP, Mungall C, Holm IA, Rageth K, Brownstein CA, Shefchek K, McMurry JA, Robinson PN, Khler S, Haendel MA.(2018)","journal":"Nat Genet","issue":"50(4):474-476","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/29632381)"}]},{"year":2017,"list":[{"title":"The Monarch Initiative: An Integrative Data and Analytic Platform Connecting Phenotypes to Genotypes across Species","authors":"Mungall CJ, McMurry JA, Khler S, Balhoff JP, Borromeo C, Brush M, Carbon S, Conlin T, Dunn N, Engelstad M, Foster E, Gourdine JP, Jacobsen JO, Keith D, Laraway B, Lewis SE, NguyenXuan J, Shefchek K, Vasilevsky N, Yuan Z,Washington N, Hochheiser H, Groza T, Smedley D, Robinson PN, Haendel MA","journal":"Nucleic Acids Research","issue":"45(D1):D71222","link":"[pubmed](https://www.ncbi.nlm.nih.gov/pubmed/27899636)"},{"title":"The Human Phenotype Ontology in 2017","authors":"Khler S, Vasilevsky NA, Engelstad M, Foster E, McMurry J, Aym S, Baynam G, Bello SM, Boerkoel CF, Boycott KM, Brudno M, Buske OJ, Chinnery PF, Cipriani V, Connell LE, Dawkins HJ, DeMare LE, Devereau AD, de Vries BB, Firth HV, Freson K, Greene D, Hamosh A, Helbig I, Hum C, Jhn JA, James R, Krause R, F Laulederkind SJ, Lochmller H, Lyon GJ, Ogishima S, Olry A, Ouwehand WH, Pontikos N, Rath A, Schaefer F, Scott RH, Segal M, Sergouniotis PI, Sever R, Smith CL, Straub V, Thompson R, Turner C, Turro E, Veltman MW, Vulliamy T, Yu J, von Ziegenweidt J, Zankl A, Zchner S, Zemojtel T, Jacobsen JO, Groza T, Smedley D, Mungall CJ, Haendel M, Robinson PN","journal":"Nucleic Acids Research","issue":"45(D1): D86576","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27899602)"},{"title":"Identifiers for the 21st Century: How to Design, Provision, and Reuse Persistent Identifiers to Maximize Utility and Impact of Life Science Data","authors":"McMurry JA, Juty N, Blomberg N, Burdett T, Conlin T, Conte N, Courtot M, Deck J, Dumontier M, Fellows DK, Gonzalez-Beltran A, Gormanns P, Grethe J, Hastings J, Hrich JK, Hermjakob H, Ison JC, Jimenez RC, Jupp S, Kunze J, Laibe C, Le Novre N, Malone J, Martin MJ, McEntyre JR, Morris C, Muilu J, Mller W, Rocca-Serra P, Sansone SA, Sariyar M, Snoep JL, Soiland-Reyes S, Stanford NJ, Swainston N, Washington N, Williams AR, Wimalaratne SM, Winfree LM, Wolstencroft K, Goble C, Mungall CJ, Haendel MA, Parkinson H","journal":"PLoS Biology","issue":"15(6):e2001414","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/28662064)"},{"title":"Disease model discovery from 3,328 gene knockouts by The International Mouse Phenotyping Consortium","authors":"Meehan TF, Conte N, West DB, Jacobsen JO, Mason J, Warren J, Chen CK, Tudose I, Relac M, Matthews P, Karp N, Santos L, Fiegel T, Ring N, Westerberg H, Greenaway S, Sneddon D, Morgan H, Codner GF, Stewart ME, Brown J, Horner N; International Mouse Phenotyping Consortium, Haendel M, Washington N, Mungall CJ, Reynolds CL, Gallegos J, Gailus-Durner V, Sorg T, Pavlovic G, Bower LR, Moore M, Morse I, Gao X, Tocchini-Valentini GP, Obata Y, Cho SY, Seong JK, Seavitt J, Beaudet AL, Dickinson ME, Herault Y, Wurst W, de Angelis MH, Lloyd KCK, Flenniken AM, Nutter LMJ, Newbigging S, McKerlie C, Justice MJ, Murray SA, Svenson KL, Braun RE, White JK, Bradley A, Flicek P, Wells S, Skarnes WC, Adams DJ, Parkinson H, Mallon AM, Brown SDM, Smedley D","journal":"Nat Genet","issue":"49(8):1231-1238","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/28650483)"},{"title":"Once Doesn\'t Count: Phenotype-Driven Gene Hunting in Cohorts","authors":"Robinson PN","journal":"Human Mutat","issue":"38(5):469","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/28425197)"},{"title":"International Cooperation to Enable the Diagnosis of All Rare Genetic Diseases","authors":"Boycott KM, Rath A, Chong JX, Hartley T, Alkuraya FS, Baynam G, Brookes AJ, Brudno M, Carracedo A, den Dunnen JT, Dyke SOM, Estivill X, Goldblatt J, Gonthier C, Groft SC, Gut I, Hamosh A, Hieter P, Hhn S, Hurles ME, Kaufmann P, Knoppers BM, Krischer JP, Macek M Jr, Matthijs G, Olry A, Parker S, Paschall J, Philippakis AA, Rehm HL, Robinson PN, Sham PC, Stefanov R, Taruscio D, Unni D, Vanstone MR, Zhang F, Brunner H, Bamshad MJ, Lochmller H","journal":"Am J Hum Genet","issue":"100(5):695-705","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/28475856)"},{"title":"Defining Disease, Diagnosis, and Translational Medicine within a Homeostatic Perturbation Paradigm: The National Institutes of Health Undiagnosed Diseases Program Experience","authors":"Gall T, Valkanas E, Bello C, Markello T, Adams C, Bone WP, Brandt AJ, Brazill JM, Carmichael L, Davids M, Davis J, Diaz-Perez Z, Draper D, Elson J, Flynn ED, Godfrey R, Groden C, Hsieh CK, Fischer R, Golas GA, Guzman J, Huang Y, Kane MS, Lee E, Li C, Links AE, Maduro V, Malicdan MCV, Malik FS, Nehrebecky M, Park J, Pemberton P, Schaffer K, Simeonov D, Sincan M, Smedley D, Valivullah Z, Wahl C, Washington N, Wolfe LA, Xu K, Zhu Y, Gahl WA, Tifft CJ, Toro C, Adams DR, He M, Robinson PN, Haendel MA, Zhai RG, Boerkoel CF","journal":"Front Med (Lausanne)","issue":"4:62","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/28603714)"},{"title":"From SNOMED CT to Uberon: Transferability of Evaluation Methodology between Similarly Structured Ontologies","authors":"Elhanan G, Ochs C, Mejino JLV Jr, Liu H, Mungall CJ, Perl Y","journal":"Artif Intell Med","issue":"79:9-14","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/28532962)"},{"title":"\\"Opposite-of\\"-Information Improves Similarity Calculations in Phenotype Ontologies","authors":"Koehler S, Robinson PN, Mungall CJ","journal":"bioRxiv","issue":"doi:10.1101/108977","link":"[bioRxiv](https://www.biorxiv.org/content/10.1101/108977v1)"},{"title":"Ontobee: A linked ontology data server to support ontology term dereferencing, linkage, query and integration","authors":"Ong E, Xiang Z, Zhao B, Liu Y, Lin Y, Zheng J, Mungall C, Courtot M, Ruttenberg A, He Y","journal":"Nucleic Acids Res","issue":"45(D1):D347-D352","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27733503)"}]},{"year":2016,"list":[{"title":"Navigating the Phenotype Frontier: The Monarch Initiative","authors":"McMurry JA, Khler S, Washington NL, Balhoff JP, Borromeo C, Brush M, Carbon S, Conlin T, Dunn N, Engelstad M, Foster E, Gourdine JP, Jacobsen JO, Keith D, Laraway B, Xuan JN, Shefchek K, Vasilevsky NA, Yuan Z, Lewis SE, Hochheiser H, Groza T, Smedley D, Robinson PN, Mungall CJ, Haendel MA","journal":"Genetics","issue":"203(4):1491-5","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27516611)"},{"title":"The health care and life sciences community profile for dataset descriptions","authors":"Dumontier M, Gray AJG, Marshall MS, Alexiev V, Ansell P, Bader G, Baran J, Bolleman JT, Callahan A, Cruz-Toledo J, Gaudet P, Gombocz EA, Gonzalez-Beltran AN, Groth P, Haendel M, Ito M, Jupp S, Juty N, Katayama T, Kobayashi N, Krishnaswami K, Laibe C, Le Novre N, Lin S, Malone J, Miller M, Mungall CJ, Rietveld L, Wimalaratne SM, Yamaguchi A","journal":"PeerJ","issue":"4:e2331","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27602295)"},{"title":"Reproducible and reusable research: are journal data sharing policies meeting the mark?","authors":"Vasilevsky NA, Minnier J, Haendel MA, Champieux RE","journal":"PeerJ","issue":"5:e3208","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/28462024)"},{"title":"Distributed Cognition and Process Management Enabling Individualized Translational Research: The NIH Undiagnosed Diseases Program Experience","authors":"Links AE, Draper D, Lee E, Guzman J, Valivullah Z, Maduro V, Lebedev V, Didenko M, Tomlin G, Brudno M, Girdea M, Dumitriu S, Haendel MA, Mungall CJ, Smedley D, Hochheiser H, Arnold AM, Coessens B, Verhoeven S, Bone W, Adams D, Boerkoel CF, Gahl WA, Sincan M","journal":"Front Med (Lausanne)","issue":"3:39","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27785453)"},{"title":"K-BOOM: A Bayesian Approach to Ontology Structure Inference, with Applications in Disease Ontology Construction","authors":"Mungall CJ, Koehler S, Robinson P, Holmes I, Haendel M","journal":"bioRxiv","issue":"doi:10.1101/048843","link":"[bioRxiv](https://www.biorxiv.org/content/10.1101/048843v3)"},{"title":"The Ontology for Biomedical Investigations","authors":"Bandrowski A, Brinkman R, Brochhausen M, Brush MH, Bug B, Chibucos MC, Clancy K, Courtot M, Derom D, Dumontier M, Fan L, Fostel J, Fragoso G, Gibson F, Gonzalez-Beltran A, Haendel MA, He Y, Heiskanen M, Hernandez-Boussard T, Jensen M, Lin Y, Lister AL, Lord P, Malone J, Manduchi E, McGee M, Morrison N, Overton JA, Parkinson H, Peters B, Rocca-Serra P, Ruttenberg A, Sansone SA, Scheuermann RH, Schober D, Smith B, Soldatova LN, Stoeckert CJ Jr, Taylor CF, Torniai C, Turner JA, Vita R, Whetzel PL, Zheng J","journal":"PLoS One","issue":"11(4):e0154556","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27128319)"},{"title":"The Cell Ontology 2016: enhanced content, modularization, and ontology interoperability","authors":"Diehl AD, Meehan TF, Bradford YM, Brush MH, Dahdul WM, Dougall DS, He Y, Osumi-Sutherland D, Ruttenberg A, Sarntivijai S, Van Slyke CE, Vasilevsky NA, Haendel MA, Blake JA, Mungall CJ","journal":"J Biomed Semantics","issue":"7(1):44","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27377652)"},{"title":"The digital revolution in phenotyping","authors":"Oellrich A, Collier N, Groza T, Rebholz-Schuhmann D, Shah N, Bodenreider O, Boland MR, Georgiev I, Liu H, Livingston K, Luna A, Mallon AM, Manda P, Robinson PN, Rustici G, Simon M, Wang L, Winnenburg R, Dumontier M","journal":"Brief Bioinform","issue":"17(5):819-30","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26420780)"},{"title":"A Whole-Genome Analysis Framework for Effective Identification of Pathogenic Regulatory Variants in Mendelian Disease","authors":"Smedley D, Schubach M, Jacobsen JOB, Khler S, Zemojtel T, Spielmann M, Jger M, Hochheiser H, Washington NL, McMurry JA, Haendel MA, Mungall CJ, Lewis SE, Groza T, Valentini G, Robinson PN","journal":"Am J Hum Genet","issue":"99(3):595-606","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/27569544)"},{"title":"Computational evaluation of exome sequence data using human and model organism phenotypes improves diagnostic efficiency","authors":"Bone WP, Washington NL, Buske OJ, Adams DR, Davis J, Draper D, Flynn ED, Girdea M, Godfrey R, Golas G, Groden C, Jacobsen J, Khler S, Lee EM, Links AE, Markello TC, Mungall CJ, Nehrebecky M, Robinson PN, Sincan M, Soldatos AG, Tifft CJ, Toro C, Trang H, Valkanas E, Vasilevsky N, Wahl C, Wolfe LA, Boerkoel CF, Brudno M, Haendel MA, Gahl WA, Smedley D","journal":"Genet Med","issue":"18(6):608-17","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26562225)"},{"title":"Next-generation diagnostics and disease-gene discovery with the Exomiser","authors":"Smedley D, Jacobsen JO, Jger M, Khler S, Holtgrewe M, Schubach M, Siragusa E, Zemojtel T, Buske OJ, Washington NL, Bone WP, Haendel MA, Robinson PN","journal":"Nat Protoc","issue":"10(12):2004-15","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26562621)"}]},{"year":2015,"list":[{"title":"Human genotype-phenotype databases: aims, challenges and opportunities","authors":"Brookes AJ, Robinson PN","journal":"Nature Rev Genet","issue":"16(12):702-15","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26553330)"},{"title":"The digital revolution in phenotyping","authors":"Oellrich A, Collier N, Groza T, Rebholz-Schuhmann D, Shah N, Bodenreider O, Boland MR, Georgiev I, Liu H, Livingston K, Luna A, Mallon AM, Manda P, Robinson PN, Rustici G, Simon M, Wang L, Winnenburg R, Dumontier M","journal":"Brief Bioinform","issue":"17(5):819-30","link":"[pubmed](https://pubmed.ncbi.nlm.nih.gov/26420780/)"},{"title":"Phenotype-driven strategies for exome prioritization of human Mendelian disease genes","authors":"Smedley D, Robinson PN","journal":"Genome Med","issue":"7(1):81","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26229552)"},{"title":"Phenolyzer: phenotype-based prioritization of candidate genes for human diseases","authors":"Yang H, Robinson PN, Wang K","journal":"Nat Methods","issue":"12(9):841-3","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26192085)"},{"title":"The Human Phenotype Ontology: Semantic Unification of Common and Rare Disease","authors":"Groza T, Khler S, Moldenhauer D, Vasilevsky N, Baynam G, Zemojtel T, Schriml LM, Kibbe WA, Schofield PN, Beck T, Vasant D, Brookes AJ, Zankl A, Washington NL, Mungall CJ, Lewis SE, Haendel MA, Parkinson H, Robinson PN","journal":"Am J Hum Genet","issue":"97(1):111-24","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26119816)"},{"title":"Disease insights through cross-species phenotype comparisons","authors":"Haendel MA, Vasilevsky N, Brush M, Hochheiser HS, Jacobsen J, Oellrich A, Mungall CJ, Washington N, Khler S, Lewis SE, Robinson PN, Smedley D","journal":"Mamm Genome","issue":"26(9-10):548-55","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/26092691)"},{"title":"Finding our way through phenotypes","authors":"Deans AR, Lewis SE, Huala E, Anzaldo SS, Ashburner M, Balhoff JP, Blackburn DC, Blake JA, Burleigh JG, Chanet B, Cooper LD, Courtot M, Cssz S, Cui H, Dahdul W, Das S, Dececchi TA, Dettai A, Diogo R, Druzinsky RE, Dumontier M, Franz NM,Friedrich F, Gkoutos GV, Haendel M, Harmon LJ, Hayamizu TF, He Y, Hines HM, Ibrahim N, Jackson LM, Jaiswal P, James-Zorn C, Khler S, Lecointre G, Lapp H, Lawrence CJ, Le Novre N, Lundberg JG, Macklin J, Mast AR, Midford PE, Mik I, Mungall CJ, Oellrich A, Osumi-Sutherland D, Parkinson H, Ramrez MJ, Richter S, Robinson PN, Ruttenberg A, Schulz KS, Segerdell E, Seltmann KC, Sharkey MJ, Smith AD, Smith B, Specht CD, Squires RB, Thacker RW, Thessen A, Fernandez-Triana J, Vihinen M, Vize PD, Vogt L, Wall CE, Walls RL, Westerfeld M, Wharton RA, Wirkner CS, Woolley JB, Yoder MJ, Zorn AM, Mabee P","journal":"PLoS Biol","issue":"13(1):e1002033","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/25562316)"}]},{"year":2014,"list":[{"title":"Deletions of chromosomal regulatory boundaries are associated with congenital disease","authors":"Ibn-Salem J, Khler S, Love MI, Chung H-R, Huang N, Hurles ME, Haendel M, Washington NL, Smedley D, Mungall CJ, Lewis SE, Ott C-E, Bauer S, Schofield PN, Mundlos S, Spielmann M and Robinson PN","journal":"Genome Biol","issue":"15(9):423","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/25315429)"},{"title":"Effective diagnosis of genetic disease by computational phenotype analysis of the disease-associated genome","authors":"Zemojtel T, Kohler S, Mackenroth L, Jager M, Hecht J, Krawitz P, Graul-Neumann L, Doelken S, Ehmke N, Spielmann M, Oien NC, Schweiger MR, Kruger U, Frommer G, Fischer B, Kornak U, Flottmann R, Ardeshirdavani A, Moreau Y, Lewis SE, Haendel M, Smedley D, Horn D, Mundlos S, Robinson PN","journal":"Sci Transl Med","issue":"6(252):252ra123","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/25186178)"},{"title":"Nose to tail, roots to shoots: spatial descriptors for phenotypic diversity in the Biological Spatial Ontology","authors":"Dahdul WM, Cui H, Mabee PM, Mungall CJ, Osumi-Sutherland D, Walls RL, Haendel MA","journal":"J Biomed Semantics","issue":"5:34","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/25140222)"},{"title":"CLO: The cell line ontology","authors":"Sarntivijai S., Lin Y, Xiang Z., Meehan TF, Diehl AD, Vempati UD, Schurer SC, Pang C, Malone J, Parkinson H, Liu Y, Takatsuki T, Saijo K, Masuy H, Nakamura Y, Brush MH, Haendel MA, Zheng J, Stoeckert CJ, Peters B, Mungall CJ, Carey TE, States DJ, Athey BD, He Y. (2014)","journal":"J Biomed Semantics","issue":"5(1):37","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/25852852)"},{"title":"Walking the interactome for candidate prioritization in exome sequencing studies of Mendelian diseases","authors":"Smedley D, Kohler S, Czeschik JC, Amberger J, Bocchini C, Hamosh A, Veldboer J, Zemojtel T, Robinson PN","journal":"Bioinformatics","issue":"30(22):3215-22","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/25078397)"},{"title":"The influence of disease categories on gene candidate predictions from model organism phenotypes","authors":"Oellrich A, Koehler S, Washington N; Sanger Mouse Genetic Project, Mungall C, Lewis S, Haendel M, Robinson PN, Smedley D","journal":"J Biomed Semantics","issue":"5(Suppl 1 Proceedings of the Bio-Ontologies Spec Interest G):S4. [ pubmed","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/25093073)"},{"title":"Improved exome prioritization of disease genes through cross-species phenotype comparison","authors":"Robinson PN, Kohler S, Oellrich A, Sanger Mouse Genetics Project, Wang K, Mungall CJ, Lewis SE, Washington N, Bauer S, Seelow D, Krawitz P, Gilissen C, Haendel M, Smedley D","journal":"Genome Res","issue":"24(2):340-8","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24162188)"},{"title":"The zebrafish anatomy and stage ontologies: representing the anatomy and development of Danio rerio","authors":"Van Slyke CE, Bradford YM, Westerfield M, Haendel MA","journal":"J Biomed Semantics","issue":"5(1):12","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24568621)"},{"title":"The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data","authors":"Khler S, Doelken SC, Mungall CJ, Bauer S, Firth HV, Bailleul-Forestier I, Black GC, Brown DL, Brudno M, Campbell J, FitzPatrick DR, Eppig JT, Jackson AP, Freson K, Girdea M, Helbig I, Hurst JA, Jhn J, Jackson LG, Kelly AM, Ledbetter DH, Mansour S, Martin CL, Moss C, Mumford A, Ouwehand WH, Park SM, Riggs ER, Scott RH, Sisodiya S, Van Vooren S, Wapner RJ, Wilkie AO, Wright CF, Vulto-van Silfhout AT, de Leeuw N, de Vries BB, Washingthon NL, Smith CL, Westerfield M, Schofield P, Ruef BJ, Gkoutos GV, Haendel M, Smedley D, Lewis SE, Robinson PN","journal":"Nucleic Acids Res","issue":"42(Database issue):D966-74","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24217912)"}]},{"year":2013,"list":[{"title":"An overview of the BioCreative 2012 Workshop Track III: interactive text mining task","authors":"Arighi CN, Carterette B, Cohen KB, Krallinger M, Wilbur WJ, Fey P, Dodson R, Cooper L, Van Slyke CE, Dahdul W, Mabee P, Li D, Harris B, Gillespie M, Jimenez S, Roberts P, Matthews L, Becker K, Drabkin H, Bello S, Licata L,Chatr-aryamontri A, Schaeffer ML, Park J, Haendel M, Van Auken K, Li Y, Chan J, Muller HM, Cui H, Balhoff JP, Chi-Yang Wu J, Lu Z, Wei CH, Tudor CO, Raja K, Subramani S, Natarajan J, Cejuela JM, Dubey P, Wu C","journal":"Database (Oxford)","issue":"2013:bas056","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23327936)"},{"title":"The environment ontology: contextualising biological and biomedical entities","authors":"Buttigieg PL, Morrison N, Smith B, Mungall CJ, Lewis SE; ENVO Consortium","journal":"J Biomed Semantics","issue":"4(1):43","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24330602)"},{"title":"Phenotypic overlap in the contribution of individual genes to CNV pathogenicity revealed by cross-species computational analysis of single-gene mutations in humans, mice and zebrafish","authors":"Doelken SC, Kohler S, Mungall CJ, Gkoutos GV, Ruef BJ, Smith C, Smedley D, Bauer S, Klopocki E, Schofield PN, Westerfield M, Robinson PN, Lewis SE","journal":"Dis Model Mech","issue":"6(2):358-72","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23104991)"},{"title":"What is an anatomy ontology?","authors":"Druzinsky R, Mungall C, Haendel M, Lapp H, Mabee P","journal":"Anat Rec (Hoboken)","issue":"296(12):1797-9","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24127438)"},{"title":"Estimating exome genotyping accuracy by comparing to data from large scale sequencing projects","authors":"Heinrich V, Kamphans T, Stange J, Parkhomchuk D, Hecht J, Dickhaus T, Robinson PN, Krawitz PM (2013)","journal":"Genome Med","issue":"5(7):69","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23902830)"},{"title":"Construction and accessibility of a cross-species phenotype ontology along with gene annotations for biomedical research","authors":"Kohler S, Doelken SC, Ruef BJ, Bauer S, Washington N, Westerfield M, Gkoutos G, Schofield P, Smedley D, Lewis SE, Robinson PN, Mungall CJ","journal":"F1000Res","issue":"2:30","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24358873)"},{"title":"A case of paroxysmal nocturnal hemoglobinuria caused by a germline mutation and a somatic mutation in PIGT","authors":"Krawitz PM, Hochsmann B, Murakami Y, Teubner B, Kruger U, Klopocki E, Neitzel H, Hoellein A, Schneider C, Parkhomchuk D, Hecht J, Robinson PN, Mundlos S, Kinoshita T, Schrezenmeier H","journal":"Blood","issue":"122(7):1312-5","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23733340)"},{"title":"A knowledge based approach to matching human neurodegenerative disease and animal models (2013)","authors":"Maynard SM, Mungall CJ, Lewis SE, Imam FT, Martone ME","journal":"Front Neuroinform","issue":"7:7","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23717278)"},{"title":"Ontology based molecular signatures for immune cell types via gene expression analysis (2013)","authors":"Meehan TF, Vasilevsky NA, Mungall CJ, Dougall DS, Haendel MA, Blake JA, Diehl AD","journal":"BMC Bioinformatics","issue":"14:263","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24004649)"},{"title":"A systematic large-scale phenotypic analysis of de novo and inherited copy number variation (2013)","authors":"Robinson PN","journal":"Hum Mutat","issue":"34(12):v","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24227377)"},{"title":"The Gene Ontology (GO) Cellular Component Ontology: integration with SAO (Subcellular Anatomy Ontology) and other recent developments","authors":"Roncaglia P, Martone ME, Hill DP, Berardini TZ, Foulger RE, Imam FT, Drabkin H, Mungall CJ, Lomax J","journal":"J Biomed Semantics","issue":"4(1):20","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24093723)"},{"title":"PhenoDigm: analyzing curated annotations to associate animal models with human diseases","authors":"Smedley D, Oellrich A, Kohler S, Ruef B; Sanger Mouse Genetics Project, Westerfield M, Robinson P, Lewis S, Mungall C","journal":"Database (Oxford)","issue":"2013:bat025","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23660285)"}]},{"year":2012,"list":[{"title":"MouseFinder: Candidate disease genes from mouse phenotype data","authors":"Chen CK, Mungall CJ, Gkoutos GV, Doelken SC, Kohler S, Ruef BJ, Smith C, Westerfield M, Robinson PN, Lewis SE, Schofield PN, Smedley D","journal":"Hum Mutat","issue":"33(5):858-66","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/22331800)"},{"title":"A unified anatomy ontology of the vertebrate skeletal system","authors":"Dahdul WM, Balhoff JP, Blackburn DC, Diehl AD, Haendel MA, Hall BK, Lapp H, Lundberg JG, Mungall CJ, Ringwald M, Segerdell E, Van Slyke CE, Vickaryous MK, Westerfield M, Mabee PM","journal":"PLoS One","issue":"7(12):e51070","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23251424)"},{"title":"Uberon, an integrative multi-species anatomy ontology","authors":"Mungall CJ, Torniai C, Gkoutos GV, Lewis SE, Haendel MA","journal":"Genome Biol","issue":"13(1):R5","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/22293552)"},{"title":"A strategy for building neuroanatomy ontologies","authors":"Osumi-Sutherland D, Reeve S, Mungall CJ, Neuhaus F, Ruttenberg A, Jefferis GS, Armstrong JD","journal":"Bioinformatics","issue":"28(9):1262-9","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/22402613)"},{"title":"On the Use of Gene Ontology Annotations to Assess Functional Similarity among Orthologs and Paralogs: A Short Report","authors":"Thomas PD, Wood V, Mungall CJ, Lewis SE, Blake JA, Gene Ontology Consortium","journal":"PLoS Comput Biol","issue":"8(2):e1002386","link":"[pubmed](http://www.ncbi.nlm.nih.gov/pubmed/22359495)"}]}]')},9993:function(e,o,t){e.exports=t.p+"img/garvan.d2f62326.png"},"99e5":function(e){e.exports=JSON.parse('{"stat":"ok","pagination":{"offset":0,"limit":50,"total":11},"monitors":[{"id":789510544,"friendly_name":"Analyzer (HP)","url":"https://legacy.monarchinitiative.org/analyze/phenotype.json?input_items=HP:0002121+HP:0001344+HP:0001251&limit=100&target_species=all","type":1,"sub_type":"","keyword_type":null,"keyword_case_type":null,"keyword_value":"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":0,"create_datetime":1634806003},{"id":789510547,"friendly_name":"API","url":"https://api.monarchinitiative.org/api/sim/search?is_feature_set=false&metric=phenodigm&id=MONDO%3A0007947&limit=100","type":1,"sub_type":"","keyword_type":null,"keyword_case_type":null,"keyword_value":"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":1,"create_datetime":1634806026},{"id":789510550,"friendly_name":"Archive","url":"https://archive.monarchinitiative.org","type":1,"sub_type":"","keyword_type":null,"keyword_case_type":null,"keyword_value":"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":2,"create_datetime":1634806055},{"id":789510553,"friendly_name":"Biolink API (homolog)","url":"https://api.monarchinitiative.org/api/bioentity/gene/NCBIGene:3630/homologs/?homology_type=O&fetch_objects=false","type":2,"sub_type":"","keyword_type":2,"keyword_case_type":0,"keyword_value":"MGI:96573","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":8,"create_datetime":1634806087},{"id":789510560,"friendly_name":"Data","url":"https://data.monarchinitiative.org","type":1,"sub_type":"","keyword_type":null,"keyword_case_type":null,"keyword_value":"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":9,"create_datetime":1634806142},{"id":789510564,"friendly_name":"Exomiser (production)","url":"http://exomiser.monarchinitiative.org/exomiser/","type":1,"sub_type":"","keyword_type":null,"keyword_case_type":null,"keyword_value":"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":2,"create_datetime":1634806163},{"id":789510570,"friendly_name":"Homolog (GOlr)","url":"https://solr.monarchinitiative.org/solr/golr/select/?defType=edismax&qt=standard&indent=on&wt=json&rows=25&start=0&fl=*,score&facet=true&facet.mincount=1&facet.sort=count&json.nl=arrarr&facet.limit=25&facet.method=enum&fq=object_category:%22gene%22&fq=relation_closure:%22RO:HOM0000001%22&fq=subject_closure:%22NCBIGene:23543%22&q=*:*","type":2,"sub_type":"","keyword_type":2,"keyword_case_type":0,"keyword_value":"\\"subject_gene_label\\":\\"RBFOX2\\"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":2,"create_datetime":1634806184},{"id":789510574,"friendly_name":"Owlsim","url":"https://legacy.monarchinitiative.org/owlsim/compareAttributeSets?a=MP:0000130&b=HP:0006165","type":1,"sub_type":"","keyword_type":null,"keyword_case_type":null,"keyword_value":"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":2,"create_datetime":1634806205},{"id":789510577,"friendly_name":"SciGraph (data)","url":"https://scigraph-data.monarchinitiative.org/scigraph/graph/neighbors?id=GO%3A0022008&depth=1&blankNodes=false&relationshipType=subClassOf&direction=BOTH&entail=false","type":2,"sub_type":"","keyword_type":2,"keyword_case_type":0,"keyword_value":"\\"lbl\\":\\"cell differentiation\\"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":2,"create_datetime":1634806229},{"id":789510578,"friendly_name":"SciGraph (ontology)","url":"https://scigraph-ontology.monarchinitiative.org/scigraph/graph/neighbors?id=GO%3A0022008&depth=1&blankNodes=false&relationshipType=subClassOf&direction=BOTH&entail=false","type":2,"sub_type":"","keyword_type":2,"keyword_case_type":0,"keyword_value":"\\"lbl\\":\\"cell differentiation\\"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":2,"create_datetime":1634806248},{"id":789510441,"friendly_name":"Website","url":"http://monarchinitiative.org","type":1,"sub_type":"","keyword_type":null,"keyword_case_type":null,"keyword_value":"","http_username":"","http_password":"","port":"","interval":300,"timeout":30,"status":0,"create_datetime":1634805277}]}')},"9ae1":function(e){e.exports=JSON.parse('{"active_lock_reason":null,"assignee":null,"assignees":[],"author_association":"NONE","body":"**Name**\\nVincent Rubinetti\\n\\n**Email**\\nvince.rubinetti@gmail.com\\n\\n**GitHub Username**\\n@vincerubinetti\\n\\n**Details**\\nPage: /help\\nBrowser: Chrome 95.0.4638.54\\nDevice: \\nOS: Windows 10 amd64\\nEngine: Blink 95.0.4638.54\\n\\n\\nI just want to tell you both good luck. We\'re all counting on yoasdasdu.asdasdads","closed_at":null,"closed_by":null,"comments":0,"comments_url":"https://api.github.com/repos/monarch-initiative/helpdesk/issues/46/comments","created_at":"2021-11-02T20:12:12Z","events_url":"https://api.github.com/repos/monarch-initiative/helpdesk/issues/46/events","html_url":"https://github.com/monarch-initiative/helpdesk/issues/46","id":1042765827,"labels":[{"color":"9BB1D7","default":false,"description":"Issue submitted via the website\'s issue form.","id":3487210531,"name":"feedback form","node_id":"LA_kwDOC6-0xc7P2pwj","url":"https://api.github.com/repos/monarch-initiative/helpdesk/labels/feedback%20form"}],"labels_url":"https://api.github.com/repos/monarch-initiative/helpdesk/issues/46/labels{/name}","locked":false,"milestone":null,"node_id":"I_kwDOC6-0xc4-J1gD","number":46,"performed_via_github_app":null,"reactions":{"+1":0,"-1":0,"confused":0,"eyes":0,"heart":0,"hooray":0,"laugh":0,"rocket":0,"total_count":0,"url":"https://api.github.com/repos/monarch-initiative/helpdesk/issues/46/reactions"},"repository_url":"https://api.github.com/repos/monarch-initiative/helpdesk","state":"open","timeline_url":"https://api.github.com/repos/monarch-initiative/helpdesk/issues/46/timeline","title":"Feedback form - Vincent Rubinetti - I just want to tell you both good luck. We\'re all countin...","updated_at":"2021-11-02T20:12:12Z","url":"https://api.github.com/repos/monarch-initiative/helpdesk/issues/46","user":{"avatar_url":"https://avatars.githubusercontent.com/in/147357?v=4","events_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/events{/privacy}","followers_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/followers","following_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/following{/other_user}","gists_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/gists{/gist_id}","gravatar_id":"","html_url":"https://github.com/apps/monarch-issues-tracker","id":93150527,"login":"monarch-issues-tracker[bot]","node_id":"BOT_kgDOBY1dPw","organizations_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/orgs","received_events_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/received_events","repos_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/repos","site_admin":false,"starred_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D/subscriptions","type":"Bot","url":"https://api.github.com/users/monarch-issues-tracker%5Bbot%5D"}}')},"9b7d":function(e,o,t){e.exports=t.p+"img/ebigene2phen.d7658285.png"},"9d3a":function(e,o,t){e.exports=t.p+"img/bryan-laraway.eedab597.jpg"},"9e71":function(e){e.exports=JSON.parse('[{"id":"MonarchArchive:#mgi","name":"Mouse Genome Informatics","image":"mgi.png","description":"MGI is the international database resource for the laboratory mouse, providing integrated genetic, genomic, and biological data to facilitate the study of human health and disease.","usage":"We list genotype-phenotype associations and asserted disease-models. We use MGI identifiers as the main hook into mouse data. Additionally, we utilize the Mouse Phenotype Ontology (MP) in our cross-species ontology, to link all mouse phenotype annotations from various sources, which is developed by MGI.","vocabulary":"ECO, MA, MP, SO","license":"http://reusabledata.org/mgi.html"},{"id":"MonarchArchive:#zfin","name":"Zebrafish Information Network","image":"zfin.png","description":"The Zebrafish Information Resource is the community database resource for the laboratory use of zebrafish which develops and supports integrated zebrafish genetic, genomic and developmental information, maintains the definitive reference data sets of zebrafish research information toward facilitation of the use of zebrafish as a model for human biology.","usage":"","vocabulary":"We integrate the curated genotype-phenotype data, including experimentally derived fish (such as via application of morpholinos), and links to the literature as evidence.","license":"http://reusabledata.org/zfin.html"},{"id":"MonarchArchive:#zfinslim","name":"Zebrafish Information Network (ZFIN Slim)","image":"zfin.png","description":"The Zebrafish Information Resource is the community database resource for the laboratory use of zebrafish which develops and supports integrated zebrafish genetic, genomic and developmental information, maintains the definitive reference data sets of zebrafish research information toward facilitation of the use of zebrafish as a model for human biology.","usage":"Zfin model only containing Gene to phenotype associations using this file: https://zfin.org/downloads/phenoGeneCleanData_fish.txt","vocabulary":"","license":"http://reusabledata.org/zfin.html"},{"id":"MonarchArchive:#wormbase","name":"WormBase database of nematode biology","image":"wormbase.png","description":"WormBase is an international consortium dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of C. elegans and related nematodes.","usage":"Wormbase curates variant (allele)-phenotype associations. The variants are both genetic (intrinsic) and induced through application of reagents such as RNAi (extrinsic). We list the variant-phenotype associations. Some data is pulled from WormBase directly, other data is routed via WormMine.","vocabulary":"WBbt, WBls, WBPhenotype","license":"http://reusabledata.org/wormbase.html"},{"id":"MonarchArchive:#flybase","name":"FlyBase","image":"flybase.png","description":"FlyBase is the model organism database providing integrated genetic, genomic, phenomic, and biological data for Drosophila melanogaster.","usage":"We integrate the genotype-phenotype associations.","vocabulary":"FBbt, FBcv, FBdv","license":"http://reusabledata.org/flybase.html"},{"id":"MonarchArchive:#impc","name":"International Mouse Phenotyping Consortium","image":"impc.png","description":"The International Mouse Phenotyping Consortium (IMPC) is generating a knockout mouse strain for every protein coding gene by using the embryonic stem cell resource generated by the International Knockout Mouse Consortium (IKMC). Systematic broad-based phenotyping is performed by each IMPC center using standardized procedures found within the International Mouse Phenotyping Resource of Standardised Screens (IMPReSS) resource. Gene-to-phenotype associations are made by a versioned statistical analysis.","usage":"We use the allele-phenotype associations recorded by the consortium. In addition, we map their allele+zygosity+background to MGI genotype identifiers. Where they do not map to MGI genotype identifiers, we create temporary identifiers for navigation purposes.","vocabulary":"MP","license":"http://reusabledata.org/impc.html"},{"id":"MonarchArchive:#mpd","name":"Mouse Phenome Database","image":"mpd.png","description":"The Mouse Phenome Database is a collaborative standardized collection of measured data on laboratory mouse strains, and includes: baseline phenotype data sets, studies of drug, diet, disease and aging effect, protocols, projects, and publications, and SNP, variation and gene expression studies. MPD collects data for classical inbred strains, other fixed-genotype strains, derived lines and populations that are openly acquirable (strain panel examples). Strains can be from JAX-Mice or from any other vendor that\'s a recognized breeding source.","usage":"genotype (strain)-phenotype association","vocabulary":"MP","license":"http://reusabledata.org/mpd.html"},{"id":"MonarchArchive:#omim","name":"Online Mendelian Inheritance in Man","image":"omim.png","description":"OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes with full-text, referenced overviews that contains information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.","usage":"We use curated disease-gene, disease-locus, and variant-disease associations, together with their annotated references. Most OMIM diseases are further curated by the HPO group. Most OMIM diseases are integrated into the Disease Ontology.","vocabulary":"OMIM","license":"http://reusabledata.org/impc.html"},{"id":"MonarchArchive:#orphanet","name":"Orphanet","image":"orphanet.png","description":"Orphanet provides reference information on rare diseases and orphan drugs to help improve the diagnosis, care and treatment of patients with rare diseases.","usage":"We use the Orphanet disease-gene associations.","vocabulary":"ORPHA","license":"http://reusabledata.org/orphanet-open.html"},{"id":"MonarchArchive:#panther","name":"PANTHER","image":"panther.png","description":"The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) according to evolutionary family/subfamily, molecular function, biological process, and pathway. The PANTHER Classifications are the result of human curation as well as sophisticated bioinformatics algorithms","usage":"We currently utilize the 12 RefGenome species, as well as HUGO HCOP species, to seed the orthology calls. Species currently include: arabidopsis, budding yeast, chicken, chimp, dog, fission yeast, fruitfly, green lizard, horse, human, macaque, mouse, opossum, pig, platypus, rat, slime mold, worms, zebrafish. We use the orthology calls to populate the orthologs tabs for genes, as well as to infer disease-model associations via homology.","vocabulary":"RO","license":"http://reusabledata.org/panther.html"},{"id":"MonarchArchive:#coriell","name":"Coriell Institute for Medical Research","image":"coriell.png","description":"The Coriell Cell Repositories provide essential research reagents to the scientific community by establishing, verifying, maintaining, and distributing cell cultures and DNA derived from cell cultures. These collections, supported by funds from the National Institutes of Health (NIH) and several foundations, are extensively utilized by research scientists around the world. NINDS and NIGMS cell line catalog. NIGMS samples represent a variety of disease states, chromosomal abnormalities, apparently healthy individuals and many distinct human populations. NINDS samples are drawn from subjects with cerebrovascular disease, epilepsy, motor neuron disease, Parkinsonism and Tourette Syndrome, as well as controls.","usage":"We link pertinent cell lines to any diseases for which they are asserted models.","vocabulary":"OMIM"},{"id":"MonarchArchive:#ctd","name":"Comparative Toxicogenomics Database","image":"ctd.png","description":"CTD promotes understanding about the effects of environmental chemicals on human health by integrating data from curated scientific literature to describe chemical interactions with genes and proteins, and associations between diseases and chemicals, and diseases and genes/proteins.","usage":"We integrate the asserted (curated) disease-gene associations, and their evidence.","vocabulary":"MESH, OMIM","license":"http://reusabledata.org/ctd.html"},{"id":"MonarchArchive:#hpoa","name":"Human Phenotype Ontology","image":"hpo.png","description":"A curated database of human hereditary syndromes from OMIM, Orphanet, and DECIPHER mapped to classes of the human phenotype ontology. Various meta-attributes such as frequency, references and negations are associated with each annotation. These are presently limited to rare mendelian diseases.","usage":"We use the HPO disease-phenotype annotations as the primary atomic description of a disease, and list them on the disease pages, together with their references. The Human Phenotype Ontology is integrated into our cross-species phenotype ontology.","vocabulary":"ECO, HPO","license":"http://reusabledata.org/hpo.html"},{"id":"MonarchArchive:#kegg","name":"Kyoto Encyclopedia of Genes and Genomes","image":"kegg.png","description":"KEGG is an integrated database resource consisting of the seventeen main databases including systems, genomic, chemical, and health information.","usage":"We list disease-gene associations, and gene-pathway associations. We utilize the KEGG Ortholog (KO) gene-pathway associations, and infer a specific-organisms\' participation in that pathway based on the gene-KO links.","vocabulary":"MESH, OMIM","license":"http://reusabledata.org/kegg-ftp.html"},{"id":"MonarchArchive:#ncbigene","name":"National Center for Biotechnology Information","image":"ncbi.png","description":"Gene integrates information from a wide range of species, and includes nomenclature, Reference Sequences (RefSeqs), maps, pathways, variations, phenotypes, and links to genome-, phenotype-, and locus-specific resources worldwide. Taxon lists the taxinomic organization of organisms. Pub2Gene serves links between genes and pubmed identifiers where they are mentioned.","usage":"We use NCBIGene ids and symbols as the primary identifier and label for human genes in our system and NCBITaxon identifiers and scientific name for species-specific labeling. For any given gene, we also list the annotated pmids from Pub2Gene.","vocabulary":"","license":"https://reusabledata.org/ncbi-gene.html"},{"id":"MonarchArchive:#biogrid","name":"BioGRID","image":"biogrid.png","usage":"Monarch indicates gene-gene/protein-protein interactions on gene pages. We also use many of the id mappings to resolve ids in our own site.","description":"BioGRID is an curated gene and protein interaction repository with for major model organism species.","vocabulary":"MI","license":"http://reusabledata.org/biogrid.html"},{"id":"MonarchArchive:#gwascatalog","name":"GWAS Catalog","image":"gwascatalog.png","description":"The NHGRI-EBI Catalog of published genome-wide association studies.","usage":"Monarch links the variants recorded here to the curated EFO-classes","vocabulary":"RO, EFO, ECO"},{"id":"MonarchArchive:#animalqtldb","name":"AnimalQTLdb","image":"animalqtldb.png","description":"The Animal Quantitative Trait Loci (QTL) database (Animal QTLdb) is designed to house publicly all available QTL and single-nucleotide polymorphism/gene association data on livestock animal species.","usage":"Monarch uses the QTL genetic maps and their computed genomic locations to create associations between the QTLs and their traits. The traits come in their internal Animal Trait ontology vocabulary, which they further map to [Vertebrate Trait](http://bioportal.bioontology.org/ontologies/VT), Product Trait, and Clinical Measurement Ontology vocabularies.","vocabulary":"RO, ECO"},{"id":"MonarchArchive:#ensembl","name":"Ensembl database of automatically annotated genomic data","image":"ensembl.png","description":"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation.","usage":"Monarch obtains equivalencies between Ensembl gene IDs and NCBI gene IDs","vocabulary":""},{"id":"MonarchArchive:#go","name":"Gene Ontology Database","image":"geneontology.png","description":"The GO defines concepts/classes used to describe gene function, and relationships between these concepts.","usage":"Monarch processes gene-process/function/subcellular/location associations.","vocabulary":"RO","license":"https://reusabledata.org/go-ann.html"},{"id":"MonarchArchive:#hgnc","name":"HUGO Gene Nomenclature Committee","image":"hgnc.png","description":"A curated online repository of HGNC-approved gene nomenclature, gene families and associated resources.","usage":"Monarch creates equivalences between HGNC identifiers and ENSEMBL and NCBIGene. We also add the links to cytogenic locations for the gene features.","vocabulary":""},{"id":"MonarchArchive:#mmrrc","name":"Mutant Mouse Resource and Research Centers","image":"mmrrc.png","description":"A repository of mouse stocks and ES cell line collections serving the world-wide genetics and biomedical research community for the benefit of human health.","usage":"Centers A repository of mouse stocks and ES cell line collections serving the world-wide genetics and biomedical research community for the benefit of human health.\\tMonarch processes the Mutant Mouse Resource and Research Center strain data, which includes: strains and their mutant alleles, phenotypes of the alleles, and descriptions of the research uses of the strains.","vocabulary":"MP"},{"id":"MonarchArchive:#reactome","name":"Reactome - a curated knowledgebase of biological pathways","image":"reactome.png","description":"A repository of mouse stocks and ES cell line collections serving the world-wide genetics and biomedical research community for the benefit of human health.","usage":"Monarch processes ensembl gene to pathway associations","vocabulary":"RO","license":"http://reusabledata.org/reactome.html"},{"id":"MonarchArchive:#bgee","name":"Gene expression data in animals","image":"bgee.png","description":"Bgee is a database to retrieve and compare gene expression patterns in multiple animal species","usage":"Monarch stores the top 20 ranked tissues where a gene is over expressed based on differential expression measures across evidence lines","vocabulary":"Uberon","license":"http://reusabledata.org/bgee.html"},{"id":"MonarchArchive:#string","name":"STRING","image":"string.png","description":"STRING is a database of known and predicted protein-protein interactions.","usage":"Monarch stores protein protein interactions with a confidence >.7","vocabulary":"RO","license":"http://reusabledata.org/string.html"},{"id":"MonarchArchive:#rgd","name":"Rat Genome Database","image":"rgd.png","description":"The Rat Genome Database (RGD)is the premier site for genetic, genomic, phenotype, and disease data generated from rat research","usage":"Monarch stores gene to phenotype associations","vocabulary":"MP, RO, RGD References, RGD genes","license":"http://reusabledata.org/rgd.html"},{"id":"MonarchArchive:#sgd","name":"Saccharomyces Genome Database","image":"sgd.png","description":"The Saccharomyces Genome Database (SGD) provides comprehensive integrated biological information for the budding yeast Saccharomyces cerevisiae along with search and analysis tools to explore these data, enabling the discovery of functional relationships between sequence and gene products in fungi and higher organisms.","usage":"Monarch stores gene to phenotype associations and gene function data","vocabulary":"APO, RO","license":"http://reusabledata.org/sgd.html"},{"id":"MonarchArchive:#eom","name":"Elements of Morphology","image":"eom.png","description":"Elements of Morphology is a resource from NHGRI that has definitions of morphological abnormalities, together with image depictions.","usage":"We pull those relationships, as well as our local mapping of equivalences between EOM and HP terminologies."},{"id":"MonarchArchive:#mgislim","name":"Mouse Genome Informatics (MGI Slim)","image":"mgi.png","description":"See \'Mouse Genome Informatics\'","usage":"Slim MGI model only containing Gene to phenotype associations. This uses mousemine: http://www.mousemine.org/mousemine/begin.do","license":"http://reusabledata.org/mgi.html"},{"id":"MonarchArchive:#monarch","name":"Monarch Initiative","image":"monarch.png","description":"This is the parser for data curated by the Monarch Initiative.","usage":"","vocabulary":"","license":"http://reusabledata.org/monarch.html"},{"id":"MonarchArchive:#monochrom","name":"Monochrom","image":"monochrom.png","description":"Ingested data represents major structural pieces of chromosomes which are often universally referenced, using physical properties/observations that remain constant over different genome builds (such as banding patterns and arms). The idea is to create a scaffold upon which we can hang build-specific chromosomal coordinates, and reason across them.","usage":"We leverage the GENO ontology and modeling patterns to build an ontology of chromosomes for any species.We currently compute bands for human, mouse, zebrafish, and rat but will be expanding in the future as needed.","vocabulary":"","license":"http://reusabledata.org/monochrom.html"},{"id":"MonarchArchive:#mychem","name":"MyChem","image":"mychem.png","description":"Chemical and drug annotation data, aggregated from many popular data resources.","usage":"","vocabulary":""},{"id":"MonarchArchive:#ucscbands","name":"UCSC Bands","image":"ucscbands.png","description":"UCSC definitions of cytogenic bands","usage":"We create nested structures to enable overlap and containment queries. We use Monochrom.py to create the OWL-classes of the chromosomal parts. Here, we simply worry about the instance-level values for particular genomebuilds.","vocabulary":""},{"id":"MonarchArchive:#omia","name":"Online Mendelian Inheritance in Animals","image":"omia.png","description":"Online Mendelian Inheritance in Animals (OMIA) is a catalogue/compendium of inherited disorders, other (single-locus) traits, and genes in 215 (non-model) animal species.","usage":"gene-disease association","vocabulary":"OMIA, OMIM","license":"http://reusabledata.org/omia.html"},{"id":"MonarchArchive:#clinvar","name":"ClinVar","image":"clinvar.png","description":"ClinVar archives and aggregates information about relationships among variation and human health. ClinVar collects reports of variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data.","usage":"disease-gene association, variant-disease association, variant definitions","vocabulary":"UMLS","distribution":"https://archive.monarchinitiative.org/latest/rdf/clinvar.nt","license":"http://reusabledata.org/clinvar.html","files":["ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz","ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/gene_condition_source_id"]},{"id":"MonarchArchive:#udp","name":"Undiagnosed Diseases Program (UDP)","image":"udp.png","description":"The National Institutes of Health (NIH) Undiagnosed Diseases Program (UDP) is part of the Undiagnosed Disease Network (UDN), an NIH Common Fund initiative that focuses on the most puzzling medical cases referred to the NIH Clinical Center in Bethesda, Maryland.","usage":"Monarch stores phenotypes for each case and variants of interest","distribution":"https://archive.monarchinitiative.org/latest/rdf/udp.ttl","vocabulary":"RO"}]')},"9f37":function(e,o,t){},"9f41":function(e,o,t){"use strict";t("89c5")},a090:function(e,o,t){e.exports=t.p+"img/renci.faa75cca.png"},a3cd:function(e,o,t){e.exports=t.p+"img/category-homolog.6babafb7.svg"},a404:function(e,o,t){e.exports=t.p+"img/ensembl.968b65f5.png"},a427:function(e,o,t){e.exports=t.p+"img/category-interaction.b57455fc.svg"},a70e:function(e,o,t){"use strict";t("18a5")},ac8f:function(e,o,t){e.exports=t.p+"img/category-model.440af6fb.svg"},adab:function(e,o,t){},aec5:function(e,o,t){e.exports=t.p+"img/category-function.79399857.svg"},aee8:function(e,o,t){},afdf:function(e,o,t){e.exports=t.p+"img/michael-gargano.ad258662.jpg"},b1a5:function(e,o,t){"use strict";t("50fa")},b28e:function(e,o,t){e.exports=t.p+"img/j-harry-caufield.98299be1.jpg"},b2e4:function(e,o,t){e.exports=t.p+"img/sebastian-kohler.d20e4da2.jpg"},b328:function(e,o,t){"use strict";t("5caa")},b499:function(e,o,t){e.exports=t.p+"img/mygene.aa0e09a9.png"},b593:function(e){e.exports=JSON.parse('[{"name":"University of Colorado Center for Health AI","image":"health-ai.png","link":"https://medschool.cuanschutz.edu/ai","members":[{"name":"Melissa Haendel","role":"Monarch PI","link":"https://tislab.org/members/melissa-haendel.html"},{"name":"Julie McMurry","role":"Associate Director","link":"https://tislab.org/members/julie-mcmurry.html"},{"name":"Monica Munoz-Torres","role":"Program Manager","link":"https://tislab.org/members/monica-munoz-torres.html"},{"name":"Tim Putman","role":"Technical Director","link":"https://tislab.org/members/tim-putman.html"},{"name":"Kent Shefchek","role":"Bioinformatics Software Engineer","link":"https://tislab.org/members/kent-shefchek.html","alumni":true},{"name":"Vincent Rubinetti","role":"Frontend and UI/UX Dev","link":"https://greenelab.com/members/vincent-rubinetti.html"},{"name":"Faisal Alquaddoomi","role":"Full Stack Dev and Dev Ops","link":"https://github.com/falquaddoomi"},{"name":"Anne Thessen","role":"Biodiversity Informaticist","link":"https://tislab.org/members/anne-thessen.html"},{"name":"Sabrina Toro","role":"Biocurator","link":"https://tislab.org/members/sabrina-toro.html"},{"name":"Nicole Vasilevsky","role":"Lead Biocurator","link":"https://tislab.org/members/nicole-vasilevsky.html"},{"name":"Victoria Alexandra Soesanto","role":"Scientific Software Engineer","link":"https://tislab.org/members/victoria-soesanto.html"},{"name":"Matthew Brush","role":"Ontologist","link":"https://tislab.org/members/matthew-brush.html"},{"name":"Shahim Essaid","role":"Ontologist","link":"https://tislab.org/members/shahim-essaid.html"},{"name":"Bryan Laraway","role":"Data Wrangler","link":"https://www.linkedin.com/in/bryanlaraway/"}]},{"name":"The Jackson Laboratory","image":"jackson.png","link":"https://www.jax.org/","members":[{"name":"Peter Robinson","link":"https://www.jax.org/research-and-faculty/faculty/peter-robinson","role":"Monarch PI"},{"name":"Leigh Carmody","role":"Scientific Curator","link":"https://www.linkedin.com/in/leighcarmody/"},{"name":"Hannah Blau","role":"Research Software Engineer","link":"https://www.jax.org/people/hannah-blau"},{"name":"Michael Gargano","role":"Software Engineer","link":"https://www.jax.org/people/michael-gargano"},{"name":"Vida Ravanmehr","role":"Postdoctoral Associate","link":"https://www.jax.org/people/vida-ravanmehr"}]},{"name":"Lawrence Berkeley National Laboratory","image":"berkeley.png","link":"http://lbl.gov/","members":[{"name":"Chris Mungall","role":"Monarch PI","link":"https://bids.berkeley.edu/people/chris-mungall"},{"name":"Seth Carbon","role":"Software engineer","link":"https://scholar.google.com/citations?hl=en&user=Ypz4s1UAAAAJ"},{"name":"Nomi Harris","role":"Program Manager","link":"https://www.linkedin.com/in/nomiharris"},{"name":"Sierra Taylor Moxon","role":"Software developer","link":"http://github.com/sierra-moxon"},{"name":"Justin Reese","role":"Software developer","link":"http://github.com/justaddcoffee"},{"name":"J. Harry Caufield","role":"Data Scientist","link":"https://harrycaufield.net/"},{"name":"Deepak Unni","role":"Bioinformatics Software Developer","alumni":true,"link":"https://www.linkedin.com/in/deepakunni3"},{"name":"Eric Douglass","role":"Software developer","alumni":true,"link":"https://www.linkedin.com/in/eric-douglass-7ba8a723/"},{"name":"Sarah Kim","role":"Web Developer/Designer","alumni":true},{"name":"Suzanna Lewis","role":"Genomics","alumni":true,"link":"https://scholar.google.com/citations?user=mS3mpAgAAAAJ&hl=en"},{"name":"Jeremy Nguyen Xuan","role":"Software Engineer","alumni":true,"link":"https://www.linkedin.com/in/jeremy-nguyen-xuan-84a26721/?originalSubdomain=au"},{"name":"Nicole Washington","role":"Lead Bioinformatician","alumni":true,"link":"https://www.linkedin.com/in/nicole-l-washington/"}]},{"name":"Oregon State University","image":"oregon-state.png","link":"https://oregonstate.edu/","alumni":true,"members":[{"name":"Jen Martin","role":"Scientific Software Engineer","link":"https://www.linkedin.com/in/jen-martin-9b73774/"},{"name":"Tom Conlin","role":"Bioinformaticist","link":"https://www.linkedin.com/in/tomconlin/"},{"name":"Daniel Keith","role":"Bioinformatics Semantic Engineer","link":"https://doctorbud.com/"}]},{"name":"Oregon Health & Science University","image":"ohsu.png","link":"https://www.ohsu.edu/","alumni":true,"members":[{"name":"Lilly Winfree","role":"Science and Data Analyst","link":"https://www.linkedin.com/in/lilly-winfree-phd/"}]},{"name":"Charit - Universittsmedizin Berlin","image":"charite.png","link":"http://compbio.charite.de/","members":[{"name":"Sebastian Khler","role":"Bioinformatician & Developer","link":"https://www.linkedin.com/in/koehlers/?originalSubdomain=de"}]},{"name":"Queen Mary University of London","image":"queen-mary.png","link":"http://www.qmul.ac.uk/","members":[{"name":"Damian Smedley","role":"Senior Scientific Manager","link":"https://www.qmul.ac.uk/whri/people/academic-staff/items/smedleydamian.html"},{"name":"Jules Jacobsen","role":"Bioinformatician","link":"https://www.linkedin.com/in/julesjacobsen/?originalSubdomain=uk"},{"name":"Tomasz Konopka","role":"Bioinformatician","alumni":true},{"name":"Pilar Cacheiro","role":"Computational Biologist","alumni":true,"link":"https://www.qmul.ac.uk/c4tb/our-team/profiles/pilar-cacheiro.html"}]},{"name":"Renaissance Computing Institute","image":"renci.png","link":"http://renci.org/","members":[{"name":"Jim Balhoff","role":"Senior Research Scientist","link":"https://renci.org/staff/james-balhoff/"}]},{"name":"European Bioinformatics Institute","image":"ebi.png","link":"https://www.ebi.ac.uk/","members":[{"name":"David Osumi-Sutherland","role":"Monarch PI","link":"https://www.ebi.ac.uk/about/people/david-osumi-sutherland"},{"name":"Nicolas Matentzoglu","role":"Semantic Web Developer","link":"https://www.linkedin.com/in/nmatentzoglu/?originalSubdomain=gr"}]},{"name":"Garvan Institute","image":"garvan.png","link":"http://garvan.org.au/","alumni":true,"members":[{"name":"Tudor Groza","role":"Phenomics Team Leader","link":"https://www.linkedin.com/in/tudorgroza/?originalSubdomain=au"},{"name":"Craig McNamara","role":"Developer","link":"https://www.linkedin.com/in/craig-mcnamara-565b8323/?originalSubdomain=au"},{"name":"Edwin Zhang","role":"Developer","link":"https://www.linkedin.com/in/edwin-zhang-64280965/"}]},{"name":"University of Pittsburgh","image":"pittsburgh.png","link":"http://pitt.edu/","alumni":true,"members":[{"name":"Harry Hochheiser","role":"Monarch PI","link":"https://www.dbmi.pitt.edu/person/harry-hochheiser-phd"},{"name":"Michael Davis","role":"Software Developer","link":"https://www.dbmi.pitt.edu/node/1902"},{"name":"Zhou (Joe) Yuan","role":"Web Application Developer","link":"https://www.dbmi.pitt.edu/node/3778"},{"name":"Charles Borromeo","role":"Interface Architect","link":"https://www.dbmi.pitt.edu/person/charles-borromeo"},{"name":"Jeremy Espino","role":"Senior Research Scientist","link":"https://www.dbmi.pitt.edu/person/jeremy-espino-md-ms-0"}]}]')},b70c:function(e,o,t){e.exports=t.p+"img/david-osumi-sutherland.35d9e480.jpg"},b82c:function(e,o,t){e.exports=t.p+"img/pilar-cacheiro.8228b784.jpg"},ba24:function(e,o,t){"use strict";t("24b8")},be35:function(e,o,t){e.exports=t.p+"img/panther.80f8dfb3.png"},bf3f:function(e,o,t){"use strict";t("9918")},bfd6:function(e,o,t){e.exports=t.p+"img/jim-balhoff.3d1cb687.jpg"},c1e2:function(e,o,t){e.exports=t.p+"img/nicole-vasilevsky.ed3bdaea.jpg"},c246:function(e){e.exports=JSON.parse('{"nodes":[{"id":"MonarchArchive:202103/#flybase","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/source":["jdbc:postgresql://chado.flybase.org:5432/flybase"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["FlyBase Monarch version 202103"],"source":["jdbc:postgresql://chado.flybase.org:5432/flybase"],"title":["FlyBase Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"http://current.geneontology.org/annotations/aspgd.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://zfin.org/downloads/phenoGeneCleanData_fish.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/flybase.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["FlyBase distribution ttl"],"title":["FlyBase distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202012/rdf/zfinslim.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2020-12-16"],"http://purl.org/dc/terms/created":["2020-12-16"],"http://purl.org/pav/version":["2020-12-01"],"http://purl.org/dc/terms/title":["Simplified ZFIN distribution ttl"],"title":["Simplified ZFIN distribution ttl"],"category":["dataset"],"version":["2020-12-01"]}},{"id":"http://www.orpha.net","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202012/#zfinslim","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2020-12-16"],"http://purl.org/pav/version":["2020-12-01"],"http://purl.org/dc/terms/created":["2020-12-16"],"http://purl.org/dc/terms/title":["Simplified ZFIN Monarch version 202012"],"category":["dataset"],"title":["Simplified ZFIN Monarch version 202012"],"version":["2020-12-01"]}},{"id":"MonarchArchive:DipperCache/flybase/species.ab.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#gwascatalog","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#gwascatalog"],"http://purl.org/dc/terms/title":["NHGRI-EBI Catalog of published genome-wide association studies"],"category":["dataset"],"title":["NHGRI-EBI Catalog of published genome-wide association studies"]}},{"id":"ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Sus_scrofa.gene_info.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Bos_taurus.gene_info.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/rainbow_trout_QTLdata.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://zfin.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/tmp/QTL_Btau_4.6.gff.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://wiki.flybase.org/wiki/FlyBase_Wiki:General_disclaimer","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.flybase.net/releases/current/precomputed_files/human_disease/disease_model_annotations_fb_2021_01.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#sgd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#sgd"],"http://purl.org/dc/terms/title":["Saccharomyces Genome Database"],"category":["dataset"],"title":["Saccharomyces Genome Database"]}},{"id":"MonarchArchive:#hgnc","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#hgnc"],"http://purl.org/dc/terms/title":["HGNC"],"category":["dataset"],"title":["HGNC"]}},{"id":"http://rest.kegg.jp/link/disease/hsa","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#ncbigene","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#ncbigene"],"http://purl.org/dc/terms/title":["National Center for Biotechnology Information"],"category":["dataset"],"title":["National Center for Biotechnology Information"]}},{"id":"MonarchLogoRepo:source-sgd.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#ucscbands","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#ucscbands"],"http://purl.org/dc/terms/title":["UCSC Cytogenic Bands"],"category":["dataset"],"title":["UCSC Cytogenic Bands"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/ONTOLOGY/disease_association.WS279.wb","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"ZFIN:warranty.html","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/species/c_elegans/PRJNA13758/c_elegans.PRJNA13758.WS279.annotations.gff3.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"MonarchArchive:202103/rdf/ctd.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Comparative Toxicogenomics Database distribution ttl"],"category":["dataset"],"title":["Comparative Toxicogenomics Database distribution ttl"],"version":["2021-03-01"]}},{"id":"MonarchArchive:DipperCache/wormbase/gaf-eco-mapping.yaml","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"http://ncbi.nih.nlm.gov/gene","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/CHECKSUMS","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"http://ftp.ncbi.nih.gov/gene/DATA/gene_history.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/ONTOLOGY/phenotype_association.WS279.wb","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"http://www.wormbase.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://ctdbase.org/about/legal.jsp","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/phenotype_fish.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/ONTOLOGY/rnai_phenotypes.WS279.wb","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"MonarchLogoRepo:source-zfin.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://tazendra.caltech.edu/~azurebrd/cgi-bin/forms/generic.cgi?action=WpaXref","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"https://zfin.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://wormbase.org/about/citing_wormbase#012--10","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#ctd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Comparative Toxicogenomics Database Monarch version 202103"],"category":["dataset"],"title":["Comparative Toxicogenomics Database Monarch version 202103"],"version":["2021-03-01"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/species/c_elegans/PRJNA13758/annotation/c_elegans.PRJNA13758.WS279.geneIDs.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"MonarchArchive:202103/rdf/string.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Known and predicted protein-protein interactions distribution ttl"],"category":["dataset"],"title":["Known and predicted protein-protein interactions distribution ttl"],"version":["2021-03-01"]}},{"id":"http://www.reactome.org/download/current/ChEBI2Reactome.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://reactome.org/license/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://reactome.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-reactome.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://ctdbase.org/reports/CTD_chemicals_diseases.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["Fri-Feb-26-19-34-10-EST-2021"],"version":["Fri-Feb-26-19-34-10-EST-2021"]}},{"id":"http://www.reactome.org/download/current/Ensembl2Reactome.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://rarediseases.info.nih.gov/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":":udp","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["udp"],"http://purl.org/dc/terms/title":["UDP"],"category":["dataset"],"title":["UDP"]}},{"id":"MonarchArchive:#ctd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#ctd"],"http://purl.org/dc/terms/title":["Comparative Toxicogenomics Database"],"category":["dataset"],"title":["Comparative Toxicogenomics Database"]}},{"id":"MonarchArchive:202103/#string","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Known and predicted protein-protein interactions Monarch version 202103"],"category":["dataset"],"title":["Known and predicted protein-protein interactions Monarch version 202103"],"version":["2021-03-01"]}},{"id":"ftp://ftp.flybase.net/releases/current/precomputed_files/alleles/fbal_to_fbgn_fb_FB2021_01.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-flybase.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#wormbase","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["WormBase Monarch version 202103"],"title":["WormBase Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/rdf/wormbase.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["WormBase distribution ttl"],"title":["WormBase distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:#coriell","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#coriell"],"http://purl.org/dc/terms/title":["Coriell Institute for Medical Research"],"category":["dataset"],"title":["Coriell Institute for Medical Research"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/letter.WS279","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"retrievedOn":["2021-03-09"],"version":["WS279"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://www.orphadata.org/data/xml/en_product6.xml","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.flybase.net/releases/current/precomputed_files/references/fbrf_pmid_pmcid_doi_fb_2021_01.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.orphadata.org/cgi-bin/index.php","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/reactome.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Reactome distribution ttl"],"title":["Reactome distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"http://www.flybase.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#wormbase","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#wormbase"],"http://purl.org/dc/terms/title":["WormBase"],"title":["WormBase"],"category":["dataset"]}},{"id":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/species/c_elegans/PRJNA13758/annotation/c_elegans.PRJNA13758.WS279.xrefs.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["WS279"],"version":["WS279"]}},{"id":"MonarchArchive:202103/#reactome","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Reactome Monarch version 202103"],"title":["Reactome Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:#monochrom","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#monochrom"],"http://purl.org/dc/terms/title":["Monarch Chromosome Ontology"],"category":["dataset"],"title":["Monarch Chromosome Ontology"]}},{"id":"MonarchArchive:#eom","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#eom"],"http://purl.org/dc/terms/title":["Elements of Morphology"],"category":["dataset"],"title":["Elements of Morphology"]}},{"id":"MonarchArchive:202103/#mmrrc","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Mutant Mouse Regional Resource Centers Monarch version 202103"],"category":["dataset"],"title":["Mutant Mouse Regional Resource Centers Monarch version 202103"],"version":["2021-03-01"]}},{"id":"https://data.omim.org/downloads/phenotypic_series_title_all.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"https://www.mmrrc.org/about/mmrrc_catalog_data.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-omim.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/mmrrc.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Mutant Mouse Regional Resource Centers distribution ttl"],"category":["dataset"],"title":["Mutant Mouse Regional Resource Centers distribution ttl"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/#coriell","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Coriell Institute for Medical Research Monarch version 202103"],"category":["dataset"],"title":["Coriell Institute for Medical Research Monarch version 202103"],"version":["2021-03-01"]}},{"id":"https://www.mmrrc.org/about/data_download.php","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"CoriellCollection:NIA/NIA.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["2021-02-26"],"version":["2021-02-26"]}},{"id":"MonarchArchive:202103/rdf/gwascatalog.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["NHGRI-EBI Catalog of published genome-wide association studies distribution ttl"],"category":["dataset"],"title":["NHGRI-EBI Catalog of published genome-wide association studies distribution ttl"],"version":["2021-03-01"]}},{"id":"CoriellCollection:NHGRI/NHGRI.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["2021-02-26"],"version":["2021-02-26"]}},{"id":"http://www.ebi.ac.uk/gwas/docs/about","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"OBO:doid.owl","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://hpo.jax.org/app/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.mmrrc.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#orphanet","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#orphanet"],"http://purl.org/dc/terms/title":["Orphanet"],"title":["Orphanet"],"category":["dataset"]}},{"id":"MonarchLogoRepo:source-wormbase.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/coriell.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Coriell Institute for Medical Research distribution ttl"],"category":["dataset"],"title":["Coriell Institute for Medical Research distribution ttl"],"version":["2021-03-01"]}},{"id":"MonarchLogoRepo:source-mmrrc.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"CoriellCollection:NINDS/NINDS.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["2021-02-26"],"version":["2021-02-26"]}},{"id":"MonarchArchive:202103/#gwascatalog","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["NHGRI-EBI Catalog of published genome-wide association studies Monarch version 202103"],"category":["dataset"],"title":["NHGRI-EBI Catalog of published genome-wide association studies Monarch version 202103"],"version":["2021-03-01"]}},{"id":"CoriellCollection:NIGMS/NIGMS.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["2021-02-26"],"version":["2021-02-26"]}},{"id":"MonarchArchive:202103/#omia","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Online Mendelian Inheritance in Animals Monarch version 202103"],"category":["dataset"],"title":["Online Mendelian Inheritance in Animals Monarch version 202103"],"version":["2021-03-01"]}},{"id":"http://www.animalgenome.org/cgi-bin/QTLdb/index","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/hpoa.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Human Phenotype Ontology distribution ttl"],"category":["dataset"],"title":["Human Phenotype Ontology distribution ttl"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/#hpoa","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Human Phenotype Ontology Monarch version 202103"],"category":["dataset"],"title":["Human Phenotype Ontology Monarch version 202103"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/rdf/omia.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Online Mendelian Inheritance in Animals distribution ttl"],"category":["dataset"],"title":["Online Mendelian Inheritance in Animals distribution ttl"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/#omim","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Online Mendelian Inheritance in Man Monarch version 202103"],"category":["dataset"],"title":["Online Mendelian Inheritance in Man Monarch version 202103"],"version":["2021-03-01"]}},{"id":"https://omia.org/dumps/omia.xml.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://omia.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://data.omim.org/downloads/morbidmap.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"MonarchLogoRepo:source-omia.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#go","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#go"],"http://purl.org/dc/terms/title":["Gene Ontology"],"title":["Gene Ontology"],"category":["dataset"]}},{"id":"MonarchArchive:202103/rdf/omim.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Online Mendelian Inheritance in Man distribution ttl"],"category":["dataset"],"title":["Online Mendelian Inheritance in Man distribution ttl"],"version":["2021-03-01"]}},{"id":"http://omim.org/help/agreement","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-animalqtldb.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/Morpholinos.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://compbio.charite.de/jenkins/job/hpo.annotations.current/lastSuccessfulBuild/artifact/current/phenotype.hpoa","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["2021-02-08"],"version":["2021-02-08"]}},{"id":"http://sydney.edu.au/disclaimer.shtml","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://hpo.jax.org/app/license","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://omia.org/curate/causal_mutations/?format=gene_table","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-hpo.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/export/trait_mappings.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202012/#impc","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2020-12-16"],"http://purl.org/dc/terms/created":["2020-12-16"],"http://purl.org/pav/version":["2020-12-01"],"http://purl.org/dc/terms/title":["International Mouse Phenotyping Consortium Monarch version 202012"],"category":["dataset"],"title":["International Mouse Phenotyping Consortium Monarch version 202012"],"version":["2020-12-01"]}},{"id":"http://phenomedoc.jax.org/MPD_downloads/measurements.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.omim.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://phenomedoc.jax.org/MPD_downloads/ontology_mappings.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#zfin","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#zfin"],"http://purl.org/dc/terms/title":["Zebra Fish Information Network"],"category":["dataset"],"title":["Zebra Fish Information Network"]}},{"id":"ftp://ftp.ebi.ac.uk/pub/databases/impc/all-data-releases/release-12.0/results/genotype-phenotype-assertions-ALL.csv.tgz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://phenomedoc.jax.org/MPD_downloads/straininfo.csv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.mousephenotype.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-mpd.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://phenomedoc.jax.org/MPD_downloads/strainmeans.csv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202012/rdf/impc.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2020-12-16"],"http://purl.org/pav/version":["2020-12-01"],"http://purl.org/dc/terms/created":["2020-12-16"],"http://purl.org/dc/terms/title":["International Mouse Phenotyping Consortium distribution ttl"],"category":["dataset"],"title":["International Mouse Phenotyping Consortium distribution ttl"],"version":["2020-12-01"]}},{"id":"PMID:15619963","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#panther","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#panther"],"http://purl.org/dc/terms/title":["Protein ANalysis THrough Evolutionary Relationships"],"category":["dataset"],"title":["Protein ANalysis THrough Evolutionary Relationships"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/canFam3/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"MonarchArchive:#animalqtldb","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#animalqtldb"],"http://purl.org/dc/terms/title":["Animal QTL db"],"title":["Animal QTL db"],"category":["dataset"]}},{"id":"ftp://ftp.ebi.ac.uk/pub/databases/impc/all-data-releases/release-12.0/results/genotype-phenotype-assertions-ALL.csv.tgz.md5","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/go.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Gene Ontology distribution ttl"],"title":["Gene Ontology distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/#ucscbands","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["UCSC Cytogenic Bands Monarch version 202103"],"category":["dataset"],"title":["UCSC Cytogenic Bands Monarch version 202103"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/#go","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Gene Ontology Monarch version 202103"],"title":["Gene Ontology Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MPD:","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/ucscbands.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["UCSC Cytogenic Bands distribution ttl"],"category":["dataset"],"title":["UCSC Cytogenic Bands distribution ttl"],"version":["2021-03-01"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/danRer11/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://current.geneontology.org/annotations/zfin.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"dctypes:Dataset","lbl":"Dataset","meta":{"types":["cliqueLeader","Node","Class","dataset"],"http://purl.org/dc/terms/issued":["2000-07-11"],"http://www.w3.org/2000/01/rdf-schema#comment":["Data encoded in a defined structure."],"http://www.w3.org/2000/01/rdf-schema#label":["Dataset"],"category":["dataset"],"http://purl.org/dc/terms/description":["Examples include lists, tables, and databases.  A dataset may be useful for direct machine processing."]}},{"id":"OBO:so.owl","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-gwascatalog.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#mgislim","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#mgislim"],"http://purl.org/dc/terms/title":["MonarchArchive:mgislim"],"category":["dataset"],"title":["MonarchArchive:mgislim"],"http://purl.org/dc/terms/description":["slim mgi model only containing Gene to phenotype associations\\nUses mousemine: http://www.mousemine.org/mousemine/begin.do\\npython lib api  http://intermine.org/intermine-ws-python/"]}},{"id":"https://hpo.jax.org/app/citation","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-ctd.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.ebi.ac.uk/gwas/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://ctdbase.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/monarch.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["The Monarch Initiative distribution ttl"],"category":["dataset"],"title":["The Monarch Initiative distribution ttl"],"version":["2021-03-01"]}},{"id":"https://ccr.coriell.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.ebi.ac.uk/pub/databases/gwas/releases/latest/gwas-catalog-associations_ontology-annotated.tsv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#mgi","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#mgi"],"http://purl.org/dc/terms/title":["Mouse Genome Informatics"],"category":["dataset"],"title":["Mouse Genome Informatics"]}},{"id":"https://github.com/monarch-initiative/mondo/releases/download/2019-04-06/mondo-minimal.json","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/hgnc_complete_set.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/mpd.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Mouse Phenome Database distribution ttl"],"category":["dataset"],"title":["Mouse Phenome Database distribution ttl"],"version":["2021-03-01"]}},{"id":"MonarchLogoRepo:source-hgnc.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#biogrid","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#biogrid"],"http://purl.org/dc/terms/title":["Biological General Repository for Interaction Datasets"],"category":["dataset"],"title":["Biological General Repository for Interaction Datasets"]}},{"id":"MonarchLogoRepo:source-coriell.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://creativecommons.org/licenses/by/4.0/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#hgnc","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["HGNC Monarch version 202103"],"title":["HGNC Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/#monarch","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["The Monarch Initiative Monarch version 202103"],"category":["dataset"],"title":["The Monarch Initiative Monarch version 202103"],"version":["2021-03-01"]}},{"id":"MonarchLogoRepo:source-monarch.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/hgnc.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["HGNC distribution ttl"],"title":["HGNC distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/#mpd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Mouse Phenome Database Monarch version 202103"],"category":["dataset"],"title":["Mouse Phenome Database Monarch version 202103"],"version":["2021-03-01"]}},{"id":"MonarchArchive:#reactome","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#reactome"],"http://purl.org/dc/terms/title":["Reactome"],"title":["Reactome"],"category":["dataset"]}},{"id":"ftp://ftp.ebi.ac.uk/pub/databases/genenames/README.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#omim","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#omim"],"http://purl.org/dc/terms/title":["Online Mendelian Inheritance in Man"],"category":["dataset"],"title":["Online Mendelian Inheritance in Man"]}},{"id":"MonarchArchive:202103/#monochrom","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Monarch Chromosome Ontology Monarch version 202103"],"category":["dataset"],"title":["Monarch Chromosome Ontology Monarch version 202103"],"version":["2021-03-01"]}},{"id":"https://project-open-data.cio.gov/unknown-license/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://current.geneontology.org/annotations/goa_pig.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-geneontology.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.informatics.jax.org/mgihome/other/copyright.shtml","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/monochrom.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Monarch Chromosome Ontology distribution ttl"],"category":["dataset"],"title":["Monarch Chromosome Ontology distribution ttl"],"version":["2021-03-01"]}},{"id":"http://www.geneontology.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"biolink:DataSetVersion","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/danRer10/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#mgislim","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["MonarchArchive:mgislim Monarch version 202103"],"http://purl.org/dc/terms/description":["slim mgi model only containing Gene to phenotype associations\\nUses mousemine: http://www.mousemine.org/mousemine/begin.do\\npython lib api  http://intermine.org/intermine-ws-python/"],"title":["MonarchArchive:mgislim Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"https://github.com/monarch-initiative/dipper","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/susScr3/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/idmapping/idmapping_selected.tab.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/oviAri4/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://current.geneontology.org/annotations/goa_cow.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":":","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-ucscbands.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://current.geneontology.org/annotations/mgi.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"jdbc:postgresql://mgi-adhoc.jax.org:5432/mgd","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["2021-03-01","6.16"],"retrievedOn":["2021-03-09"],"version":["2021-03-01","6.16"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://current.geneontology.org/annotations/goa_dog.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.w3.org/TR/turtle/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/oviAri3/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/mgislim.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["MonarchArchive:mgislim distribution ttl"],"http://purl.org/dc/terms/description":["slim mgi model only containing Gene to phenotype associations\\nUses mousemine: http://www.mousemine.org/mousemine/begin.do\\npython lib api  http://intermine.org/intermine-ws-python/"],"title":["MonarchArchive:mgislim distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchLogoRepo:source-mgi.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.mousemine.org/mousemine/service","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://creativecommons.org/licenses/by/4.0/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-monochrom.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.informatics.jax.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#panther","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-10"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-10"],"http://purl.org/dc/terms/title":["Protein ANalysis THrough Evolutionary Relationships Monarch version 202103"],"category":["dataset"],"title":["Protein ANalysis THrough Evolutionary Relationships Monarch version 202103"],"version":["2021-03-01"]}},{"id":"MonarchArchive:#hpoa","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#hpoa"],"http://purl.org/dc/terms/title":["Human Phenotype Ontology"],"category":["dataset"],"title":["Human Phenotype Ontology"]}},{"id":"MonarchArchive:#mpd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#mpd"],"http://purl.org/dc/terms/title":["Mouse Phenome Database"],"category":["dataset"],"title":["Mouse Phenome Database"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/cytoBand.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://current.geneontology.org/annotations/wb.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.genenames.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://current.geneontology.org/annotations/goa_chicken.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/felCat9/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://geneontology.org/page/use-and-license","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/mm10/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://current.geneontology.org/annotations/sgd.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://current.geneontology.org/annotations/rgd.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/susScr11/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://current.geneontology.org/annotations/pombase.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://raw.githubusercontent.com/mpi2/PhenotypeArchive/master/LICENSE","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MPD:about/termsofuse","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#mgi","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Mouse Genome Informatics Monarch version 202103"],"category":["dataset"],"title":["Mouse Genome Informatics Monarch version 202103"],"version":["2021-03-01"]}},{"id":"http://rest.genome.jp/list/disease","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://rest.kegg.jp/link/pathway/hsa","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://rest.genome.jp/link/omim/hsa","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/galGal6/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"MonarchArchive:DipperCache/go/gaf-eco-mapping.yaml","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.genome.jp/kegg/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/equCab2/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://current.geneontology.org/annotations/goa_human.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"dcat:Distribution","lbl":null,"meta":{"types":["cliqueLeader","Node","Class"]}},{"id":"http://genome.ucsc.edu/license/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://current.geneontology.org/annotations/fb.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/mgi.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Mouse Genome Informatics distribution ttl"],"category":["dataset"],"title":["Mouse Genome Informatics distribution ttl"],"version":["2021-03-01"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/bosTau7/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"http://rest.kegg.jp/link/pathway/ds","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"OBO:upheno/monarch.owl","lbl":null,"meta":{"http://purl.org/dc/elements/1.1/title":["MONARCH top importer, alpha version"],"types":["Ontology","cliqueLeader","Node"],"http://purl.org/dc/elements/1.1/description":["\\n        This is an ontology that collects a subset of core ontologies\\n        required by MONARCH. The set imported will continue to grow.\\n       "],"http://www.w3.org/2000/01/rdf-schema#comment":["Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/components/mondo-tags.owl>) VersionIRI(<null>))) [Axioms: 16743 Logical Axioms: 0]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/zfa_import.owl>) VersionIRI(<null>))) [Axioms: 26985 Logical Axioms: 621]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/zp.owl>) VersionIRI(<http://purl.obolibrary.org/obo/zp/releases/2020-08-02/zp.owl>))) [Axioms: 433967 Logical Axioms: 158572]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/bfo-axioms.owl>) VersionIRI(<null>))) [Axioms: 18 Logical Axioms: 7]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-ma.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-ma/2014-05-27/uberon/bridge/cl-bridge-to-ma.owl>))) [Axioms: 144 Logical Axioms: 8]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/cl/cl-simple.owl>) VersionIRI(<http://purl.obolibrary.org/obo/cl/2021-03-05/cl-simple.owl>))) [Axioms: 21751 Logical Axioms: 3626]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-zfa.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-zfa/2014-05-27/uberon/bridge/cl-bridge-to-zfa.owl>))) [Axioms: 2182 Logical Axioms: 416]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/generated-axioms.owl>) VersionIRI(<null>))) [Axioms: 18 Logical Axioms: 9]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/el-constraints.owl>) VersionIRI(<null>))) [Axioms: 6 Logical Axioms: 2]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/so.owl>) VersionIRI(<http://purl.obolibrary.org/obo/so/2021-02-16/so.owl>))) [Axioms: 25062 Logical Axioms: 3312]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/hp.owl>) VersionIRI(<http://purl.obolibrary.org/obo/hp/releases/2021-02-28/hp.owl>))) [Axioms: 325525 Logical Axioms: 57925]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/ro_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/ro_import.owl>))) [Axioms: 1555 Logical Axioms: 639]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/fbbt_phenotype.owl>) VersionIRI(<null>))) [Axioms: 25280 Logical Axioms: 8426]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/pw.owl>) VersionIRI(<http://purl.obolibrary.org/obo/pw/7.52/pw.owl>))) [Axioms: 22149 Logical Axioms: 3253]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/axioms.owl>) VersionIRI(<null>))) [Axioms: 11 Logical Axioms: 3]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-fma.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-fma/2014-05-27/uberon/bridge/cl-bridge-to-fma.owl>))) [Axioms: 3608 Logical Axioms: 700]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/go/extensions/go-plus.owl>) VersionIRI(<http://purl.obolibrary.org/obo/go/releases/2021-02-01/extensions/go-plus.owl>))) [Axioms: 820839 Logical Axioms: 260618]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/geno.owl>) VersionIRI(<http://purl.obolibrary.org/obo/geno/releases/2020-03-08/geno.owl>))) [Axioms: 3533 Logical Axioms: 727]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/eco.owl>) VersionIRI(<http://purl.obolibrary.org/obo/eco/releases/2021-03-05/eco.owl>))) [Axioms: 25376 Logical Axioms: 7584]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/chebi_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2021-02-12/chebi_import.owl>))) [Axioms: 955 Logical Axioms: 425]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/sepio.owl>) VersionIRI(<http://purl.obolibrary.org/obo/sepio-v20170929.owl>))) [Axioms: 1827 Logical Axioms: 305]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/foodon_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/foodon_import.owl>))) [Axioms: 1893 Logical Axioms: 364]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-wbbt.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-wbbt/2021-02-16/uberon/bridge/uberon-bridge-to-wbbt.owl>))) [Axioms: 273 Logical Axioms: 54]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/mf_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/mf_import.owl>))) [Axioms: 490 Logical Axioms: 54]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/ro_import.owl>) VersionIRI(<null>))) [Axioms: 1124 Logical Axioms: 484]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/uberon_import.owl>) VersionIRI(<null>))) [Axioms: 40892 Logical Axioms: 23831]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/clo-core.owl>) VersionIRI(<null>))) [Axioms: 252484 Logical Axioms: 40532]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/wbphenotype/wbphenotype-full.owl>) VersionIRI(<http://purl.obolibrary.org/obo/wbphenotype/releases/2021-03-10/wbphenotype-full.owl>))) [Axioms: 95125 Logical Axioms: 19377]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/metazoa.owl>) VersionIRI(<http://purl.obolibrary.org/obo/upheno/2014-12-17/metazoa.owl>))) [Axioms: 0 Logical Axioms: 0]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/ro_extra.owl>) VersionIRI(<null>))) [Axioms: 26 Logical Axioms: 12]","Includes Ontology(OntologyID(OntologyIRI(<http://biohackathon.org/resource/faldo>) VersionIRI(<null>))) [Axioms: 132 Logical Axioms: 49]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/dpo.owl>) VersionIRI(<http://purl.obolibrary.org/obo/dpo/releases/2021-01-14/dpo.owl>))) [Axioms: 13114 Logical Axioms: 3200]","Includes Ontology(OntologyID(OntologyIRI(<http://xmlns.com/foaf/0.1/>) VersionIRI(<null>))) [Axioms: 551 Logical Axioms: 161]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/ecto_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/ecto_import.owl>))) [Axioms: 8413 Logical Axioms: 2857]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-nifstd.owl>) VersionIRI(<null>))) [Axioms: 18 Logical Axioms: 6]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/ro_import.owl>) VersionIRI(<null>))) [Axioms: 1313 Logical Axioms: 358]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/iao.owl>) VersionIRI(<http://purl.obolibrary.org/obo/iao/2020-12-09/iao.owl>))) [Axioms: 3565 Logical Axioms: 510]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/upheno_root_alignments.owl>) VersionIRI(<null>))) [Axioms: 2 Logical Axioms: 2]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-wbbt.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-wbbt/2014-05-27/uberon/bridge/cl-bridge-to-wbbt.owl>))) [Axioms: 252 Logical Axioms: 30]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/xco.owl>) VersionIRI(<http://purl.obolibrary.org/obo/xco/4.46/xco.owl>))) [Axioms: 6322 Logical Axioms: 820]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mp.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mp/releases/2021-01-12>))) [Axioms: 504123 Logical Axioms: 102485]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/rohom.owl>) VersionIRI(<null>))) [Axioms: 731 Logical Axioms: 148]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/temporal-intervals.owl>) VersionIRI(<null>))) [Axioms: 146 Logical Axioms: 45]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/mondo.owl>))) [Axioms: 627573 Logical Axioms: 137746]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/auto>) VersionIRI(<null>))) [Axioms: 1479 Logical Axioms: 295]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/hp-mp/mp_hp-align-equiv.owl>) VersionIRI(<null>))) [Axioms: 5770 Logical Axioms: 1442]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/go_cc_import.owl>) VersionIRI(<null>))) [Axioms: 49 Logical Axioms: 14]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/pato_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2021-02-12/pato_import.owl>))) [Axioms: 433 Logical Axioms: 143]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/zfa.owl>) VersionIRI(<http://purl.obolibrary.org/obo/zfa/releases/2020-04-14/zfa.owl>))) [Axioms: 36534 Logical Axioms: 12036]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/chebi_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/chebi_import.owl>))) [Axioms: 9189 Logical Axioms: 2241]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/pato_import.owl>) VersionIRI(<null>))) [Axioms: 20 Logical Axioms: 6]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/bfo-classes-minimal.owl>) VersionIRI(<null>))) [Axioms: 56 Logical Axioms: 13]","\\n        MONARCH members are free to add new imports. Care must be taken to avoid causing unsatisfiable\\n        classes.\\n      ","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/uberon_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/uberon_import.owl>))) [Axioms: 50011 Logical Axioms: 18581]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-fbbt.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-fbbt/2014-05-27/uberon/bridge/cl-bridge-to-fbbt.owl>))) [Axioms: 412 Logical Axioms: 62]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-fma.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-fma/2021-02-16/uberon/bridge/uberon-bridge-to-fma.owl>))) [Axioms: 29923 Logical Axioms: 5993]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/annotations.owl>) VersionIRI(<null>))) [Axioms: 60 Logical Axioms: 0]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/other_import.owl>) VersionIRI(<null>))) [Axioms: 104 Logical Axioms: 26]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/go_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2021-02-12/go_import.owl>))) [Axioms: 6779 Logical Axioms: 4011]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/chemical.owl>) VersionIRI(<null>))) [Axioms: 9 Logical Axioms: 3]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/envo_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2013-10-21/envo_import.owl>))) [Axioms: 28 Logical Axioms: 7]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/pato.owl>) VersionIRI(<http://purl.obolibrary.org/obo/pato/releases/2020-08-02/pato.owl>))) [Axioms: 23470 Logical Axioms: 2881]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/core.owl>) VersionIRI(<null>))) [Axioms: 291 Logical Axioms: 62]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/ext.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2021-02-12/ext.owl>))) [Axioms: 237512 Logical Axioms: 47946]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/cl_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/cl_import.owl>))) [Axioms: 5146 Logical Axioms: 1256]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/pco.owl>) VersionIRI(<http://purl.obolibrary.org/obo/pco/releases/2020-08-26/pco.owl>))) [Axioms: 3582 Logical Axioms: 602]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/so_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/so_import.owl>))) [Axioms: 434 Logical Axioms: 95]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/pr_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2021-02-12/pr_import.owl>))) [Axioms: 55 Logical Axioms: 17]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/hp_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/hp_import.owl>))) [Axioms: 14834 Logical Axioms: 1383]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.org/oban/oban_core.owl>) VersionIRI(<null>))) [Axioms: 44 Logical Axioms: 11]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-zfa.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-zfa/2021-02-16/uberon/bridge/uberon-bridge-to-zfa.owl>))) [Axioms: 10356 Logical Axioms: 2069]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/envo_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/envo_import.owl>))) [Axioms: 3722 Logical Axioms: 1441]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/go_phenotype.owl>) VersionIRI(<null>))) [Axioms: 53179 Logical Axioms: 10635]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/hgnc_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/hgnc_import.owl>))) [Axioms: 11344 Logical Axioms: 3781]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/mpath_phenotype.owl>) VersionIRI(<null>))) [Axioms: 1019 Logical Axioms: 203]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/ncbitaxon_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/ncbitaxon_import.owl>))) [Axioms: 12702 Logical Axioms: 1357]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-fbbt.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-fbbt/2021-02-16/uberon/bridge/uberon-bridge-to-fbbt.owl>))) [Axioms: 1995 Logical Axioms: 397]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/go_mf_import.owl>) VersionIRI(<null>))) [Axioms: 29 Logical Axioms: 9]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/mammal.owl>) VersionIRI(<null>))) [Axioms: 0 Logical Axioms: 0]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/go_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/go_import.owl>))) [Axioms: 30470 Logical Axioms: 9667]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro.owl>) VersionIRI(<http://purl.obolibrary.org/obo/ro/releases/2021-03-08/ro.owl>))) [Axioms: 4380 Logical Axioms: 1096]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl>) VersionIRI(<null>))) [Axioms: 127486 Logical Axioms: 14365]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-ma.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-ma/2021-02-16/uberon/bridge/uberon-bridge-to-ma.owl>))) [Axioms: 15360 Logical Axioms: 3073]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/TEMP>) VersionIRI(<null>))) [Axioms: 29331 Logical Axioms: 4226]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/nbo_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2013-10-21/nbo_import.owl>))) [Axioms: 30 Logical Axioms: 9]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/cl_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2021-02-12/cl_import.owl>))) [Axioms: 25632 Logical Axioms: 6366]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/uberon/ncbitaxon_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/uberon/releases/2021-02-12/ncbitaxon_import.owl>))) [Axioms: 1665 Logical Axioms: 937]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/vertebrate.owl>) VersionIRI(<null>))) [Axioms: 12 Logical Axioms: 0]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ro/go-biotic.owl>) VersionIRI(<null>))) [Axioms: 42 Logical Axioms: 16]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/fbbt_import.owl>) VersionIRI(<null>))) [Axioms: 44631 Logical Axioms: 27184]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/imports/uberon_phenotype.owl>) VersionIRI(<null>))) [Axioms: 51894 Logical Axioms: 10378]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/mondo/imports/pato_import.owl>) VersionIRI(<http://purl.obolibrary.org/obo/mondo/releases/2021-03-03/imports/pato_import.owl>))) [Axioms: 1347 Logical Axioms: 489]","Includes Ontology(OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/upheno/dpo/dpo-bridge.owl>) VersionIRI(<null>))) [Axioms: 1 Logical Axioms: 1]"],"http://www.w3.org/2000/01/rdf-schema#seeAlso":["http://genomebiology.com/2012/13/1/R5"],"http://purl.org/dc/elements/1.1/creator":["Chris Mungall"]}},{"id":"http://rest.genome.jp/list/pathway","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://rest.kegg.jp/link/pathway/pubmed","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.kegg.jp/kegg/legal.html","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#monarch","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#monarch"],"http://purl.org/dc/terms/title":["The Monarch Initiative"],"category":["dataset"],"title":["The Monarch Initiative"]}},{"id":"MonarchArchive:#mmrrc","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#mmrrc"],"http://purl.org/dc/terms/title":["Mutant Mouse Regional Resource Centers"],"category":["dataset"],"title":["Mutant Mouse Regional Resource Centers"]}},{"id":"MonarchArchive:202103/#animalqtldb","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Animal QTL db Monarch version 202103"],"title":["Animal QTL db Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchLogoRepo:source-ncbi.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#rgd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#rgd"],"http://purl.org/dc/terms/title":["Rat Genome Database"],"category":["dataset"],"title":["Rat Genome Database"]}},{"id":"MonarchArchive:202103/rdf/biogrid.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Biological General Repository for Interaction Datasets distribution ttl"],"category":["dataset"],"title":["Biological General Repository for Interaction Datasets distribution ttl"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/rdf/animalqtldb.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"rights":["\'https://www.animalgenome.org/QTLdb/faq#32\'"],"http://purl.org/dc/terms/title":["Animal QTL db distribution ttl"],"category":["dataset"],"title":["Animal QTL db distribution ttl"],"version":["2021-03-01"],"http://purl.org/dc/terms/rights":["\'https://www.animalgenome.org/QTLdb/faq#32\'"]}},{"id":"MonarchArchive:202103/#biogrid","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Biological General Repository for Interaction Datasets Monarch version 202103"],"category":["dataset"],"title":["Biological General Repository for Interaction Datasets Monarch version 202103"],"version":["2021-03-01"]}},{"id":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/chicken_QTLdata.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#flybase","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#flybase"],"http://purl.org/dc/terms/title":["FlyBase"],"title":["FlyBase"],"category":["dataset"]}},{"id":"https://downloads.thebiogrid.org/Download/BioGRID/Latest-Release/BIOGRID-IDENTIFIERS-LATEST.tab.zip","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:DipperCache/Equus_caballus.gene_info.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://ftp.ncbi.nih.gov/gene/DATA/gene_info.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.genome.gov/copyright.cfm","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://ftp.ncbi.nih.gov/gene/DATA/gene2pubmed.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/rn6/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-orphanet.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#kegg","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#kegg"],"http://purl.org/dc/terms/title":["Kyoto Encyclopedia of Genes and Genomes"],"category":["dataset"],"title":["Kyoto Encyclopedia of Genes and Genomes"]}},{"id":"https://creativecommons.org/publicdomain/mark/1.0/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://data.omim.org/downloads/mimTitles.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-10","2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-10","2021-03-09"]}},{"id":"http://hgdownload.cse.ucsc.edu/goldenPath/galGal4/database/cytoBandIdeo.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://pantherdb.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/tmp/QTL_EquCab2.0.gff.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://downloads.thebiogrid.org/Download/LICENSE.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/horse_QTLdata.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.pantherdb.org/ortholog/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/sheep_QTLdata.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://www.pantherdb.org/terms/disclaimer.jsp","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"jdbc:postgresql://nif-db.crbs.ucsd.edu:5432/disco_crawler","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["2021-03-09"],"version":["2021-03-09"]}},{"id":"ftp://ftp.pantherdb.org/ortholog/current_release/RefGenomeOrthologs.tar.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/master/src/mappings/hp-to-eom-mapping.tsv","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"MonarchLogoRepo:source-impc.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-panther.jpg","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-eom.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#ncbigene","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["National Center for Biotechnology Information Monarch version 202103"],"category":["dataset"],"title":["National Center for Biotechnology Information Monarch version 202103"],"version":["2021-03-01"]}},{"id":"https://monarchinitiative.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://current.geneontology.org/annotations/dictybase.gaf.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#eom","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Elements of Morphology Monarch version 202103"],"category":["dataset"],"title":["Elements of Morphology Monarch version 202103"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/rdf/panther.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-10"],"http://purl.org/dc/terms/created":["2021-03-10"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Protein ANalysis THrough Evolutionary Relationships distribution ttl"],"category":["dataset"],"title":["Protein ANalysis THrough Evolutionary Relationships distribution ttl"],"version":["2021-03-01"]}},{"id":"http://pantherdb.org/publications.jsp#HowToCitePANTHER","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/eom.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Elements of Morphology distribution ttl"],"category":["dataset"],"title":["Elements of Morphology distribution ttl"],"version":["2021-03-01"]}},{"id":"ftp://ftp.pantherdb.org/ortholog/current_release/Orthologs_HCOP.tar.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-10"],"http://purl.org/pav/retrievedOn":["2021-03-10"]}},{"id":"MonarchArchive:DipperCache/impc/impc_evidence_stable_key.tsv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-kegg.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://rest.genome.jp/list/hsa","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#orphanet","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Orphanet Monarch version 202103"],"title":["Orphanet Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/rdf/ncbigene.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["National Center for Biotechnology Information distribution ttl"],"category":["dataset"],"title":["National Center for Biotechnology Information distribution ttl"],"version":["2021-03-01"]}},{"id":"http://elementsofmorphology.nih.gov","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://ftp.ncbi.nih.gov/gene/DATA/gene_group.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.ncbi.nlm.nih.gov/home/about/policies/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#omia","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#omia"],"http://purl.org/dc/terms/title":["Online Mendelian Inheritance in Animals"],"category":["dataset"],"title":["Online Mendelian Inheritance in Animals"]}},{"id":"MonarchArchive:202103/rdf/orphanet.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Orphanet distribution ttl"],"title":["Orphanet distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchArchive:#string","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#string"],"http://purl.org/dc/terms/title":["Known and predicted protein-protein interactions"],"category":["dataset"],"title":["Known and predicted protein-protein interactions"]}},{"id":"ZFIN:downloads/genotype_backgrounds.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/CRISPR.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/fish_model_disease.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/gene.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/pub_to_pubmed_id_translation.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/wildtypes_fish.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2020-12-16"],"http://purl.org/pav/retrievedOn":["2020-12-16"]}},{"id":"OBO:zp//id_map_zfin.tsv","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://0.0.0.0","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2020-12-16"],"http://purl.org/pav/retrievedOn":["2020-12-16"]}},{"id":"MonarchArchive:202103/#bgee","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Bgee Gene expression data in animals Monarch version 202103"],"category":["dataset"],"title":["Bgee Gene expression data in animals Monarch version 202103"],"version":["2021-03-01"]}},{"id":"http://uswest.ensembl.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:DipperCache/reactome/gaf-eco-mapping.yaml","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/bgee.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Bgee Gene expression data in animals distribution ttl"],"category":["dataset"],"title":["Bgee Gene expression data in animals distribution ttl"],"version":["2021-03-01"]}},{"id":"https://string-db.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/download/protein.links.detailed.v11.0/7955.protein.links.detailed.v11.0.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#ensembl","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["ENSEMBL Monarch version 202103"],"title":["ENSEMBL Monarch version 202103"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchLogoRepo:source-string.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://uswest.ensembl.org/biomart/martservice?","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-09"],"http://purl.org/pav/retrievedOn":["2021-03-09"]}},{"id":"MonarchArchive:202103/rdf/ensembl.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["ENSEMBL distribution ttl"],"title":["ENSEMBL distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"MonarchLogoRepo:source-ensembl.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/features.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/genotype_features.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/gene_marker_relationship.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#zfinslim","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#zfinslim"],"http://purl.org/dc/terms/title":["Simplified ZFIN"],"category":["dataset"],"title":["Simplified ZFIN"]}},{"id":"MonarchArchive:202012/#zfin","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2020-12-16"],"http://purl.org/dc/terms/created":["2020-12-16"],"http://purl.org/pav/version":["2020-12-01"],"http://purl.org/dc/terms/title":["Zebra Fish Information Network Monarch version 202012"],"category":["dataset"],"title":["Zebra Fish Information Network Monarch version 202012"],"version":["2020-12-01"]}},{"id":"https://www.animalgenome.org/QTLdb/tmp/QTL_GG_5.0.gff.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202012/rdf/zfin.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2020-12-16"],"http://purl.org/dc/terms/created":["2020-12-16"],"http://purl.org/pav/version":["2020-12-01"],"http://purl.org/dc/terms/title":["Zebra Fish Information Network distribution ttl"],"category":["dataset"],"title":["Zebra Fish Information Network distribution ttl"],"version":["2020-12-01"]}},{"id":"MonarchArchive:DipperCache/Ovis_aries.gene_info.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/genbank.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/tmp/QTL_SS_11.1.gff.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/zfinpubs.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/pig_QTLdata.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://downloads.thebiogrid.org/Download/BioGRID/Latest-Release/BIOGRID-ALL-LATEST.mitab.zip","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/pav/version":["4.3.195"],"version":["4.3.195"]}},{"id":"https://www.animalgenome.org/QTLdb/tmp/QTL_OAR_4.0.gff.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://wiki.thebiogrid.org/doku.php/terms_and_conditions","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Non-mammalian_vertebrates/Gallus_gallus.gene_info.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-biogrid.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/cattle_QTLdata.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://thebiogrid.org","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:DipperCache/Oncorhynchus_mykiss.gene_info.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/fish_components_fish.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/download/protein.links.detailed.v11.0/9606.protein.links.detailed.v11.0.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/cgi/access.pl?footer_active_subpage=licensing","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/download/protein.links.detailed.v11.0/10090.protein.links.detailed.v11.0.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/download/protein.links.detailed.v11.0/10116.protein.links.detailed.v11.0.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/download/protein.links.detailed.v11.0/7227.protein.links.detailed.v11.0.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/mapping_files/entrez/celegans.entrez_2_string.2018.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#bgee","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#bgee"],"http://purl.org/dc/terms/title":["Bgee Gene expression data in animals"],"category":["dataset"],"title":["Bgee Gene expression data in animals"]}},{"id":"https://string-db.org/mapping_files/entrez/human.entrez_2_string.2018.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/download/protein.links.detailed.v11.0/6239.protein.links.detailed.v11.0.txt.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/mapping_files/entrez/yeast.entrez_2_string.2018.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/mapping_files/entrez/zebrafish.entrez_2_string.2018.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/mapping_files/entrez/mouse.entrez_2_string.2018.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/human_orthos.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/E_zfin_gene_alias.gff3","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#kegg","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Kyoto Encyclopedia of Genes and Genomes Monarch version 202103"],"category":["dataset"],"title":["Kyoto Encyclopedia of Genes and Genomes Monarch version 202103"],"version":["2021-03-01"]}},{"id":"MonarchArchive:202103/rdf/kegg.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Kyoto Encyclopedia of Genes and Genomes distribution ttl"],"category":["dataset"],"title":["Kyoto Encyclopedia of Genes and Genomes distribution ttl"],"version":["2021-03-01"]}},{"id":"http://rest.genome.jp/conv/ncbi-geneid/hsa","lbl":null,"meta":{"types":["cliqueLeader","Node"],"retrievedOn":["2021-03-10"],"http://purl.org/pav/retrievedOn":["2021-03-10"]}},{"id":"http://rest.kegg.jp/link/pathway/ko","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"http://rest.genome.jp/list/orthology","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:#impc","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#impc"],"http://purl.org/dc/terms/title":["International Mouse Phenotyping Consortium"],"category":["dataset"],"title":["International Mouse Phenotyping Consortium"]}},{"id":"http://rest.genome.jp/link/disease/omim","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://string-db.org/mapping_files/entrez/fly.entrez_2_string.2018.tsv.gz","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#sgd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Saccharomyces Genome Database Monarch version 202103"],"category":["dataset"],"title":["Saccharomyces Genome Database Monarch version 202103"],"version":["2021-03-01"]}},{"id":"https://sites.google.com/view/yeastgenome-help/about","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://downloads.yeastgenome.org/curation/literature/phenotype_data.tab","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://www.yeastgenome.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/mappings.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/TALEN.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/pheno_environment_fish.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/features-affected-genes.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:Downloads/stage_ontology.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://wiki.zfin.org/display/general/ZFIN+db+information","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ZFIN:downloads/uniprot.txt","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://rgd.mcw.edu/wg/citing-rgd/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-rgd.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://rgd.mcw.edu/wg/disclaimer/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/sgd.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/title":["Saccharomyces Genome Database distribution ttl"],"category":["dataset"],"title":["Saccharomyces Genome Database distribution ttl"],"version":["2021-03-01"]}},{"id":"http://rgd.mcw.edu/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchLogoRepo:source-bgee.png","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"https://bgee.org/?page=about","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/rdf/rgd.ttl","lbl":null,"meta":{"types":["cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Rat Genome Database distribution ttl"],"title":["Rat Genome Database distribution ttl"],"category":["dataset"],"version":["2021-03-01"]}},{"id":"http://bgee.org/","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"ftp://ftp.rgd.mcw.edu/pub/data_release/annotated_rgd_objects_by_ontology/rattus_genes_mp","lbl":null,"meta":{"types":["cliqueLeader","Node"]}},{"id":"MonarchArchive:202103/#rgd","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"created":["2021-03-09"],"http://purl.org/pav/version":["2021-03-01"],"http://purl.org/dc/terms/created":["2021-03-09"],"http://purl.org/dc/terms/title":["Rat Genome Database Monarch version 202103"],"category":["dataset"],"title":["Rat Genome Database Monarch version 202103"],"version":["2021-03-01"]}},{"id":"owl:Thing","lbl":"Thing","meta":{"types":["cliqueLeader","Node","Class"],"http://www.w3.org/2000/01/rdf-schema#label":["Thing"]}},{"id":"dc:DCMIType","lbl":"DCMI Type Vocabulary","meta":{"types":["cliqueLeader","Node","NamedIndividual"],"http://purl.org/dc/terms/issued":["2000-07-11"],"http://www.w3.org/2000/01/rdf-schema#comment":["The set of classes specified by the DCMI Type Vocabulary, used to categorize the nature or genre of the resource."],"http://www.w3.org/2000/01/rdf-schema#label":["DCMI Type Vocabulary"]}},{"id":"dctypes:","lbl":null,"meta":{"types":["cliqueLeader","Node"],"http://purl.org/dc/terms/modified":["2012-06-14"],"http://purl.org/dc/terms/title":["DCMI Type Vocabulary"],"title":["DCMI Type Vocabulary"]}},{"id":"MonarchArchive:#ensembl","lbl":null,"meta":{"types":["Ontology","cliqueLeader","Node","dataset"],"http://purl.org/dc/terms/identifier":["MonarchArchive:#ensembl"],"http://purl.org/dc/terms/title":["ENSEMBL"],"title":["ENSEMBL"],"category":["dataset"]}}],"edges":[{"sub":"MonarchArchive:#panther","obj":"MonarchLogoRepo:source-panther.jpg","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":"MonarchArchive:202103/rdf/panther.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/#impc","obj":"MonarchArchive:#impc","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#ucscbands","obj":"MonarchArchive:202103/#ucscbands","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#panther","obj":"ftp://ftp.pantherdb.org/ortholog/current_release/RefGenomeOrthologs.tar.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#impc","obj":"ftp://ftp.ebi.ac.uk/pub/databases/impc/all-data-releases/release-12.0/results/genotype-phenotype-assertions-ALL.csv.tgz.md5","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#impc","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#panther","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":"OBO:so.owl","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#mgi","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/#impc","obj":"MonarchArchive:202012/rdf/impc.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":"http://www.ebi.ac.uk/gwas/docs/about","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":"https://raw.githubusercontent.com/mpi2/PhenotypeArchive/master/LICENSE","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"MonarchArchive:#ucscbands","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/canFam3/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:#panther","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/danRer11/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:#ucscbands","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["type"]}},{"sub":"MonarchArchive:#ncbigene","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"http://ftp.ncbi.nih.gov/gene/DATA/gene_group.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/felCat9/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":"https://www.ncbi.nlm.nih.gov/home/about/policies/","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/mm10/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"MonarchArchive:202103/rdf/ncbigene.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/susScr11/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/equCab2/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":"http://genome.ucsc.edu/license/","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/galGal6/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":"MonarchArchive:202103/rdf/ucscbands.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["license"]}},{"sub":"MonarchArchive:202012/#impc","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#impc","obj":"MonarchArchive:202012/#impc","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#omia","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#hgnc","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#mmrrc","obj":"MonarchArchive:#mmrrc","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#hgnc","obj":"MonarchLogoRepo:source-hgnc.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#mmrrc","obj":"MonarchArchive:202103/rdf/mmrrc.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:#hpoa","obj":"https://hpo.jax.org/app/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["source"]}},{"sub":"MonarchArchive:#animalqtldb","obj":"MonarchLogoRepo:source-animalqtldb.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#omim","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["Format"]}},{"sub":"MonarchArchive:202012/#impc","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":"MonarchArchive:202103/rdf/mmrrc.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202012/#impc","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#mmrrc","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#omim","obj":"http://www.omim.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#mmrrc","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":"https://www.mmrrc.org/about/data_download.php","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#panther","obj":"MonarchArchive:202103/rdf/panther.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#panther","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["license"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["type"]}},{"sub":"MonarchArchive:#mmrrc","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#panther","obj":"MonarchArchive:#panther","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:#mmrrc","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#panther","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/#impc","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["type"]}},{"sub":"MonarchArchive:#impc","obj":"http://www.mousephenotype.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":"http://www.pantherdb.org/terms/disclaimer.jsp","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#panther","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/#impc","obj":"ftp://ftp.ebi.ac.uk/pub/databases/impc/all-data-releases/release-12.0/results/genotype-phenotype-assertions-ALL.csv.tgz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#impc","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":"MonarchArchive:202103/rdf/gwascatalog.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/ONTOLOGY/phenotype_association.WS279.wb","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/rdf/impc.ttl","obj":"MonarchArchive:202012/rdf/impc.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":"MonarchArchive:#gwascatalog","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["type"]}},{"sub":"MonarchArchive:#wormbase","obj":"MonarchLogoRepo:source-wormbase.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"MonarchArchive:202103/rdf/ucscbands.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:#biogrid","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#biogrid","obj":"http://thebiogrid.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#panther","obj":"http://pantherdb.org/publications.jsp#HowToCitePANTHER","pred":"cito:citesAsAuthority","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["http://purl.org/spar/cito/citesAsAuthority"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":"https://wiki.thebiogrid.org/doku.php/terms_and_conditions","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#panther","obj":"ftp://ftp.pantherdb.org/ortholog/current_release/Orthologs_HCOP.tar.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/panther.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#panther","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["license"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/fish_components_fish.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"MonarchArchive:202012/rdf/zfin.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/zfinpubs.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:#mgi","obj":"MonarchLogoRepo:source-mgi.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/genbank.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:#mgi","obj":"http://www.informatics.jax.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["source"]}},{"sub":"MonarchArchive:#panther","obj":"MonarchArchive:202103/#panther","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:#zfin","obj":"MonarchArchive:202012/#zfin","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":"https://creativecommons.org/publicdomain/mark/1.0/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"http://ftp.ncbi.nih.gov/gene/DATA/gene2pubmed.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"http://ftp.ncbi.nih.gov/gene/DATA/gene_info.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"MonarchArchive:#ncbigene","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#ucscbands","obj":"http://hgdownload.cse.ucsc.edu","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["Format"]}},{"sub":"MonarchArchive:#ucscbands","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":"MonarchArchive:202103/rdf/ncbigene.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#ucscbands","obj":"MonarchLogoRepo:source-ucscbands.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/cytoBand.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"https://data.omim.org/downloads/mimTitles.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/ncbigene.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ucscbands.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["type"]}},{"sub":"MonarchArchive:#hgnc","obj":"MonarchArchive:202103/#hgnc","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:#ncbigene","obj":"MonarchLogoRepo:source-ncbi.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#ncbigene","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/oviAri4/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:#monochrom","obj":"MonarchArchive:202103/#monochrom","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#panther","obj":"ftp://ftp.pantherdb.org/ortholog/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":"https://downloads.thebiogrid.org/Download/LICENSE.txt","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#biogrid","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/susScr3/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":"MonarchArchive:202103/rdf/biogrid.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/danRer10/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:#panther","obj":"http://pantherdb.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#panther"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/equCab2/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":"MonarchArchive:#biogrid","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["type"]}},{"sub":"MonarchArchive:#biogrid","obj":"MonarchArchive:202103/#biogrid","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/cytoBand.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"MonarchArchive:202103/rdf/monochrom.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/mm10/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/ONTOLOGY/disease_association.WS279.wb","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:#wormbase","obj":"MonarchArchive:202103/#wormbase","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#orphanet","obj":"MonarchArchive:#orphanet","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#flybase","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#flybase","obj":"MonarchLogoRepo:source-flybase.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#flybase","obj":"http://www.flybase.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":"https://wiki.zfin.org/display/general/ZFIN+db+information","pred":"cito:citesAsAuthority","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://purl.org/spar/cito/citesAsAuthority"]}},{"sub":"MonarchArchive:202103/#reactome","obj":"MonarchArchive:202103/rdf/reactome.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#reactome","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#reactome","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#reactome","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":"MonarchArchive:202103/rdf/reactome.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#reactome","obj":"http://www.reactome.org/download/current/Ensembl2Reactome.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":"https://wormbase.org/about/citing_wormbase#012--10","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"http://tazendra.caltech.edu/~azurebrd/cgi-bin/forms/generic.cgi?action=WpaXref","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:#zfinslim","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/ONTOLOGY/rnai_phenotypes.WS279.wb","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:#wormbase","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":"MonarchArchive:202103/rdf/wormbase.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/species/c_elegans/PRJNA13758/annotation/c_elegans.PRJNA13758.WS279.geneIDs.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["type"]}},{"sub":"MonarchArchive:#reactome","obj":"http://reactome.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["source"]}},{"sub":"MonarchArchive:#reactome","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#reactome","obj":"MonarchLogoRepo:source-reactome.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#reactome","obj":"MonarchArchive:#reactome","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#reactome","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#reactome","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#reactome","obj":"http://www.reactome.org/download/current/ChEBI2Reactome.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/reactome.ttl","obj":"https://reactome.org/license/","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["Rights"]}},{"sub":"MonarchArchive:#string","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["type"]}},{"sub":"MonarchArchive:#zfinslim","obj":"https://zfin.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["source"]}},{"sub":"MonarchArchive:#zfinslim","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#zfinslim","obj":"MonarchLogoRepo:source-zfin.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":"https://zfin.org/downloads/phenoGeneCleanData_fish.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/mapping_files/entrez/human.entrez_2_string.2018.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/download/protein.links.detailed.v11.0/10116.protein.links.detailed.v11.0.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/download/protein.links.detailed.v11.0/7227.protein.links.detailed.v11.0.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#string","obj":"MonarchArchive:202103/#string","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":":udp","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["_:5bf83e82e809f7c93fb6007bbd13030e"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/download/protein.links.detailed.v11.0/6239.protein.links.detailed.v11.0.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":"MonarchArchive:202103/rdf/string.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"MonarchArchive:202103/rdf/flybase.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#flybase","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"ftp://ftp.flybase.net/releases/current/precomputed_files/references/fbrf_pmid_pmcid_doi_fb_2021_01.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#orphanet","obj":"http://www.orphadata.org/data/xml/en_product6.xml","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"ftp://ftp.flybase.net/releases/current/precomputed_files/human_disease/disease_model_annotations_fb_2021_01.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#string","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/mapping_files/entrez/yeast.entrez_2_string.2018.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#string","obj":"MonarchArchive:202103/rdf/string.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/mapping_files/entrez/celegans.entrez_2_string.2018.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/download/protein.links.detailed.v11.0/7955.protein.links.detailed.v11.0.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":"https://string-db.org/cgi/access.pl?footer_active_subpage=licensing","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#string","obj":"MonarchArchive:#string","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/download/protein.links.detailed.v11.0/10090.protein.links.detailed.v11.0.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":"ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/hgnc_complete_set.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["source"]}},{"sub":"MonarchArchive:#hgnc","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":"ftp://ftp.ebi.ac.uk/pub/databases/genenames/README.txt","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#omia","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":"MonarchArchive:202103/rdf/hgnc.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#omia","obj":"http://omia.org/curate/causal_mutations/?format=gene_table","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":"https://data.omim.org/downloads/mimTitles.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/rainbow_trout_QTLdata.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:#ctd","obj":"MonarchLogoRepo:source-ctd.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":"http://sydney.edu.au/disclaimer.shtml","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#omia","obj":"https://omia.org/dumps/omia.xml.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":"MonarchArchive:202103/rdf/omia.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/tmp/QTL_SS_11.1.gff.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#omia","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#string","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/download/protein.links.detailed.v11.0/9606.protein.links.detailed.v11.0.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/mapping_files/entrez/zebrafish.entrez_2_string.2018.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#string","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/mapping_files/entrez/mouse.entrez_2_string.2018.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#string","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#ensembl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#omia","obj":"MonarchArchive:#omia","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#omia","obj":"https://data.omim.org/downloads/mimTitles.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["type"]}},{"sub":"MonarchArchive:#omia","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#hgnc","obj":"MonarchArchive:#hgnc","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"CoriellCollection:NIA/NIA.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":"MonarchArchive:202103/rdf/hgnc.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#omia","obj":"MonarchArchive:202103/rdf/omia.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#mmrrc","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/hgnc.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#mmrrc","obj":"https://www.mmrrc.org/about/mmrrc_catalog_data.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["source"]}},{"sub":"MonarchArchive:#omia","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["Format"]}},{"sub":"MonarchArchive:#omia","obj":"MonarchLogoRepo:source-omia.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/export/trait_mappings.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ensembl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#ensembl","obj":"MonarchArchive:#ensembl","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#ensembl","obj":"MonarchArchive:202103/#ensembl","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:#string","obj":"https://string-db.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/string.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#bgee","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#bgee","obj":"MonarchArchive:202103/rdf/bgee.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:#ensembl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#ensembl","obj":"http://uswest.ensembl.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["source"]}},{"sub":"MonarchArchive:#ensembl","obj":"MonarchLogoRepo:source-ensembl.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#ensembl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/species/c_elegans/PRJNA13758/c_elegans.PRJNA13758.WS279.annotations.gff3.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/wormbase.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"MonarchArchive:202103/rdf/wormbase.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":"http://www.orphadata.org/cgi-bin/index.php","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/CHECKSUMS","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"MonarchArchive:#wormbase","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"MonarchArchive:DipperCache/wormbase/gaf-eco-mapping.yaml","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"ftp://ftp.flybase.net/releases/current/precomputed_files/alleles/fbal_to_fbgn_fb_FB2021_01.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/letter.WS279","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ensembl","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":":udp","obj":"https://rarediseases.info.nih.gov/","pred":"foaf:page","meta":{"owlType":["Annotation"],"isDefinedBy":["_:5bf83e82e809f7c93fb6007bbd13030e"],"lbl":["page"]}},{"sub":"MonarchArchive:#reactome","obj":"MonarchArchive:202103/#reactome","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:#bgee","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:#bgee","obj":"MonarchArchive:202103/#bgee","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#coriell","obj":"MonarchLogoRepo:source-coriell.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":"https://bgee.org/?page=about","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["Rights"]}},{"sub":"MonarchArchive:#coriell","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#ctd","obj":"http://ctdbase.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#omia","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["type"]}},{"sub":"MonarchArchive:#animalqtldb","obj":"http://www.animalgenome.org/cgi-bin/QTLdb/index","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["license"]}},{"sub":"MonarchArchive:#omia","obj":"MonarchArchive:202103/#omia","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"CoriellCollection:NIGMS/NIGMS.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"CoriellCollection:NHGRI/NHGRI.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":"MonarchArchive:202103/rdf/animalqtldb.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#wormbase","obj":"http://www.wormbase.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["type"]}},{"sub":"MonarchArchive:#orphanet","obj":"http://www.orpha.net","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["type"]}},{"sub":"MonarchArchive:#wormbase","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":"https://hpo.jax.org/app/citation","pred":"cito:citesAsAuthority","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://purl.org/spar/cito/citesAsAuthority"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":"OBO:doid.owl","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:#coriell","obj":"https://ccr.coriell.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/omia.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["type"]}},{"sub":"MonarchArchive:#animalqtldb","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#bgee","obj":"MonarchArchive:#bgee","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#reactome","obj":"MonarchArchive:DipperCache/reactome/gaf-eco-mapping.yaml","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#reactome"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":"MonarchArchive:202103/rdf/bgee.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#bgee","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/bgee.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["Format"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":"OBO:zp//id_map_zfin.tsv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["source"]}},{"sub":"MonarchArchive:#gwascatalog","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":"MonarchArchive:202103/rdf/gwascatalog.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#gwascatalog","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":"https://github.com/monarch-initiative/mondo/releases/download/2019-04-06/mondo-minimal.json","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/human_orthos.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:#zfin","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/genotype_features.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/features.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/gene.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/CRISPR.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/genotype_backgrounds.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/wildtypes_fish.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"OBO:zp//id_map_zfin.tsv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/pub_to_pubmed_id_translation.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/mappings.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/uniprot.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"https://wiki.zfin.org/display/general/ZFIN+db+information","pred":"cito:citesAsAuthority","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://purl.org/spar/cito/citesAsAuthority"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:Downloads/stage_ontology.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/features-affected-genes.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/gene_marker_relationship.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"MonarchArchive:#zfin","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"http://0.0.0.0","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:#ctd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#ctd","obj":"http://ctdbase.org/reports/CTD_chemicals_diseases.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ctd","obj":"MonarchArchive:202103/rdf/ctd.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/tmp/QTL_GG_5.0.gff.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"MonarchArchive:DipperCache/Oncorhynchus_mykiss.gene_info.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/cattle_QTLdata.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ctd","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"MonarchArchive:202103/rdf/animalqtldb.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Bos_taurus.gene_info.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":"MonarchArchive:202103/rdf/hpoa.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Sus_scrofa.gene_info.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:#hpoa","obj":"MonarchArchive:202103/#hpoa","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/pig_QTLdata.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":"https://downloads.thebiogrid.org/Download/BioGRID/Latest-Release/BIOGRID-IDENTIFIERS-LATEST.tab.zip","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"MonarchArchive:#animalqtldb","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"MonarchArchive:DipperCache/Ovis_aries.gene_info.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/phenotype_fish.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":"http://creativecommons.org/licenses/by/4.0/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["license"]}},{"sub":"MonarchArchive:#zfin","obj":"https://zfin.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/tmp/QTL_Btau_4.6.gff.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":"MonarchArchive:202103/rdf/hpoa.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":"https://wiki.flybase.org/wiki/FlyBase_Wiki:General_disclaimer","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/Morpholinos.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"MonarchArchive:DipperCache/flybase/species.ab.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["type"]}},{"sub":"MonarchArchive:#flybase","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#flybase","obj":"MonarchArchive:#flybase","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#flybase","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#flybase","obj":"MonarchArchive:202103/#flybase","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:#ctd","obj":"MonarchArchive:202103/#ctd","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#ctd","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:#animalqtldb","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["type"]}},{"sub":"MonarchArchive:#ncbigene","obj":"MonarchArchive:202103/#ncbigene","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":"https://downloads.thebiogrid.org/Download/BioGRID/Latest-Release/BIOGRID-ALL-LATEST.mitab.zip","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["Format"]}},{"sub":"MonarchArchive:#zfin","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#monarch","obj":"MonarchArchive:202103/#monarch","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["type"]}},{"sub":"MonarchArchive:#animalqtldb","obj":"MonarchArchive:202103/#animalqtldb","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#biogrid","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["type"]}},{"sub":"MonarchArchive:#zfin","obj":"MonarchLogoRepo:source-zfin.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/galGal4/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":"MonarchArchive:202103/rdf/monarch.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#orphanet","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#mgislim","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["type"]}},{"sub":"MonarchArchive:#ncbigene","obj":"http://ncbi.nih.nlm.gov/gene","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"MonarchArchive:#monochrom","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["Is Version Of"]}},{"sub":"dctypes:Dataset","obj":"owl:Thing","pred":"subClassOf","meta":{"isDefinedBy":["http://purl.obolibrary.org/obo/upheno/monarch.owl"],"lbl":["subClassOf"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"MonarchArchive:DipperCache/Equus_caballus.gene_info.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/chicken_QTLdata.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#ctd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/bosTau7/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ncbigene","obj":"http://ftp.ncbi.nih.gov/gene/DATA/gene_history.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ncbigene"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/tmp/QTL_EquCab2.0.gff.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ctd","obj":"MonarchArchive:#ctd","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#ctd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":"MonarchArchive:202103/rdf/ctd.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["Format"]}},{"sub":"MonarchArchive:#ctd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Non-mammalian_vertebrates/Gallus_gallus.gene_info.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/animalqtldb.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/ctd.ttl","obj":"http://ctdbase.org/about/legal.jsp","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ctd"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/tmp/QTL_OAR_4.0.gff.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/sheep_QTLdata.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#animalqtldb","obj":"https://www.animalgenome.org/QTLdb/export/KSUI8GFHOT6/horse_QTLdata.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#animalqtldb"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#omim","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:#hpoa","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["type"]}},{"sub":"MonarchArchive:#coriell","obj":"MonarchArchive:202103/#coriell","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":"MonarchArchive:202103/rdf/omim.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#omim","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#omim","obj":"MonarchArchive:202103/#omim","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:#omim","obj":"MonarchLogoRepo:source-omim.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#coriell","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":"MonarchArchive:202103/rdf/coriell.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#coriell","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#rgd","obj":"MonarchArchive:202103/rdf/rgd.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:#rgd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#coriell","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"MonarchArchive:202103/rdf/coriell.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#rgd","obj":"https://rgd.mcw.edu/wg/citing-rgd/","pred":"cito:citesAsAuthority","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://purl.org/spar/cito/citesAsAuthority"]}},{"sub":"MonarchArchive:202103/rdf/coriell.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#rgd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"CoriellCollection:NINDS/NINDS.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#coriell","obj":"MonarchArchive:#coriell","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#coriell"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#rgd","obj":"ftp://ftp.rgd.mcw.edu/pub/data_release/annotated_rgd_objects_by_ontology/rattus_genes_mp","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#ensembl","obj":"http://uswest.ensembl.org/biomart/martservice?","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["source"]}},{"sub":"MonarchArchive:#rgd","obj":"MonarchArchive:202103/#rgd","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:#bgee","obj":"http://bgee.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/hpoa.ttl","obj":"https://hpo.jax.org/app/license","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":"http://compbio.charite.de/jenkins/job/hpo.annotations.current/lastSuccessfulBuild/artifact/current/phenotype.hpoa","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#hpoa","obj":"MonarchArchive:#hpoa","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/rdf/flybase.ttl","obj":"MonarchArchive:202103/rdf/flybase.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#flybase"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#omim","obj":"https://data.omim.org/downloads/mimTitles.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["source"]}},{"sub":"MonarchArchive:#omim","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/fish_model_disease.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#biogrid","obj":"MonarchLogoRepo:source-biogrid.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["http://schema.org/logo"]}},{"sub":"dctypes:Dataset","obj":"dc:DCMIType","pred":"http://purl.org/dc/dcam/memberOf","meta":{"owlType":["Annotation"],"isDefinedBy":["http://purl.obolibrary.org/obo/upheno/monarch.owl"],"lbl":["http://purl.org/dc/dcam/memberOf"]}},{"sub":"MonarchArchive:#rgd","obj":"http://rgd.mcw.edu/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["source"]}},{"sub":"MonarchArchive:#hpoa","obj":"MonarchLogoRepo:source-hpo.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#rgd","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:#hpoa","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hpoa"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":"https://rgd.mcw.edu/wg/disclaimer/","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#rgd","obj":"MonarchArchive:#rgd","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#rgd","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#omim","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/E_zfin_gene_alias.gff3","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#omim","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#omim","obj":"https://data.omim.org/downloads/phenotypic_series_title_all.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#omim","obj":"MonarchArchive:202103/rdf/omim.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":"http://omim.org/help/agreement","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#omim","obj":"https://data.omim.org/downloads/morbidmap.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#string","obj":"https://string-db.org/mapping_files/entrez/fly.entrez_2_string.2018.tsv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/omim.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#sgd","obj":"MonarchArchive:#sgd","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#string","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#omim","obj":"MonarchArchive:#omim","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omim"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#string","obj":"MonarchLogoRepo:source-string.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#string"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#sgd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#sgd","obj":"MonarchArchive:202103/#sgd","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":"MonarchArchive:202103/rdf/sgd.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#ensembl","obj":"MonarchArchive:202103/rdf/ensembl.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:#rgd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#ensembl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"http://phenomedoc.jax.org/MPD_downloads/strainmeans.csv.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["license"]}},{"sub":"MonarchArchive:#monarch","obj":"MonarchLogoRepo:source-monarch.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#monarch","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#monarch","obj":"https://monarchinitiative.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["source"]}},{"sub":"MonarchArchive:#mmrrc","obj":"https://www.mmrrc.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"PMID:15619963","pred":"cito:citesAsAuthority","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://purl.org/spar/cito/citesAsAuthority"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#sgd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#sgd","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#sgd","obj":"MonarchArchive:202103/rdf/sgd.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#sgd","obj":"https://downloads.yeastgenome.org/curation/literature/phenotype_data.tab","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/sgd.ttl","obj":"https://sites.google.com/view/yeastgenome-help/about","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["license"]}},{"sub":"MonarchArchive:#rgd","obj":"MonarchLogoRepo:source-rgd.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"MonarchArchive:202103/rdf/kegg.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/bosTau7/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#monochrom","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":"MonarchArchive:202103/rdf/monochrom.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"http://phenomedoc.jax.org/MPD_downloads/measurements.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#mmrrc","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/mmrrc.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":"MonarchArchive:202103/rdf/mpd.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#mpd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#ucscbands","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ucscbands"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/oviAri3/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#mpd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"MonarchArchive:202103/rdf/mpd.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.genome.jp/link/disease/omim","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.genome.jp/conv/ncbi-geneid/hsa","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.kegg.jp/link/pathway/ko","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"https://data.omim.org/downloads/mimTitles.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:#kegg","obj":"MonarchArchive:202103/#kegg","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["type"]}},{"sub":"MonarchArchive:#sgd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#sgd","obj":"https://www.yeastgenome.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.genome.jp/list/disease","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.kegg.jp/link/pathway/pubmed","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":"https://creativecommons.org/licenses/by/4.0/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#gwascatalog","obj":"ftp://ftp.ebi.ac.uk/pub/databases/gwas/releases/latest/gwas-catalog-associations_ontology-annotated.tsv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#wormbase","obj":"ftp://ftp.wormbase.org/pub/wormbase/releases/current-production-release/species/c_elegans/PRJNA13758/annotation/c_elegans.PRJNA13758.WS279.xrefs.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#wormbase"],"lbl":["source"]}},{"sub":"MonarchArchive:#gwascatalog","obj":"http://www.ebi.ac.uk/gwas/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["source"]}},{"sub":"MonarchArchive:#gwascatalog","obj":"MonarchLogoRepo:source-gwascatalog.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.kegg.jp/link/pathway/ds","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.genome.jp/list/pathway","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/ensembl.ttl","obj":"MonarchArchive:202103/rdf/ensembl.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#ensembl"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.genome.jp/list/orthology","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#kegg","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#kegg","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.kegg.jp/link/pathway/hsa","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["type"]}},{"sub":"MonarchArchive:#mmrrc","obj":"MonarchLogoRepo:source-mmrrc.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mmrrc"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#gwascatalog","obj":"MonarchArchive:202103/#gwascatalog","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#monarch","obj":"MonarchArchive:202103/rdf/monarch.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/gwascatalog.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#gwascatalog"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#monarch","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#monarch","obj":"MonarchArchive:#monarch","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#monarch","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/monarch.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#monarch","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":"MonarchArchive:202103/rdf/kegg.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.genome.jp/link/omim/hsa","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#rgd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"MonarchArchive:#kegg","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#sgd","obj":"MonarchLogoRepo:source-sgd.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.kegg.jp/link/disease/hsa","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/kegg.ttl","obj":"http://www.kegg.jp/kegg/legal.html","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#kegg","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#kegg","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#orphanet","obj":"MonarchArchive:202103/rdf/orphanet.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#go","obj":"ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/idmapping/idmapping_selected.tab.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":"https://creativecommons.org/publicdomain/mark/1.0/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":"MonarchArchive:202103/rdf/go.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#go","obj":"MonarchArchive:DipperCache/go/gaf-eco-mapping.yaml","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:#eom","obj":"http://elementsofmorphology.nih.gov","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/sgd.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#orphanet","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#go","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#orphanet","obj":"MonarchArchive:202103/#orphanet","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:#mpd","obj":"MonarchArchive:202103/#mpd","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202012/#zfin","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/goa_dog.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:#go","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":"dcat:Distribution","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#go","obj":"MonarchArchive:202103/rdf/go.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/goa_cow.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:#kegg","obj":"MonarchLogoRepo:source-kegg.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#kegg","obj":"http://www.genome.jp/kegg/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#bgee","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#bgee","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["type"]}},{"sub":"MonarchArchive:#bgee","obj":"MonarchLogoRepo:source-bgee.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#bgee"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/rgd.ttl","obj":"MonarchArchive:202103/rdf/rgd.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#rgd"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/pheno_environment_fish.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:#monochrom","obj":"https://monarchinitiative.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#orphanet","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["type"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"ZFIN:downloads/TALEN.txt","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/#orphanet","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["Format"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":"MonarchArchive:202012/rdf/zfin.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#orphanet","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#go","obj":"MonarchArchive:#go","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/rdf/biogrid.ttl","obj":"MonarchArchive:202103/rdf/biogrid.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#biogrid"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":"MonarchArchive:202103/rdf/orphanet.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#go","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":"http://www.genome.gov/copyright.cfm","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/goa_pig.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/#zfin","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#orphanet","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":"ZFIN:warranty.html","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/rdf/monochrom.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["type"]}},{"sub":"MonarchArchive:#orphanet","obj":"MonarchLogoRepo:source-orphanet.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#mgislim","obj":"MonarchArchive:202103/#mgislim","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202012/rdf/zfin.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#zfin"],"lbl":["type"]}},{"sub":"MonarchArchive:#monarch","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#monarch"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#monochrom","obj":"http://hgdownload.cse.ucsc.edu/goldenPath/rn6/database/cytoBandIdeo.txt.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["source"]}},{"sub":"MonarchArchive:#go","obj":"http://www.geneontology.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/orphanet.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#orphanet"],"lbl":["Format"]}},{"sub":"MonarchArchive:#go","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#go","obj":"MonarchLogoRepo:source-geneontology.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#kegg","obj":"http://rest.genome.jp/list/hsa","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#kegg"],"lbl":["source"]}},{"sub":"MonarchArchive:#sgd","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#sgd","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#sgd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#eom","obj":"jdbc:postgresql://nif-db.crbs.ucsd.edu:5432/disco_crawler","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:#mpd","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:#omia","obj":"https://omia.org","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#omia"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#eom","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["type"]}},{"sub":"MonarchArchive:#mmrrc","obj":"MonarchArchive:202103/#mmrrc","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202012/#impc","obj":"MonarchArchive:DipperCache/impc/impc_evidence_stable_key.tsv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["source"]}},{"sub":"MonarchArchive:#eom","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["type"]}},{"sub":"MonarchArchive:#mpd","obj":"MonarchLogoRepo:source-mpd.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:#eom","obj":"MonarchArchive:202103/#eom","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:#monochrom","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"http://phenomedoc.jax.org/MPD_downloads/straininfo.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["source"]}},{"sub":"MonarchArchive:#monochrom","obj":"MonarchLogoRepo:source-monochrom.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#monochrom"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"MonarchArchive:#mpd","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#hgnc","obj":"https://www.genenames.org/","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#hgnc"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#mpd","obj":"http://phenomedoc.jax.org/MPD_downloads/ontology_mappings.csv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["source"]}},{"sub":"MonarchArchive:#mpd","obj":"MPD:","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#eom","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#go","obj":"MonarchArchive:202103/#go","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:#impc","obj":"MonarchLogoRepo:source-impc.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#impc"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#eom","obj":"MonarchArchive:202103/rdf/eom.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":"MonarchArchive:202103/rdf/eom.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/rdf/mpd.ttl","obj":"MPD:about/termsofuse","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/wb.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#eom","obj":"MonarchArchive:#eom","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/pombase.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#eom","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/rgd.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/eom.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#eom","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/zfin.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#eom","obj":"https://raw.githubusercontent.com/obophenotype/human-phenotype-ontology/master/src/mappings/hp-to-eom-mapping.tsv","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#mpd","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mpd"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#eom","obj":"MonarchLogoRepo:source-eom.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":"http://geneontology.org/page/use-and-license","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["Rights"]}},{"sub":"MonarchArchive:#eom","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#eom"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/goa_chicken.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#go","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":"http://www.informatics.jax.org/mgihome/other/copyright.shtml","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":"MonarchArchive:202103/rdf/mgislim.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:#mgislim","obj":"MonarchLogoRepo:source-mgi.png","pred":"schema:logo","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["http://schema.org/logo"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/aspgd.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/rdf/go.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/#go","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/goa_human.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/fb.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/dictybase.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:#mgislim","obj":"http://www.mousemine.org/mousemine/service","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["source"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":"MonarchArchive:202012/rdf/zfinslim.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":"ZFIN:warranty.html","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":"MonarchArchive:#zfinslim","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":"MonarchArchive:202012/rdf/zfinslim.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202012/#zfinslim","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["license"]}},{"sub":"MonarchArchive:202012/rdf/zfinslim.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#zfinslim"],"lbl":["Format"]}},{"sub":"MonarchArchive:#zfinslim","obj":"MonarchArchive:202012/#zfinslim","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/#mgislim","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/#mgi","obj":"biolink:DataSetVersion","pred":"biolink:category","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["https://w3id.org/biolink/vocab/category"]}},{"sub":"MonarchArchive:#mgi","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":"https://project-open-data.cio.gov/unknown-license/","pred":"dc:license","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["license"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":"http://www.informatics.jax.org/mgihome/other/copyright.shtml","pred":"dc:rights","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["Rights"]}},{"sub":"MonarchArchive:202103/#mgi","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#mgi","obj":"jdbc:postgresql://mgi-adhoc.jax.org:5432/mgd","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["source"]}},{"sub":"MonarchArchive:#mgi","obj":"MonarchArchive:202103/#mgi","pred":"versionIRI","meta":{"lbl":["versionIRI"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":"MonarchArchive:202103/rdf/mgi.ttl","pred":"dcat:downloadURL","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["http://www.w3.org/ns/dcat#downloadURL"]}},{"sub":"MonarchArchive:202103/#go","obj":"http://current.geneontology.org/annotations/mgi.gaf.gz","pred":"dc:source","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#go"],"lbl":["source"]}},{"sub":"MonarchArchive:#mgislim","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#mgislim","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["Creator"]}},{"sub":"MonarchArchive:#mgi","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/#mgislim","obj":"MonarchArchive:#mgislim","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:#mgislim","obj":"dctypes:Dataset","pred":"type","meta":{"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["type"]}},{"sub":"MonarchArchive:202103/#mgi","obj":"MonarchArchive:#mgi","pred":"dc:isVersionOf","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["Is Version Of"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":":","pred":"dc:Publisher","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["http://purl.org/dc/terms/Publisher"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#mgislim","obj":"MonarchArchive:202103/rdf/mgislim.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"MonarchArchive:202103/rdf/mgislim.ttl","obj":"https://www.w3.org/TR/turtle/","pred":"dc:format","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgislim"],"lbl":["Format"]}},{"sub":"MonarchArchive:202103/#mgi","obj":":","pred":"dc:creator","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["Creator"]}},{"sub":"MonarchArchive:202103/rdf/mgi.ttl","obj":"https://github.com/monarch-initiative/dipper","pred":"pav:createdWith","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["http://purl.org/pav/createdWith"]}},{"sub":"MonarchArchive:202103/#mgi","obj":"MonarchArchive:202103/rdf/mgi.ttl","pred":"dcat:distribution","meta":{"owlType":["Annotation"],"isDefinedBy":["https://archive.monarchinitiative.org/#mgi"],"lbl":["http://www.w3.org/ns/dcat#distribution"]}},{"sub":"dctypes:Dataset","obj":"dctypes:","pred":"rdfs:isDefinedBy","meta":{"owlType":["Annotation"],"isDefinedBy":["http://purl.obolibrary.org/obo/upheno/monarch.owl"],"lbl":["http://www.w3.org/2000/01/rdf-schema#isDefinedBy"]}},{"sub":"dctypes:Dataset","obj":"OBO:upheno/monarch.owl","pred":"isDefinedBy","meta":{"lbl":["isDefinedBy"]}}],"meta":{}}')},c581:function(e,o,t){e.exports=t.p+"img/sierra-taylor-moxon.d850820e.jpg"},c587:function(e,o,t){e.exports=t.p+"img/eom.be9dfb85.png"},c7a0:function(e,o,t){e.exports=t.p+"img/category-variant.5ac7f60d.svg"},c7d6:function(e,o,t){e.exports=t.p+"img/victoria-alexandra-soesanto.12e4f23c.jpg"},c7fe:function(e,o,t){"use strict";t("6ec3")},cd49:function(e,o,t){"use strict";t.r(o);var i=t("2909"),r=t("b85c"),n=t("3835"),a=(t("e260"),t("e6cf"),t("cca6"),t("a79d"),t("4fad"),t("7a23"));function l(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("TheHeader"),s=Object(a["resolveComponent"])("router-view"),c=Object(a["resolveComponent"])("TheFloatButtons"),p=Object(a["resolveComponent"])("TheFooter");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(l),Object(a["createElementVNode"])("main",null,[Object(a["createVNode"])(s),Object(a["createVNode"])(c)]),Object(a["createVNode"])(p)],64)}t("f5df1");var s=t("ecee"),c=t("c074"),p=t("f2d1");s["d"].add(p["a"],p["b"],c["a"],c["b"],c["c"],c["d"],c["e"],c["f"],c["g"],c["h"],c["i"],c["j"],c["k"],c["l"],c["m"],c["n"],c["o"],c["p"],c["q"],c["r"],c["s"],c["t"],c["u"],c["v"],c["w"],c["x"],c["y"],c["z"],c["A"],c["B"],c["C"],c["D"],c["E"],c["F"],c["G"],c["H"],c["I"],c["J"],c["K"],c["L"],c["M"],c["N"],c["O"],c["P"],c["Q"],c["R"],c["S"],c["T"]);t("71d9");var d=function(e){return Object(a["pushScopeId"])("data-v-c137e8fe"),e=e(),Object(a["popScopeId"])(),e},h=["data-home"],u={class:"title"},b=Object(a["createTextVNode"])(" Monarch "),g=d((function(){return Object(a["createElementVNode"])("br",null,null,-1)})),m=Object(a["createTextVNode"])(" Initiative "),y=["aria-expanded","aria-label"],v=["data-home","data-expanded"],f=Object(a["createTextVNode"])(" Explore "),w=Object(a["createTextVNode"])(" About "),O=Object(a["createTextVNode"])(" Help ");function A(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("TheNexus"),s=Object(a["resolveComponent"])("Logo"),c=Object(a["resolveComponent"])("AppLink"),p=Object(a["resolveComponent"])("AppIcon"),d=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createElementBlock"])("header",{"data-home":e.home},[e.home?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0})):Object(a["createCommentVNode"])("",!0),Object(a["createElementVNode"])("div",u,[Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(c,{to:e.home?"":"/",class:"logo","data-home":e.home},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(s,{class:"image"}),(Object(a["openBlock"])(),Object(a["createBlock"])(Object(a["resolveDynamicComponent"])(e.home?"h1":"div"),{class:"text"},{default:Object(a["withCtx"])((function(){return[b,g,m]})),_:1}))]})),_:1},8,["to","data-home"])),[[d,e.home?"":"Homepage"]]),Object(a["createElementVNode"])("button",{class:"button",onClick:o[0]||(o[0]=function(o){return e.expanded=!e.expanded}),"aria-expanded":e.expanded,"aria-label":e.expanded?"Close navigation menu":"Expand navigation menu"},[Object(a["createVNode"])(p,{icon:e.expanded?"times":"bars"},null,8,["icon"])],8,y)]),Object(a["createElementVNode"])("nav",{"data-home":e.home,"data-expanded":e.expanded},[Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(c,{class:"link",to:"/explore"},{default:Object(a["withCtx"])((function(){return[f]})),_:1})),[[d,"Dive right in and use Monarch"]]),Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(c,{class:"link",to:"/about"},{default:Object(a["withCtx"])((function(){return[w]})),_:1})),[[d,"Citing, licensing, sources, and other info"]]),Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(c,{class:"link",to:"/help"},{default:Object(a["withCtx"])((function(){return[O]})),_:1})),[[d,"Feedback, docs, guides, contact, and more"]])],8,v)],8,h)}t("b0c0");var M={ref:"canvas"};function j(e,o,t,i,r,n){return Object(a["openBlock"])(),Object(a["createElementBlock"])("canvas",M,null,512)}t("99af"),t("a434"),t("4e82"),t("4de4"),t("d3b7"),t("7db0"),t("a15b"),t("cb29");var k,B=t("2ef0"),D=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:0,o=arguments.length>1&&void 0!==arguments[1]?arguments[1]:0,t=arguments.length>2&&void 0!==arguments[2]?arguments[2]:0,i=arguments.length>3&&void 0!==arguments[3]?arguments[3]:0;return Math.sqrt(Math.pow(t-e,2)+Math.pow(i-o,2))},x=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:0;return Math.sin(2*Math.PI*e/360)},C=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:0;return Math.cos(2*Math.PI*e/360)},I=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:0,o=arguments.length>1&&void 0!==arguments[1]?arguments[1]:0,t=arguments.length>2&&void 0!==arguments[2]?arguments[2]:10;return((e-o)%(t-o)+(t-o))%(t-o)+o},_=function(e){var o=e.x,t=e.y,i=e.z,r=arguments.length>1&&void 0!==arguments[1]?arguments[1]:0;return{x:o,y:t*C(r)-i*x(r),z:t*x(r)+i*C(r)}},T=function(e){var o=e.x,t=e.y,i=e.z,r=arguments.length>1&&void 0!==arguments[1]?arguments[1]:0;return{x:o*C(r)+i*x(r),y:t,z:i*C(r)-o*x(r)}},N=function(e,o){var t=arguments.length>2&&void 0!==arguments[2]?arguments[2]:1;return{x:e.x+o.x*t,y:e.y+o.y*t,z:e.z+o.z*t}},S=function(e){var o=arguments.length>1&&void 0!==arguments[1]?arguments[1]:0,t=arguments.length>2&&void 0!==arguments[2]?arguments[2]:0,i=arguments.length>3&&void 0!==arguments[3]?arguments[3]:0,r=arguments.length>4&&void 0!==arguments[4]?arguments[4]:0,n={x:i,y:r,z:0};e=N(e,n,-1),e=_(e,o),e=T(e,t),e=N(e,n);var a=e,l=a.x,s=a.y;return{x:l,y:s}},R=function(e,o){return{x:(e.x+o.x)/2,y:(e.y+o.y)/2,z:(e.z+o.z)/2}},P=4,L=50,E=[25,143,154],V=[127,193,199],z=0,H=0,F=0,G=0,q=null,W=null,J=0,U=0,Y=[],K=[],Z=function(){if(q&&W){var e=window.devicePixelRatio;J=q.clientWidth,U=q.clientHeight,q.width=J*e,q.height=U*e,W.scale(e,e)}},Q=Object(B["debounce"])((function(){Y=[];for(var e=0;e<=J;e+=L)for(var o=0;o<=U;o+=L)if(Math.random()>.4&&(Math.abs(e-J/2)>2*L||Math.abs(o-U/2)>L)){var t=360*Math.random();Y.push({point:{x:e+x(t)*(L/4),y:o+C(t)*(L/4),z:-L+2*Math.random()*L},color:[].concat(E),colorTarget:[].concat(E)})}while(Y.length>200)Y.splice(Math.floor(Y.length*Math.random()),1);Y.sort((function(e,o){return e.point.z-o.point.z})),K=[];for(var i=0;i<Y.length;i++)for(var r=0;r<Y.length;r++)if(i>r){var n=Y[i],a=Y[r];D(n.point.x,n.point.y,a.point.x,a.point.y)<1.5*L&&K.push({from:n,to:a,color:[].concat(E),colorTarget:[].concat(E)})}Y=Y.filter((function(e){return K.find((function(o){var t=o.from,i=o.to;return e===t||e===i}))}))}),50),X=function(e){var o="clientX"in e?e.clientX:e.touches[0].clientX,t="clientY"in e?e.clientY:e.touches[0].clientY;F=90*(.5-t/window.innerHeight),G=90*-(.5-o/window.innerWidth)},$=function(){z+=(F-z)/100,H+=(G-H)/100;var e,o=Object(r["a"])(Y);try{for(o.s();!(e=o.n()).done;){var t=e.value;t.projected=S(t.point,z,H,J/2,U/2)}}catch(c){o.e(c)}finally{o.f()}for(var n=0,a=[].concat(Object(i["a"])(Y),Object(i["a"])(K));n<a.length;n++)for(var l=a[n],s=0;s<3;s++)l.color[s]+=(l.colorTarget[s]-l.color[s])/20},ee=function(){q&&W&&W.clearRect(0,0,J||0,U||0)},oe=function(){if(W){var e,o=Object(r["a"])(K);try{for(o.s();!(e=o.n()).done;){var t=e.value,i=t.from,n=t.to,a=t.color;i.projected&&n.projected&&(W.strokeStyle="rgb("+a.join(",")+")",W.lineWidth=P/3,W.beginPath(),W.moveTo(i.projected.x,i.projected.y),W.lineTo(n.projected.x,n.projected.y),W.stroke())}}catch(h){o.e(h)}finally{o.f()}var l,s=Object(r["a"])(Y);try{for(s.s();!(l=s.n()).done;){var c=l.value,p=c.projected,d=c.color;p&&(W.fillStyle="rgb("+d.join(",")+")",W.beginPath(),W.arc(p.x,p.y,P,0,2*Math.PI),W.fill())}}catch(h){s.e(h)}finally{s.f()}}},te=function(){for(var e=function(){var e=t[o],i="point"in e?e.point:R(e.from.point,e.to.point),r=.2,n=D(i.x-J/2,i.y-U/2)/r,a=n+100/r;window.setTimeout((function(){return e.colorTarget=[].concat(V)}),n),window.setTimeout((function(){return e.colorTarget=[].concat(E)}),a)},o=0,t=[].concat(Object(i["a"])(Y),Object(i["a"])(K));o<t.length;o++)e()},ie=function(){$(),ee(),oe()};window.setInterval(ie,1e3/60),window.setInterval(te,1e4),window.addEventListener("mousemove",X),window.addEventListener("touchmove",X);var re=Object(a["defineComponent"])({mounted:function(){q=this.$refs.canvas,W=q.getContext("2d"),k=new ResizeObserver((function(){Z(),Q()})),k.observe(q)},beforeUnmount:function(){k.disconnect&&k.disconnect()}}),ne=(t("d08c"),t("6b0d")),ae=t.n(ne);const le=ae()(re,[["render",j],["__scopeId","data-v-1652ac66"]]);var se=le,ce={viewBox:"-50 -30 100 60"},pe=Object(a["createStaticVNode"])('<mask id="logo-mask" data-v-672340fe><rect x="-50" y="-50" width="100" height="100" fill="white" data-v-672340fe></rect><rect x="-10" y="7.5" width="20" height="5" fill="black" data-v-672340fe></rect><rect x="7.5" y="-10" width="5" height="20" fill="black" data-v-672340fe></rect></mask><g transform="scale(0.8) rotate(-45)" data-v-672340fe><path class="logo-path" fill="none" stroke="currentColor" stroke-width="5" stroke-linecap="round" mask="url(#logo-mask)" d="M 5 10 L 5 -20 A 25 25 0 1 0 -20 5 L -15 5" data-v-672340fe></path><path class="logo-path" fill="none" stroke="currentColor" stroke-width="5" stroke-linecap="round" mask="url(#logo-mask)" d="M -5 10 L -5 -20 A 15 15 0 1 0 -20 -5 L -15 -5" data-v-672340fe></path><path class="logo-path" fill="none" stroke="currentColor" stroke-width="5" stroke-linecap="round" mask="url(#logo-mask)" d="M -5 15 L -5 20 A 25 25 0 1 0 20 -5 L 10 -5" data-v-672340fe></path><path class="logo-path" fill="none" stroke="currentColor" stroke-width="5" stroke-linecap="round" mask="url(#logo-mask)" d="M 5 15 L 5 20 A 15 15 0 1 0 20 5 L 10 5" data-v-672340fe></path></g>',2),de=[pe];function he(e,o){return Object(a["openBlock"])(),Object(a["createElementBlock"])("svg",ce,de)}t("d0ee");const ue={},be=ae()(ue,[["render",he],["__scopeId","data-v-672340fe"]]);var ge=be,me=Object(a["defineComponent"])({components:{TheNexus:se,Logo:ge},data:function(){return{expanded:!1}},computed:{home:function(){return"home"===String(this.$route.name).toLowerCase()}},watch:{$route:function(){this.expanded=!1}}});t("bf3f");const ye=ae()(me,[["render",A],["__scopeId","data-v-c137e8fe"]]);var ve=ye,fe=function(e){return Object(a["pushScopeId"])("data-v-37bed414"),e=e(),Object(a["popScopeId"])(),e},we={class:"social"},Oe={class:"license"},Ae=fe((function(){return Object(a["createElementVNode"])("span",null,"  ",-1)})),Me=Object(a["createTextVNode"])("BSD-3");function je(e,o){var t=Object(a["resolveComponent"])("AppIcon"),i=Object(a["resolveComponent"])("AppLink"),r=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createElementBlock"])("footer",null,[Object(a["createElementVNode"])("div",we,[Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(i,{to:"https://substack.com/"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,{icon:"envelope-open-text"})]})),_:1})),[[r,"Newsletter"]]),Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(i,{to:"https://github.com/monarch-initiative"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,{icon:"github"})]})),_:1})),[[r,"GitHub"]]),Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(i,{to:"https://twitter.com/MonarchInit"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,{icon:"twitter"})]})),_:1})),[[r,"Twitter"]]),Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(i,{to:"https://medium.com/@MonarchInit"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,{icon:"blog"})]})),_:1})),[[r,"Blog"]])]),Object(a["createElementVNode"])("div",Oe,[Object(a["createElementVNode"])("span",null,"Monarch Intiative "+Object(a["toDisplayString"])((new Date).getFullYear()),1),Ae,Object(a["createVNode"])(i,{to:"/terms",class:"license"},{default:Object(a["withCtx"])((function(){return[Me]})),_:1})])])}t("3467");const ke={},Be=ae()(ke,[["render",je],["__scopeId","data-v-37bed414"]]);var De=Be;function xe(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppButton"),s=Object(a["resolveComponent"])("TheFeedbackForm"),c=Object(a["resolveComponent"])("AppModal"),p=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{class:"float",style:Object(a["normalizeStyle"])({bottom:e.nudge+"px"})},[Object(a["createVNode"])(a["Transition"],{name:"fade"},{default:Object(a["withCtx"])((function(){return[e.showJump?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0,design:"circle",class:"button",icon:"angle-up",onClick:o[0]||(o[0]=function(o){return e.jump()})},null,512)),[[p,"Jump to top of page"]]):Object(a["createCommentVNode"])("",!0)]})),_:1}),Object(a["createVNode"])(a["Transition"],{name:"fade"},{default:Object(a["withCtx"])((function(){return[e.showFeedback?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0,design:"circle",class:"button",icon:"comment",onClick:o[1]||(o[1]=function(o){return e.showModal=!0})},null,512)),[[p,"Give us feedback on this page!"]]):Object(a["createCommentVNode"])("",!0)]})),_:1}),Object(a["createVNode"])(c,{modelValue:e.showModal,"onUpdate:modelValue":o[2]||(o[2]=function(o){return e.showModal=o}),label:"Feedback form"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(s,{modal:!0})]})),_:1},8,["modelValue"])],4)}var Ce=["aria-label"];function Ie(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppButton"),s=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createBlock"])(a["Teleport"],{to:"body"},[Object(a["createVNode"])(a["Transition"],{name:"fade"},{default:Object(a["withCtx"])((function(){return[e.modelValue?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{key:0,class:"overlay",onMousedown:o[2]||(o[2]=function(){return e.close&&e.close.apply(e,arguments)}),onTouchstart:o[3]||(o[3]=function(){return e.close&&e.close.apply(e,arguments)})},[Object(a["createElementVNode"])("div",{ref:"modal",class:"modal",onMousedown:o[0]||(o[0]=Object(a["withModifiers"])((function(){}),["stop"])),onTouchstart:o[1]||(o[1]=Object(a["withModifiers"])((function(){}),["stop"])),role:"dialog","aria-modal":"true","aria-label":e.label},[Object(a["withDirectives"])(Object(a["createVNode"])(l,{class:"close",design:"circle",icon:"times",onClick:e.close},null,8,["onClick"]),[[s,"Close dialog (esc)"]]),Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)],40,Ce)],32)):Object(a["createCommentVNode"])("",!0)]})),_:3})])}var _e=t("e181"),Te=Object(a["defineComponent"])({props:{modelValue:Boolean,label:{type:String,required:!0}},data:function(){return{originalFocus:null}},emits:["update:modelValue"],methods:{close:function(){this.$emit("update:modelValue",!1)},keyDown:function(e){"Escape"===e.key&&this.close()}},mounted:function(){window.addEventListener("keydown",this.keyDown)},updated:function(){var e,o=this.$refs.modal,t=null===(e=document)||void 0===e?void 0:e.querySelector("#app");if(this.modelValue){null===t||void 0===t||t.setAttribute("aria-hidden","true"),null===t||void 0===t||t.setAttribute("inert","true"),Object(_e["a"])(o);var i="input, textarea, button, select",r=o.querySelector(i);null===r||void 0===r||r.focus()}},beforeUpdate:function(){var e,o,t=this,i=this.$refs.modal,r=null===(e=document)||void 0===e?void 0:e.querySelector("#app");this.modelValue?this.originalFocus=null===(o=document)||void 0===o?void 0:o.activeElement:(null===r||void 0===r||r.removeAttribute("aria-hidden"),null===r||void 0===r||r.removeAttribute("inert"),Object(_e["b"])(i),Object(a["nextTick"])((function(){var e;return null===(e=t.originalFocus)||void 0===e?void 0:e.focus()})))},beforeUnmount:function(){window.removeEventListener("keydown",this.keyDown)}});t("ebfc");const Ne=ae()(Te,[["render",Ie],["__scopeId","data-v-0324f226"]]);var Se=Ne,Re=(t("9911"),function(e){return Object(a["pushScopeId"])("data-v-a06ee782"),e=e(),Object(a["popScopeId"])(),e}),Pe=Object(a["createTextVNode"])("Feedback Form"),Le={key:1},Ee={class:"fields"},Ve={class:"details"},ze=Object(a["createTextVNode"])(" Submitting this form starts a "),He=Re((function(){return Object(a["createElementVNode"])("strong",null,"public",-1)})),Fe=Object(a["createTextVNode"])(" discussion on our "),Ge=Object(a["createTextVNode"])("help desk"),qe=Object(a["createTextVNode"])(" on GitHub with "),We=Re((function(){return Object(a["createElementVNode"])("em",null,"all of the information above",-1)})),Je=Object(a["createTextVNode"])(". You'll get a link to the discussion once it's created. "),Ue=Object(a["createTextVNode"])("View your submitted feedback here.");function Ye(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("AppInput"),c=Object(a["resolveComponent"])("AppLink"),p=Object(a["resolveComponent"])("AppStatus"),d=Object(a["resolveComponent"])("AppButton");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[e.modal?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0,level:1},{default:Object(a["withCtx"])((function(){return[Pe]})),_:1})):Object(a["createCommentVNode"])("",!0),e.status?Object(a["createCommentVNode"])("",!0):(Object(a["openBlock"])(),Object(a["createElementBlock"])("p",Le," Request a feature, report a bug, or chat with us about anything Monarch-related. ")),Object(a["createElementVNode"])("form",{onSubmit:o[4]||(o[4]=Object(a["withModifiers"])((function(){return e.onSubmit&&e.onSubmit.apply(e,arguments)}),["prevent"]))},[Object(a["createElementVNode"])("div",Ee,[Object(a["createVNode"])(s,{title:"Name",description:"So we can address you",placeholder:"Jane Smith",modelValue:e.name,"onUpdate:modelValue":o[0]||(o[0]=function(o){return e.name=o}),modelModifiers:{trim:!0}},null,8,["modelValue"]),Object(a["createVNode"])(s,{title:"Email",description:"So we can follow up with you",placeholder:"jane.smith@gmail.com",type:"email",modelValue:e.email,"onUpdate:modelValue":o[1]||(o[1]=function(o){return e.email=o}),modelModifiers:{trim:!0}},null,8,["modelValue"]),Object(a["createVNode"])(s,{title:"GitHub username",description:"So we can tag you",placeholder:"@janesmith",modelValue:e.github,"onUpdate:modelValue":o[2]||(o[2]=function(o){return e.github=o}),modelModifiers:{trim:!0}},null,8,["modelValue"]),Object(a["createVNode"])(s,{class:"feedback",title:"Feedback",description:"Please give us as many details as possible!",placeholder:"",required:!0,multi:!0,modelValue:e.feedback,"onUpdate:modelValue":o[3]||(o[3]=function(o){return e.feedback=o})},null,8,["modelValue"])]),Object(a["createElementVNode"])("div",Ve,[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.details,(function(e,o){return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:o},[Object(a["createElementVNode"])("span",null,Object(a["toDisplayString"])(o),1),Object(a["createElementVNode"])("span",null,Object(a["toDisplayString"])(e),1)],64)})),128))]),Object(a["createElementVNode"])("p",null,[ze,He,Fe,Object(a["createVNode"])(c,{to:"https://github.com/monarch-initiative/helpdesk"},{default:Object(a["withCtx"])((function(){return[Ge]})),_:1}),qe,We,Je]),e.status?(Object(a["openBlock"])(),Object(a["createBlock"])(p,{key:0,status:e.status},{default:Object(a["withCtx"])((function(){return[e.link?(Object(a["openBlock"])(),Object(a["createBlock"])(c,{key:0,to:e.link},{default:Object(a["withCtx"])((function(){return[Ue]})),_:1},8,["to"])):Object(a["createCommentVNode"])("",!0)]})),_:1},8,["status"])):(Object(a["openBlock"])(),Object(a["createBlock"])(d,{key:1,text:"Submit",icon:"paper-plane",type:"submit"}))],32)],64)}var Ke=t("1da1"),Ze=(t("96cf"),t("d81d"),t("2b80")),Qe=t.n(Ze),Xe=(t("a4d3"),t("e01a"),{class:"label"}),$e={key:0,class:"title"},eo={class:"input"},oo=["value","placeholder","required"],to=["value","placeholder","type","required"],io={class:"icon"},ro={key:1,class:"description"};function no(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon");return Object(a["openBlock"])(),Object(a["createElementBlock"])("label",Xe,[e.title?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",$e,[Object(a["createTextVNode"])(Object(a["toDisplayString"])(e.title)+" ",1),e.required?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0,icon:"asterisk",class:"asterisk"})):Object(a["createCommentVNode"])("",!0)])):Object(a["createCommentVNode"])("",!0),Object(a["createElementVNode"])("div",eo,[e.multi?(Object(a["openBlock"])(),Object(a["createElementBlock"])("textarea",{key:0,ref:"input",value:e.modelValue,onFocus:o[0]||(o[0]=function(){return e.onFocus&&e.onFocus.apply(e,arguments)}),onInput:o[1]||(o[1]=function(){return e.onInput&&e.onInput.apply(e,arguments)}),onChange:o[2]||(o[2]=function(){return e.onChange&&e.onChange.apply(e,arguments)}),placeholder:e.placeholder,required:e.required},"\n      ",40,oo)):(Object(a["openBlock"])(),Object(a["createElementBlock"])("input",{key:1,ref:"input",value:e.modelValue,onFocus:o[3]||(o[3]=function(){return e.onFocus&&e.onFocus.apply(e,arguments)}),onInput:o[4]||(o[4]=function(){return e.onInput&&e.onInput.apply(e,arguments)}),onChange:o[5]||(o[5]=function(){return e.onChange&&e.onChange.apply(e,arguments)}),placeholder:e.placeholder,type:e.type,required:e.required},null,40,to)),Object(a["createElementVNode"])("div",io,[e.icon?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0,icon:e.icon},null,8,["icon"])):Object(a["createCommentVNode"])("",!0)])]),e.description?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",ro,Object(a["toDisplayString"])(e.description),1)):Object(a["createCommentVNode"])("",!0)])}var ao=Object(a["defineComponent"])({emits:["update:modelValue","focus","input","change"],props:{modelValue:String,placeholder:String,type:String,title:String,description:String,required:Boolean,multi:Boolean,icon:String},methods:{focus:function(){this.$refs.input.focus()},onFocus:function(){this.$emit("focus")},onInput:function(e){var o=e.target;this.$emit("update:modelValue",o.value),this.$emit("input")},onChange:function(e){var o=this;Object(a["nextTick"])((function(){var t;null!==(t=document)&&void 0!==t&&t.contains(e.target)&&o.$emit("change")}))}}});t("5c21");const lo=ae()(ao,[["render",no],["__scopeId","data-v-8a90bf92"]]);var so=lo,co=(t("fb6a"),t("498a"),t("ac1f"),t("5319"),function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",o=arguments.length>1&&void 0!==arguments[1]?arguments[1]:20;return e.length>o?e.slice(0,o-3)+"...":e}),po=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"";return e.replace(/\s+/g," ").trim()},ho=(t("d9e2"),t("bee2")),uo=t("d4ec"),bo=t("257e"),go=t("262e"),mo=t("2caf"),yo=t("9072"),vo=t("ade3"),fo=(t("3ca3"),t("ddb0"),t("9861"),t("1276"),t("25f0"),t("5b81"),"https://api.monarchinitiative.org/api"),wo=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(){var o,t,i,a,l,s,c,p,d,h,u,b,g,m,y,v,f=arguments;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:for(o=f.length>0&&void 0!==f[0]?f[0]:"",t=f.length>1&&void 0!==f[1]?f[1]:{},i=f.length>2&&void 0!==f[2]?f[2]:{},a=new URLSearchParams,l=0,s=Object.entries(t);l<s.length;l++){c=Object(n["a"])(s[l],2),p=c[0],d=c[1],h=Object(r["a"])(String(d).split(","));try{for(h.s();!(u=h.n()).done;)b=u.value,a.append(p,b)}catch(w){h.e(w)}finally{h.f()}}return a.sort(),g="?"+a.toString(),m=o+g,console.info("Making request",m.replaceAll(" ","%20")),e.next=11,fetch(m,i);case 11:if(y=e.sent,y.ok){e.next=14;break}throw new Ao("Response not OK - ".concat(y.statusText));case 14:return e.next=16,y.json();case 16:return v=e.sent,e.abrupt("return",v);case 18:case"end":return e.stop()}}),e)})));return function(){return e.apply(this,arguments)}}(),Oo=function(e){return e instanceof Ao||(e=new Ao("Error: "+e.message)),console.groupCollapsed(e.text),console.info(e),console.groupEnd(),e},Ao=function(e){Object(go["a"])(t,e);var o=Object(mo["a"])(t);function t(){var e,i=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",r=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"error";return Object(uo["a"])(this,t),e=o.call(this),Object(vo["a"])(Object(bo["a"])(e),"code",void 0),Object(vo["a"])(Object(bo["a"])(e),"text",void 0),e.name="ApiError",e.text=String(i),e.code=r,e}return Object(ho["a"])(t)}(Object(yo["a"])(Error)),Mo="https://us-central1-monarch-initiative.cloudfunctions.net/monarch-gh-issue-post",jo=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(){var o,t,i,r,n,a=arguments;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:if(o=a.length>0&&void 0!==a[0]?a[0]:"",t=a.length>1&&void 0!==a[1]?a[1]:"",e.prev=2,o&&t){e.next=5;break}throw new Error("Title or body not specified");case 5:return e.next=7,wo(Mo,{title:o,body:t},{method:"POST"});case 7:if(i=e.sent,r=i.error,!r){e.next=11;break}throw new Error(r);case 11:if(n=i.html_url,!n){e.next=14;break}return e.abrupt("return",n);case 14:throw new Error("Unknown problem submitting feedback");case 17:throw e.prev=17,e.t0=e["catch"](2),Oo(e.t0);case 20:case"end":return e.stop()}}),e,null,[[2,17]])})));return function(){return e.apply(this,arguments)}}(),ko={class:"text"};function Bo(e,o,t,i,r,n){var l,s,c,p=Object(a["resolveComponent"])("AppIcon"),d=Object(a["resolveComponent"])("AppLink");return Object(a["openBlock"])(),Object(a["createBlock"])(d,{to:(null===(l=e.status)||void 0===l?void 0:l.link)||"",class:"status","data-code":(null===(s=e.status)||void 0===s?void 0:s.code)||"","aria-label":(null===(c=e.status)||void 0===c?void 0:c.code)||""},{default:Object(a["withCtx"])((function(){var o;return[Object(a["createVNode"])(p,{class:"icon",icon:e.icon},null,8,["icon"]),Object(a["createElementVNode"])("span",ko,[Object(a["createTextVNode"])(Object(a["toDisplayString"])((null===(o=e.status)||void 0===o?void 0:o.text)||"")+" ",1),Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)])]})),_:3},8,["to","data-code","aria-label"])}var Do={loading:"loading",paused:"pause-circle",success:"check-circle",warning:"exclamation-circle",error:"times-circle",unknown:"question-circle"},xo=Object(a["defineComponent"])({props:{status:Object},computed:{icon:function(){var e;return Do[(null===(e=this.status)||void 0===e?void 0:e.code)||"unknown"]}}});t("b1a5");const Co=ae()(xo,[["render",Bo],["__scopeId","data-v-5b04cbe0"]]);var Io=Co,_o=Object(a["defineComponent"])({components:{AppInput:so,AppStatus:Io},props:{modal:Boolean},data:function(){return{name:"",email:"",github:"",feedback:"",status:null,link:""}},computed:{details:function(){var e=Qe()(),o=e.browser,t=e.device,i=e.os,r=e.engine,n=e.cpu,a=function(){for(var e=arguments.length,o=new Array(e),t=0;t<e;t++)o[t]=arguments[t];return o.filter((function(e){return e&&"()"!==e})).join(" ")};return{Page:this.$route.fullPath,Browser:a(o.name,o.version),Device:a(t.vendor,t.model,t.type),OS:a(i.name,i.version,n.architecture),Engine:a(r.name,r.version)}}},persist:{keys:["name","email","github","feedback"],namespace:"feedback-form"},methods:{onSubmit:function(){var e=this;return Object(Ke["a"])(regeneratorRuntime.mark((function o(){var t,r,a,l;return regeneratorRuntime.wrap((function(o){while(1)switch(o.prev=o.next){case 0:if(r=null===(t=document)||void 0===t?void 0:t.activeElement,!r||"submit"===r.getAttribute("type")){o.next=3;break}return o.abrupt("return");case 3:return a=["Feedback form",co(e.name,20),co(po(e.feedback),60)].join(" - "),l=["**Name**",e.name,"","**Email**",e.email,"","**GitHub Username**",e.github,"","**Details**"].concat(Object(i["a"])(Object.entries(e.details).map((function(e){var o=Object(n["a"])(e,2),t=o[0],i=o[1];return"".concat(t,": ").concat(i)}))),["","",e.feedback]).join("\n"),e.status={code:"loading",text:"Submitting feedback"},o.prev=6,o.next=9,jo(a,l);case 9:e.link=o.sent,e.status={code:"success",text:"Feedback submitted successfully!"},null===e||void 0===e||e.clearPersist(),o.next=17;break;case 14:o.prev=14,o.t0=o["catch"](6),e.status=o.t0;case 17:case"end":return o.stop()}}),o,null,[[6,14]])})))()}},mounted:function(){this.status=null}});t("d3ae");const To=ae()(_o,[["render",Ye],["__scopeId","data-v-a06ee782"]]);var No,So=To,Ro=Object(a["defineComponent"])({components:{AppModal:Se,TheFeedbackForm:So},data:function(){return{showJump:!1,showFeedback:!0,showModal:!1,nudge:0}},methods:{update:function(){var e,o=null===(e=document)||void 0===e?void 0:e.querySelector("footer");if(o){var t=o.getBoundingClientRect();this.showJump=window.scrollY>.1*window.innerHeight,this.showFeedback="feedback"!==(this.$route.name||"").toLowerCase(),this.nudge=Math.max(0,t.height+window.innerHeight-t.bottom)}},jump:function(){window.scrollTo(0,0)}},watch:{$route:{handler:function(){Object(a["nextTick"])(this.update)},deep:!0}},mounted:function(){var e;Object(a["nextTick"])(this.update),window.addEventListener("scroll",this.update),window.addEventListener("resize",this.update),No=new MutationObserver(this.update),No.observe(null===(e=document)||void 0===e?void 0:e.body,{subtree:!0,childList:!0})},beforeUnmount:function(){window.removeEventListener("scroll",this.update),window.removeEventListener("resize",this.update),No&&No.disconnect()}});t("32bb");const Po=ae()(Ro,[["render",xe],["__scopeId","data-v-69ba876f"]]);var Lo=Po,Eo=Object(a["defineComponent"])({props:{large:Boolean},components:{TheHeader:ve,TheFooter:De,TheFloatButtons:Lo}});const Vo=ae()(Eo,[["render",l]]);var zo=Vo,Ho={key:0};function Fo(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon");return Object(a["openBlock"])(),Object(a["createBlock"])(Object(a["resolveDynamicComponent"])(e.component),{class:"button",to:e.to,onClick:o[0]||(o[0]=function(o){return e.copy?e.copyToClipboard():e.click}),"data-design":e.design,"data-color":e.color,"data-text":!!e.text,"data-notification":e.notification},{default:Object(a["withCtx"])((function(){return[e.text?(Object(a["openBlock"])(),Object(a["createElementBlock"])("span",Ho,Object(a["toDisplayString"])(e.copied?"Copied!":e.text),1)):Object(a["createCommentVNode"])("",!0),e.icon?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:1,icon:e.icon},null,8,["icon"])):Object(a["createCommentVNode"])("",!0)]})),_:1},8,["to","data-design","data-color","data-text","data-notification"])}var Go=Object(a["defineComponent"])({props:{text:String,icon:String,to:String,click:Function,design:{default:"normal",type:String},color:{default:"primary",type:String},notification:{default:!1,type:Boolean},copy:{default:!1,type:Boolean}},data:function(){return{copied:!1}},computed:{component:function(){return this.to?"AppLink":this.$attrs.onClick||"submit"===this.$attrs.type||this.copy?"button":"span"}},methods:{copyToClipboard:function(){var e=this;return Object(Ke["a"])(regeneratorRuntime.mark((function o(){return regeneratorRuntime.wrap((function(o){while(1)switch(o.prev=o.next){case 0:return o.next=2,window.navigator.clipboard.writeText(e.text||"");case 2:e.copied=!0,window.setTimeout((function(){return e.copied=!1}),1e3);case 4:case"end":return o.stop()}}),o)})))()}}});t("a70e"),t("53c8");const qo=ae()(Go,[["render",Fo],["__scopeId","data-v-6b0c6d5a"]]);var Wo=qo,Jo=["data-direction","data-gap"];function Uo(e,o,t,i,r,n){return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{class:"flex","data-direction":e.direction,"data-gap":e.gap},[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)],8,Jo)}var Yo=Object(a["defineComponent"])({props:{direction:{default:"row",type:String},gap:{default:"medium",type:String}}});t("9223");const Ko=ae()(Yo,[["render",Uo],["__scopeId","data-v-75cb2fc3"]]);var Zo=Ko,Qo=["data-size"];function Xo(e,o,t,i,r,n){return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{class:"gallery","data-size":e.size},[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)],8,Qo)}var $o=Object(a["defineComponent"])({props:{size:{default:"medium",type:String}}});t("5ed6");const et=ae()($o,[["render",Xo],["__scopeId","data-v-b2ba259a"]]);var ot=et;function tt(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon"),s=Object(a["resolveComponent"])("AppLink");return Object(a["openBlock"])(),Object(a["createBlock"])(Object(a["resolveDynamicComponent"])(e.tag||"div"),{ref:"heading",id:e.link},{default:Object(a["withCtx"])((function(){return[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0),e.link?(Object(a["openBlock"])(),Object(a["createBlock"])(s,{key:0,to:"#"+e.link,class:"anchor","aria-label":"Link to this section"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,{icon:"link"})]})),_:1},8,["to"])):Object(a["createCommentVNode"])("",!0)]})),_:3},8,["id"])}t("a9e3");function it(){var e;if(this.level)return"h"+this.level;var o=null===this||void 0===this||null===(e=this.$refs)||void 0===e?void 0:e.heading,t=o.parentElement,i=o.matches("*:first-child"),r=t.matches("*:first-child");return r?i?"h1":"h2":i?"h2":"h3"}function rt(){var e,o=null===this||void 0===this||null===(e=this.$refs)||void 0===e?void 0:e.heading;return Object(B["kebabCase"])(o.textContent||"")}var nt=Object(a["defineComponent"])({props:{level:Number},data:function(){return{tag:"",link:""}},mounted:function(){this.tag=it.call(this),this.link=rt.call(this)},updated:function(){this.tag=it.call(this),this.link=rt.call(this)}}),at=nt;t("da13");const lt=ae()(at,[["render",tt],["__scopeId","data-v-73ff4025"]]);var st=lt;function ct(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("InlineSvg"),s=Object(a["resolveComponent"])("FontAwesomeIcon");return e.custom?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0,src:e.custom,class:"icon","aria-hidden":"true"},null,8,["src"])):e.fa?(Object(a["openBlock"])(),Object(a["createBlock"])(s,{key:1,icon:e.fa,"aria-hidden":"true"},null,8,["icon"])):Object(a["createCommentVNode"])("",!0)}var pt=t("ad3d"),dt=t("6d5e"),ht=t.n(dt),ut=Object(a["defineComponent"])({props:{icon:{type:String,required:!0},fallback:String},components:{FontAwesomeIcon:pt["a"],InlineSvg:ht.a},computed:{custom:function(){try{return t("6f32")("./".concat(this.icon,".svg"))}catch(e){return!!this.fallback&&t("6f32")("./".concat(this.fallback,".svg"))}},fa:function(){for(var e=0,o=["fas","far","fab"];e<o.length;e++){var t=o[e],i=Object(s["b"])({prefix:t,iconName:this.icon});if(i)return i}return null}}});t("f919");const bt=ae()(ut,[["render",ct],["__scopeId","data-v-618079af"]]);var gt=bt,mt={key:0},yt=["href"];function vt(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon"),s=Object(a["resolveComponent"])("router-link");return e.to?e.isAbsolute?(Object(a["openBlock"])(),Object(a["createElementBlock"])("a",{key:1,href:e.to,target:"_blank"},[e.isExternal&&e.isPlainText?(Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:0},[Object(a["createElementVNode"])("span",null,[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)]),Object(a["createVNode"])(l,{class:"icon",icon:"external-link-alt"})],64)):Object(a["renderSlot"])(e.$slots,"default",{key:1},void 0,!0)],8,yt)):(Object(a["openBlock"])(),Object(a["createBlock"])(s,{key:2,to:e.to},{default:Object(a["withCtx"])((function(){return[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)]})),_:3},8,["to"])):(Object(a["openBlock"])(),Object(a["createElementBlock"])("span",mt,[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)]))}t("2ca0"),t("8a79"),t("2b3d");var ft=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"";return["http:","https:","ftp:","mailto:"].some((function(o){return e.startsWith(o)}))},wt=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"";return ft(e)&&!Ot(e).endsWith("monarchinitiative.org")},Ot=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"";try{return new URL(e).hostname}catch(o){return""}},At=Object(a["defineComponent"])({props:{to:String},computed:{isExternal:function(){return wt(this.to)},isAbsolute:function(){return ft(this.to)},isPlainText:function(){return this.$slots.default&&1===this.$slots.default().length&&"string"===typeof this.$slots.default()[0].children}}});t("d8df");const Mt=ae()(At,[["render",vt],["__scopeId","data-v-4f883a7e"]]);var jt=Mt;function kt(e,o,t,i,r,n){return Object(a["openBlock"])(),Object(a["createBlock"])(Object(a["resolveDynamicComponent"])(e.component||"div"),{innerHTML:e.html},null,8,["innerHTML"])}var Bt=t("45cf"),Dt=Object(a["defineComponent"])({props:{source:{type:String,required:!0},component:String},computed:{html:function(){return Object(Bt["a"])(this.source||"").replaceAll("<p>","").replaceAll("</p>","")}}});const xt=ae()(Dt,[["render",kt]]);var Ct=xt,It={class:"placeholder"};function _t(e,o){return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",It)}t("0bae");const Tt={},Nt=ae()(Tt,[["render",_t],["__scopeId","data-v-56131e3e"]]);var St=Nt,Rt=["data-width","data-design"];function Pt(e,o,t,i,r,n){return Object(a["openBlock"])(),Object(a["createElementBlock"])("section",{"data-width":e.width,"data-design":e.design},[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)],8,Rt)}var Lt=Object(a["defineComponent"])({props:{width:{default:"medium",type:String},design:{default:"normal",type:String}}});t("da68");const Et=ae()(Lt,[["render",Pt],["__scopeId","data-v-7d2e0965"]]);var Vt=Et,zt=["data-design"],Ht={class:"title"},Ft={key:0,class:"subtitle"};function Gt(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppButton");return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{class:"tile","data-design":e.design},[Object(a["createVNode"])(l,{design:"circle",to:e.to||"",class:"button",icon:e.icon,"aria-label":e.title},null,8,["to","icon","aria-label"]),Object(a["createElementVNode"])("div",Ht,Object(a["toDisplayString"])(e.title),1),e.subtitle?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",Ft,Object(a["toDisplayString"])(e.subtitle),1)):Object(a["createCommentVNode"])("",!0)],8,zt)}var qt=Object(a["defineComponent"])({props:{to:String,icon:{type:String,required:!0},title:{type:String,required:!0},subtitle:String,design:{default:"big",type:String}}});t("3274");const Wt=ae()(qt,[["render",Gt],["__scopeId","data-v-0e9ef62a"]]);var Jt=Wt,Ut={AppButton:Wo,AppFlex:Zo,AppGallery:ot,AppHeading:st,AppIcon:gt,AppLink:jt,AppMarkdown:Ct,AppPlaceholder:St,AppSection:Vt,AppTile:Jt},Yt=Ut,Kt=(t("e9c4"),window.localStorage),Zt=Object(a["defineComponent"])({created:function(){var e=this;if(this.$options.persist){var o,t=this.$options.persist,i=t.keys,n=t.namespace,a=n+"-"+this.$route.name.toLowerCase(),l=Object(r["a"])(i);try{for(l.s();!(o=l.n()).done;){var s=o.value,c=Qt("".concat(a,"-").concat(s));null!==c&&(this[s]=c)}}catch(u){l.e(u)}finally{l.f()}var p,d=Object(r["a"])(i);try{var h=function(){var o=p.value;e.$watch(o,(function(e){return ei("".concat(a,"-").concat(o),e)}))};for(d.s();!(p=d.n()).done;)h()}catch(u){d.e(u)}finally{d.f()}}},methods:{clearPersist:function(){if(this.$options.persist){var e,o=this.$options.persist,t=o.keys,i=o.namespace,n=Object(r["a"])(t);try{for(n.s();!(e=n.n()).done;){var a=e.value;$t("".concat(i,"-").concat(a))}}catch(l){n.e(l)}finally{n.f()}}}}}),Qt=function(e){var o=Kt.getItem(e)||"";if(!o)return null;try{return JSON.parse(o)}catch(t){return console.error('Error parsing key "'.concat(e,'" from storage')),null}},Xt=function(e,o){if(e){var t;try{t=JSON.stringify(o)}catch(i){return void console.error('Error stringifying key "'.concat(e,'" for storage"'))}try{Kt.setItem(e,t)}catch(i){return void console.error('Error setting key "'.concat(e,'" to storage'))}}},$t=function(e){return Kt.removeItem(e)},ei=Object(B["debounce"])(Xt,1e3),oi=[Zt],ti=oi,ii=(t("841c"),t("6c02")),ri=t("6f9a"),ni=Object(a["createTextVNode"])("What is Monarch"),ai=Object(a["createTextVNode"])("Features");function li(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("ExploreTabs"),s=Object(a["resolveComponent"])("AppSection"),c=Object(a["resolveComponent"])("AppHeading"),p=Object(a["resolveComponent"])("AppPlaceholder"),d=Object(a["resolveComponent"])("AppButton");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(s,{design:"fill"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l)]})),_:1}),Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(c,null,{default:Object(a["withCtx"])((function(){return[ni]})),_:1}),Object(a["createVNode"])(p),Object(a["createVNode"])(d,{to:"/about",text:"Learn more",icon:"arrow-right"})]})),_:1}),Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(c,null,{default:Object(a["withCtx"])((function(){return[ai]})),_:1}),Object(a["createVNode"])(p)]})),_:1})],64)}var si=Object(a["createTextVNode"])("Text Annotator"),ci=Object(a["createTextVNode"])("Phenotype Explorer");function pi(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("NodeSearch"),s=Object(a["resolveComponent"])("AppTabs");return Object(a["openBlock"])(),Object(a["createBlock"])(s,{name:"Explore Mode",tabs:[{id:"node-search",text:"Node Search",icon:"search"},{id:"text-annotator",text:"Text Annotator",icon:"subscript"},{id:"phenotype-explorer",text:"Phenotype Explorer",icon:"bars-progress"}],default:"node-search",onChange:e.onChange},{"node-search":Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l)]})),"text-annotator":Object(a["withCtx"])((function(){return[si]})),"phenotype-explorer":Object(a["withCtx"])((function(){return[ci]})),_:1},8,["onChange"])}var di=t("5530");function hi(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppButton"),s=Object(a["resolveComponent"])("AppFlex");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(s,{role:"tablist"},{default:Object(a["withCtx"])((function(){return[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.tabs,(function(o,t){return Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:t,id:"tab-".concat(e.id,"-").concat(o.id),text:o.text,icon:o.icon,onClick:function(t){return e.selected=o.id},design:"circle",color:e.selected===o.id?"primary":"none","aria-selected":e.selected===o.id,tabindex:e.selected===o.id?0:-1,"aria-controls":"panel-".concat(e.id,"-").concat(o.id),role:"tab",tooltip:o.tooltip,"aria-label":e.name,onKeydown:e.onKeydown},null,8,["id","text","icon","onClick","color","aria-selected","tabindex","aria-controls","tooltip","aria-label","onKeydown"])})),128))]})),_:1}),Object(a["createVNode"])(s,{gap:"big",id:"panel-".concat(e.id,"-").concat(e.selected),class:"panel","aria-labelledby":"tab-".concat(e.id,"-").concat(e.selected),role:"tabpanel"},{default:Object(a["withCtx"])((function(){return[Object(a["renderSlot"])(e.$slots,e.selected,{},void 0,!0)]})),_:3},8,["id","aria-labelledby"])],64)}t("caad"),t("c740");var ui=Object(a["defineComponent"])({emits:["change"],props:{tabs:{type:Array,required:!0},default:String,name:{type:String,required:!0}},data:function(){return{id:Object(B["uniqueId"])(),selected:this.getHash()||this.$props.default||this.$props.tabs[0].id||""}},methods:{onKeydown:function(e){var o=this;if(["ArrowLeft","ArrowRight","Home","End"].includes(e.key)){e.preventDefault();var t=this.tabs.findIndex((function(e){return e.id===o.selected}));"ArrowLeft"===e.key&&t--,"ArrowRight"===e.key&&t++,"Home"===e.key&&(t=0),"End"===e.key&&(t=this.tabs.length-1),this.selected=this.tabs[I(t,0,this.tabs.length)].id}},getHash:function(){var e=this.$route.hash.slice(1);if(this.tabs.find((function(o){return o.id===e})))return e}},watch:{selected:function(){var e=this;return Object(Ke["a"])(regeneratorRuntime.mark((function o(){var t,i,r;return regeneratorRuntime.wrap((function(o){while(1)switch(o.prev=o.next){case 0:return i="#tab-".concat(e.id,"-").concat(e.selected),r=null===(t=document)||void 0===t?void 0:t.querySelector(i),null===r||void 0===r||r.focus(),o.next=5,e.$router.replace(Object(di["a"])(Object(di["a"])({},e.$route),{},{hash:"#"+e.selected}));case 5:e.$emit("change",e.selected);case 6:case"end":return o.stop()}}),o)})))()},$route:function(){var e=this.getHash();e&&(this.selected=e)}}});t("ba24");const bi=ae()(ui,[["render",hi],["__scopeId","data-v-b34db8ce"]]);var gi=bi,mi=(t("b64b"),function(e){return Object(a["pushScopeId"])("data-v-73fb87d2"),e=e(),Object(a["popScopeId"])(),e}),yi=mi((function(){return Object(a["createElementVNode"])("span",null,"Try:",-1)})),vi={class:"title"},fi=["innerHTML"],wi={class:"truncate-3",tabindex:"0"},Oi={key:0,class:"names truncate-1",tabindex:"0"},Ai={key:1,class:"ids truncate-1",tabindex:"0"},Mi=Object(a["createTextVNode"])(" to "),ji=Object(a["createTextVNode"])(" of "),ki=Object(a["createTextVNode"])(" results "),Bi=["onClick","disabled"];function Di(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppInput"),s=Object(a["resolveComponent"])("AppButton"),c=Object(a["resolveComponent"])("AppFlex"),p=Object(a["resolveComponent"])("AppSelectMulti"),d=Object(a["resolveComponent"])("AppStatus"),h=Object(a["resolveComponent"])("AppIcon"),u=Object(a["resolveComponent"])("AppLink"),b=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(l,{ref:"searchBox",placeholder:"Search for a gene, disease, phenotype, etc.",icon:"search",modelValue:e.search,"onUpdate:modelValue":o[0]||(o[0]=function(o){return e.search=o}),onInput:e.onInput,onChange:e.onChange,onFocus:e.onFocus},null,8,["modelValue","onInput","onChange","onFocus"]),e.results.length||e.status?Object(a["createCommentVNode"])("",!0):(Object(a["openBlock"])(),Object(a["createBlock"])(c,{key:0},{default:Object(a["withCtx"])((function(){return[yi,(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.examples,(function(o,t){return Object(a["openBlock"])(),Object(a["createBlock"])(s,{key:t,text:o,design:"small",onClick:function(t){return e.doExample(o)}},null,8,["text","onClick"])})),128))]})),_:1})),e.home?Object(a["createCommentVNode"])("",!0):(Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:1},[Object.keys(e.availableFilters).length?(Object(a["openBlock"])(),Object(a["createBlock"])(c,{key:0},{default:Object(a["withCtx"])((function(){return[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.availableFilters,(function(o,t,i){return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:i},[o.length?(Object(a["openBlock"])(),Object(a["createBlock"])(p,{key:0,name:t,options:e.availableFilters[t],modelValue:e.activeFilters[t],"onUpdate:modelValue":function(o){return e.activeFilters[t]=o},onChange:e.onFilterChange},null,8,["name","options","modelValue","onUpdate:modelValue","onChange"])):Object(a["createCommentVNode"])("",!0)],64)})),128))]})),_:1})):Object(a["createCommentVNode"])("",!0),e.status&&e.search?(Object(a["openBlock"])(),Object(a["createBlock"])(d,{key:1,ref:"status",status:e.status},null,8,["status"])):Object(a["createCommentVNode"])("",!0),(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.results,(function(o,t){return Object(a["openBlock"])(),Object(a["createBlock"])(c,{key:t,direction:"col",gap:"small",class:"result"},{default:Object(a["withCtx"])((function(){var t,i;return[Object(a["createElementVNode"])("div",vi,[Object(a["withDirectives"])(Object(a["createVNode"])(h,{icon:"category-".concat(e.kebabCase(o.category)),fallback:"category-unknown",class:"type"},null,8,["icon"]),[[b,"Category: ".concat(e.capitalize(o.category||"unknown"))]]),Object(a["createVNode"])(u,{to:"/".concat(e.kebabCase(o.category||"unknown"),"/").concat(o.id),class:"name"},{default:Object(a["withCtx"])((function(){return[Object(a["createElementVNode"])("span",{innerHTML:o.highlight},null,8,fi)]})),_:2},1032,["to"]),Object(a["withDirectives"])(Object(a["createVNode"])(s,{class:"id",text:o.id,icon:"hashtag",design:"small",copy:!0,color:"secondary"},null,8,["text"]),[[b,"Node ID (click to copy)"]])]),Object(a["createElementVNode"])("p",wi,Object(a["toDisplayString"])(o.description||"No description available"),1),null!==(t=o.altNames)&&void 0!==t&&t.length?(Object(a["openBlock"])(),Object(a["createElementBlock"])("p",Oi,Object(a["toDisplayString"])(o.altNames.join("  ")),1)):Object(a["createCommentVNode"])("",!0),null!==(i=o.altIds)&&void 0!==i&&i.length?(Object(a["openBlock"])(),Object(a["createElementBlock"])("p",Ai,Object(a["toDisplayString"])(o.altIds.join("  ")),1)):Object(a["createCommentVNode"])("",!0)]})),_:2},1024)})),128)),e.results.length?(Object(a["openBlock"])(),Object(a["createBlock"])(c,{key:2,direction:"col"},{default:Object(a["withCtx"])((function(){return[Object(a["createElementVNode"])("div",null,[Object(a["createElementVNode"])("b",null,Object(a["toDisplayString"])(e.from+1),1),Mi,Object(a["createElementVNode"])("b",null,Object(a["toDisplayString"])(e.to+1),1),ji,Object(a["createElementVNode"])("b",null,Object(a["toDisplayString"])(e.count),1),ki]),Object(a["createVNode"])(c,{gap:"small"},{default:Object(a["withCtx"])((function(){return[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.pages,(function(o){return Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createElementBlock"])("button",{key:o,onClick:function(t){return e.page=o},class:"nav-button",disabled:o===e.page},[Object(a["createTextVNode"])(Object(a["toDisplayString"])(o+1),1)],8,Bi)),[[b,"Go to page ".concat(o+1," of results")]])})),128))]})),_:1})]})),_:1})):Object(a["createCommentVNode"])("",!0)],64))],64)}var xi={"NCBITaxon:28377":"Anolis carolinensis","NCBITaxon:3702":"Arabidopsis thaliana","NCBITaxon:9913":"Bos taurus","NCBITaxon:9788":"Equidae","NCBITaxon:6237":"Caenorhabditis","NCBITaxon:6239":"Caenorhabditis elegans","NCBITaxon:7955":"Danio rerio","NCBITaxon:7227":"Drosophila melanogaster","NCBITaxon:9796":"Equus caballus","NCBITaxon:9031":"Gallus gallus","NCBITaxon:9606":"Homo sapiens","NCBITaxon:9544":"Macaca mulatta","NCBITaxon:40674":"Mammalia","NCBITaxon:13616":"Monodelphis domestica","NCBITaxon:10090":"Mus musculus","NCBITaxon:10088":"Mus <mouse, genus>","NCBITaxon:39442":"Mus musculus musculus","NCBITaxon:10092":"Mus musculus domesticus","NCBITaxon:10091":"Mus musculus castaneus","NCBITaxon:57486":"Mus musculus molossinus","NCBITaxon:10102":"Mus setulosus","NCBITaxon:477815":"Mus musculus musculus x M. m. domesticus","NCBITaxon:10096":"Mus spretus","NCBITaxon:35531":"Mus musculus bactrianus","NCBITaxon:9258":"Ornithorhynchus anatinus","NCBITaxon:9598":"Pan troglodytes","NCBITaxon:10116":"Rattus norvegicus","NCBITaxon:4932":"Saccharomyces cerevisiae","NCBITaxon:559292":"Saccharomyces cerevisiae S288C","NCBITaxon:9823":"Sus scrofa","NCBITaxon:8364":"Xenopus (Silurana) tropicalis","NCBITaxon:9845":"Ruminantia","NCBITaxon:9615":"Canis lupus familiaris","NCBITaxon:9681":"Felidae","NCBITaxon:9685":"Felis catus","NCBITaxon:33554":"Carnivora","NCBITaxon:9822":"Sus","NCBITaxon:9821":"Suidae","NCBITaxon:9895":"Bovidae","NCBITaxon:33090":"Viridiplantae","NCBITaxon:9783":"Elephas maximus","NCBITaxon:9940":"Ovis aries","NCBITaxon:9545":"Macaca nemestrina","NCBITaxon:10029":"Cricetulus griseus","NCBITaxon:54600":"Macaca nigra","NCBITaxon:9538":"Erythrocebus patas","NCBITaxon:9986":"Oryctolagus cuniculus","NCBITaxon:78454":"Saguinus labiatus","NCBITaxon:9487":"Saguinus fuscicollis","NCBITaxon:9519":"Lagothrix lagotricha","NCBITaxon:9521":"Saimiri sciureus","NCBITaxon:9523":"Plecturocebus moloch","NCBITaxon:9888":"Muntiacus muntjak","OBI:0100026":"Organism"},Ci=function(e){for(var o=0,t=Object.entries(xi);o<t.length;o++){var i=Object(n["a"])(t[o],2),r=i[0],a=i[1];if(a===e)return r}return""},Ii=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(){var o,t,i,r,a,l,s,c,p,d,h,u,b,g,m,y,v,f,w,O,A,M,j,k,B,D,x,C,I,_,T,N,S,R,P=arguments;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:if(o=P.length>0&&void 0!==P[0]?P[0]:"",t=P.length>1&&void 0!==P[1]?P[1]:{},i=P.length>2&&void 0!==P[2]?P[2]:{},r=P.length>3&&void 0!==P[3]?P[3]:0,e.prev=4,o.trim()){e.next=7;break}throw new Ao("No results","warning");case 7:for(a={},l=0,s=Object.entries(i);l<s.length;l++)c=Object(n["a"])(s[l],2),p=c[0],d=c[1],h=(d||[]).map((function(e){var o=e.value;return String(o)})).filter((function(e){return e.trim()})),"taxon"===p&&(h=h.map(Ci)),u=h.join(","),b=i[p].length===t[p].length,u&&!b&&(a[p]=u);return a=Object(di["a"])(Object(di["a"])({},a),{},{boost_q:"category:disease^5,category:phenotype^5,category:gene^0,category:genotype^-10,category:variant^-35",prefix:"-OMIA",min_match:"67%",rows:10,start:r}),e.next=12,wo("".concat(fo,"/search/entity/").concat(o),a);case 12:if(g=e.sent,m=g.numFound,y=void 0===m?0:m,v=g.docs,f=void 0===v?[]:v,w=g.facet_counts,O=void 0===w?{}:w,A=g.highlighting,M=void 0===A?{}:A,j=f.map((function(e){return{id:e.id||"",altIds:e.equivalent_curie||[],name:(e.label||[])[0]||"",altNames:(e.label||[]).slice(1),category:(e.category||[])[0]||"",description:(e.definition||[])[0]||"",score:e.score||0,prefix:e.prefix||"",highlight:M[e.id].highlight}})),j.length){e.next=17;break}throw new Ao("No results","warning");case 17:for(k={},B=0,D=Object.entries(O);B<D.length;B++)for(x=Object(n["a"])(D[B],2),C=x[0],I=x[1],k[C]=[],_=0,T=Object.entries(I);_<T.length;_++)N=Object(n["a"])(T[_],2),S=N[0],R=N[1],k[C].push({value:S,count:R});return k.taxon_label&&(k.taxon=k.taxon_label,delete k.taxon_label),e.abrupt("return",{count:y,results:j,facets:k});case 23:throw e.prev=23,e.t0=e["catch"](4),Oo(e.t0);case 26:case"end":return e.stop()}}),e,null,[[4,23]])})));return function(){return e.apply(this,arguments)}}(),_i=(t("2532"),function(e){return Object(a["pushScopeId"])("data-v-07b1ee78"),e=e(),Object(a["popScopeId"])(),e}),Ti=["id","aria-label","aria-expanded","aria-controls","aria-activedescendant"],Ni={class:"button-label"},Si={class:"button-more"},Ri=Object(a["createTextVNode"])("none selected"),Pi=Object(a["createTextVNode"])(" all selected "),Li=["id"],Ei=["id","aria-label","data-selected","data-highlighted"],Vi={class:"option-check"},zi=["data-checked"],Hi=_i((function(){return Object(a["createElementVNode"])("td",{class:"option-label"},"All",-1)})),Fi=_i((function(){return Object(a["createElementVNode"])("td",{class:"option-count"},null,-1)})),Gi=_i((function(){return Object(a["createElementVNode"])("tr",null,[Object(a["createElementVNode"])("td",{class:"option-spacer",colspan:"3"})],-1)})),qi=["id","aria-selected","data-selected","data-highlighted","onClick","onMouseenter"],Wi={class:"option-check"},Ji=["data-checked"],Ui={class:"option-label"},Yi={class:"option-count"};function Ki(e,o,t,i,r,n){var l,s=Object(a["resolveComponent"])("AppIcon");return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{class:"select",style:Object(a["normalizeStyle"])({width:e.hasSlot?"":e.width})},[e.hasSlot?Object(a["renderSlot"])(e.$slots,"default",{key:0,id:"select-".concat(e.id),role:"combobox",ariaLabel:e.name,ariaMultiselectable:"true",ariaExpanded:e.expanded,ariaControls:"list-".concat(e.id),ariaHaspopup:"listbox",ariaActivedescendant:"option-".concat(e.id,"-").concat(e.highlighted),onClick:o[0]||(o[0]=function(){return e.onClick&&e.onClick.apply(e,arguments)}),onKeydown:o[1]||(o[1]=function(){return e.onKeydown&&e.onKeydown.apply(e,arguments)}),onBlur:o[2]||(o[2]=function(){return e.onBlur&&e.onBlur.apply(e,arguments)}),notification:!e.allSelected},void 0,!0):(Object(a["openBlock"])(),Object(a["createElementBlock"])("button",{key:1,id:"select-".concat(e.id),class:"button",role:"combobox","aria-label":e.name,"aria-multiselectable":"true","aria-expanded":e.expanded,"aria-controls":"list-".concat(e.id),"aria-haspopup":"listbox","aria-activedescendant":"option-".concat(e.id,"-").concat(e.highlighted),onClick:o[3]||(o[3]=function(){return e.onClick&&e.onClick.apply(e,arguments)}),onKeydown:o[4]||(o[4]=function(){return e.onKeydown&&e.onKeydown.apply(e,arguments)}),onBlur:o[5]||(o[5]=function(){return e.onBlur&&e.onBlur.apply(e,arguments)})},[Object(a["createElementVNode"])("span",Ni,[Object(a["createTextVNode"])(Object(a["toDisplayString"])(e.startCase(e.name))+" ",1),Object(a["createElementVNode"])("span",Si,[0===e.selected.length?(Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:0},[Ri],64)):e.selected.length===e.options.length?(Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:1},[Pi],64)):1===e.selected.length?(Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:2},[Object(a["createTextVNode"])(Object(a["toDisplayString"])(null===(l=e.options[e.selected[0]])||void 0===l?void 0:l.value),1)],64)):(Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:3},[Object(a["createTextVNode"])(Object(a["toDisplayString"])(e.selected.length)+" selected",1)],64))])]),Object(a["createVNode"])(s,{class:"button-icon",icon:e.expanded?"angle-up":"angle-down"},null,8,["icon"])],40,Ti)),e.expanded?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{key:2,id:"list-".concat(e.id),class:"list",role:"listbox",tabindex:"0"},[Object(a["createElementVNode"])("table",null,[Object(a["createElementVNode"])("tr",{id:"option-".concat(e.id,"--1"),class:"option",role:"menuitem","aria-label":e.allSelected?"Deselect all":"Select all","data-selected":e.allSelected,"data-highlighted":-1===e.highlighted,onClick:o[6]||(o[6]=function(){return e.toggleSelect(-1)}),onMouseenter:o[7]||(o[7]=function(o){return e.highlighted=-1}),onMousedown:o[8]||(o[8]=Object(a["withModifiers"])((function(){}),["prevent"])),onFocusin:o[9]||(o[9]=function(){return null}),onKeydown:o[10]||(o[10]=function(){return null}),tabindex:"0"},[Object(a["createElementVNode"])("td",Vi,[Object(a["createElementVNode"])("div",{"data-checked":e.allSelected},null,8,zi)]),Hi,Fi],40,Ei),Gi,(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.options,(function(t,i){return Object(a["openBlock"])(),Object(a["createElementBlock"])("tr",{key:i,id:"option-".concat(e.id,"-").concat(i),class:"option",role:"option","aria-selected":e.selected.includes(i),"data-selected":e.selected.includes(i),"data-highlighted":i===e.highlighted,onClick:function(o){return e.toggleSelect(i,o.shiftKey)},onMouseenter:function(o){return e.highlighted=i},onMousedown:o[11]||(o[11]=Object(a["withModifiers"])((function(){}),["prevent"])),onFocusin:o[12]||(o[12]=function(){return null}),onKeydown:o[13]||(o[13]=function(){return null}),tabindex:"0"},[Object(a["createElementVNode"])("td",Wi,[Object(a["createElementVNode"])("div",{"data-checked":e.selected.includes(i)},null,8,Ji)]),Object(a["createElementVNode"])("td",Ui,Object(a["toDisplayString"])(e.startCase(String(t.value))),1),Object(a["createElementVNode"])("td",Yi,Object(a["toDisplayString"])(t.count),1)],40,qi)})),128))])],8,Li)):Object(a["createCommentVNode"])("",!0)],4)}var Zi=Object(a["defineComponent"])({emits:["update:modelValue","change"],props:{name:{type:String,required:!0},modelValue:{type:Array,required:!0},options:{type:Array,required:!0},width:{type:String}},data:function(){return{id:Object(B["uniqueId"])(),expanded:!1,selected:[],highlighted:0}},methods:{open:function(){this.expanded=!0,this.highlighted=this.selected[0]||0},close:function(){this.expanded=!1},onClick:function(){var e,o;this.expanded?this.close():this.open(),null===(e=document)||void 0===e||null===(o=e.querySelector("#select-".concat(this.id)))||void 0===o||o.focus()},onBlur:function(){this.close()},onKeydown:function(e){if(["ArrowUp","ArrowDown","Home","End"].includes(e.key)&&this.expanded){e.preventDefault();var o=this.highlighted;"ArrowUp"===e.key&&o--,"ArrowDown"===e.key&&o++,"Home"===e.key&&(o=0),"End"===e.key&&(o=this.options.length-1),this.highlighted=I(o,-1,this.options.length)}!this.expanded||"Enter"!==e.key&&" "!==e.key||(e.preventDefault(),this.toggleSelect(this.highlighted)),this.expanded&&"Escape"===e.key&&this.close()},getSelected:function(){var e=this;return this.options.map((function(o,t){return e.modelValue.find((function(e){return o.value===e.value}))?t:-1})).filter((function(e){return-1!==e}))},getModel:function(){var e=this;return this.options.filter((function(o,t){return e.selected.includes(t)}))},toggleSelect:function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:-1,o=arguments.length>1&&void 0!==arguments[1]&&arguments[1];-1===e?this.allSelected?this.selected=[]:this.selected=Array(this.options.length).fill(0).map((function(e,o){return o})):o?this.selected=[e]:this.selected.includes(e)?this.selected=this.selected.filter((function(o){return o!==e})):this.selected.push(e),this.selected.sort(),this.$emit("change")},startCase:B["startCase"]},watch:{modelValue:{handler:function(){Object(B["isEqual"])(this.selected,this.getSelected())||(this.selected=this.getSelected())},deep:!0,immediate:!0},selected:{handler:function(){this.$emit("update:modelValue",this.getModel())},deep:!0},highlighted:function(){var e;null===(e=document.querySelector("#option-".concat(this.id,"-").concat(this.highlighted)))||void 0===e||e.scrollIntoView({block:"nearest"})}},computed:{hasSlot:function(){return!!this.$slots.default},allSelected:function(){return this.selected.length===this.options.length}}});t("cf86");const Qi=ae()(Zi,[["render",Ki],["__scopeId","data-v-07b1ee78"]]);var Xi=Qi,$i=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(o,t){var i,r,n,a,l;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:if(er.cancel(),!t){e.next=6;break}return i={},this.search&&(i.search=this.search),e.next=6,this.$router.push(Object(di["a"])(Object(di["a"])({},this.$route),{},{name:"Explore",query:i}));case 6:return this.status={code:"loading",text:"Loading results"},this.results=[],e.prev=8,e.next=11,Ii(this.search,o?void 0:this.availableFilters,o?void 0:this.activeFilters,o?void 0:this.from);case 11:r=e.sent,n=r.count,a=r.results,l=r.facets,this.results=a,this.originalSearch=this.search,this.count=n,o&&(this.availableFilters=Object(di["a"])({},l),this.activeFilters=Object(di["a"])({},l)),this.status=null,e.next=27;break;case 22:e.prev=22,e.t0=e["catch"](8),this.search.trim()?this.status=e.t0:this.status=null,this.results=[],o&&(this.availableFilters={},this.activeFilters={});case 27:case"end":return e.stop()}}),e,this,[[8,22]])})));return function(o,t){return e.apply(this,arguments)}}(),er=Object(B["debounce"])($i,1e3),or=["Marfan Syndrome","Multicystic Kidney Dysplasia","SSH"],tr=Object(a["defineComponent"])({components:{AppInput:so,AppStatus:Io,AppSelectMulti:Xi},data:function(){return{search:String(this.$route.query.search||""),originalSearch:"",results:[],count:0,page:0,perPage:10,status:null,availableFilters:{},activeFilters:{},examples:or}},watch:{$route:function(){var e=String(this.$route.query.search||"");this.search!==e&&(this.search=e,$i.call(this,!0,!1))},from:function(){$i.call(this,!1,!1)}},methods:{onFocus:function(){var e=this;return Object(Ke["a"])(regeneratorRuntime.mark((function o(){var t;return regeneratorRuntime.wrap((function(o){while(1)switch(o.prev=o.next){case 0:return o.next=2,e.$router.push(Object(di["a"])(Object(di["a"])({},e.$route),{},{name:"Explore"}));case 2:null===(t=document.querySelector("input"))||void 0===t||t.focus();case 3:case"end":return o.stop()}}),o)})))()},onInput:function(){er.call(this,!0,!0)},onChange:function(){this.page=0,this.search!==this.originalSearch&&$i.call(this,!0,!0)},onFilterChange:function(){this.page=0,er.call(this,!1,!1)},doExample:function(e){this.search=e,$i.call(this,!0,!0)},kebabCase:B["kebabCase"],capitalize:B["capitalize"]},computed:{home:function(){return"home"===String(this.$route.name).toLowerCase()},from:function(){return this.page*this.perPage},to:function(){return this.from+this.results.length-1},pages:function(){return Array(Math.ceil(this.count/10)).fill(0).map((function(e,o){return o}))}},mounted:function(){$i.call(this,!0,!1)}}),ir=tr;t("8697"),t("e443");const rr=ae()(ir,[["render",Di],["__scopeId","data-v-73fb87d2"]]);var nr=rr,ar=Object(a["defineComponent"])({components:{AppTabs:gi,NodeSearch:nr},methods:{onChange:function(){this.$router.push(Object(di["a"])(Object(di["a"])({},this.$route),{},{name:"Explore"}))}}});const lr=ae()(ar,[["render",pi]]);var sr=lr,cr=Object(a["defineComponent"])({components:{ExploreTabs:sr}});const pr=ae()(cr,[["render",li]]);var dr=pr,hr=Object(a["createTextVNode"])("Explore");function ur(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("ExploreTabs"),c=Object(a["resolveComponent"])("AppSection");return Object(a["openBlock"])(),Object(a["createBlock"])(c,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[hr]})),_:1}),Object(a["createVNode"])(s)]})),_:1})}var br=Object(a["defineComponent"])({components:{ExploreTabs:sr}});const gr=ae()(br,[["render",ur]]);var mr=gr,yr=Object(a["createTextVNode"])("About"),vr=Object(a["createElementVNode"])("p",null," The Monarch Initiative is an extensive, comprehensive knowledge graph and ecosystem of tools made for the benefit of clinicians, researchers, and scientists. The knowledge graph consists of XX genes, diseases, phenotypes, and other entities, imported from over XX sources. Monarch connects phenotypes to genotypes across species and bridges basic and applied research with semantics-based analysis. ",-1),fr=Object(a["createTextVNode"])("Our Vision"),wr=Object(a["createElementVNode"])("ol",null,[Object(a["createElementVNode"])("li",null," Connect cross-species gene, genotype, variant, disease, and phenotype data "),Object(a["createElementVNode"])("li",null,"Provide a portal for exploration of phenotype-based similarity"),Object(a["createElementVNode"])("li",null," Facilitate identification of animal models of human disease through phenotypic similarity "),Object(a["createElementVNode"])("li",null,"Enable quantitative comparison of cross-species phenotypes"),Object(a["createElementVNode"])("li",null,"Develop embeddable widgets for data exploration"),Object(a["createElementVNode"])("li",null,"Influence genotype and phenotype reporting standards"),Object(a["createElementVNode"])("li",null,"Improve ontologies to better curate genotype-phenotype data")],-1),Or=Object(a["createTextVNode"])("Stay up to date"),Ar=Object(a["createElementVNode"])("p",null," Be the first to know when we have important updates or other fun news. ",-1);function Mr(e,o){var t=Object(a["resolveComponent"])("AppHeading"),i=Object(a["resolveComponent"])("AppSection"),r=Object(a["resolveComponent"])("AppTile"),n=Object(a["resolveComponent"])("AppFlex");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[yr]})),_:1}),vr,Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[fr]})),_:1}),wr]})),_:1}),Object(a["createVNode"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(n,{gap:"big"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(r,{to:"/overview",icon:"cogs",title:"Overview",subtitle:"How all the pieces of Monarch fit together"}),Object(a["createVNode"])(r,{to:"https://github.com/biolink/biolink-api",icon:"code",title:"API",subtitle:"The API behind this website"}),Object(a["createVNode"])(r,{to:"/sources",icon:"database",title:"Sources",subtitle:"Datasets, ontologies, and downloads"}),Object(a["createVNode"])(r,{to:"/cite",icon:"feather-alt",title:"Cite",subtitle:"How to cite and attribute Monarch"}),Object(a["createVNode"])(r,{to:"/team",icon:"users",title:"Team",subtitle:"The people behind Monarch"}),Object(a["createVNode"])(r,{to:"/publications",icon:"newspaper",title:"Publications",subtitle:"Monarch-related published works"}),Object(a["createVNode"])(r,{to:"/terms",icon:"balance-scale",title:"Terms of Use",subtitle:"Disclaimers and agreements"})]})),_:1})]})),_:1}),Object(a["createVNode"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[Or]})),_:1}),Ar,Object(a["createVNode"])(n,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(r,{to:"https://substack.com/",icon:"envelope-open-text",title:"Newsletter",subtitle:"Occasional, detailed updates",design:"small"}),Object(a["createVNode"])(r,{to:"https://github.com/monarch-initiative",icon:"github",title:"GitHub",subtitle:"Star and follow for new releases",design:"small"}),Object(a["createVNode"])(r,{to:"https://twitter.com/MonarchInit",icon:"twitter",title:"Twitter",subtitle:"Quick updates and musings",design:"small"}),Object(a["createVNode"])(r,{to:"https://medium.com/@MonarchInit",icon:"blog",title:"Blog",subtitle:"Our blog posts on Medium",design:"small"})]})),_:1})]})),_:1})],64)}const jr={},kr=ae()(jr,[["render",Mr]]);var Br=kr,Dr=Object(a["createTextVNode"])("Overview"),xr=Object(a["createElementVNode"])("p",null," The Monarch Initiative is more than just this website, it is a whole ecosystem of related tools. This diagram illustrates the architecture of how it all fits together. ",-1);function Cr(e,o){var t=Object(a["resolveComponent"])("AppHeading"),i=Object(a["resolveComponent"])("AppPlaceholder"),r=Object(a["resolveComponent"])("AppSection");return Object(a["openBlock"])(),Object(a["createBlock"])(r,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[Dr]})),_:1}),xr,Object(a["createVNode"])(i)]})),_:1})}const Ir={},_r=ae()(Ir,[["render",Cr]]);var Tr=_r,Nr=function(e){return Object(a["pushScopeId"])("data-v-3e100142"),e=e(),Object(a["popScopeId"])(),e},Sr=Object(a["createTextVNode"])("Sources"),Rr=["src","alt"],Pr=["innerHTML"],Lr={key:3},Er=Nr((function(){return Object(a["createElementVNode"])("strong",null,"Ingested Files:",-1)})),Vr=[Er],zr={key:4,class:"files"},Hr=Nr((function(){return Object(a["createElementVNode"])("p",null," A listing of all data that Monarch archives for use in its knowledge graph and tools: ",-1)})),Fr=Object(a["createTextVNode"])("Note about licensing"),Gr=Object(a["createTextVNode"])(" Each of these sources has its own license. We have described this licensing challenge extensively on "),qr=Object(a["createTextVNode"])("reusabledata.org"),Wr=Object(a["createTextVNode"])(" and our "),Jr=Object(a["createTextVNode"])("2018 PlosOne publication"),Ur=Object(a["createTextVNode"])(". Many of the specific data resources we use in Monarch have been evaluated according to our reusabledata.org rubric; see the "),Yr=Object(a["createTextVNode"])("corpus of evaluations here"),Kr=Object(a["createTextVNode"])(". ");function Zr(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("AppCheckbox"),c=Object(a["resolveComponent"])("AppFlex"),p=Object(a["resolveComponent"])("AppStatus"),d=Object(a["resolveComponent"])("AppButton"),h=Object(a["resolveComponent"])("AppMarkdown"),u=Object(a["resolveComponent"])("AppLink"),b=Object(a["resolveComponent"])("AppAccordion"),g=Object(a["resolveComponent"])("AppSection"),m=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(g,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Sr]})),_:1}),Object(a["createVNode"])(c,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(s,{modelValue:e.showDatasets,"onUpdate:modelValue":o[0]||(o[0]=function(o){return e.showDatasets=o}),text:"Datasets (".concat(e.datasetCount,")"),icon:"database"},null,8,["modelValue","text"]),Object(a["createVNode"])(s,{modelValue:e.showOntologies,"onUpdate:modelValue":o[1]||(o[1]=function(o){return e.showOntologies=o}),text:"Ontologies (".concat(e.ontologyCount,")"),icon:"puzzle-piece"},null,8,["modelValue","text"])]})),_:1}),e.status?(Object(a["openBlock"])(),Object(a["createBlock"])(p,{key:0,status:e.status},null,8,["status"])):Object(a["createCommentVNode"])("",!0),Object(a["createVNode"])(c,{direction:"col"},{default:Object(a["withCtx"])((function(){return[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.filteredSources,(function(o,t){return Object(a["openBlock"])(),Object(a["createBlock"])(b,{key:t,text:o.name||o.id||"",icon:"dataset"===o.type?"database":"puzzle-piece"},{default:Object(a["withCtx"])((function(){var t,i;return[Object(a["createVNode"])(c,null,{default:Object(a["withCtx"])((function(){return[o.link?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(d,{key:0,design:"small",icon:"home",text:"Home",to:o.link},null,8,["to"])),[[m,"Homepage or repository for this source"]]):Object(a["createCommentVNode"])("",!0),o.license?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(d,{key:1,design:"small",icon:"balance-scale",text:"License",to:o.license},null,8,["to"])),[[m,"Link to licensing information for this source"]]):Object(a["createCommentVNode"])("",!0),o.distribution?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(d,{key:2,design:"small",icon:"download",text:"RDF",to:o.distribution},null,8,["to"])),[[m,"Download Resource Description Framework file"]]):Object(a["createCommentVNode"])("",!0),o.date?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(d,{key:3,design:"small",color:"secondary",icon:"calendar-alt",text:o.date},null,8,["text"])),[[m,"Date when this source was ingested into Monarch"]]):Object(a["createCommentVNode"])("",!0)]})),_:2},1024),e.getSrc(o.image)?(Object(a["openBlock"])(),Object(a["createElementBlock"])("img",{key:0,class:"image",src:e.getSrc(o.image),alt:o.name},null,8,Rr)):Object(a["createCommentVNode"])("",!0),o.description?(Object(a["openBlock"])(),Object(a["createElementBlock"])("p",{key:1,innerHTML:o.description},null,8,Pr)):Object(a["createCommentVNode"])("",!0),o.usage?(Object(a["openBlock"])(),Object(a["createBlock"])(h,{key:2,source:o.usage,component:"p"},null,8,["source"])):Object(a["createCommentVNode"])("",!0),null!==(t=o.files)&&void 0!==t&&t.length?(Object(a["openBlock"])(),Object(a["createElementBlock"])("p",Lr,Vr)):Object(a["createCommentVNode"])("",!0),null!==(i=o.files)&&void 0!==i&&i.length?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",zr,[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(o.files,(function(o,t){return Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(u,{key:t,to:o,class:"truncate"},{default:Object(a["withCtx"])((function(){return[Object(a["createTextVNode"])(Object(a["toDisplayString"])(e.getFilename(o)),1)]})),_:2},1032,["to"])),[[m,o,void 0,{break:!0}]])})),128))])):Object(a["createCommentVNode"])("",!0)]})),_:2},1032,["text","icon"])})),128))]})),_:1})]})),_:1}),Object(a["createVNode"])(g,null,{default:Object(a["withCtx"])((function(){return[Hr,Object(a["createVNode"])(d,{icon:"download",text:"All Downloads",to:"https://archive.monarchinitiative.org/latest/"})]})),_:1}),Object(a["createVNode"])(g,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Fr]})),_:1}),Object(a["createElementVNode"])("p",null,[Gr,Object(a["createVNode"])(u,{to:"https://reusabledata.org/"},{default:Object(a["withCtx"])((function(){return[qr]})),_:1}),Wr,Object(a["createVNode"])(u,{to:"https://doi.org/10.1371/journal.pone.0213090"},{default:Object(a["withCtx"])((function(){return[Jr]})),_:1}),Ur,Object(a["createVNode"])(u,{to:"https://reusabledata.org/#our-sources-data"},{default:Object(a["withCtx"])((function(){return[Yr]})),_:1}),Kr])]})),_:1})],64)}var Qr={class:"checkbox"},Xr=["checked"],$r={class:"text"},en={key:0};function on(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon");return Object(a["openBlock"])(),Object(a["createElementBlock"])("label",Qr,[Object(a["createElementVNode"])("input",{type:"checkbox",checked:e.modelValue,onChange:o[0]||(o[0]=function(){return e.onChange&&e.onChange.apply(e,arguments)})},null,40,Xr),Object(a["createElementVNode"])("span",$r,[e.text?(Object(a["openBlock"])(),Object(a["createElementBlock"])("span",en,Object(a["toDisplayString"])(e.text),1)):Object(a["createCommentVNode"])("",!0),e.icon?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:1,class:"icon",icon:e.icon},null,8,["icon"])):Object(a["createCommentVNode"])("",!0)])])}var tn=Object(a["defineComponent"])({props:{modelValue:Boolean,text:String,icon:String},emits:["update:modelValue"],methods:{onChange:function(e){this.$emit("update:modelValue",null===e||void 0===e?void 0:e.target.checked)}}});t("8c9a");const rn=ae()(tn,[["render",on],["__scopeId","data-v-3b0b175f"]]);var nn=rn,an=["aria-expanded","aria-label"],ln={class:"text"},sn={key:0},cn={key:0,class:"content"};function pn(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createElementVNode"])("button",{class:"title",onClick:o[0]||(o[0]=function(o){return e.expanded=!e.expanded}),"aria-expanded":e.expanded,"aria-label":e.expanded?"Collapse section":"Expand section"},[Object(a["createElementVNode"])("span",ln,[e.text?(Object(a["openBlock"])(),Object(a["createElementBlock"])("span",sn,Object(a["toDisplayString"])(e.text),1)):Object(a["createCommentVNode"])("",!0),e.icon?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:1,class:"icon",icon:e.icon},null,8,["icon"])):Object(a["createCommentVNode"])("",!0)]),Object(a["createVNode"])(l,{class:"caret",icon:"angle-down","data-expanded":e.expanded},null,8,["data-expanded"])],8,an),e.expanded?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",cn,[Object(a["renderSlot"])(e.$slots,"default",{},void 0,!0)])):Object(a["createCommentVNode"])("",!0)],64)}var dn=Object(a["defineComponent"])({props:{text:String,icon:String},data:function(){return{expanded:!1}}});t("c7fe");const hn=ae()(dn,[["render",pn],["__scopeId","data-v-027d534e"]]);var un=hn,bn=(t("13d5"),t("4c53"),t("159b"),t("9e71")),gn=(t("07ac"),function(e,o){var t=arguments.length>2&&void 0!==arguments[2]&&arguments[2],i={},r=function(){var e=!(arguments.length>0&&void 0!==arguments[0])||arguments[0];return function(o){var t=o.id,r=void 0===t?"":t;(i[r]||e)&&(i[r]=Object(B["merge"])(i[r]||{},o))}};return e.forEach(r(!0)),o.forEach(r(!t)),Object.values(i)}),mn=[["MonarchArchive:","https://archive.monarchinitiative.org/"],["CoriellCollection:","https://catalog.coriell.org/1/"],["OBO:","http://purl.obolibrary.org/obo/"],["ZFIN:","http://zfin.org/"]],yn=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"";return mn.reduce((function(e,o){var t=Object(n["a"])(o,2),i=t[0],r=t[1];return e.replaceAll(i,r)}),e)},vn=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(){var o,t,i,r,n;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return e.prev=0,e.next=3,wo("".concat(fo,"/metadata/datasets"));case 3:return o=e.sent,t=o.nodes,i=o.edges,r=i.filter((function(e){return"dc:isVersionOf"===e.pred})).map((function(e){return t.find((function(o){return o.id===e.sub}))})).filter((function(e){return e})),n=r.map((function(e){var o,t;return{id:(null===(o=i.find((function(o){return o.obj===e.id&&"versionIRI"===o.pred})))||void 0===o?void 0:o.sub)||"",date:e.meta["http://purl.org/dc/terms/created"][0],distribution:yn(null===(t=i.find((function(o){return o.sub===e.id&&"dcat:distribution"===o.pred})))||void 0===t?void 0:t.obj),files:i.filter((function(o){return o.sub===e.id&&"dc:source"===o.pred})).map((function(e){return yn(e.obj)}))}})),n=gn(bn,n),n.forEach((function(e){return e.type="dataset"})),e.abrupt("return",n);case 13:throw e.prev=13,e.t0=e["catch"](0),Oo(e.t0);case 16:case"end":return e.stop()}}),e,null,[[0,13]])})));return function(){return e.apply(this,arguments)}}(),fn=t("1778"),wn="https://obofoundry.org/registry/ontologies.jsonld",On=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(){var o,t;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return e.prev=0,e.next=3,wo(wn);case 3:return o=e.sent,t=o.ontologies.map((function(e){var o;return{id:e.id,name:e.title,link:e.homepage,license:null===(o=e.license)||void 0===o?void 0:o.url,image:e.depicted_by,description:e.description}})),t=gn(fn,t,!0),t.forEach((function(e){return e.type="ontology"})),e.abrupt("return",t);case 10:throw e.prev=10,e.t0=e["catch"](0),Oo(e.t0);case 13:case"end":return e.stop()}}),e,null,[[0,10]])})));return function(){return e.apply(this,arguments)}}(),An=Object(a["defineComponent"])({components:{AppCheckbox:nn,AppAccordion:un,AppStatus:Io},data:function(){return{sources:[],showDatasets:!0,showOntologies:!0,status:null}},methods:{getSrc:function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"";try{return e.startsWith("http")?e:t("fccd")("./".concat(e))}catch(o){return!1}},getFilename:function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"";return e.split("/").filter((function(e){return e})).pop()}},computed:{filteredSources:function(){var e=this;return this.sources.filter((function(o){return"dataset"===o.type&&e.showDatasets||"ontology"===o.type&&e.showOntologies}))},datasetCount:function(){return this.sources.filter((function(e){return"dataset"===e.type})).length},ontologyCount:function(){return this.sources.filter((function(e){return"ontology"===e.type})).length}},mounted:function(){var e=this;return Object(Ke["a"])(regeneratorRuntime.mark((function o(){var t,r;return regeneratorRuntime.wrap((function(o){while(1)switch(o.prev=o.next){case 0:return e.status={code:"loading",text:"Loading sources"},o.prev=1,o.next=4,vn();case 4:return t=o.sent,o.next=7,On();case 7:r=o.sent,e.sources=[].concat(Object(i["a"])(t),Object(i["a"])(r)),e.sources.sort((function(e,o){return((null===e||void 0===e?void 0:e.name)||(null===e||void 0===e?void 0:e.id)||"").toLowerCase()<((null===o||void 0===o?void 0:o.name)||(null===o||void 0===o?void 0:o.id)||"").toLowerCase()?-1:1})),e.status=null,o.next=16;break;case 13:o.prev=13,o.t0=o["catch"](1),e.status=o.t0;case 16:case"end":return o.stop()}}),o,null,[[1,13]])})))()}});t("2a5a");const Mn=ae()(An,[["render",Zr],["__scopeId","data-v-3e100142"]]);var jn=Mn,kn=Object(a["createTextVNode"])("Cite"),Bn=Object(a["createTextVNode"])("Acknowledgement"),Dn=Object(a["createElementVNode"])("p",null," If you use Monarch, please acknolwedge us with a paragraph like the following: ",-1),xn=Object(a["createElementVNode"])("blockquote",null,[Object(a["createElementVNode"])("code",null,"YOUR WORK"),Object(a["createTextVNode"])(" uses and displays data and algorithms from the Monarch Initiative. The Monarch Initiative (https://monarchinitiative.org) makes biomedical knowledge exploration more efficient and effective by providing tools for genotype-phenotype analysis, genomic diagnostics, and precision medicine across broad areas of disease. ")],-1),Cn=Object(a["createTextVNode"])("In Publications"),In=Object(a["createElementVNode"])("p",null," To cite the use of Monarch in publications, please use these two references: ",-1),_n=Object(a["createTextVNode"])("Data or Result"),Tn=Object(a["createTextVNode"])(" Because most of the association content in Monarch comes from external sources, please cite the original database records (e.g. "),Nn=Object(a["createTextVNode"])("MGI:1915196"),Sn=Object(a["createTextVNode"])(") or the original publications (e.g. PMID:25605924) where possible. "),Rn=Object(a["createTextVNode"])(" The data in Monarch is constantly expanding and improving. The data is versioned with each database release, and archived database releases are available upon request. The evolution of individual records in Monarch is not currently tracked. More documentation about our identifiers can be found "),Pn=Object(a["createTextVNode"])("on GitHub"),Ln=Object(a["createTextVNode"])(". "),En=Object(a["createTextVNode"])("Page or Algorithmic Results"),Vn=Object(a["createTextVNode"])(" To link to a specific page or to algorithmic results presented on this website, please use the URL in your browser's address bar. For example, you would cite the prefixed identifier "),zn=Object(a["createTextVNode"])("OMIM:154700"),Hn=Object(a["createTextVNode"])(" as: "),Fn=Object(a["createElementVNode"])("blockquote",null," Monarch Consortium. Accessed 2016-09-22. Marfan Syndrome. https://monarchinitiative.org/OMIM:154700 ",-1);function Gn(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("AppSection"),c=Object(a["resolveComponent"])("AppCitation"),p=Object(a["resolveComponent"])("AppLink");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[kn]})),_:1})]})),_:1}),Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Bn]})),_:1}),Dn,xn]})),_:1}),Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Cn]})),_:1}),In,Object(a["createVNode"])(c,{title:"Navigating the Phenotype Frontier: The Monarch Initiative",authors:"McMurry et al.",details:["GENETICS","August 1, 2016 vol. 203 no. 4 1491-1495","10.1534/genetics.116.188870"]},null,8,["details"]),Object(a["createVNode"])(c,{title:"The Monarch Initiative in 2019: an integrative data and analytic platform connecting phenotypes to genotypes across species",authors:"Shefchek et al.",details:["Nucleic Acids Res.","2020 Jan 8; 48 (Database issue): D704D715. Published online 2019 Nov 08.","10.1093/nar/gkz997"]},null,8,["details"])]})),_:1}),Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[_n]})),_:1}),Object(a["createElementVNode"])("p",null,[Tn,Object(a["createVNode"])(p,{to:"http://www.informatics.jax.org/accession/MGI:1915196"},{default:Object(a["withCtx"])((function(){return[Nn]})),_:1}),Sn]),Object(a["createElementVNode"])("p",null,[Rn,Object(a["createVNode"])(p,{to:"https://github.com/monarch-initiative/dipper/blob/master/README.md#identifiers"},{default:Object(a["withCtx"])((function(){return[Pn]})),_:1}),Ln])]})),_:1}),Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[En]})),_:1}),Object(a["createElementVNode"])("p",null,[Vn,Object(a["createVNode"])(p,{to:"OMIM:154700"},{default:Object(a["withCtx"])((function(){return[zn]})),_:1}),Hn]),Fn]})),_:1})],64)}var qn={class:"citation"},Wn={key:0,class:"truncate-2",tabindex:"0"},Jn={key:1,class:"truncate-2",tabindex:"0"};function Un(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppMarkdown");return Object(a["openBlock"])(),Object(a["createElementBlock"])("blockquote",qn,[e.title?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",Wn,Object(a["toDisplayString"])(e.title),1)):Object(a["createCommentVNode"])("",!0),e.authors?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",Jn,Object(a["toDisplayString"])(e.authors),1)):Object(a["createCommentVNode"])("",!0),e.detailsString?(Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:2,class:"truncate-2",tabindex:"0",source:e.detailsString},null,8,["source"])):Object(a["createCommentVNode"])("",!0)])}var Yn=Object(a["defineComponent"])({props:{title:String,authors:String,details:Array},computed:{detailsString:function(){return(this.details||[]).filter((function(e){return e})).join("&nbsp;  &nbsp;")}}});t("b328");const Kn=ae()(Yn,[["render",Un],["__scopeId","data-v-70247124"]]);var Zn=Kn,Qn=Object(a["defineComponent"])({components:{AppCitation:Zn}});const Xn=ae()(Qn,[["render",Gn]]);var $n=Xn,ea=function(e){return Object(a["pushScopeId"])("data-v-9e88833c"),e=e(),Object(a["popScopeId"])(),e},oa=Object(a["createTextVNode"])("Team"),ta=ea((function(){return Object(a["createElementVNode"])("p",null," Monarch is made possible thanks to a diverse and dedicated team of biologists, scientists, and programmers from various schools and institutes. ",-1)})),ia=["href"],ra={key:0},na=["src","alt"],aa=Object(a["createTextVNode"])("Funding"),la=Object(a["createTextVNode"])(" OFFICE OF THE DIRECTOR, NATIONAL INSTITUTES OF HEALTH "),sa=ea((function(){return Object(a["createElementVNode"])("br",null,null,-1)})),ca=Object(a["createTextVNode"])(" The Monarch Initiative: Linking diseases to model organism resources"),pa=ea((function(){return Object(a["createElementVNode"])("br",null,null,-1)})),da=Object(a["createTextVNode"])(" 2R24OD011883-10A1 "),ha=Object(a["createTextVNode"])(" NATIONAL HUMAN GENOME RESEARCH INSTITUTE, Center of Excellence in Genome Sciences "),ua=ea((function(){return Object(a["createElementVNode"])("br",null,null,-1)})),ba=Object(a["createTextVNode"])(" A phenomics-first resource for interpretation of variants "),ga=ea((function(){return Object(a["createElementVNode"])("br",null,null,-1)})),ma=Object(a["createTextVNode"])(" 7RM1HG010860-02 "),ya=Object(a["createTextVNode"])(" NATIONAL HUMAN GENOME RESEARCH INSTITUTE "),va=ea((function(){return Object(a["createElementVNode"])("br",null,null,-1)})),fa=Object(a["createTextVNode"])(" The Human Phenotype Ontology: Accelerating Computational Integration of Clinical Data for Genomics"),wa=ea((function(){return Object(a["createElementVNode"])("br",null,null,-1)})),Oa=Object(a["createTextVNode"])(" 1U24HG011449-01A1 ");function Aa(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("AppSection"),c=Object(a["resolveComponent"])("AppIcon"),p=Object(a["resolveComponent"])("AppLink"),d=Object(a["resolveComponent"])("AppMember"),h=Object(a["resolveComponent"])("AppGallery"),u=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[oa]})),_:1}),ta,Object(a["createElementVNode"])("p",null,[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.team,(function(o,t){return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:t},[Object(a["createElementVNode"])("a",{href:"#"+e.kebabCase(o.name)},Object(a["toDisplayString"])(o.name),9,ia),t!==e.team.length-1?(Object(a["openBlock"])(),Object(a["createElementBlock"])("span",ra,"  ")):Object(a["createCommentVNode"])("",!0)],64)})),128))])]})),_:1}),(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.team,(function(o,t){return Object(a["openBlock"])(),Object(a["createBlock"])(s,{key:t,width:"big"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Object(a["createTextVNode"])(Object(a["toDisplayString"])(o.name)+" ",1),o.alumni?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(c,{key:0,class:"icon",icon:"history"},null,512)),[[u,"Alumni group"]]):Object(a["createCommentVNode"])("",!0)]})),_:2},1024),o.link?(Object(a["openBlock"])(),Object(a["createBlock"])(p,{key:0,to:o.link,"aria-label":o.name},{default:Object(a["withCtx"])((function(){return[e.getSrc(o.image)?(Object(a["openBlock"])(),Object(a["createElementBlock"])("img",{key:0,class:"image",src:e.getSrc(o.image),alt:o.name,loading:"lazy"},null,8,na)):Object(a["createCommentVNode"])("",!0)]})),_:2},1032,["to","aria-label"])):Object(a["createCommentVNode"])("",!0),Object(a["createVNode"])(h,null,{default:Object(a["withCtx"])((function(){return[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(o.members,(function(e,t){return Object(a["openBlock"])(),Object(a["createBlock"])(d,{key:t,name:e.name,role:e.role,link:e.link,alumni:o.alumni||e.alumni},null,8,["name","role","link","alumni"])})),128))]})),_:2},1024)]})),_:2},1024)})),128)),Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[aa]})),_:1}),Object(a["createElementVNode"])("ul",null,[Object(a["createElementVNode"])("li",null,[la,sa,Object(a["createVNode"])(p,{to:"https://reporter.nih.gov/project-details/10173498"},{default:Object(a["withCtx"])((function(){return[ca]})),_:1}),pa,da]),Object(a["createElementVNode"])("li",null,[ha,ua,Object(a["createVNode"])(p,{to:"https://reporter.nih.gov/project-details/10448140"},{default:Object(a["withCtx"])((function(){return[ba]})),_:1}),ga,ma]),Object(a["createElementVNode"])("li",null,[ya,va,Object(a["createVNode"])(p,{to:"https://reporter.nih.gov/project-details/10269338"},{default:Object(a["withCtx"])((function(){return[fa]})),_:1}),wa,Oa])])]})),_:1})],64)}var Ma={class:"image"},ja={class:"portrait"},ka=["src","alt"],Ba={key:0,class:"text"},Da={class:"name"},xa={key:0,class:"role"};function Ca(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon"),s=Object(a["resolveComponent"])("AppLink"),c=Object(a["resolveDirective"])("tooltip");return Object(a["openBlock"])(),Object(a["createBlock"])(s,{to:e.link,class:"member"},{default:Object(a["withCtx"])((function(){return[Object(a["createElementVNode"])("div",Ma,[Object(a["createElementVNode"])("div",ja,[Object(a["createElementVNode"])("img",{src:e.src,alt:e.name,loading:"lazy"},null,8,ka)]),e.alumni?Object(a["withDirectives"])((Object(a["openBlock"])(),Object(a["createBlock"])(l,{key:0,icon:"history",class:"icon"},null,512)),[[c,"Alumni team member"]]):Object(a["createCommentVNode"])("",!0)]),e.role?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",Ba,[Object(a["createElementVNode"])("div",Da,Object(a["toDisplayString"])(e.name),1),e.role?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",xa,Object(a["toDisplayString"])(e.role),1)):Object(a["createCommentVNode"])("",!0)])):Object(a["createCommentVNode"])("",!0)]})),_:1},8,["to"])}var Ia=Object(a["defineComponent"])({props:{name:String,role:String,link:String,alumni:Boolean},computed:{src:function(){var e=Object(B["kebabCase"])(Object(B["deburr"])((this.name||"").toLowerCase()));try{return t("0879")("./".concat(e,".jpg"))}catch(o){return t("397d")}}}});t("1052");const _a=ae()(Ia,[["render",Ca],["__scopeId","data-v-54e67079"]]);var Ta=_a,Na=t("b593"),Sa=Object(a["defineComponent"])({components:{AppMember:Ta},data:function(){return{team:Na}},methods:{getSrc:function(e){try{return t("1802")("./".concat(e))}catch(o){return!1}},kebabCase:B["kebabCase"]}});t("9f41");const Ra=ae()(Sa,[["render",Aa],["__scopeId","data-v-9e88833c"]]);var Pa=Ra,La=Object(a["createTextVNode"])("Publications"),Ea=["href"],Va={key:0};function za(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("AppSection"),c=Object(a["resolveComponent"])("AppCitation"),p=Object(a["resolveComponent"])("AppGallery");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(s,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[La]})),_:1}),Object(a["createElementVNode"])("p",null,[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.publications,(function(o,t){return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],{key:t},[Object(a["createElementVNode"])("a",{href:"#"+o.year},Object(a["toDisplayString"])(o.year),9,Ea),t!==e.publications.length-1?(Object(a["openBlock"])(),Object(a["createElementBlock"])("span",Va,"  ")):Object(a["createCommentVNode"])("",!0)],64)})),128))])]})),_:1}),(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.publications,(function(e,o){return Object(a["openBlock"])(),Object(a["createBlock"])(s,{key:o,width:"big"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Object(a["createTextVNode"])(Object(a["toDisplayString"])(e.year),1)]})),_:2},1024),Object(a["createVNode"])(p,{size:"big"},{default:Object(a["withCtx"])((function(){return[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.list,(function(e,o){return Object(a["openBlock"])(),Object(a["createBlock"])(c,{key:o,title:e.title,authors:e.authors,details:[e.journal,e.issue,e.link]},null,8,["title","authors","details"])})),128))]})),_:2},1024)]})),_:2},1024)})),128))],64)}var Ha=t("993b"),Fa=Object(a["defineComponent"])({components:{AppCitation:Zn},data:function(){return{publications:Ha}}});const Ga=ae()(Fa,[["render",za]]);var qa=Ga,Wa=Object(a["createTextVNode"])("Terms of Use"),Ja=Object(a["createTextVNode"])("License"),Ua=Object(a["createTextVNode"])(" This web application and its source code are licensed under the "),Ya=Object(a["createTextVNode"])("3-Clause BSD License"),Ka=Object(a["createTextVNode"])(". Licensing information for the data sources and ontologies that make up the Monarch knowledge graph can be found on the "),Za=Object(a["createTextVNode"])("sources page"),Qa=Object(a["createTextVNode"])(". Licensing information for tools, APIs, algorithms, and workflows can be found on the "),Xa=Object(a["createTextVNode"])("overview page"),$a=Object(a["createTextVNode"])(". "),el=Object(a["createTextVNode"])("Tracking"),ol=Object(a["createElementVNode"])("p",null," This web application tracks user behavior such as navigation and clicks through the use of HotJar and Google Analytics. These background scripts/services take significant steps to remove identifying information from the data they collect to ensure anonymity. ",-1),tl=Object(a["createElementVNode"])("p",null," The Monarch Initiative pledges to never inspect or disclose any private, identifying information collected, and to only use it for the purposes of improving Monarch. ",-1),il=Object(a["createTextVNode"])(" To learn more about how Hotjar maintains privacy and complies with relevant privacy laws, see "),rl=Object(a["createTextVNode"])("their explanation here"),nl=Object(a["createTextVNode"])(". "),al=Object(a["createTextVNode"])("Definitions"),ll=Object(a["createTextVNode"])(' The first-party software and resources used and referenced on this website  including but not limited to the Monarch Website (this website itself), BioLink, and Exomiser  are hereby refered to as the "Monarch Tools". The associated '),sl=Object(a["createTextVNode"])("Universities, Institutions, and other groups"),cl=Object(a["createTextVNode"])(' that develop and maintain the Monarch Tools are hereby collectively refered to as the "Monarch Initiative". '),pl=Object(a["createTextVNode"])("Intended use"),dl=Object(a["createElementVNode"])("p",null," Monarch Tools should not be used for direct diagnostic use or medical decision-making. If you have any health questions about the information contained on this website, please contact a medical professional. ",-1),hl=Object(a["createTextVNode"])("Disclaimer"),ul=Object(a["createElementVNode"])("p",null,' YOU ACKNOWLEDGE THAT THE MONARCH TOOLS, INCLUDING WITHOUT LIMITATION, THE DATABASE OF INFORMATION CONTAINED THEREIN, IS EXPERIMENTAL AND ACADEMIC IN NATURE, AND IS NOT LICENSED BY THE U.S. FOOD AND DRUG ADMINISTRATION OR ANY OTHER REGULATORY BODY. THE MONARCH TOOLS ARE PROVIDED ON AN "AS IS" AND AS AVAILABLE BASIS WITH ALL FAULTS, WITHOUT WARRANTY OR REPRESENTATION OF ANY KIND, EXPRESS OR IMPLIED. THE MONARCH INITIATIVE DISCLAIMS ALL WARRANTIES AND REPRESENTATIONS, EXPRESS OR IMPLIED, WITH RESPECT TO THE MONARCH TOOLS, INCLUDING WITHOUT LIMITATION, ANY WARRANTIES OF TITLE, MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT OF THIRD PARTY RIGHTS. THIS DISCLAIMER APPLIES TO THE FULLEST EXTENT POSSIBLE. THE MONARCH INITIATIVE DOES NOT WARRANT THAT THE MONARCH TOOLS WILL MEET YOUR REQUIREMENTS OR THAT THE USE OF THE MONARCH TOOLS WILL BE FREE OF INFECTION OR VIRUSES, ERROR-FREE, UNINTERRUPTED, SECURE OR WILL PRODUCE ACCURATE RESULTS. YOU SHALL BEAR TOTAL AND EXCLUSIVE RESPONSIBILITY AND RISK FOR THE USE OF THE MONARCH TOOLS. YOU SHALL BE SOLELY RESPONSIBLE FOR ANY RESULTING DAMAGE TO YOUR COMPUTER SYSTEMS OR LOSS OF DATA. IF YOU ARE DISSATISFIED WITH THE MONARCH TOOLS, OR WITH ANY OF THESE TERMS, YOUR SOLE AND EXCLUSIVE REMEDY IS TO DISCONTINUE USING THE MONARCH TOOLS. ',-1),bl=Object(a["createTextVNode"])("Limitation of liability"),gl=Object(a["createElementVNode"])("p",null,' IN NO EVENT SHALL THE MONARCH INITIATIVE, OR THEIR RESPECTIVE DIRECTORS, OFFICERS, EMPLOYEES, AGENTS, LICENSEES, LICENSORS, CONTRIBUTORS, FOUNDERS OR ADVISORS, OR ANY OTHER PERSON WHO HAS BEEN INVOLVED IN THE CREATION OR PRODUCTION OF THE MONARCH TOOLS (EACH A RELEASED PARTY AND COLLECTIVELY, "RELEASED PARTIES") BE LIABLE FOR ANY CLAIMS, DEMANDS, LIABILITIES, COSTS, LOSSES, DAMAGES OR EXPENSES (INCLUDING WITHOUT LIMITATION LEGAL COSTS AND ATTORNEYS FEES) CAUSED TO OR SUFFERED BY ANY PERSONS OR ENTITIES (INCLUDING WITHOUT LIMITATION, YOU), THAT DIRECTLY OR INDIRECTLY ARISE OUT OF OR RESULT FROM THE USE OF, THE INABILITY TO USE, OR ERRORS, OMISSIONS, INTERRUPTIONS OR OTHER INACCURACIES IN, THE MONARCH TOOLS OR VIOLATION OF ANY RIGHTS OF ANY THIRD PARTY. WITHOUT LIMITING THE FOREGOING, THE RELEASED PARTIES SHALL NOT BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, CONSEQUENTIAL, SPECIAL, PUNITIVE OR EXEMPLARY DAMAGES, OR ANY OTHER DAMAGES OF ANY KIND, INCLUDING WITHOUT LIMITATION, DAMAGES FOR LOSS OF BUSINESS, LOSS OF DATA, LOST PROFITS OR BUSINESS INTERRUPTION, UNDER ANY CONTRACT, NEGLIGENCE, STRICT LIABILITY OR OTHER THEORY, EVEN IF A RELEASED PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES. THIS LIMITATION OF LIABILITY APPLIES TO THE FULLEST EXTENT POSSIBLE. ',-1),ml=Object(a["createTextVNode"])("Variant interpretation"),yl=Object(a["createElementVNode"])("p",null," The Monarch Initiative sources variant interpretations as provided by the original source, with no consolidation of results, and always has a link out to the original reference. While we display the variants and interpretations as originally reported in the variant databases, some variants in the same genes have been reported on different transcripts. Whenever possible, Monarch will attempt to map all variants in a particular gene to a common transcript to aid in viewing and organizing the variants. However, to aid in variant searches, we will also continue to report the original mapping information. All interpretations were performed by the original submitters and have not been modified in any way. As such, there might be discordant interpretations for the same variant from different archives. For further information or to request a review of the variant, please contact the submitter of the original reference. Variant interpretations are not static as they are based on the current understanding of the variant at the time it was observed in a patient or in the literature. Any reported interpretation may not be up-to-date, and some information may not be reported. Many interpretations will change over time as more information about the variant becomes available. If you have any questions about a variant interpretation or would like use any variant interpretations for publication purposes, please contact the submitter of the original reference. As always, we value your feedback. Please email us if you have any problems, concerns, or suggestions. ",-1);function vl(e,o){var t=Object(a["resolveComponent"])("AppHeading"),i=Object(a["resolveComponent"])("AppSection"),r=Object(a["resolveComponent"])("AppLink");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[Wa]})),_:1})]})),_:1}),Object(a["createVNode"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[Ja]})),_:1}),Object(a["createElementVNode"])("p",null,[Ua,Object(a["createVNode"])(r,{to:"https://opensource.org/licenses/BSD-3-Clause"},{default:Object(a["withCtx"])((function(){return[Ya]})),_:1}),Ka,Object(a["createVNode"])(r,{to:"/sources"},{default:Object(a["withCtx"])((function(){return[Za]})),_:1}),Qa,Object(a["createVNode"])(r,{to:"/overview"},{default:Object(a["withCtx"])((function(){return[Xa]})),_:1}),$a]),Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[el]})),_:1}),ol,tl,Object(a["createElementVNode"])("p",null,[il,Object(a["createVNode"])(r,{to:"https://www.hotjar.com/legal/compliance/gdpr-commitment/"},{default:Object(a["withCtx"])((function(){return[rl]})),_:1}),nl])]})),_:1}),Object(a["createVNode"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[al]})),_:1}),Object(a["createElementVNode"])("p",null,[ll,Object(a["createVNode"])(r,{to:"/team"},{default:Object(a["withCtx"])((function(){return[sl]})),_:1}),cl]),Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[pl]})),_:1}),dl,Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[hl]})),_:1}),ul,Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[bl]})),_:1}),gl]})),_:1}),Object(a["createVNode"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[ml]})),_:1}),yl]})),_:1})],64)}const fl={},wl=ae()(fl,[["render",vl]]);var Ol=wl,Al=function(e){return Object(a["pushScopeId"])("data-v-c26e7474"),e=e(),Object(a["popScopeId"])(),e},Ml=Object(a["createTextVNode"])("Help"),jl=Al((function(){return Object(a["createElementVNode"])("p",null," Request a feature, report a bug, or chat with us about anything Monarch-related. ",-1)})),kl=Object(a["createTextVNode"])("How to use this website"),Bl=Object(a["createTextVNode"])("Status"),Dl=Object(a["createTextVNode"])(" If you still need help, or for general inquires, you can "),xl=Object(a["createTextVNode"])("email us"),Cl=Object(a["createTextVNode"])(" or "),Il=Object(a["createTextVNode"])("tweet us"),_l=Object(a["createTextVNode"])(". ");function Tl(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("AppTile"),c=Object(a["resolveComponent"])("AppFlex"),p=Object(a["resolveComponent"])("AppSection"),d=Object(a["resolveComponent"])("AppPlaceholder"),h=Object(a["resolveComponent"])("AppGallery"),u=Object(a["resolveComponent"])("AppStatus"),b=Object(a["resolveComponent"])("AppButton"),g=Object(a["resolveComponent"])("AppLink");return Object(a["openBlock"])(),Object(a["createElementBlock"])(a["Fragment"],null,[Object(a["createVNode"])(p,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Ml]})),_:1}),jl,Object(a["createVNode"])(c,{gap:"big"},{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(s,{icon:"comment",title:"Feedback Form",subtitle:"Right here, no account required, one-way",to:"/feedback"}),Object(a["createVNode"])(s,{icon:"comments",title:"Help Desk",subtitle:"On GitHub, requires account, two-way",to:"https://github.com/monarch-initiative/helpdesk"})]})),_:1})]})),_:1}),Object(a["createVNode"])(p,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[kl]})),_:1}),Object(a["createVNode"])(h,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(d),Object(a["createVNode"])(d),Object(a["createVNode"])(d),Object(a["createVNode"])(d),Object(a["createVNode"])(d),Object(a["createVNode"])(d)]})),_:1})]})),_:1}),Object(a["createVNode"])(p,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Bl]})),_:1}),e.status?(Object(a["openBlock"])(),Object(a["createBlock"])(u,{key:0,status:e.status},null,8,["status"])):(Object(a["openBlock"])(),Object(a["createBlock"])(h,{key:1,size:"small"},{default:Object(a["withCtx"])((function(){return[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.uptimes,(function(e,o){return Object(a["openBlock"])(),Object(a["createBlock"])(u,{class:"status",key:o,status:e},null,8,["status"])})),128))]})),_:1})),Object(a["createVNode"])(b,{to:"https://stats.uptimerobot.com/XPRo9s4BJ5",text:"More Details",icon:"arrow-right"})]})),_:1}),Object(a["createVNode"])(p,null,{default:Object(a["withCtx"])((function(){return[Object(a["createElementVNode"])("p",null,[Dl,Object(a["createVNode"])(g,{to:"mailto:info@monarchinitiative.org"},{default:Object(a["withCtx"])((function(){return[xl]})),_:1}),Cl,Object(a["createVNode"])(g,{to:"https://twitter.com/MonarchInit"},{default:Object(a["withCtx"])((function(){return[Il]})),_:1}),_l])]})),_:1})],64)}var Nl,Sl="https://api.uptimerobot.com/v2/getMonitors",Rl="ur1488940-1c05ba09e0aef926989d6593",Pl="https://stats.uptimerobot.com/XPRo9s4BJ5";(function(e){e[e["paused"]=0]="paused",e[e["unchecked"]=1]="unchecked",e[e["up"]=2]="up",e[e["seems_down"]=8]="seems_down",e[e["down"]=9]="down"})(Nl||(Nl={}));var Ll=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(){var o,t,i,r,n,a,l,s;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return e.prev=0,t={api_key:Rl},i={method:"POST"},e.next=5,wo(Sl,t,i);case 5:return r=e.sent,n=r.monitors,a=void 0===n?[]:n,o={},Object(vo["a"])(o,Nl.paused,"paused"),Object(vo["a"])(o,Nl.unchecked,"unknown"),Object(vo["a"])(o,Nl.up,"success"),Object(vo["a"])(o,Nl.seems_down,"error"),Object(vo["a"])(o,Nl.down,"error"),l=o,s=a.map((function(e){return{code:"undefined"!==typeof e.status?l[e.status]:"unknown",text:e.friendly_name,link:Pl+"/"+(e.id||"")}})),e.abrupt("return",s);case 12:throw e.prev=12,e.t0=e["catch"](0),Oo(e.t0);case 15:case"end":return e.stop()}}),e,null,[[0,12]])})));return function(){return e.apply(this,arguments)}}(),El=Object(a["defineComponent"])({components:{AppStatus:Io},data:function(){return{uptimes:[],status:null}},mounted:function(){var e=this;return Object(Ke["a"])(regeneratorRuntime.mark((function o(){return regeneratorRuntime.wrap((function(o){while(1)switch(o.prev=o.next){case 0:return e.status={code:"loading",text:"Loading service statuses"},o.prev=1,o.next=4,Ll();case 4:e.uptimes=o.sent,e.status=null,o.next=11;break;case 8:o.prev=8,o.t0=o["catch"](1),e.status=o.t0;case 11:case"end":return o.stop()}}),o,null,[[1,8]])})))()}});t("54f9");const Vl=ae()(El,[["render",Tl],["__scopeId","data-v-c26e7474"]]);var zl=Vl,Hl=Object(a["createTextVNode"])("Feedback Form");function Fl(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppHeading"),s=Object(a["resolveComponent"])("TheFeedbackForm"),c=Object(a["resolveComponent"])("AppSection");return Object(a["openBlock"])(),Object(a["createBlock"])(c,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(l,null,{default:Object(a["withCtx"])((function(){return[Hl]})),_:1}),Object(a["createVNode"])(s)]})),_:1})}var Gl=Object(a["defineComponent"])({components:{TheFeedbackForm:So}});const ql=ae()(Gl,[["render",Fl]]);var Wl=ql,Jl=Object(a["createTextVNode"])("Node Page");function Ul(e,o){var t=Object(a["resolveComponent"])("AppHeading"),i=Object(a["resolveComponent"])("AppSection");return Object(a["openBlock"])(),Object(a["createBlock"])(i,null,{default:Object(a["withCtx"])((function(){return[Object(a["createVNode"])(t,null,{default:Object(a["withCtx"])((function(){return[Jl]})),_:1})]})),_:1})}const Yl={},Kl=ae()(Yl,[["render",Ul]]);var Zl=Kl;var Ql={class:"select"},Xl=["id","aria-label","aria-expanded","aria-controls","aria-activedescendant"],$l={class:"button-label"},es=["id"],os=["id","aria-selected","data-selected","data-highlighted","onClick","onMouseenter"];function ts(e,o,t,i,r,n){var l=Object(a["resolveComponent"])("AppIcon");return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",Ql,[Object(a["createElementVNode"])("button",{id:"select-".concat(e.id),class:"button",role:"combobox","aria-label":e.name,"aria-expanded":e.expanded,"aria-controls":"list-".concat(e.id),"aria-haspopup":"listbox","aria-activedescendant":"option-".concat(e.id,"-").concat(e.highlighted),onClick:o[0]||(o[0]=function(){return e.onClick&&e.onClick.apply(e,arguments)}),onKeydown:o[1]||(o[1]=function(){return e.onKeydown&&e.onKeydown.apply(e,arguments)}),onBlur:o[2]||(o[2]=function(){return e.onBlur&&e.onBlur.apply(e,arguments)})},[Object(a["createElementVNode"])("span",$l,Object(a["toDisplayString"])(e.modelValue),1),Object(a["createVNode"])(l,{class:"button-icon",icon:e.expanded?"angle-up":"angle-down"},null,8,["icon"])],40,Xl),e.expanded?(Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{key:0,id:"list_".concat(e.id),class:"list",role:"listbox",tabindex:"0"},[(Object(a["openBlock"])(!0),Object(a["createElementBlock"])(a["Fragment"],null,Object(a["renderList"])(e.options,(function(t,i){return Object(a["openBlock"])(),Object(a["createElementBlock"])("div",{key:i,id:"option-".concat(e.id,"-").concat(i),class:"option",role:"option","aria-selected":t===e.modelValue,"data-selected":t===e.modelValue,"data-highlighted":i===e.highlighted,onClick:function(o){return e.selected=i},onMouseenter:function(o){return e.highlighted=i},onMousedown:o[3]||(o[3]=Object(a["withModifiers"])((function(){}),["prevent"])),onFocusin:o[4]||(o[4]=function(){return null}),onKeydown:o[5]||(o[5]=function(){return null}),tabindex:"0"},Object(a["toDisplayString"])(t),41,os)})),128))],8,es)):Object(a["createCommentVNode"])("",!0)])}var is=Object(a["defineComponent"])({props:{name:{type:String,required:!0},modelValue:{type:String,required:!0},options:{type:Array,required:!0}},data:function(){return{id:Object(B["uniqueId"])(),expanded:!1,selected:this.getSelected(),highlighted:0}},methods:{open:function(){this.expanded=!0,this.highlighted=this.selected},close:function(){this.expanded=!1},onClick:function(){var e,o;this.expanded?this.close():this.open(),null===(e=document)||void 0===e||null===(o=e.querySelector("#select-".concat(this.id)))||void 0===o||o.focus()},onBlur:function(){this.close()},onKeydown:function(e){if(["ArrowUp","ArrowDown","Home","End"].includes(e.key)){e.preventDefault();var o=this.expanded?"highlighted":"selected",t=this[o];"ArrowUp"===e.key&&t--,"ArrowDown"===e.key&&t++,"Home"===e.key&&(t=0),"End"===e.key&&(t=this.options.length-1),this[o]=I(t,0,this.options.length)}!this.expanded||"Enter"!==e.key&&" "!==e.key||(e.preventDefault(),this.selected=this.highlighted),this.expanded&&"Escape"===e.key&&this.close()},getSelected:function(){var e=this;return this.options.findIndex((function(o){return o===e.modelValue}))}},watch:{modelValue:function(){this.selected=this.getSelected()},selected:function(){this.$emit("update:modelValue",this.options[this.selected]),this.close()},highlighted:function(){var e;null===(e=document.querySelector("#option-".concat(this.id,"-").concat(this.highlighted)))||void 0===e||e.scrollIntoView({block:"nearest"})}}});t("8684");const rs=ae()(is,[["render",ts],["__scopeId","data-v-298388da"]]);var ns=rs;Object(a["defineComponent"])({components:{AppStatus:Io,AppTabs:gi,AppSelectSingle:ns,AppSelectMulti:Xi},data:function(){for(var e=[],o=0,t=["normal","circle","small"];o<t.length;o++)for(var i=t[o],r=0,a=["primary","secondary"];r<a.length;r++){for(var l=a[r],s=[],c=0,p=[["Text",""],["Text","download"],["","download"]];c<p.length;c++){var d=Object(n["a"])(p[c],2),h=d[0],u=d[1];s.push([i,l,h,u])}e.push(s)}return{buttons:e,singleSelectOptions:["apple","banana","cherry","durian","elderberry","fig","grape","honeydew"],singleSelectValue:"durian",multiSelectOptions:[{value:"fruits",count:0},{value:"vegetables",count:7},{value:"colors",count:42},{value:"animals",count:999},{value:"cars"},{value:"schools"},{value:"appliances"}],multiSelectValue:[{value:"vegetables"}]}},methods:{alert:function(){window.alert("alert")}}});var as=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(){var o,t=arguments;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return o=t.length>0&&void 0!==t[0]?t[0]:0,e.abrupt("return",new Promise((function(e){return window.setTimeout(e,o)})));case 2:case"end":return e.stop()}}),e)})));return function(){return e.apply(this,arguments)}}(),ls=["anatomy","case","cell-line","disease","function","gene","genotype","homolog","interaction","model","ortholog-disease","ortholog-phenotype","pathway","phenotype","publication","variant"],ss=function(){var e=window.sessionStorage.redirect;if(e)return delete window.sessionStorage.redirect,e},cs=function(e){if(e.fullPath.startsWith("/disease/Orphanet:")){var o=Array.isArray(e.params.id)?e.params.id[0]:e.params.id;return{path:"/disease/".concat(o.replace("Orphanet","ORPHA"))}}},ps=[];var ds=[{path:"/",name:"Home",component:dr,beforeEnter:ss},{path:"/home",redirect:"/"},{path:"/explore",name:"Explore",component:mr},{path:"/about",name:"About",component:Br},{path:"/overview",name:"Overview",component:Tr},{path:"/sources",name:"Sources",component:jn},{path:"/cite",name:"Cite",component:$n},{path:"/team",name:"Team",component:Pa},{path:"/publications",name:"Publications",component:qa},{path:"/terms",name:"Terms",component:Ol},{path:"/help",name:"Help",component:zl},{path:"/feedback",name:"Feedback",component:Wl}].concat(Object(i["a"])(ls.map((function(e){return{path:"/".concat(e,"/:id"),name:"Node ".concat(Object(B["startCase"])(e)),component:Zl,beforeEnter:cs}}))),Object(i["a"])(ps)),hs=function(){var e=Object(Ke["a"])(regeneratorRuntime.mark((function e(o,t,i){var r,n,a,l,s;return regeneratorRuntime.wrap((function(e){while(1)switch(e.prev=e.next){case 0:return e.next=2,as();case 2:if(!i){e.next=4;break}return e.abrupt("return",i);case 4:if(!o.hash){e.next=11;break}if(n=null===(r=document)||void 0===r?void 0:r.getElementById(o.hash.slice(1)),!n){e.next=11;break}return l=n.parentElement,"SECTION"===(null===l||void 0===l?void 0:l.tagName)&&n.matches(":first-child")&&(n=l),s=(null===(a=document.querySelector("header"))||void 0===a?void 0:a.clientHeight)||0,e.abrupt("return",{el:n,top:s});case 11:case"end":return e.stop()}}),e)})));return function(o,t,i){return e.apply(this,arguments)}}(),us=Object(ii["a"])({history:Object(ii["b"])("/"),routes:ds,scrollBehavior:hs});us.afterEach((function(e){var o=e.name,t=e.query,i=e.hash;Object(a["nextTick"])((function(){var e="string"===typeof o?o:"",r=Object(B["lowerCase"])(i.slice(1)),n="";t.search&&(n='"'.concat(t.search,'"')),document.title=["Monarch",e,r,n].filter((function(e){return e})).join(" - ")}))})),us.beforeEach((function(){Object(ri["b"])()}));for(var bs=us,gs=t("5502"),ms=Object(gs["a"])({state:{},mutations:{},actions:{},modules:{}}),ys=[bs,ms],vs=ys,fs=(t("52df"),function(e,o){e.getAttribute("aria-label")||e.setAttribute("aria-label",(e.innerText+" - "+o).toLowerCase())}),ws=function(e){var o,t,i=arguments.length>1&&void 0!==arguments[1]&&arguments[1],r=null===(o=e._tippy)||void 0===o||null===(t=o.popper)||void 0===t?void 0:t.querySelector(".tippy-content");r&&(r.dataset.break=String(i))},Os=function(e,o){var t=o.value,i=o.modifiers;t&&(Object(ri["a"])(e,{content:t,delay:100,duration:200}),ws(e,i.break),fs(e,t))},As=function(e,o){var t=o.value,i=o.modifiers,r=e._tippy;t?null===r||void 0===r||r.setContent(t):null===r||void 0===r||r.destroy(),ws(e,i.break),fs(e,t)},Ms=function(e){var o=e._tippy;null===o||void 0===o||o.destroy()},js={mounted:Os,updated:As,beforeUnmount:Ms},ks=js,Bs={tooltip:ks},Ds=Bs,xs=(t("beb9"),t("9788")),Cs=t("2db1"),Is=t("c246"),_s=t("8670"),Ts=t("99e5"),Ns=t("9ae1"),Ss=t("26df"),Rs=[Cs["c"].get(/metadata\/datasets/i,(function(e,o,t){return o(t.status(200),t.json(Is))})),Cs["c"].get(/obo.+ontologies/i,(function(e,o,t){return o(t.status(200),t.json(_s))})),Cs["c"].post(/api\.uptimerobot\.com/i,(function(e,o,t){return o(t.status(200),t.json(Ts))})),Cs["c"].post(/monarch-gh-issue-post/i,(function(e,o,t){return o(t.status(200),t.json(Ns))})),Cs["c"].get(/search\/entity/i,(function(e,o,t){return o(t.status(200),t.json(Ss))}))],Ps=Object(a["createApp"])(zo),Ls=0,Es=Object.entries(Ds);Ls<Es.length;Ls++){var Vs=Object(n["a"])(Es[Ls],2),zs=Vs[0],Hs=Vs[1];Ps=Ps.directive(zs,Hs)}var Fs,Gs=Object(r["a"])(ti);try{for(Gs.s();!(Fs=Gs.n()).done;){var qs=Fs.value;Ps=Ps.mixin(qs)}}catch($s){Gs.e($s)}finally{Gs.f()}var Ws,Js=Object(r["a"])(vs);try{for(Js.s();!(Ws=Js.n()).done;){var Us=Ws.value;Ps=Ps.use(Us)}}catch($s){Js.e($s)}finally{Js.f()}for(var Ys=0,Ks=Object.entries(Yt);Ys<Ks.length;Ys++){var Zs=Object(n["a"])(Ks[Ys],2),Qs=Zs[0],Xs=Zs[1];Ps=Ps.component(Qs,Xs)}Ps.mount("#app"),"true"===Object({NODE_ENV:"production",VUE_APP_TITLE:"Monarch Initiative",VUE_APP_TITLE_SHORT:"Monarch",VUE_APP_DESCRIPTION:"The Monarch Initiative integrates, aligns, and re-distributes cross-species gene, genotype, variant, disease, and phenotype data.",VUE_APP_URL:"https://monarchinitiative.org/",BASE_URL:"/"}).VUE_APP_MOCK_API&&xs["a"].apply(void 0,Object(i["a"])(Rs)).start()},cd5a:function(e,o,t){e.exports=t.p+"img/oregon-state.50dd4211.png"},cd5b:function(e,o,t){e.exports=t.p+"img/zhou-joe-yuan.38f232c2.jpg"},cf86:function(e,o,t){"use strict";t("e735")},d025:function(e,o,t){},d071:function(e,o,t){e.exports=t.p+"img/kent-shefchek.864e3f05.jpg"},d08c:function(e,o,t){"use strict";t("43fc")},d0b7:function(e,o,t){e.exports=t.p+"img/berkeley.38a13b77.png"},d0ee:function(e,o,t){"use strict";t("1a7a")},d154:function(e,o,t){e.exports=t.p+"img/clinvar.6b4ca139.png"},d347:function(e,o,t){e.exports=t.p+"img/omia.c03df1df.png"},d3ae:function(e,o,t){"use strict";t("83ef")},d57f:function(e,o,t){e.exports=t.p+"img/impc.4097d519.png"},d78b:function(e,o,t){e.exports=t.p+"img/hpo.81d8b869.png"},d8df:function(e,o,t){"use strict";t("2dc1")},da13:function(e,o,t){"use strict";t("940f")},da19:function(e,o){e.exports="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAYAAAA7MK6iAAAEJGlDQ1BJQ0MgUHJvZmlsZQAAOBGFVd9v21QUPolvUqQWPyBYR4eKxa9VU1u5GxqtxgZJk6XtShal6dgqJOQ6N4mpGwfb6baqT3uBNwb8AUDZAw9IPCENBmJ72fbAtElThyqqSUh76MQPISbtBVXhu3ZiJ1PEXPX6yznfOec7517bRD1fabWaGVWIlquunc8klZOnFpSeTYrSs9RLA9Sr6U4tkcvNEi7BFffO6+EdigjL7ZHu/k72I796i9zRiSJPwG4VHX0Z+AxRzNRrtksUvwf7+Gm3BtzzHPDTNgQCqwKXfZwSeNHHJz1OIT8JjtAq6xWtCLwGPLzYZi+3YV8DGMiT4VVuG7oiZpGzrZJhcs/hL49xtzH/Dy6bdfTsXYNY+5yluWO4D4neK/ZUvok/17X0HPBLsF+vuUlhfwX4j/rSfAJ4H1H0qZJ9dN7nR19frRTeBt4Fe9FwpwtN+2p1MXscGLHR9SXrmMgjONd1ZxKzpBeA71b4tNhj6JGoyFNp4GHgwUp9qplfmnFW5oTdy7NamcwCI49kv6fN5IAHgD+0rbyoBc3SOjczohbyS1drbq6pQdqumllRC/0ymTtej8gpbbuVwpQfyw66dqEZyxZKxtHpJn+tZnpnEdrYBbueF9qQn93S7HQGGHnYP7w6L+YGHNtd1FJitqPAR+hERCNOFi1i1alKO6RQnjKUxL1GNjwlMsiEhcPLYTEiT9ISbN15OY/jx4SMshe9LaJRpTvHr3C/ybFYP1PZAfwfYrPsMBtnE6SwN9ib7AhLwTrBDgUKcm06FSrTfSj187xPdVQWOk5Q8vxAfSiIUc7Z7xr6zY/+hpqwSyv0I0/QMTRb7RMgBxNodTfSPqdraz/sDjzKBrv4zu2+a2t0/HHzjd2Lbcc2sG7GtsL42K+xLfxtUgI7YHqKlqHK8HbCCXgjHT1cAdMlDetv4FnQ2lLasaOl6vmB0CMmwT/IPszSueHQqv6i/qluqF+oF9TfO2qEGTumJH0qfSv9KH0nfS/9TIp0Wboi/SRdlb6RLgU5u++9nyXYe69fYRPdil1o1WufNSdTTsp75BfllPy8/LI8G7AUuV8ek6fkvfDsCfbNDP0dvRh0CrNqTbV7LfEEGDQPJQadBtfGVMWEq3QWWdufk6ZSNsjG2PQjp3ZcnOWWing6noonSInvi0/Ex+IzAreevPhe+CawpgP1/pMTMDo64G0sTCXIM+KdOnFWRfQKdJvQzV1+Bt8OokmrdtY2yhVX2a+qrykJfMq4Ml3VR4cVzTQVz+UoNne4vcKLoyS+gyKO6EHe+75Fdt0Mbe5bRIf/wjvrVmhbqBN97RD1vxrahvBOfOYzoosH9bq94uejSOQGkVM6sN/7HelL4t10t9F4gPdVzydEOx83Gv+uNxo7XyL/FtFl8z9ZAHF4bBsrEwAAAAlwSFlzAAALEwAACxMBAJqcGAAAActpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IlhNUCBDb3JlIDUuNC4wIj4KICAgPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4KICAgICAgPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIKICAgICAgICAgICAgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx4bXA6Q3JlYXRvclRvb2w+QWRvYmUgSW1hZ2VSZWFkeTwveG1wOkNyZWF0b3JUb29sPgogICAgICAgICA8dGlmZjpPcmllbnRhdGlvbj4xPC90aWZmOk9yaWVudGF0aW9uPgogICAgICA8L3JkZjpEZXNjcmlwdGlvbj4KICAgPC9yZGY6UkRGPgo8L3g6eG1wbWV0YT4KKS7NPQAABa5JREFUSA2tVwtQlFUU/qTlJbKQ6+JCggsqKEKKD3ygIqmhUGljliUqM040mTqpafZuspRyshwdRxunccakKdPwlZVGhamAEyoPcZmUFVZdXFBYYHfZXbR7Lt5/n9KOeWb+/9x7zrnn3PO49z9/r7sM8ABgsthQWt2AWl0z9E1tuNXagdttJrS2m9Fh6oSZ8VMSY7Bt9TNetft5pfpA7B3kj7QRAzFMrbyvdMn5y9h58KxX/gMbJm0BskcwZYQai7JGQt4n2KuB3QdL0Nph8eD9L8NCW2xkX7y1KB3Jg6MEScImFvavCj29fiiGyQqFfs2CycjJGiMZFYN9v5SLoYQfmmGhcV5GEjYsewrBbCMCqNgOn7okphz36qmqtfoW1De2uCyI6R8OtSrchVZ8QYuqK42Q9w5E8iAVRgxWoVrbiFc37Eer0QS7/Q6mpw3FlpWOCpe5aHCbfHe8ArsLT7tRgXfyZiIncyQu/KPH8s8OwNBsdJF5clIiN7J93VwseHsP5527pHOR8SnU6mgldr4/HxNTBvHFe4/9zfG6bUe5UcFfnpPRbaRGB4pWUlx/LH0xndOa2OZut5n5mF49eiyk+gQH8WOzv6hCkLi32gYDn+cvy+bhpaO19NlxkgwNXpk9FkVltajUNKCSpYNkCHzyuKq2AcPm5uPXvy7yRVNHD3E5m5RTyjPJiOfTPcVcll6vPZ/Gx/UsCgJ88lipkCN7chJfM3NCPPeO8iuAxlR0uXMm4ujJKo+ck5f9+spx7aajFnwy3F8RhjcXThF2OCYvKbcUbsr10nlpeCxCDpO500VOTNLHxIshxz4ZdlnhNKHcUlWT8bWbCyVOUnw0XpjxuDSnQaQyFG3tjk31aDg7JhwJE+IQqVJISoylF2G71R2y4amJOLJkKopYeHUGIwYo5Vw2eewQ9HY761ThZyrqJT09Gg48VIyQzQXwSx0O5EyDbtt+1Cz/nC+OfO4JKGaNhybrC4SUVSPhnkra0in2DPkoD+r3cu9RPZFPVU3LyFNno0n7PnbRRhsZ9dNm9JuRyumGI2TeAc7eEtUnwxbdTZRnvM61hCQMhLtRh3rHqE9y92UjKLVXG8WQYz/qJP4LLNcNsJnbuViH5iqaj5V4LLnxQxHKs1ah6XgZ5/XNGC3J0C12hjUFVPUC/H4/V8cveKu9S9C8YgplOOWagWblFg8Z9aqXMOPuaQzbuprzKnI+gEnTXUzrdx3ntBinguOhrq4z4PvfKvkXxUMjIwRFKTF0x1rEfbiEs8lr7frdLqKd9XoeiZY/z0l0e0s7Ck5cQMn5Ok5LZpUtQMqxxWpH0dk6bP+xFOWaG4LPcdCACMjY7UVVLLzWbiyA3emrJEJNmIAiVK9UYGtBMZ/HRkfg0VBHe9TryGnN3XaTFW0mC+sM7TBZrGg3WxEeGojMqDBEmkzwZ9edfFwiV0DhM1+5zsfBcVEgr8S55kT2InpdYOCDfY/1t9qx4/ptREeEIztBge6bmrU4CTH8EUa8Yeo28r8+hE6ro3Cnp7pdmQEyKdredOCyrgmbvtEjMVaFxVkpUClCvcoJIqXqwInzYspxCOtMnmYdiDPI0lNiUaM1oPKynofamek8pg6ipEqL6akJWDgrhbU5Ac5sliIbNu09ifJL9S50mszLHOVBk3ouMyuu0iod+zuo5zm22Gz8b8DK6J029nSyx25n/VMXZKyfXjF/EjJS4rjCZqMZ+Xv+QGOT0eVPgkLt7y/Dz1tfRlhIkItxKc7BATJMHaVG3pyxGBkf6SLkPqEC/ISdzbyNB2BkhamQB7NURLiL8Xnu7PEeRokheey+qrahGYdZx6FhaXD3mC6bLvYQpr+JNYunITN1MG9hdxWW+PTvdF/DYiNlNdfwLWvIbxjapFA7G+5iresd9t+XPSUJ7+ZmgDb8xpeHoOoXhk0rsrx6S7r/Bam2cdsasjADAAAAAElFTkSuQmCC"},da68:function(e,o,t){"use strict";t("867a")},db89:function(e,o,t){e.exports=t.p+"img/vincent-rubinetti.38745f2b.jpg"},dd49:function(e,o,t){e.exports=t.p+"img/ohsu.9f08625e.png"},df54:function(e,o,t){e.exports=t.p+"img/vida-ravanmehr.0436aac7.jpg"},dfd0:function(e,o,t){e.exports=t.p+"img/ctd.cf0737f3.png"},e443:function(e,o,t){"use strict";t("5762")},e6ee:function(e,o,t){e.exports=t.p+"img/category-ortholog-phenotype.283fb951.svg"},e735:function(e,o,t){},e8a7:function(e,o,t){e.exports=t.p+"img/anne-thessen.f4f0b98b.jpg"},e93b:function(e,o,t){e.exports=t.p+"img/geneontology.e7751a99.png"},e97b:function(e,o,t){e.exports=t.p+"img/jackson.aa5515c2.png"},ea31:function(e,o,t){},ebfc:function(e,o,t){"use strict";t("62f8")},ec4d:function(e,o,t){e.exports=t.p+"img/queen-mary.6375a3ba.png"},f05d:function(e,o,t){e.exports=t.p+"img/category-ortholog-disease.157285b2.svg"},f409:function(e,o){e.exports="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHMAAAAkCAYAAAC6yAWpAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1wIFEzU1ZBM+2QAAAB10RVh0Q29tbWVudABDcmVhdGVkIHdpdGggVGhlIEdJTVDvZCVuAAAG6ElEQVR42u2bW0hUXRTH19yy6eKl7AIFmoJmF6ULQRSEBV2goiCsh7GnkrBIoYbIGfKhkpKiqB4EkV4GpAslomTYgxLVR9QwZnmZJouJDKmmsZnG5uL5f08zjJ09t3PO3L7PAwscnb3+a+/fdp991lmbAJDf7HY7ERHibW1tbQjWTYSm36TQnD9/fsTvyGQyPHz4cEo/Y7WYYwtu3N/fH/fBPHLkCLODLpeLjEYjjh8/DrlcHneYfs3Xr1+H1ZTL5Thz5gyGh4fh8XgIAHm9Xvrx4wf19vaitrYWWVlZzLaZmZkYHh4WBdTj8ZDZbIZer4dCoeDFNTQ0FIhrSsPOzs7Al3NyctDQ0ICXL1/i+/fv5G8QjdlsNlq6dCmvc4WFhXA4HBHbX7p0KSKIUB232WxkMpnQ1NSEkpKSqH2E0tTr9YimvwcOHGC2X7lyJZxOJ4kB6rf6+vqwcU350NTUBCJCcXExRkdHVwkVPXjwIK9TCoUCz58/j2qWms1mQTD/Nq/XS6dPn47KRyhNk8kUlS7HcaTVapk+Dh06BClgGo3GgM++vr7wMOvq6iCXy/HmzRvB4gaDgdkhnU4XtU+Hw0FSwPRbdXV1RB+hNO12uyT3uWvXrokG+vPnz4D/8fFxCguzsrIS5eXlgkWtVuvF7OxsXkfWrVuHWJZpn88nKUyHw0ELFiwI6yOUps/nkwSmUqnE06dPRQH1er0ULq4pH7Zs2YLGxkZBghzH0datW3mdUKvVGBwchBQ7OTEDodPpIvqQQtPfbuHChTxfixcvFnX7CvbP/Fvwh4KCAjx79kzQoF29epU5I2/evAmptuViBuHdu3cJhdnT0wOlUsnzt3nzZni93vjDrKioELTz6u/vR0ZGBi/w7du3g+M4SgWYidIMbnf9+nXmBK+pqUHcYQoxt9tNZWVlvIDnzZuHL1++bE3mwCYbJgDSaDRMoK2trUg5mKG243fv3kWyBzYVYLpcLlqzZg3P7+zZs/H27VukDMze3l5m5qSyshLJHNixsTESA0JKmADo48eP/7BSgEVFRcxHjITDHB8fp7y8PF6AeXl5MQco9cB2dXUhlWACoO7u7inpOL/t27cvpn1FXGAePnyYmcfs6elBspe8Y8eOpRxMANTY2Mi8JTU0NCBpMO/fv88MSqvVItn3r4GBAajV6pSEGS7V+eTJEyQc5ujo6CrW+l9aWgq3252UzcifP3/IYrHgxo0byM3NFQ0injCdTietXr2ap5Obmwur1XoxoTB37tzJCyQjI0NUPleSd3kSgognTABksViQk5PD09qwYUPEfwjJYN66dYs5aFeuXEEiHhP+KzAB0KNHj5hPAlVVVYg7zKGhocC9KNjKy8sFZ3mkGliXy0VWq/WiwWBAcXFxWsAEQBcuXGBOwtu3byNuML1eL61fv54nmpWVFdU6n8gNkMPhoOXLl6cFTI7jaP/+/TzNmTNnwmg0Ii4w9Xo9cwYZDAakYjamtbU1LWACoF+/fhGrImLZsmWw2WzSwnzx4gXzYbeiogKpmlr7/PlzTbrA9N/CMjMzedq7du3C5OSkNDCdTicVFhbyRJYsWcKcNakC0+PxUDrBBEDt7e2QyWQ8/XPnzkESmEePHmWWEHZ3d6dF0judYP5dsBU83h0dHRAFs729nXmfPHnyZNwH9f8Kk+M42r17Ny+G7OxsfPjwwSkI5tjYGLHKHlasWIGJiQlRA+bxeOjx48eYhsk2u91ORUVFvDjKysrgcrlih7l3716eM5VKFXK7HIt1dnYiPz9/GmaEEpc5c+YwXy3GBLO5uVl0Zj/S25Y9e/ZMwxT4MiNqmBaLhTkjNm3axNsiC7Fv376RWq1GXV3dNMwo7OzZs8Jg+nw+2rhxI6/R3LlzMTIyMiJFcP7kQ6T6l+D60GCTYkLFUzNSXWusNjk5STt27Igd5vnz55mNWlpaJJllAwMDgQq+SLUvoU6jSXVmIxbN379/C/IhRbWF/xxLQUFB9DBfvXoFlUrFLGmQCmR+fj6ICDNmzIhYN9rX18ecWGazOW5LbShNi8UStabJZAp7FkRMbLNmzYoIUz4xMUEajca/RASuRYsWUXNzs4wEXG63m75+/bqqq6sLVVVVWLt2LX369ImIiEpKSkipVIZt39bWxvz9gwcPKF6XFJr37t0L/NzR0SFZbKWlpbKWlpbIXzxx4kTCDroSETQaDcKdQ/QfXgp1VrK6uhpGo1H0826wpk6nY+afo9H0eDw0ODgIrVY7JR2nUqlQX1+P9+/fi6pgD7ZTp06FX2ZZ+cB42uXLlyHli+hkvPwW6kMsTJ/PR9u2bQvpS8lxnIySfAGQpaNmouNWKBR0584dWW1tLVh//xfjU3zG0QOJpwAAAABJRU5ErkJggg=="},f89b:function(e,o,t){e.exports=t.p+"img/sabrina-toro.99cf072b.jpg"},f919:function(e,o,t){"use strict";t("8af4")},f99d:function(e,o,t){e.exports=t.p+"img/kegg.5e5a8851.png"},fccd:function(e,o,t){var i={"./animalqtldb.png":"499e","./bgee.png":"0b8f","./biogrid.png":"5adf","./clinvar.png":"d154","./coriell.png":"fccf","./ctd.png":"dfd0","./decipher.png":"9808","./ebigene2phen.png":"9b7d","./ensembl.png":"a404","./eom.png":"c587","./flybase.png":"4c96","./geneontology.png":"e93b","./genereviews.png":"5dac","./gwascatalog.png":"14a5","./hgnc.png":"3dd4","./hpo.png":"d78b","./impc.png":"d57f","./kegg.png":"f99d","./mgi.png":"5b33","./mmrrc.png":"6aa4","./monarch.png":"675f","./monochrom.png":"4031","./mpd.png":"fda9","./mychem.png":"62eb","./mydrug.png":"2f9c","./mygene.png":"b499","./ncbi.png":"9318","./omia.png":"d347","./omim.png":"6e7f","./orphanet.png":"2634","./panther.png":"be35","./pharmgkb.png":"da19","./reactome.png":"fdb5","./rgd.png":"9184","./sgd.png":"3b32","./string.png":"4354","./ucscbands.png":"58ab","./udp.png":"1f1c","./vega.png":"3b48","./wormbase.png":"6da3","./zfin.png":"f409"};function r(e){var o=n(e);return t(o)}function n(e){if(!t.o(i,e)){var o=new Error("Cannot find module '"+e+"'");throw o.code="MODULE_NOT_FOUND",o}return i[e]}r.keys=function(){return Object.keys(i)},r.resolve=n,e.exports=r,r.id="fccd"},fccf:function(e,o,t){e.exports=t.p+"img/coriell.09236315.png"},fd9b:function(e,o,t){e.exports=t.p+"img/category-disease.0caa37d5.svg"},fda9:function(e,o,t){e.exports=t.p+"img/mpd.e83603f9.png"},fdb5:function(e,o){e.exports="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAASkAAABGCAYAAACQTGK5AAAACXBIWXMAAC4jAAAuIwF4pT92AAAPq0lEQVR42u1dzW8bxxX/LT/1ackxbSUFDNEoCgQFLG3QY0CIvvlQIMq9qEnAyDX0qcfQf0Hoq2GA1B9QhLrlUoQCL0GaNku5KIrGQcg6sC2LsbiSRUqiyO1BQ3s8Wu7nkCKl9wMMGzQ5b/a9N795b3bmjWIYBs4Cm9t7yaWrsyUQCASCBUJnQU4ACgCqAJJkAgKBMBIktbm9F2fktMI+qpL6CQTCmZPU5vbePIAMgC9I3QQCYaRIanN7LwUgB2COVE0gEEaGpNi6Uw7AMqmYQCCMDEmxdaccgE9ktPdo82kSwPzdpetFMhWBQCTlh5x6604ZGando82ncQBZAHcA3AdAJEUgEEl5JqgUI5RFSdFTVhbZEQiEC0xSbN0pi7dbCvyS0ypLFRfJLAQCwTNJsdQux1IxGeSksvZWyBwEAsEXSW1u70lLxY67RujR5tOCLLIjEAgXmKQ2t/ekpmI/7uwf/LTTjL83Gf7Yb1ufff14nqWdjYe3b2bJpATCBSIpk6MsvlBvHuHb5w293enOhQLKsQSCyjCCmsPJW0ACgXARSIqtO2UBfC5DSLPdwbfPdur64XFMRqr42dePk6DNogTCxSSpze09PjrxhXa3i8fbe/Wa3ooBiEkgpzgkbhYlEAhjRFJcCRUp605VvdWqvNxVuoYhg5zokDKBcFFJiqV2BVnRSb15hO9f6PXWcScmo70nO/t/wElZF9+RnZovzwOAlk40yPTnC8y2qwBKWjpRJY2cHwQAqDIIqtnuoPzLq3r5l1eQQVD15hH+Vqtv/aw3lyURVAqAxp6XcP5QAJAH8HNvMiKcs3TPK9rdLv77al9/srMf7hr+152a7Q42t3e3nr8+XACwMBMO1nySE20WvRjgJ1oVQIlUQiSFqt5q/au+d9TudKUssnNktyAp/Je2M55AIIwRSdWbR6hs727tHh4vAJj024lnrw+a/9zabcsgO0ZQWdAhZQLh4pFUu9vFP17ob1Ixv8L1wza+f6H3yE4GOdEhZQLhopLUs9cHzb8/15WuYfgmlHa3i8rL1+HDThcyyO6w08V/fn39J9AWBQLh4pJUvXm03TUM3xHKjzv7B/+uvzYCCtoL01FfbXW6Bv6316pvN49iAH5L5iQQLni65wf8uT0ACCiKr/ae7x8e/LLXMgwJbxQt0sd4vz03ar4cBxDH27sDqwA0LZ3QXMro/T6Ot9d8VQe114e9UFCZvDj7WAPQ0NKJ0qD9gNPbO888DNmS+68C6G11KDH9abJsZLaXj9kuiXe30ZTc6I71XxXbYL7bGEWbK5WXu0kA39h9cfPlbu2nRtN1JCWc23vLjgGltjAdtW1vJhyszUXDb763e3SMJzv7+nHX8LoofsuJctR8uYiT19rrWjqxKpBKFv23NDzQ0omMA2LKwHp/Wg0nZZNzfgmL2+iYgf15x3UARS2dKEh20hT7syhDNttawu+H4n34HiNeEZ6IhPU/w3Ro1X+ds5nmUVcl5ltrWjqRcuhzALAGIGsxqTppY521oUmyeZbpbM5GZs5qTA6MpIRze6dDOJckddjp4udGs757dOw3cnJKUm/un9fSCcXFloYNLZ1IWhiuAPd7tu5r6UTWo7N4fZlQA5DyG+F4fNNqKVvNlwvwvrXEdhIRyD0Lbwft1wFk3E4wJn6XcyFfZzILwjO43YpzT0sncj4mRC8622A2rw6FpLhzexN980yHJDUZCjzdb3cuPX99KGs7gWuSAnCZhcTLgkNoeLtpsEdMWbP2GVkUTAZrjbXTm716obiomwqApNOQ3MY5K70UhQvB1T5RluNBbSK/aELIot6sZJsOFjVfrsL7G9x3ImObSK1oIkfsv5XNdDbwil5IikVHd0z8hR/IZhNeWksnCswGJRPdim2oJn7p2u4W8pzaXGc+rg2UpB5v72092dm3fWPnhKT2252WftBWDGBCEkHVmBKqLp1lnUvNKoyI3DiemY4tQ+s+4bkjourjLDojrVy/33NpmRj5vEk9fDirpd4sZJ+KIhmBZLh1DpjoyewZq05SMdZ+SehHjfW/YGNns5Qq7TR9FvzONjJjujY7fP8R+/yOEK30m0TNqp986tTPfdg8K/TxFFFJJymn37MiqcNOF/pBe6vdNRYkkZPOnDPrYqCZOYurwcoZr8oZ/1RIbvP7DIAv3UQCJulQ31DawnmKgsO5SZNE+W5+aybbNvoV7HXLa5pqYi/X6XafqPkjJ2s9ffzOluRMiLUmRHa2vtunDdVJ9M6tpbkeK4zci/3kBjBC6HQN/No62qo3jyCRoNYAxL2u6fghKAbxmvlVN4vSLN1Jcx99wr0RNDN4SiCINS2dSLpZG2HfTbKZsIfPreQKDndHGOAZn7KHeSi8KNgr7dZ3WOSQZBPSm3Y9Hny+58RfGAHyel4UJraUwzZWhTacpMYprwTF5JYEfS2yCAsjQ1Jdw8Du4bG+tX/YPDjuyiKnDTZ7pSS8WtUFB3A6o6omA9b1DM+cdJ37KGuzDvUm3PZIrL1yNkk2q/XghFz5vm14mRw42feZw+eG4YeMYFcEexU86s9swLv1oZqbZ2d93TD5r4xLwlhz+VvR5imP+soKk2J8JEiqddxpbu0f6XtHx3MGMCVp3SnNogdNUjcLHokuIzicn2iOb2uFEaCIlBi1+Xlo9sy8wy2yVMbpIE/5ka2lE1mvJOsRvgnWZMA/4G3oMpryIl8k1XUPW1j4Npat+sz8YdELIfbJGiqi/54ZSRlA5OX+4darVnuqaxgy3tzpbOZVZe7x4VIAL1gV0j4/Dl8VoqnVPiTFh9xVvw/OBtqGQ+JZFQZ5FWMCNmuv+CQIO6KZczlxaBJ8teTR5nCYbos29xsY5EaCpMIBZSscCARkrTtdioTqv78yU2Az7yB2zXrZACi+1i1K6AffRtIk1VuWRYoWjmO1LpV0mRqOElaFqLckxXFO/HHNof7MUiAv8vwSHfqkjcOwOe/jK0MnqYCi6PMT4da16ehCKIADv+1FgoH6h1dm8OGVmdhMJDSwksAeiS8uOL2MqIJ3uHmL2U6XmOqKs/Fcn1QTAklqGC+okieUfvobxkuADcntWRHrokybs7FW4yf7oZ3dm42E9JlIcM7vmT0AUBQcxC9NGVenIrExcfp59opWJpYtHEkqQWjpREPNlyuczPk+6ZKvKOCMwfdfdppatbDb2MJkssqp+bKMpnn/mh84SUWDgfrliXAsGFCk7Bj/YCaq/2Z6Yi4YUMbJnnMYbvnixhm0Gcf5gTbi7Y0KxMlqID4+sHQvqCj12FQEsalITAahTIWD9eVrl3B9dnLcCGpQoBubx4Xx6HYir6ho6URJeiQVCCjT8xPh1nQ4KCUVCyrKzu/em758KRKKjbGy+x46HtIMN4w2q+docEi9yMHJJtgxRUMg44FED1IjqalwsP7+dLQ1HQ5OymozNhXZvRQJjaMBtSGnQiVhkMnGsk20ULVZrxh1VAdoL769yjmKELVh2FwKSYUDSv396SguT4RjCtClKPUUSS2KC8sDljcn02GsNnCK4Tn37+QY22tVctvJPnLORUo2aJv7IqmAouixqQiuTUdjtE5kGlnwR0pSA5bXEBwmI7F5p30vDet5TaD7/H1RmFSkDDiu4KCZjs4DBm5zTySlAK3ZSEj/YCY6Fw0GQOiLAk8aQ7hZl5d3R0b0xgar0xuu+U2fy0Nei9H8zOhsUuH3F2Ul9YsvPaND/h6sUfLxgdjcNcNMhAJb789EJy9FQ3SnnTsDzkHuLvB+8vQ+0YHXKCDnY6AXhnjleVVCusYT0work+NHfyrerfOUO29v+ti61EBt7pikggFl79p0BFcmIwsyNmQGFakRmD6KuT4btPeF6CbllTDs1pnYAMgIM1vBB0EV4H7zIf98i/BeosTtQmxReO6UB3uVhAH3pQ97iW8Ia0OYpM4KWYk2T3ohqRqAT/fbnb+GA/6JJRxQtmKTEUyEpJHUGgD14e2bIxlGs5P0/FpRntWtdmO4FIsUfmA1w63kFfDuWbE7ar5ccuM0LE0sCWlezQUx3+MjEgAlN6mnmi/H1XxZY89bcii3KPQx52LBH0IUpgv2yrq01ypOV/ZcPa/7pUyqPXixucps/o1o84BNdHIfgLp0ddY3AQQURZ+PhlvXpqILktaxKgBuPbx9M/Xw9s3qiNsxKTj+52q+XFXz5ZRdGQxmsDzn8E4MnxGIcQVAVc2Xs1aOw8ghy6JSPoJKw8U+KFZygyfKZQAakz9vEy2K8t1EUykhvf5KzZcLbsiKq2XF2+uLnr3sogBmr69wunDeed113tNbxsLmcRtyKgD4oZ/NQxbRSXbp6qyUwT8bCekzYTnn9pjzZB7evlkYIwM2uOhkmQuL82ym3mADs4GTTZMqzI8YVODgrR2Tl8S7FzHMsfWRL9g5vBKcXcTQK+qfcvnMKTVfbuDtrSG8fP557eS7Ktim5stpptc3kSSLJnn/yVoVk9PSCY3pryDai0XBA7uIYcyJKsX03M/nNG6ySzK7m+rMiqQ2GDmVZHQ6GgzUL0flndtjkV3u4e2bjTE0YAOAyiIFsWj+CqzPPbmu0d4rWMfuDxSvtFp2sNa0gZNa7JqPZ84w+QVBvt3zepbPCLUB85t5egMnY7c+xBFVFu9ezzTnsP+errQaUTRcEpVXnzO9OzDADYL00tXZpF+CMoBAUFHqsckIYpMRWfun1gHceHj7ZnaIBLUu/C1zjeoGI9yKg4Gaho8a7Vo6UdTSiThrp+LwuT81qWxa4NamNBfyS5x8O13q7Du3/FRWZZFLnK2NmT1zwWE7DZbG3GBrLjUH/V/DSdnqVY8Etc4NWK/oRW5+6mLlOHsXfficnc1rTLc3WKnvUzpTKi935wFg6eqs5eB/tPnULAI4ZSRta7d81On+0e5Barut2tb+oe2tMpFg4MF3f/44gzOA1TXrEmXMm627DPLacRYh8HIdXbPe7/pvCfJLkHhNuY2efV1hz10Z3vvT4PQ30GvWx7ENwean3NxJ+4phGI6EOCCpBwCy3z1rZByQmWOSgo/bewkEwvhDxsndDQCpu0vXqwDw3TOqSkEgEEaDpGqMnEqkRgKBMEokpQPI3l26niP1EQiEUSOpNQCZu0vXKacjEAijRVJ3l65nJcpdqzePXgD4C5mAQCBYYdh1VjbAjrJ0DOOA1E8gEGSne15RA5Adp6MsBALhYpCUjpOdq2N5lIVAIJxvklrHyUHgKqmZQCCMEklpOFl3KklqjyIwAoFISh4kFp+r4aTMRYnMRCAQSY0SbOv9EAiEi4NRu+rlAU7KkhBBEQiEkYqkNlhqVyWTEAiEUSKpGk6qFxbJFAQCYdRIqkh1oggEgh3+D+K+rSsofs46AAAAAElFTkSuQmCC"},ffdb:function(e,o,t){e.exports=t.p+"img/melissa-haendel.851dabc0.jpg"}});
//# sourceMappingURL=app.1ebbc673.js.map